-->
Pharmacogenetic and Pharmacodynamic Testing
Clinical Policy Bulletins Medical Clinical Policy Bulletins
Print
opens a dialog
 
Share
opens in a new window

Number: 0715

Table Of Contents
PolicyApplicable CPT / HCPCS / ICD-10 CodesBackgroundReferences
Policy
Scope of Policy

This Clinical Policy Bulletin addresses pharmacogenetic and pharmacodynamic testing.

Medical Necessity

Aetna considers the following tests medically necessary (unless otherwise stated):

ABCD1 gene mutation testing for members who are considering elivaldogene autotemcel (Skysona) for treatment of cerebral adrenoleukodystrophy (CALD), see CPB 1017 - Elivaldogene Autotemcel (Skysona);
Anaplastic lymphoma kinase (ALK) fusion gene (e.g., the Vysis ALK Break Apart FISH Probe Kit; Ventana ALK (D5F3) CDx Assay) for members who are considering crizotinib (Xalkori), alectinib (Alecensa) or ceritinib (Zykadia) for the treatment of non-small cell lung cancer (NSCLC);

Aetna considers FoundationOne testing panels (FoundationOne CDx and FoundationOne Liquid CDx) not medically necessary for assessing candidacy of members with non-small cell lung cancer for treatment with alectinib (Alecensa), because there is no proven advantage of the FoundationOne CDx gene panels over targeted ALK mutation testing or small targeted panels for this indication;

ALK gene rearrangement for members who are considering pembrolizumab (Keytruda) for the treatment of NSCLC;
BCR/ABL mutation testing (e.g., MRDx BCR-ABL Test) for members with chronic myeloid leukemia being considered for treatment with nilotinib (Tasigna);
BRAF and NRAS mutations (e.g., cobas KRAS Mutation Test; therascreen KRAS RGQ PCR Kit, Dako EGFR pharmDx Kit) for members with colorectal cancer being considered for treatment with cetuximab (Erbitux) or panitumumab (Vectibix);
BRAF gene (V600E or V600K) mutation testing (e.g., the THxID BRAF test, cobas 4800 BRAF V600 mutation test, Qiagen therascreen BRAF V600E RGQ polymerase chain reaction (PCR) Kit) for the following indications:
Members with unresectable or metastatic melanoma who are being considered for treatment with vemurafenib (Zelboraf), dabrafenib (Tafinlar), trametinib (Mekinist), cobimetinib (Cotellic), binimetinib (Mektovi) or encorafenib (Braftovi); or
Members with metastatic colorectal cancer being considered for treatment with encorafenib (Braftovi); 

Aetna considers FoundationOne Liquid CDx not medically necessary for assessing candidacy of members with melanoma for treatment with encorafenib (Braftovi), because there is no proven advantage of the FoundationOne Liquid CDx gene panels over targeted BRAF V600E mutation testing or small targeted panels for this indication; or

Members with recurrent or metastatic non-small cell lung cancer who are being considered for treatment with dabrafenib (Tafinlar), pembrolizumab (Keytruda), or vemurafenib (Zelboraf); or
Members with thyroid carcinoma who are being considered for treatment with dabrafenib (Tafinlar) or vemurafenib (Zelboraf);

BRCA testing

BRCA testing (e.g., BRACAnalysis CDx) for women with advanced epithelial ovarian cancer, fallopian tube or primary peritoneal cancer who have been treated with three or more prior lines of chemotherapy and are being considered for olaparib (Lynparza), and for women with advanced epithelial ovarian cancer, fallopian tube or primary peritoneal cancer who are in complete or partial response to first-line platinum based chemotherapy and are being considered for maintenance treatment with olaparib;

Aetna considers FoundationOne testing (FoundationOne CDx and FoundationOne Liquid CDx) not medically necessary for assessing candidacy of persons with ovarian cancer for treatment with olaparib, because there is no proven advantage of the FoundationOne CDx gene panel over targeted BRCA mutation testing or small targeted panels for this indication;

BRCA testing (e.g., BRACAnalysis CDx) for members with metastatic pancreatic carcinoma whose disease has not progressed on first-line platinum treatment, and are being considered for maintenance treatment with olaparib;
BRCA testing (e.g., FoundationOne CDx) for men with advanced, recurrent or metastatic prostate cancer who have been treated with androgen-receptor directed therapy and are being considered for treatment with olaparib (Lynparza);
Germline BRCA testing (e.g., BRACAnalysis CDx) for women with human epidermal growth factor receptor 2 (HER2)-negative breast cancer who are being considered for olaparib (Lynparza or talazoparib (Talzenna), either as adjuvant therapy or for those who have previously been treated with chemotherapy in the neoadjuvant, adjuvant or metastatic setting (regardless of family history);
Somatic/tumor BRCA testing (e.g., myChoice CDx) for women with advanced epithelial ovarian, fallopian tube or primary peritoneal cancer who have been treated with three or more prior lines of chemotherapy and are being considered for niraparib (Zejula);
Somatic/tumor BRCA testing (e.g., myChoice CDx) for women with advanced epithelial ovarian, fallopian tube or primary peritoneal cancer who are in a complete or partial response to two or more lines of platinum-based chemotherapy and are being considered for maintenance treatment with niraparib;
Somatic/tumor BRCA testing (e.g., FoundationFocus™ CDx BRCA LOH test) for women with epithelial ovarian cancer, fallopian tube or primary peritoneal cancer who are being considered for treatment with rucaparib (Rubraca) after two or more previous lines of chemotherapy, except when the individual has previously tested positive for a deleterious or suspected deleterious germline BRCA mutation;

Aetna considers FoundationOne Liquid CDx not medically necessary for assessing candidacy of persons with ovarian cancer being considered for treatment with rucaparib because there is no proven advantage of the FoundationOne Liquid CDx gene panel over targeted BRCA mutation testing or small targeted panels for this indication. For more information on BRCA testing, see CPB 0227 - BRCA Testing, Prophylactic Mastectomy, and Prophylactic Oophorectomy;

Somatic/tumor BRCA testing (e.g., FoundationOne CDx, FoundationOne Liquid CDx) for men with advanced, recurrent or metastatic prostate cancer who have been treated with androgen-receptor directed therapy and a taxane-based chemotherapy and are being considered for treatment with rucaparib;

c-KIT testing for members with gastrointestinal stromal tumors (GIST) (e.g., Dako c-KIT pharmDx) or aggressive systemic mastocytosis (KIT D816V Mutation Detection by PCR) being considered for treatment with imatinib mesylate (Gleevec);
Claudin 18 isoform 2 (CLDN18.2) testing (e.g., VENTANA CLDN18 (43-14A) RxDx Assay) for members with gastric cancer or esophagogastric junction adenocarcinoma who are being considered for zolbetuximab (Vyloy) therapy;
CRCdx RAS Mutation Detection Kit for detection of certain RAS gene (i.e., KRAS and NRAS genes) mutations in members who are considering panitumumab for treatment of anal adenocarcinoma, colorectal cancer, or small bowel adenocarcinoma;
CTFR gene mutations for members with cystic fibrosis who are being considered for treatment with ivacaftor (Kalydeco):
G551D
G1244E
G1349D
G178R
G551S
R117H
S1251N
S1255P
S549N
S549R

Cytochrome P450 polymorphisms:

CYP2C19 polymorphism genotyping for members who have been prescribed clopidogrel (Plavix) - repeat CYP2C19 genotyping has no proven value;
CYP2D6 polymorphism genotyping for members who have been prescribed doses of tetrabenazine (Xenazine) greater than 50 mg per day - repeat CYP2D6 genotyping has no proven value;
CYP2D6 polymorphism genotyping for members with Gaucher disease type 1 who are being considered for treatment with eliglustat (Cerdelga) - repeat CYP2D6 genotyping has no proven value;
del(17p)/TP53 mutation testing (e.g., Vysis CLL FISH Probe Kit) for members with chronic lymphocytic leukemia/small lymphocytic lymphoma being considered for treatment with venetoclax (Venclexta);
DPYD gene variants testing for members prior to considering fluoropymidine therapy (i.e., fluorouracil or capecitabine);
Epidermal growth factor receptor (EGFR) exon 19 deletions or exon 21 (L858R) substitution mutations (e.g., cobas EGFR Mutation Test, therascreen EGFR RGQ PCR Kit) for members with non-small cell lung cancer (NSCLC) being considered for treatment with osimertinib (Tagrisso), erlotinib, dacomitinib (Vizimpro), gefitinib (Iressa) or afatinib (Gilotrif);
Aetna considers Guardant360CDx testing panel not medically necessary for assessing candidacy of members with non-small cell lung cancer (NSCLC) for treatment with osimertinib (Tagrisso), because there is no proven advantage of the Guardant360 CDx gene panel over targeted EGFR mutation testing or small targeted panels for this indication;
Aetna considers FoundationOne Liquid CDx not medically necessary for assessing candidacy of members with NSCLC for treatment with osimertinib (Tagrisso), erlotinib or gefitinib (Iressa) because there is no proven advantage of the FoundationOne Liquid CDx over targeted EGFR mutation testing or small targeted panels for this indication;
Aetna considers oncoReveal Dx Lung and Colon Cancer Assay (O/RDx-LCCA) not medically necessary for assessing candidacy of members with NSCLC for treatment with erlotinib, afatinib (Gilotrif), gefitinib (Iressa), or dacomitinib (Vizimpro) because there is no proven advantage of the O/RDx-LCCA over targeted EGFR mutation testing for this indication;
Epidermal growth factor receptor (EGFR) exon 19 deletions or L858R mutations for members with non-small cell lung cancer (NSCLC) being considered for treatment with pembrolizumab (Keytruda), see CPB 0890 - Pembrolizumab (Keytruda);
Epidermal growth factor receptor (EGFR) exon 20 insertion mutations for members with non-small cell lung cancer (NSCLC) being considered for treatment with amivantamab-vmjw (Rybrevant) or mobocertinib (Exkivity);
Aetna considers Guardant360 CDx gene panel not medically necessary for assessing candidacy of members with NSCLC for amivantamab-vmjw, because there is no proven advantage of Guardant360 CDx gene panel over targeted EGFR mutation testing or small targeted panels for this indication;
Aetna considers FoundationOne Liquid CDx gene panel not medically necessary for assessing candidacy of members with NSCLC for mobocertinib, because there is no proven advantage of the FoundationOne Liquid CDx gene panel over targeted EGFR mutation testing or small targeted panels for this indication;
Epidermal growth factor receptor (EGFR) exon deletions and L858R mutations for members with non-small cell lung cancer (NSCLC) being considered for treatment with ipilimumab (Yervoy) or nivolumab (Opdivo), see CPB 0815 - Ipilimumab (Yervoy), CPB 0892 - Nivolumab (Opdivo);
Epidermal growth factor receptor (EGFR) mutation testing for predicting response to EGFR-targeting tyrosine kinase inhibitors (erlotinib, gefitinib [Iressa], afatinib [Gilotrif], osimertinib [Tagrisso]) in non-small cell lung cancer (NSCLC);
Epidermal growth factor receptor (EGFR) T790 mutation testing (e.g., cobas EGFR Mutation Test v2) for members with non-small cell lung cancer (NSCLC) being considered for treatment with osimertinib (Tagrisso);
ERBB2 (HER2) amplification testing for members with breast, colorectal, esophageal, gastric, gastroesophageal junction, non-small cell lung (NSCLC), salivary gland, and uterine serous cancer being considered for treatment with trastuzumab (Herceptin and biosimilars), ado-trastuzumab emtansine (Kadcyla), or pertuzumab (Perjeta), see CPB 0313 - Trastuzumab (Herceptin and biosimilars), Trastuzumab and Hyaluronidase-oysk (Herceptin Hylecta);
ERBB2 (HER2) mutations in persons with breast, colorectal, esophageal, gastric, gastroesophageal junction, or non-small cell lung cancer (NSCLC) being considered for treatment with fam-trastuzumab deruxtecan-nxki (Enhertu), see CPB 0966 - Fam-trastuzumab Deruxtecan-nxki (Enhertu);

Aetna considers Guardant360 CDx gene panel not medically necessary for assessing candidacy of persons with NSCLC for Enhertu, because there is no proven advantage of Guardant360 CDx gene panel over targeted ERBB2 mutation testing or small targeted panels for this indication;

Estrogen receptor 1 (ESR1) gene mutations (e.g., Guardant360 CDx assay) in persons with advanced or metastatic HER2 negative, ER positive breast cancer who have progressed on endocrine therapy and are being considered for treatment with elacestrant (Orserdu);
EZH2 mutation testing for members with follicular lymphoma being considered for treatment with tazemetostat (Tazverik);
F508del mutation in the CTFR gene for members with cystic fibrosis who are being considered for treatment with lumacaftor/ivacaftor (Orkambi);
Fibroblast growth factor receptor 2 (FGFR2) mutation testing (e.g., FoundationOne CDx) for members with cholangiocarcinoma being considered for treatment with pemigatinib (Pemazyre);
Fibroblast growth factor receptor 2 (FGFR2) and FGFR3 mutation testing for members with urothelial carcinoma being considered for treatment with erdafitinib (Balversa);
Folate receptor-alpha testing (e.g., VENTANA FOLR1 (FOLR-2.1) RxDx Assay) for members who are considering mirvetuximab soravtansine-gynx (Elahere) for the treatment of epithelial ovarian, fallopian tube, or primary peritoneal cancer;
FTL3 mutation assay (e.g., LeukoStrat CDx FLT Mutation Assay) for members with acute myeloid leukemia (AML) being considered for treatment with midostaurin (Rydapt), gilternib (Xospata) or sorafenib (Nexavar);
HLA class 1 genotyping for allopurinol hypersensitivity reaction prior to initiation of therapy;
HLA-A*02:01 genotyping for members with uveal melanoma before commencing treatment with tebentafusp-tebn (Kimmtrak), see CPB 1005 - Tebentafusp-tebn (Kimmtrak);
HLA-B*1502 genotyping for members of Asian ancestry before commencing treatment with carbamazepine (Tegretol);
HLA-B*5701 screening for members infected with HIV-1 before commencing treatment with abacavir (Ziagen);
HLA-B*58:01 genotyping for Asian members prior to commencing allopurinol therapy;
Isocitrate dehydrogenase-1 (IDH1) mutation or isocitrate dehydrogenase-2 (IDH2) mutation (e.g., Abbott Real Time IDH1 and IDH2) for  members with acute myeloid leukemia (AML) being considered for treatment with enasidenib mesylate (Idhifa) or ivosidenib (Tibsovo);
KRAS G12C mutation testing (including the QIAGEN therascreen KRAS RGQ PCR) for members with non-small cell lung cancer (NSCLC) being considered for treatment with sotorasib (Lumakras);

Aetna considers Guardant360 CDx gene panel not medically necessary for assessing candidacy of persons with NSCLC for sotorasib, because there is no proven advantage of Guardant360 CDx gene panel over targeted KRAS G12C mutation testing or small targeted panels for this indication;

KRAS G12C mutation testing (including QIAGEN therascreen KRAS RGQ PCR Kit) for members with non-small cell lung cancer (NSCLC) being considered for treatment with adagrasib (Krazati);

Aetna considers the Agilent Resolution ctDx FIRST assay not medically necessary for assessing candidacy of members with NSCLC for adagrasib, because there is no proven advantage of the Agilent Resolution ctDx FIRST assay over targeted KRAS G12C mutation testing or small targeted panels for this indication;

KRAS mutation analysis, with BRAF reflex testing, to predict non-response to cetuximab (Erbitux) and panitumumab (Vectibix) in the treatment of anal adenocarcinoma, metastatic colorectal cancer and small bowel adenocarcinoma;
KRAS mutation analysis to predict non-response to erlotinib in the treatment of non-small cell lung cancer;
Mass spectrometry and tandem mass spectrometry for busulfan dose monitoring for Hodgkin lymphoma and multiple myeloma;
Mesenchymal-epithelial transition (MET) exon 14 skipping mutation testing (e.g., FoundationOneCDx) for members with advanced, recurrent or metastatic non-small cell lung cancer being considered for treatment with the kinase inhibitor capmatinib (Tabrecta);

Aetna considers FoundationOne Liquid CDx gene panel not medically necessary for assessing candidacy of members with NSCLC for capmatinib, because there is no proven advantage of the FoundationOne Liquid CDx gene panel over targeted EGFR mutation testing or small targeted panels for this indication;

MGMT (O(6)-methylguanine-DNA methyltransferase) gene methylation assay for predicting response to temozolomide (Temodar) in persons with glioblastoma;
Microsatellite instability (MSI-H) and mismatch repair deficiency (DMMR)
For members with metastatic colorectal cancer being considered for treatment with ipilimumab (Yervoy);
For members with metastatic colorectal cancer being considered for treatment with nivolumab (Opdivo);
For members with unresectable or metastatic solid tumors being considered for treatment with dostarlimab-gxly (Jemperli);
For members with unresectable or metastatic solid tumors being considered for treatment with pembrolizumab (Keytruda), see CPB 0890 - Pembrolizumab (Keytruda);
FoundationOne CDx is considered not medically necessary for MSI-H/dMMR testing for members with solid tumors being considered for treatment with pembrolizumab, because there is no proven advantage of the FoundationOne CDx gene panel over targeted MSI-H/dMMR testing or small targeted panels for this indication;
Neurofilament light chain for persons with amyotrophic lateral sclerosis being considered for togersen (Qalsody);
Neurotrophic receptor tyrosine kinase (NTRK)1/2/3 gene fusion for members with solid tumors being considered for treatment with larotrectinib (Vitrakvi) or entrectinib (Rozyltrek);

Aetna considers FoundationOne testing (FoundationOne CDx and FoundationOne Liquid CDx) not medically necessary for this indication because there is no proven advantage of this testing over targeted NTRK testing;

Neurotrophic receptor tyrosine kinase (NTRK) 1/2/3 gene fusion for members with solid tumors being considered for treatment with larotrectinib (Vitrakvi):

Aetna considers TruSight Oncology Comprehensive (TSO Comp) testing not medically necessary for this indication because there is no proven advantage of this testing over targeted NTRK testing;

NS3 Q80K polymorphism testing of virus for members with hepatitis C virus (HCV) genotype 1a infection being considered for treatment with simeprevir (Olysio);
NS5A resistance-associated polymorphisms testing of virus for members with hepatitis C virus genotype 1, 3 and 4 infections being considered for treatment with daclatasvir (Daklinza) or elbasvir and grazoprevir (Zepatier);
oncoReveal CDx panel is considered not medically necessary for assessing candidacy of members with either of the following because there is no proven advantage over targeted variant testing for these indications:
EGFR exon 19 deletions and exon 21 (L858R) substitution mutation for members with non-small cell lung cancer (NSCLC) being considered for EFGR tyrosine kinase inhibitors (e.g., erlotinib [Tarceva], osimertinib [Tagrisso], gefitinib [Iressa]); or
KRAS G12D and G13D variant for members with colorectal cancer (CRC) being considered for cetuximab (Erbitux) or panitumumab (Vectibix) therapy;
oncoReveal Dx Lung and Colon Cancer Assay (O/RDx-LCCA) is considered not medically necessary for assessing candidacy of members with colorectal cancer (CRC) for treatment with cetuximab (Erbitux) or panitumumab (Vectibix) because there is no proven advantage of the O/RDx-LCCA over targeted KRAS mutation testing for this indication;
PD-L1 expression (e.g., Ventana PD-L1 (SP263) Assay) for members with urothelial carcinoma who are being considered for treatment with durvalumab (Imfinzi);
PD-L1 expression (e.g., Ventana PD-L1 (SP142) Assay) for members with triple-negative breast carcinoma (TNBC), non-small cell lung cancer and urothelial carcinoma who are being considered for treatment with atezolizumab (Tecentriq);
PD-L1 expression (e.g., PD-L1 IHC 22C3 pharmDx) for members with non-small cell lung cancer being considered for treatment with pembrolilzumab (Keytruda), see CPB 0890 - Pembrolizumab (Keytruda);
PIK3CA mutation testing (e.g., FoundationOne Liquid CDx) for members with breast cancer being considered for treatment with inavolisib (Itovebi);
PIK3CA mutation testing (e.g., therascreen PIK3CA RGQ PCR Kit) for members with breast cancer being considered for treatment with alpelizib (Piqray);

Aetna considers FoundationOne CDx testing panel not medically necessary for assessing candidacy of members with breast cancer for treatment with alpelisib (Piqray), because there is no proven advantage of the FoundationOne CDx panel over targeted PIK3CA mutation testing for this indication;

Detection of PIK3CA/AKT1 activating mutations or PTEN loss of function alterations (e.g., FoundationOne CDx) for members with hormone receptor positive, HER2 negative locally advanced or metastatic breast cancer being considered for treatment with capivasertib (Truqap) and fulvestrant (Faslodex);
Platelet-derived growth factor receptor-beta (PDGFRß) gene rearrangements (e.g., PDGFRB FISH) for members with chronic myelomonocytic leukemia and myelodysplastic syndrome/myeloproliferative disease being considered for treatment with imatinib mesylate (Gleevec);
Praxis Extended RAS Panel for members with colorectal cancer who do not have specific mutations in RAS genes [KRAS (exons 2, 3, and 4) and NRAS (exons 2, 3, and 4)] who are being considered for treatment with panitumumab (Vectibix);
RET fusion testing (e.g., Oncomine Dx Target Test) for members with non-small cell lung cancer (NSCLC) or thyroid cancer being considered for treatment with pralsetinib (Gavreto);
RET fusion for members with non-small cell lung cancer (NSCLC) being considered for treatment with selpercatinib (Retevmo):

Aetna considers TruSight Oncology Comprehensive (TSO Comp) testing not medically necessary for this indication because there is no proven advantage of this testing over targeted RET testing;

ROS1 fusion testing, for members with non-small cell lung cancer being considered for treatment with entrectinib (Rozyltrek);

Aetna considers FoundationOne Liquid CDx testing panel not medically necessary for assessing candidacy of members with non-small cell lung cancer being considered for treatment with entrectinib, because there is no proven advantage of the FoundationOne Liquid CDx panel over targeted ROS1 mutation testing for this indication;

ROS-1 to predict response to crizotinib (Xalkori) for the treatment of non-small cell lung cancer (NSCLC);
Superoxide dismutase 1 (SOD1) mutation testing in adult members with amyotrophic lateral sclerosis (ALS) who are considering tofersen (Qalsody) treatment;
Tumor mutational burden (TMB) (including FoundationOne CDx) for members with unresectable or metastatic solid tumors that have progressed with standard treatment and are considering last resort treatment with pembrolizumab  (Keytruda), see CPB 0890 - Pembrolizumab (Keytruda).
Very long chain fatty acids (VLCFA) testing for members who are considering elivaldogene autotemcel (Skysona) for treatment of cerebral adrenoleukodystrophy (CALD), see CPB 1017 - Elivaldogene Autotemcel (Skysona).
xT CDx test is considered not medically necessary for assessing candidacy of members with colorectal cancer (CRC) for treatment with cetuximab (Erbitux) or panitumumab (Vectibix), because there is no proven advantage of the xT CDx test over targeted KRAS or NRAS mutation analysis or small targeted panels for this indication.
Experimental, Investigational, or Unproven

Aetna considers the following tests experimental, investigational, or unproven because their clinical value has not been established:

Multi-gene pharmacogenetics panels (i.e., diagnostic tests to identify specific genetic variations that may be linked to reduced/enhanced metabolism and/or severe side effects of multiple classes of drugs). These tests include, but are not limited to the following:

Aura Genetics Pharmacogenomics (PGx) Test
EffectiveRX Comprehensive Panel
Genecept Assay
GeneSight ADHD
GeneSight Analgesic
GeneSight MTHFR
GeneSight Psychotropic
GeneSightRx
GENETWORx Neuropsychiatric Panel
Genomind Pharmacogenetics Report
INFINITI Neural Response Panel
IDgenetix
Millennium PGT
MindX One TM Blood Test
MyGenVar Pharmacogenomics Test
OneOme RightMed Pharmacogenomic Test
PersonaGene Genetic Panels
PersonalisedRx
Proove Profiles
Psych HealthPGx Panel by RPRD Diagnostics
RightMed Comprehensive Test
RightMed Gene Report
RightMed Gene Test Exclude F2 and F5
RightMed Mental Health Gene Report
RightMed Mental Health Medication Report
RightMed Oncology Gene Report
RightMed Oncology Medication Report
RightMed PGx16 Test
rxSEEK Epilepsy Drug Metabolism
Tempus nP;
A1555G genotyping for prescribing aminoglycosides;
Aegis Drug-Drug Interaction Test;
Amerigene PGT pharmacogenetic testing panels;
Apolipoprotein E (Apo E) genotyping for determining therapeutic response to lipid-lowering medications;
Beta adrenergic receptor genotyping for evaluating persons with treatment resistant asthma and for all other indications;
Catechol-O-Methyltransferase (COMT) Genotype, Varies Test;
Circulating tumor DNA (ctDNA) (also referred to as a liquid biopsy) for predicting response (e.g., Guardant360 Response) in persons with a solid tumor undergoing immunotherapy or targeted treatment. Note: additional information on ctDNA/liquid biopsy testing can be found in CPB 0352 - Tumor Markers; 
ComplyRX urine test (Claro Labs) for drug monitoring;
CQuentia pharmacogenetic comprehensive panel for uses associated with drug therapy including antidepressants;
CYP2C19 polymorphisms testing for fluoxetine;
CYP2D6 genotyping for identifying individuals with Alzheimer's disease with different clinical response to donepezil (Aricept);
CYP2C9 genotyping to inform genotype-guided dosing of coumarin derivatives;
CYP2D6 genotyping for predicting response to beta blockers;
CYP2D6 genotyping for uses associated with opioid medications;
EpiSwitch CiRT (Checkpoint-inhibitor Response Test);
Genotyping for other cytochrome P450 polymorphisms (diagnostic tests to identify specific genetic variations that may be linked to reduced/enhanced effect or severe side effects of drugs metabolized by the cytochrome P450 system including opioid analgesics, warfarin, tamoxifen, proton pump inhibitors, antipsychotic medications, and selective serotonin reuptake inhibitors);
IL28B polymorphism genotyping for interferon therapy for hepatitis C;
Interferon-lambda 3 (IFNL3) genotyping for predicting virological response to pegylated-interferon-alpha and ribavirin combination therapy;
Laboratory testing to allow area under the curve (AUC)-targeted 5-fluorouracil dosing (e.g., Myriad Genetics’ OnDose);
Methotrexate polyglutamates (Avise PG test)  for evaluating response to methotrexate therapy in rheumatoid arthritis or other conditions;
Methylenetetrahydrofolate reductase (MTHFR) genotyping for determining therapeutic response to antifolate chemotherapy and for guiding antidepressant therapy;
Natera Signatera Molecular Monitoring (MRD) for breast cancer;
NeuroIDgenetix test for guiding medication selection for anxiety, depression and other psychiatric disorders;
OncolyticAssuranceRX (Firstox) for adherence monitoring to oral anticancer medications;
PGxOnePlus genetic testing for anxiety and gastroesophageal reflux disease;
Pharmacogenetic studies for prediction of response to medications for chronic kidney disease;
Platelet reactivity/function testing (VerifyNow P2Y12 Assay, Ultegra System Rapid Platelet Function Assay-ASA) for individuals who have undergone percutaneous coronary intervention;
rs3798220 allele testing for selecting persons for chronic aspirin therapy or other indications;
Serotonin Receptor Genotype (HTR2A and HTR2C);
Signatera for individuals with stage II/III colorectal cancer who are considering adjuvant chemotherapy and/or who are being monitored for relapse post-treatment;
Snapshot Oral Fluid Compliance (Ethos) for adherence monitoring to prescription drugs;
Thromboxane metabolites in urine (e.g., AspirinWorks) to evaluate aspirin resistance;
Thymidylate synthase genetic polymorphisms to predict 5-fluorouracil toxicity;
UCSF Pharmacogenomics Panel;
UGT1A1 molecular assay (a screening test for determining the proper dosage of irinotecan for persons with colorectal cancer or other types of cancer (e.g., non-small-cell lung cancer);
UrSure tenofovir quantification test for adherence monitoring to pre-exposure prophylaxis;
VKORC1 polymorphism genotyping (diagnostic tests to identify specific genetic variations that may be linked to reduced/enhanced effect or severe side effects of drugs metabolized by the vitamin K epoxide reductase complex subunit 1 gene including warfarin).
Related Policies
CPB 0140 - Genetic Testing
CPB 0227 - BRCA Testing, Prophylactic Mastectomy, and Prophylactic Oophorectomy
CPB 0249 - Inflammatory Bowel Disease: Serologic Markers and Pharmacogenomic and Metabolic Assessment of Thiopurine Therapy
CPB 0313 - Trastuzumab (Herceptin and biosimilars), Trastuzumab and Hyaluronidase-oysk (Herceptin Hylecta)
CPB 0352 - Tumor Markers
CPB 0381 - Cardiovascular Disease Risk Tests
CPB 0763 - Homocysteine Testing
CPB 0815 - Ipilimumab (Yervoy)
CPB 0890 - Pembrolizumab (Keytruda)
CPB 0892 - Nivolumab (Opdivo)
CPB 1005 - Tebentafusp-tebn (Kimmtrak)
CPB 1017 - Elivaldogene Autotemcel (Skysona)
Table:
CPT Codes / HCPCS Codes / ICD-10 Codes
Code	Code Description

ABCD1 gene mutation and Very long chain fatty acids (VLCFA) testing:


CPT codes covered if selection criteria are met:

82726	Very long chain fatty acids

Other HCPCS codes related to the CPB:

Elivaldogene autotemcel (Skysona) –no specific code

ICD-10 codes covered if selection criteria are met:

E71.520	Childhood cerebral X-linked adrenoleukodystrophy
E71.521	Adolescent X-linked adrenoleukodystrophy
E71.528	Other X-linked adrenoleukodystrophy
E71.529	X-linked adrenoleukodystrophy, unspecified type

Genotyping for CYP2C19:


CPT codes covered if selection criteria are met:

81225	CYP2C19 (cytochrome P450, family 2, subfamily C, polypeptide 19) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *8, *17)

CPT codes not covered for indications listed in the CPB:

81227	CYP2C9 (cytochrome P450, family 2, subfamily C, polypeptide 9) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *5, *6)

ICD-10 codes covered if selection criteria are met:

I25.10 - I25.9	Chronic ischemic heart disease

Genotyping for CYP2D6:


CPT codes covered if selection criteria are met:

0028U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, copy number variants, common variants with reflex to targeted sequence analysis
0070U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, common and select rare variants (ie, *2, *3, *4, *4N, *5, *6, *7, *8, *9, *10, *11, *12, *13, *14A, *14B, *15, *17, *29, *35, *36, *41, *57, *61, *63, *68, *83, *xN)
0071U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, full gene sequence (List separately in addition to code for primary procedure)
0072U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, targeted sequence analysis (ie, CYP2D6-2D7 hybrid gene) (List separately in addition to code for primary procedure)
0073U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, targeted sequence analysis (ie, CYP2D7-2D6 hybrid gene) (List separately in addition to code for primary procedure)
0074U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, targeted sequence analysis (ie, non-duplicated gene when duplication/multiplication is trans) (List separately in addition to code for primary procedure)
0075U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, targeted sequence analysis (ie, 5’ gene duplication/multiplication) (List separately in addition to code for primary procedure)
0076U	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) gene analysis, targeted sequence analysis (ie, 3’ gene duplication/ multiplication) (List separately in addition to code for primary procedure)
81226	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *5, *6, *9, *10, *17, *19, *29, *35, *41, *1XN, *2XN, *4XN)

Other HCPCS codes related to the CPB:

J1800	Injection, propranolol HCl, up to 1 mg

ICD-10 codes covered if selection criteria are met:

E75.22	Gaucher disease [for persons with Gaucher's disease type 1 who are being considered for treatment with eligustat (Cerdelga)]
G10	Huntington's disease [Tetrabenazine (Xenazine) is indicated for the treatment of chorea associated with Huntington’s disease]

Genotyping for cytochrome P450:


CPT code not covered for indications listed in the CPB:

Genotyping for cytochrome P450 – no specific code
0029U	Drug metabolism (adverse drug reactions and drug response), targeted sequence analysis (ie, CYP1A2, CYP2C19, CYP2C9, CYP2D6, CYP3A4, CYP3A5, CYP4F2, SLCO1B1, VKORC1 and rs12777823)
0030U	Drug metabolism (warfarin drug response), targeted sequence analysis (ie, CYP2C9, CYP4F2, VKORC1, rs12777823)
0031U	CYP1A2 (cytochrome P450 family 1, subfamily A, member 2)(eg, drug metabolism) gene analysis, common variants (ie, *1F, *1K, *6, *7)
0476U	Drug metabolism, psychiatry (eg, major depressive disorder, general anxiety disorder, attention deficit hyperactivity disorder (ADHD), schizophrenia), whole blood, buccal swab, and pharmacogenomic genotyping of 14 genes and CYP2D6 copy number variant analysis and reported phenotypes
81230	CYP3A4 (cytochrome P450 family 3 subfamily A member 4) (eg, drug metabolism), gene analysis, common variant(s) (eg, *2, *22)
81231	CYP3A5 (cytochrome P450 family 3 subfamily A member 5) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *5, *6, *7)
81346	TYMS (thymidylate synthetase) (eg, 5-fluorouracil/5-FU drug metabolism), gene analysis, common variant(s) (eg, tandem repeat variant)

CPT codes related to the CPB:

81400 - 81408	Molecular pathology

HCPCS codes not covered for indications listed in the CPB:

G9143	Warfarin responsiveness testing by genetic technique using any method, any number of specimen(s)

Other HCPCS codes related to the CPB:

C9037	Injection, risperidone (perseris), 0.5 mg
J0640	Injection, leucovorin calcium, per 50 mg
J1630	Injection, haloperidol, up to 5 mg
J1631	Injection, haloperidol decanoate, per 50 mg
J2270	Injection, morphine sulfate, up to 10 mg
J2271	Injection, morphine sulfate, 100 mg
J2272	Injection, morphine sulfate (fresenius kabi) not therapeutically equivalent to J2270, up to 10 mg
J2275	Injection, morphine sulfate (preservative-free sterile solution), per 10 mg
J2794	Injection, risperidone, long acting, 0.5 mg
J3310	Injection, perphenazine, up to 5 mg
J3360	Injection, diazepam, up to 5 mg
J8530	Cyclophosphamide, oral, 25 mg
J9073	Injection, cyclophosphamide (ingenus), 5 mg
J9074	Injection, cyclophosphamide (sandoz), 5 mg
J9075	Injection, cyclophosphamide, not otherwise specified, 5mg
J9076	Injection, cyclophosphamide (baxter), 5 mg
J9190	Fluorouracil, 500 mg
J9206	Irinotecan, 20 mg
J9263	Injection, oxaliplatin, 0.5 mg
Q0175	Perphenazine, 4 mg, oral, FDA approved prescription anti-emetic, for use as a complete therapeutic substitute for an IV anti-emetic at the time of chemotherapy treatment, not to exceed a 48-hour dosage regimen
S0093	Injection, morphine sulfate, 500 mg (loading doe for infusion pump)
S0187	Tamoxifen citrate, oral, 10 mg

ICD-10 codes not covered for indications listed in the CPB:

G30. 0 - G30.9	Alzheimer's disease

CYP2C19 polymorphisms testing for fluoxetine:


CPT codes not covered for indications listed in the CPB:

81225	CYP2C19 (cytochrome P450, family 2, subfamily C, polypeptide 19) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *8, *17)

UCSF Pharmacogenomics Panel:


CPT codes not covered for indications listed in the CPB:

0533U	Drug metabolism (adverse drug reactions and drug response), genotyping of 16 genes (ie, ABCG2, CYP2B6, CYP2C9, CYP2C19, CYP2C, CYP2D6, CYP3A5, CYP4F2, DPYD, G6PD, GGCX, NUDT15, SLCO1B1, TPMT, UGT1A1, VKORC1), reported as metabolizer status and transporter function

UGT1A1 molecular assay:


CPT codes not covered for indications listed in the CPB:

81350	UGT1A1 (UDP glucuronosyltransferase 1 family, polypeptide A1) (eg, irinotecan metabolism), gene analysis, common variants (eg, *28, *36, *37)

Other HCPCS codes related to the CPB:

J9206	Irinotecan, 20 mg

ICD-10 codes not covered for indications listed in the CPB:


C00.0 - C7A.8, C7B.1, C76.0 - C86.61,
C88.40 - C94.32, C94.80 - C96.4,
C96.6 - C96.9
	Malignant neoplasms
D00.00 - D09.9	In situ neoplasms
D45	Polycythemia vera

VKORC1 polymorphism:


CPT codes not covered for indications listed in the CPB:

81355	VKORC1 (vitamin K epoxide reductase complex, subunit 1) (eg, warfarin metabolism), gene analysis, common variant(s) (eg, -1639G>A, c.173+1000C>T)

HLA class 1 genotyping:


CPT codes covered if selection criteria are met:

81381	HLA Class I typing, high resolution (ie, alleles or allele groups); one allele or allele group (eg, B*57:01P), each

ICD-10 codes covered if selection criteria are met:

T50.45XA - T50.45XS	Adverse effect of drugs affecting uric acid metabolism [allopurinol hypersensitivity reaction prior to initiation of therapy]

Genotyping for HLA-A*02:01:


CPT codes covered if selection criteria are met:

81381	HLA Class I typing, high resolution (ie, alleles or allele groups); one allele or allele group (eg, B*57:01P), each

Other HCPCS codes related to the CPB:

J9274	Injection, tebentafusp-tebn, 1 microgram

ICD-10 codes covered if selection criteria are met:

C69.30 – C69.32	Malignant neoplasm of choroid
C69.40 – C69.42	Malignant neoplasm of ciliary body

HLA-B*1502:


CPT codes covered if selection criteria are met:

HLA-B*1502 - No specific codes

HLA-B*5701:


CPT codes covered if selection criteria are met:

HLA-B*5701 - No specific code

ICD-10 codes covered if selection criteria are met:

B20	Human immunodeficiency virus [HIV] disease
Z21	Asymptomatic human immunodeficiency virus [HIV] infection status

Genotyping for apolipoprotein E (Apo E):


CPT codes not covered for indications listed in the CPB:

Genotyping for apolipoprotein E (Apo E) - No specific code

Other CPT codes related to the CPB :

88271 - 88275	Molecular cytogenetics

ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):

E71.30	Disorder of fatty-acid metabolism, unspecified
E75.5 - E75.6	Other and unspecified lipid storage disorders

E78.00 - E78.5, E78.70, E78.79 - E78.9
	Disorders of lipoprotein metabolism and other lipidemias
E88.2	Lipomatosis, not elsewhere classified
E88.89	Other specified metabolic disorders

Genotyping for methylenetetrahydrofolate reductase (MTHFR):


CPT codes not covered for indications listed in the CPB :

81291	MTHFR (5,10-methylenetetrahydrofolate reductase) (eg, hereditary hypercoagulability) gene analysis, common variants (eg, 677T, 1298C)

ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):


C00.0 - C7A.8, C7B.1, C76.0 - C86.61,
C88.40 - C94.32, C94.80 - C96.4,
C96.6 - C96.9
	Malignant neoplasms
D00.00 - D09.9	In situ neoplasms
D45	Polycythemia vera

Signatera:


CPT codes not covered for indications listed in the CPB:

0306U	Oncology (minimal residual disease [MRD]), next-generation targeted sequencing analysis, cell-free DNA, initial (baseline) assessment to determine a patient specific panel for future comparisons to evaluate for MRD
0307U	Oncology (minimal residual disease [MRD]), next-generation targeted sequencing analysis of a patient-specific panel, cell-free DNA, subsequent assessment with comparison to previously analyzed patient specimens to evaluate for MRD
0340U	Oncology (pan-cancer), analysis of minimal residual disease (MRD) from plasma, with assays personalized to each patient based on prior next-generation sequencing of the patient’s tumor and germline DNA, reported as absence or presence of MRD, with disease-burden correlation, if appropriate

ICD-10 codes not covered for indications listed in the CPB:

C18.0 – C21.8	Malignant neoplasm of colon, rectosigmoid junction, rectum, anus and anal canal
C50.011 – C50.929	Malignant neoplasm of breast

Measurement of thromboxane metabolites in urine:


CPT codes not covered for indications listed in the CPB:

84431	Thromboxane metabolite(s), including thromboxane if performed, urine

BRAF V600E or V600K mutations (e.g., the THxID BRAF test, Qiagen therascreen BRAF V600E RGQ polymerase chain reaction (PCR) Kit))::


CPT codes covered if selection criteria are met:

Qiagen therascreen BRAF V600E RGQ polymerase chain reaction (PCR) Kit – no specific code
81210	BRAF (B-Raf proto-oncogene, serine/threonine kinase) (eg, colon cancer, melanoma), gene analysis, V600 variant(s)

Other HCPCS codes related to the CPB:


Dabrafenib (Tafinlar); Vemurafenib (Zelboraf); Encorafenib (Braftovi), Cobimetinib (Cotellic), Binimetinib (Mektovi), Trametinib (Mekinist)- no specific code:

J9271	Injection, pembrolizumab, 1 mg

ICD-10 codes covered by indications listed in the CPB:

C18.0 - C20	Malignant neoplasm of colon, rectosigmoid junction and rectum
C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer]
C43.0 - C43.9	Malignant melanoma of skin
C73	Malignant neoplasm of thyroid

Neurofilament light chain:


CPT codes covered if selection criteria are met:

0361U	Neurofilament light chain, digital immunoassay, plasma, quantitative
0443U	Neurofilament light chain (NfL), ultra-sensitive immunoassay, serum or cerebrospinal fluid
0547U	Neurofilament light chain (NfL), chemiluminescent enzyme
83884	Neurofilament light chain (NfL)

Other CPT codes related to the CPB:

96450	Chemotherapy administration, into CNS (eg, intrathecal), requiring and including spinal puncture

Other HCPCS codes related to the CPB:

J1304	Injection, tofersen, 1 mg

ICD-10 codes covered if selection criteria are met:

G12.21	Amyotrophic lateral sclerosis

NS3 Q80K polymorphism, NS5A resistance-associated polymorphisms:


CPT codes covered if selection criteria are met:

87900	Infectious agent drug susceptibility phenotype prediction using regularly updated genotypic bioinformatics [for persons with hepatitis C virus (HCV) genotype 1a infection being considered for treatment with simeprevir (Olysio)] [for persons with hepatitis C virus (HCV) genotype 1, 3 and 4 infections being considered for treatment with daclatasvir (Daklinza) or elbasvir and grazoprevir (Zepatier)]
87902	Infectious agent genotype analysis by nucleic acid (DNA or RNA); Hepatitis C virus [for persons with hepatitis C virus (HCV) genotype 1a infection being considered for treatment with simeprevir (Olysio)] [for persons with hepatitis C virus (HCV) genotype 1, 3 and 4 infections being considered for treatment with daclatasvir (Daklinza) or elbasvir and grazoprevir (Zepatier)]

ICD-10 codes covered if selection criteria are met:

B17.10 -B17.11	Acute hepatitis C without/with hepatic coma [genotype 1, 1a, 3, 4 infections]
B18.2	Chronic viral hepatitis C [genotype 1, 1a, 3, 4 infections]
B19.20 - B19.21	Unspecified viral hepatitis C [genotype 1, 1a, 3, 4 infections]
Z22.52	Carrier of viral hepatitis C [genotype 1, 1a, 3, 4 infections]

BRACAnalysis CDx, FoundationOne CDx, MyChoice CDx:


CPT codes covered if selection criteria are met:

0037U	Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden
0172U	Oncology (solid tumor as indicated by the label), somatic mutation analysis of BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) and analysis of homologous recombination deficiency pathways, DNA, formalin-fixed paraffin-embedded tissue, algorithm quantifying tumor genomic instability score
81162	BRCA1, BRCA2 (breast cancer 1 and 2) (eg, hereditary breast and ovarian cancer) gene analysis; full sequence analysis and full duplication/deletion analysis
81163	BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full sequence analysis
81165	BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full sequence analysis
81212	BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; 185delAG, 5385insC, 6174delT variants
81215	BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; known familial variant
81216	BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full sequence analysis
81217	BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; known familial variant

CPT codes not covered for indications listed in the CPB:

81164	BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full duplication/deletion analysis (ie, detection of large gene rearrangements)
81166	BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full duplication/deletion analysis (ie, detection of large gene rearrangements)
81167	BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene analysis; full duplication/deletion analysis (ie, detection of large gene rearrangements

Other HCPCS codes related to the CPB:

Alectinib (Alecensa), Capivasertib (Truqap) –no specific code
J9395	Injection, fulvestrant, 25 mg

ICD-10 codes covered if selection criteria are met:

C25.0 - C25.9	Malignant neoplasm of pancreas
C48.0 - C48.8	Malignant neoplasm of retroperitoneum and peritoneum
C50.011 - C50.929	Malignant neoplasm of breast [HER2-negative]
C56.1 - C56.9	Malignant neoplasm of ovary
C57.00 - C57.02	Malignant neoplasm of fallopian tube
C61	Malignant neoplasm of prostate

ICD-10 codes not covered for indications listed in the CPB:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer]

Other HCPCS codes related to the CPB:


Olaparib (Lynparza), Talazoparib (Talzenna), Rucaparib (Rubraca), Niraparib (Zejula) - no specific code:

J9043	Injection, cabazitaxel, 1 mg
J9045	Injection, carboplatin, 50 mg
J9060	Injection, cisplatin, powder or solution, 10 mg
J9064	Injection, cabazitaxel (sandoz), not therapeutically equivalent to j9043, 1 mg
J9171	Injection, docetaxel, 1 mg
J9172	Injection, docetaxel (ingenus) not therapeutically equivalent to j9171, 1 mg
J9258	Injection, paclitaxel protein-bound particles (teva) not therapeutically equivalent to j9264, 1 mg
J9259	Injection, paclitaxel protein-bound particles (american regent) not therapeutically equivalent to j9264, 1 mg
J9263	Injection, oxaliplatin, 0.5 mg
J9264	Injection, paclitaxel protein-bound particles, 1 mg
J9267	Injection, paclitaxel, 1 mg

FoundationOne Liquid CDx:


CPT codes covered if selection criteria are met:

0239U	Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free DNA, analysis of 311 or more genes, interrogation for sequence variants, including substitutions, insertions, deletions, select rearrangements, and copy number variations

Other HCPCS codes related to the CPB:

Alectinib (Alecensa), Encorafenib (Braftovi), Capmatinib (Tabrecta), Mobocertinib (Exkivity) – no specific code

ICD-10 codes covered if selection criteria are met:

C61	Malignant neoplasm of prostate

ICD-10 codes not covered for indications listed in the CPB:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer]
C43.0 - C43.9	Malignant melanoma of skin

OncoReveal CDx panel:


CPT codes not covered if selection criteria are met:

0523U	Oncology (solid tumor), DNA, qualitative, next-generation sequencing (NGS) of single-nucleotide variants (SNV) and insertion/deletions in 22 genes utilizing formalin-fixed paraffin-embedded tissue, reported as presence or absence of mutation(s), location of mutation(s), nucleotide change, and amino acid change

Other CPT codes related to the CPB:

81235	EGFR (epidermal growth factor receptor) (eg, non-small cell lung cancer) gene analysis, common variants (eg, exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q)
81275	KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; variants in exon 2 (eg, codons 12 and 13)

Other HCPCS codes related to the CPB:

EFGR tyrosine kinase inhibitors Erlotinib [Tarceva] and Osimertinib [Tagrisso] – no specific code
J8565	Gefitinib, oral, 250 mg
J9055	Injection, cetuximab, 10 mg
J9303	Injection, panitumumab, 10 mg

ICD-10 codes not covered for indications listed in the CPB:

C18.0 – C21.8	Malignant neoplasm of colon, rectosigmoid junction, rectum, anus and anal canal Malignant neoplasm of colon and rectum
C34.00 – C34.92	Malignant neoplasm of bronchus and lung [Non-small cell lung cancer]

OncoReveal Dx Lung and Colon Cancer Assay (O/RDx-LCCA):


CPT codes not covered if selection criteria are met:

0406U	Oncology (lung), flow cytometry, sputum, 5 markers (meso-tetra [4- carboxyphenyl] porphyrin [TCPP], CD206, CD66b, CD3, CD19), algorithm reported as likelihood of lung cancer
0414U	Oncology (lung), augmentative algorithmic analysis of digitized whole slide imaging for 8 genes (ALK, BRAF, EGFR, ERBB2, MET, NTRK1-3, RET, ROS1), and KRAS G12C and PD-L1, if performed, formalin-fixed paraffin- embedded (FFPE) tissue, reported as positive or negative for each biomarker
0436U	Oncology (lung), plasma analysis of 388 proteins, using aptamer- based proteomics technology, predictive algorithm reported as clinical benefit from immune checkpoint inhibitor therapy
81525	Oncology (colon), mRNA, gene expression profiling by real-time RT-PCR of 12 genes (7 content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a recurrence score
81538	Oncology (lung), mass spectrometric 8-protein signature, including amyloid A, utilizing serum, prognostic and predictive algorithm reported as good versus poor overall survival

Other HCPCS codes related to the CPB:

Erlotinib (Tarceva), afatinib (Gilotrif) and dacomitinib (Vizimpro) – no specific code
J8565	Gefitinib, oral, 250 mg
J9055	Injection, cetuximab, 10 mg
J9303	Injection, panitumumab, 10 mg

ICD-10 codes not covered for indications listed in the CPB:

C18.0 - C20	Malignant neoplasm of colon, rectosigmoid junction and rectum
C34.00 - C34.92	Malignant neoplasm of bronchus and lung [Non-small cell lung cancer (NSCLC)]

BRAF and NRAS mutations (e.g., cobas KRAS Mutation Test; therascreen KRAS RGQ PCR Kit):


CPT codes covered if selection criteria are met:

0471U	Oncology (colorectal cancer), qualitative real-time PCR of 35 variants of KRAS and NRAS genes (exons 2, 3, 4), formalin- fixed paraffin-embedded (FFPE), predictive, identification of detected mutations
81210	BRAF (B-Raf proto-oncogene, serine/threonine kinase) (eg, colon cancer, melanoma), gene analysis, V600 variant(s)
81311	NRAS (neuroblastoma RAS viral [v-ras] oncogene homolog) (eg, colorectal carcinoma), gene analysis, variants in exon 2 (eg, codons 12 and 13) and exon 3 (eg, codon 61)

CPT codes not covered for indications listed in the CPB:

0473U	Oncology (solid tumor), next- generation sequencing (NGS) of DNA from formalin-fixed paraffin- embedded (FFPE) tissue with comparative sequence analysis from a matched normal specimen (blood or saliva), 648 genes, interrogation for sequence variants, insertion and deletion alterations, copy number variants, rearrangements, microsatellite instability, and tumor-mutation burden

Other HCPCS codes related to the CPB:

J9055	Injection, cetuximab, 10 mg
J9303	Injection, panitumumab, 10 mg

ICD-10 codes covered if selection criteria are met:

C17.0 - C17.9	Malignant neoplasm of small intestine including duodenum [Small bowel adenocarcinoma]
C18.0 - C20	Malignant neoplasm of colon, rectosigmoid junction and rectum
C21.0 - C21.8	Malignant neoplasm of anus and anal canal [Anal adenocarcinoma]

DPYD gene variants:


CPT codes covered if selection criteria are met:

81232	DPYD (dihydropyrimidine dehydrogenase) (eg, 5-fluorouracil/5-FU and capecitabine drug metabolism), gene analysis, common variant(s) (eg, *2A, *4, *5, *6)

Other CPT codes related to the CPB::

96409 - 96411	Chemotherapy administration; intravenous, push technique
96413 - 96417	Chemotherapy administration, intravenous infusion technique

Other HCPCS codes related to the CPB:

J8522	Capecitabine, oral, 50 mg
J9190	Injection, fluorouracil, 500 mg

Epidermal growth factor receptor (EGFR) T790 mutation; exon 19 deletions or exon 21 (L858R) substitution mutations (e.g., cobas EGFR Mutation Test, therascreen EGFR RGQ PCR Kit):


CPT codes covered if selection criteria are met:

81235	EGFR (epidermal growth factor receptor) (eg, non-small cell lung cancer) gene analysis, common variants (eg, exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q)

Other HCPCS codes related to the CPB:


Dacomitinib (Vizimpro), Osimertinib (Tagrisso), Erlotinib (Tarceva), Afatinib (Gilotrif), Amivantamab (Rybrevent) - - no specific code:

J8565	Gefitinib, oral, 250 mg
J9228	Injection, ipilimumab, 1 mg
J9271	Injection, pembrolizumab, 1 mg
J9299	Injection, nivolumab, 1 mg

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung carcinoma]

Guardant 360CDx testing panel, Circulating tumor DNA (ctDNA) [Liquid biopsy], Trusight Oncology Comprehensive testing:


CPT codes covered if selection criteria are met:

0242U	Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA analysis of 55-74 genes, interrogation for sequence variants, gene copy number amplifications, and gene rearrangements

CPT codes not covered for indications listed in the CPB:

0091U	Oncology (colorectal) screening, cell enumeration of circulating tumor cells, utilizing whole blood, algorithm, for the presence of adenoma or cancer, reported as a positive or negative result
0179U	Oncology (non-small cell lung cancer), cell-free DNA, targeted sequence analysis of 23 genes (single nucleotide variations, insertions and deletions, fusions without prior knowledge of partner/breakpoint, copy number variations), with report of significant mutation(s)
0326U	Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA analysis of 83 or more genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden
0388U	Oncology (non-small cell lung cancer), next-generation sequencing with identification of single nucleotide variants, copy number variants, insertions and deletions, and structural variants in 37 cancer-related genes, plasma, with report for alteration detection
0422U	Oncology (pan-solid tumor), analysis of DNA biomarker response to anti-cancer therapy using cell-free circulating DNA, biomarker comparison to a previous baseline pre-treatment cell-free circulating DNA analysis using next-generation sequencing, algorithm reported as a quantitative change from baseline, including specific alterations, if appropriate [Response test]
81445	Solid organ neoplasm, genomic sequence analysis panel, 5-50 genes, interrogation for sequence variants and copy number variants or rearrangements, if performed; DNA analysis or combined DNA and RNA analysis
81450	Hematolymphoid neoplasm or disorder, genomic sequence analysis panel, 5-50 genes, interrogation for sequence variants, and copy number variants or rearrangements, or isoform expression or mRNA expression levels, if performed; DNA analysis or combined DNA and RNA analysis
81455	Solid organ or hematolymphoid neoplasm or disorder, 51 or greater genes, genomic sequence analysis panel, interrogation for sequence variants and copy number variants or rearrangements, or isoform expression or mRNA expression levels, if performed; DNA analysis or combined DNA and RNA analysis
81457	Solid organ neoplasm, genomic sequence analysis panel, interrogation for sequence variants; DNA analysis, microsatellite instability
81458	     DNA analysis, copy number variants and microsatellite instability
81459	     DNA analysis or combined DNA and RNA analysis, copy number variants, microsatellite instability, tumor mutation burden, and rearrangements
81462	Solid organ neoplasm, genomic sequence analysis panel, cell-free nucleic acid (eg, plasma), interrogation for sequence variants; DNA analysis or combined DNA and RNA analysis, copy number variants and rearrangements
81463	     DNA analysis, copy number variants, and microsatellite instability
81464	     DNA analysis or combined DNA and RNA analysis, copy number variants, microsatellite instability, tumor mutation burden, and rearrangements
86152	Cell enumeration using immunologic selection and identification in fluid specimen (eg, circulating tumor cells in blood)
86153	Cell enumeration using immunologic selection and identification in fluid specimen (eg, circulating tumor cells in blood); physician interpretation and report, when required

Other HCPCS codes related to the CPB:


Osimertinib (Tagrisso), Elacestrant (Orserdu), Iarotrectinib - no specific code:

J9358	Injection, fam-trastuzumab deruxtecan-nxki, 1 mg

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung carcinoma]
C50.011 - C50.929	Malignant neoplasm of breast

ICD-10 codes not covered for indications listed in the CPB:

C00.0 – C14.8	Malignant neoplasm of lip, oral cavity and pharynx
C15.3 – C26.9	Malignant neoplasm of digestive organs
C30.0 - C39.9	Malignant neoplasm of respiratory and intrathoracic organs
C40.00 - C41.9	Malignant neoplasm of bone and articular cartilage
C43.0 - C44.99	Malignant melanoma and other malignant neoplasms of skin
C45.0 - C49.A9	Malignant neoplasms of mesothelial and soft tissue
C50.011 - C50.929	Malignant neoplasm of female and male breast
C51.0 - C58	Malignant neoplasm of female genital organs
C60.0 - C63.9	Malignant neoplasm of male genital organs
C64.1 - C68.9	Malignant neoplasm of kidney and other and unspecified urinary organs
C69.00 - C72.9	Malignant neoplasm of eye, brain and other parts of central nervous system
C73 – C75.9	Malignant neoplasms of thyroid and other endocrine glands
C7A.00 - C7A.8	Malignant neuroendocrine tumors
C7B.00 – C7B.8	Secondary neuroendocrine tumors
C76.0 – C80.2	Malignant neoplasms of ill-defined, other secondary and unspecified sites
C81.00 – C88.91	Hodgkin lymphoma, follicular lymphoma, non- follicular lymphoma, mature T/NK-cell lymphomas, other specified and unspecified types of non-Hodgkin lymphoma, other specified types of T/NK-cell lymphoma, and malignant immunoproliferative diseases and certain other B-cell lymphomas
C90.00 – C90.02	Multiple myeloma
C90.20 – C90.32	Extamedullary plasmacytoma and solitary plasmacytoma
C96.0 – C96.9	Other and unspecified malignant neoplasms of lymphoid, hematopoietic and related tissue
D00.00 - D09.9	Carcinoma in situ

PD-L1 expression (e.g., PD-L1 IHC 22C3 pharmDx, Ventana PD-L1 (SP263) Assay):


CPT codes covered if selection criteria are met:

88360	Morphometric analysis, tumor immunohistochemistry (eg, Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, per specimen, each single antibody stain procedure; manual
88361	    using computer-assisted technology

Other HCPCS codes related to the CPB:

J9022	Injection, atezolizumab, 10 mg
J9271	Injection, pembrolizumab, 1 mg

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung carcinoma]
C50.011 - C50.929	Malignant neoplasm of breast
C65.1 - C65.9	Malignant neoplasm of renal pelvis [urothelial carcinoma]
C66.1 - C66.9	Malignant neoplasm of ureter [urothelial carcinoma]
C67.0 - C67.9	Malignant neoplasm of bladder, unspecified [urothelial carcinoma]
C68.0 - C68.9	Malignant neoplasm of other and unspecified urinary organs [urothelial carcinoma]

ROS1 fusion testing:


CPT codes covered if selection criteria are met:

ROS1 fusion testing-no specific code

CPT codes not covered for indications listed in the CPB:

0239U	Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free DNA, analysis of 311 or more genes, interrogation for sequence variants, including substitutions, insertions, deletions, select rearrangements, and copy number variations

Other HCPCS codes related to the CPB:

Entrectinib (Rozyltrek), Crizotinib (Xalkori) -no specific code

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [Non-small cell lung cancer]

Superoxide dismutase 1 (SOD1):


CPT codes covered if selection criteria are met:

81404	Molecular pathology procedure, Level 5 (eg, analysis of 2-5 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 6-10 exons, or characterization of a dynamic mutation disorder/triplet repeat by Southern blot analysis) ACADS (acyl-CoA dehydrogenase, C-2 to C-3 short chain) (eg, short chain acyl-CoA dehydrogenase deficiency), targeted sequence analysis (eg, exons 5 and 6) AQP2 (aquaporin 2 [collecting duct]) (eg, nephrogenic diabetes insipidus), full gene sequence ARX (aristaless related homeobox) (eg, X-linked lissencephaly with ambiguous genitalia, X-linked mental retardation), full gene sequence AVPR2 (arginine vasopressin receptor 2) (eg, nephrogenic diabetes insipidus), full gene sequence BBS10 (Bardet-Biedl syndrome 10) (eg, Bardet-Biedl syndrome), full gene sequence BTD (biotinidase) (eg, biotinidase deficiency), full gene sequence C10orf2 (chromosome 10 open reading frame 2) (eg, mitochondrial DNA depletion syndrome), full gene sequence CAV3 (caveolin 3) (eg, CAV3-related distal myopathy, limb-girdle muscular dystrophy type 1C), full gene sequence CD40LG (CD40 ligand) (eg, X-linked hyper IgM syndrome), full gene sequence CDKN2A (cyclin-dependent kinase inhibitor 2A) (eg, CDKN2A-related cutaneous malignant melanoma, familial atypical mole-malignant melanoma syndrome), full gene sequence CLRN1 (clarin 1) (eg, Usher syndrome, type 3), full gene sequence COX6B1 (cytochrome c oxidase subunit VIb polypeptide 1) (eg, mitochondrial respiratory chain complex IV deficiency), full gene sequence CPT2 (carnitine palmitoyltransferase 2) (eg, carnitine palmitoyltransferase II deficiency), full gene sequence CRX (cone-rod homeobox) (eg, cone-rod dystrophy 2, Leber congenital amaurosis), full gene sequence CYP1B1 (cytochrome P450, family 1, subfamily B, polypeptide 1) (eg, primary congenital glaucoma), full gene sequence EGR2 (early growth response 2) (eg, Charcot-Marie-Tooth), full gene sequence EMD (emerin) (eg, Emery-Dreifuss muscular dystro

Other HCPCS codes related to the CPB:

J1304	Injection, tofersen, 1 mg

ICD-10 codes covered if selection criteria are met:

G12.21	Amyotrophic lateral sclerosis (ALS)

Tumor mutational burden-high (TMB-H):


CPT codes covered if selection criteria are met:

0037U	Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden

Other HCPCS codes related to the CPB:

J9271	Injection, pembrolizumab, 1 mg

ICD-10 codes covered if selection criteria are met:


C00.0 - C07,
C09.0 - C10.9,
C12 - C14.8
	Malignant neoplasm of lip, oral cavity, and pharynx [except nasopharynx] [not covered for salivary glands]
C15.3 - C15.9	Malignant neoplasm of esophagus [gastroesophageal junction adenocarcinoma]
C16.0 - C16.9	Malignant neoplasm of stomach
C17.0 - C17.9	Malignant neoplasm of small intestine including duodenum [small bowel adenocarcinoma]
C18.0 - C20	Malignant neoplasm of colon and rectum
C21.0 - C21.8	Malignant neoplasm of anus and anal canal [anal adenocarcinoma]
C22.0	Liver cell carcinoma [hepatocellular carcinoma]
C22.1	Intrahepatic bile duct carcinoma [cholangiocarcinoma]
C23	Malignant neoplasm of gallbladder
C24.0 - C24.9	Malignant neoplasm of other and unspecified parts of biliary tract
C25.0 - C25.9	Malignant neoplasm of pancreas
C26.0 - C26.9	Malignant neoplasm of intestinal tract, part unspecified
C30.0 - C30.1	Malignant neoplasm of nasal cavity and middle ear
C31.0 - C31.9	Malignant neoplasms of accessory sinuses
C32.0 - C32.9	Malignant neoplasm of larynx
C34.00 - C34.92	Malignant neoplasm of bronchus and lung
C37	Malignant neoplasm of thymus
C40.00 - C41.9	Malignant neoplasm of bone and articular cartilage [osteosarcoma, ewing sarcoma, osteosarcoma]
C43.0 - C43.9	Malignant melanoma of skin
C4A.0 - C4A.9	Merkel cell carcinoma

C44.00, C44.02, C44.09
	Unspecified malignant neoplasm, squamous cell carcinoma, or other specified malignant neoplasm of skin of lip
C44.101 - C44.1992	Other and unspecified malignant neoplasm of skin of eyelid including canthus
C44.201 - C44.299	Other and unspecified malignant neoplasm of skin of ear and external auricular canal
C44.300 - C44.399	Other and unspecified malignant neoplasm of skin of other and unspecified parts of face
C44.40 - C44.49	Other and unspecified malignant neoplasm of skin of scalp and neck
C45.0	Mesothelioma of pleura
C48.1 - C48.2	Malignant neoplasm of peritoneum [primary peritoneal cancer]
C50.011 - C50.929	Malignant neoplasm of female and male breast
C51.0 - C51.9	Malignant neoplasm of vulva
C53.0 - C53.9	Malignant neoplasm of cervix uteri
C56.1 - C56.9	Malignant neoplasm of ovary
C57.00 - C57.02	Malignant neoplasm of fallopian tube
C58	Malignant neoplasm of placenta [gestational trophoblastic neoplasia]
C60.1 - C60.9	Malignant neoplasm of penis
C62.90 - C62.92	Malignant neoplasm of testis, unspecified whether descended or undescended
C64.1 - C64.9	Malignant neoplasm of kidney, except renal pelvis
C65.1 - C65.9	Malignant neoplasm of renal pelvis
C66.1 - C66.9	Malignant neoplasm of ureter
C67.0 - C67.9	Malignant neoplasm of bladder
C68.0	Malignant neoplasm of urethra
C69.40 - C69.42	Malignant neoplasm of ciliary body [uveal melanoma]
C73	Malignant neoplasm of thyroid gland
C74.00 - C74.92	Malignant neoplasm of adrenal gland
C76.0	Malignant neoplasm of head, face and neck
C78.89	Secondary malignant neoplasm of other digestive organs
C7A.010 - C7A.8	Malignant neuroendocrine tumors
C7B.8	Other secondary neuroendocrine tumors
C81.10 - C81.79	Hodgkin lymphoma
C84.00 - C84.09	Mycosis fungoides [non-Hodgkin's lymphoma] [not covered for Stage III mycosis fungoides]
C84.10 - C84.19	Sezary disease [non-Hodgkin's lymphoma] [not covered for Stage IV Sezary syndrome]
C85.20 - C85.29	Mediastinal (thymic) large B-cell lymphoma
C86.00 - C86.01	Extranodal NK/T-cell lymphoma, nasal type
C86.60 - C86.61	Primary cutaneous CD30-hyphenpositive T-cell proliferations

Platelet-derived growth factor receptor-beta (PDGFRß) gene rearrangements (e.g., PDGFRB FISH):


CPT codes covered if selection criteria are met:

88271	Molecular cytogenetics; DNA probe, each (eg, FISH)
88275	    interphase in situ hybridization, analyze 100-300 cells

Other HCPCS codes related to the CPB:

S0088	Imatinib, 100 mg

ICD-10 codes covered if selection criteria are met:

C93.10 - C93.12	Chronic myelomonocytic leukemia
D46.0 - D46.9	Myelodysplastic syndromes
D47.1	Chronic myeloproliferative disease

C-KIT testing/ KIT D816V Mutation Detection by PCR:


CPT codes covered if selection criteria are met:

81272	KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, gastrointestinal stromal tumor [GIST], acute myeloid leukemia, melanoma), gene analysis, targeted sequence analysis (eg, exons 8, 11, 13, 17, 18)
81273	KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, mastocytosis), gene analysis, D816 variant(s)

Other HCPCS codes related to the CPB:

S0088	Imatinib, 100 mg

ICD-10 codes covered if selection criteria are met:

C49.A0 - C49.A9	Gastrointestinal stromal tumors
C96.21	Aggressive systemic mastocytosis

Folate receptor alpha testing:


CPT codes covered if selection criteria are met:

Folate receptor alpha testing –no specific code

Other HCPCS codes related to the CPB:

J9063	Injection, mirvetuximab soravtansine-gynx, 1 mg

ICD10 codes covered if selection criteria are met:

C48.0 – C48.8	Malignant neoplasm of retroperitoneum and peritoneum
C56.1 – C56.9	Malignant neoplasm of ovary
C57.00 – C57.02	Malignant neoplasm of fallopian tube

FTL3 mutation assay (e.g., LeukoStrat CDx FLT Mutation Assay):


CPT codes covered if selection criteria are met:

0023U	Oncology (acute myelogenous leukemia), DNA, genotyping of internal tandem duplication, p.D835, p.I836, using mononuclear cells, reported as detection or non-detection of FLT3 mutation and indication for or against the use of midostaurin
81245	FLT3 (fms-related tyrosine kinase 3) (eg, acute nyeloid leukemia), gene analysis; internal tandem duplication (ITD) variants (ie, exons 14, 15)
81246	    tyrosine kinase domain (TKD) variants (eg D835, I836)

Other HCPCS codes related to the CPB:


Midostaurin (Rydapt), Gilternib (Xospata), Sorafenib (Nexavar) – no specific code:


ICD-10 codes covered if selection criteria are met:

C92.00 - C92.02	Myeloid leukemia

Isocitrate dehydrogenase-1 (IDH1), Isocitrate dehydrogenase-2 (IDH2) mutations:


CPT codes covered if selection criteria are met:

81120	IDH1 (isocitrate dehydrogenase 1 [NADP+], soluble) (eg, glioma), common variants (eg, R132H, R132C)
81121	IDH2 (isocitrate dehydrogenase 2 [NADP+], mitochondrial) (eg, glioma), common variants (eg, R140W, R172M)

Other HCPCS codes related to the CPB:


Enasidenib mesylate (Idhifa), Ivosidenib (Tibsovo) - no specific code:


ICD-10 codes covered if selection criteria are met:

C92.00 - C92.02	Myeloid leukemia

Del(17p)/TP53 mutation (e,g,, Vysis CLL FISH Probe Kit):


CPT codes covered if selection criteria are met:


Del(17p)/TP53 mutation (e,g,, Vysis CLL FISH Probe Kit) - no specific code:


ICD-10 codes covered if selection criteria are met:


C91.10 - C91.12, C91.90 - C91.91
	Chronic lymphoid leukemia [chronic lymphocytic]

Other HCPCS codes related to the CPB:


Venetoclax (Venclexta) - no specific code:


BCR/ABL mutations (e.g., MRDx BCR-ABL Test):


CPT codes covered if selection criteria are met:


BCR/ABL mutations (e.g., MRDx BCR-ABL Test ) - no specific code:


ICD-10 codes covered if selection criteria are met:

C92.10 - C92.92	Myeloid leukemia

Other HCPCS codes related to the CPB:


Nilotinib (Tasigna) – no specific code:


FGFR3 mutation testing:


CPT codes covered if selection criteria are met:

0154U	FGFR3 (fibroblast growth factor receptor 3) gene analysis (ie, p.R248C [c.742C>T], p.S249C [c.746C>G], p.G370C [c.1108G>T], p.Y373C [c.1118A>G], FGFR3-TACC3v1, and FGFR3-TACC3v3)

Other HCPCS codes related to the CPB:


Erdafitinib (Balversa), Pemigatinib (Pemazyre) - no specific code:


ICD-10 codes covered if selection criteria are met:

C65.1 - C65.9	Malignant neoplasm of renal pelvis [urothelial carcinoma]
C66.1 - C66.9	Malignant neoplasm of ureter [urothelial carcinoma]
C67.0 - C67.9	Malignant neoplasm of bladder, unspecified [urothelial carcinoma]
C68.0 - C68.9	Malignant neoplasm of other and unspecified urinary organs [urothelial carcinoma]

PIK3CA mutation testing:


CPT codes covered if selection criteria are met:

0037U	Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden
0155U	PIK3CA (phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha) (eg, breast cancer) gene analysis (ie, p.C420R, p.E542K, p.E545A, p.E545D [g.1635G>T only], p.E545G, p.E545K, p.Q546E, p.Q546R, p.H1047L, p.H1047R, p.H1047Y)
0177U	Oncology (breast cancer), DNA, PIK3CA (phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha) gene analysis of 11 gene variants utilizing plasma, reported as PIK3CA gene mutation status
0239U	Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free DNA, analysis of 311 or more genes, interrogation for sequence variants, including substitutions, insertions, deletions, select rearrangements, and copy number variations [FoundationOne Liquid CDx]
81309	PIK3CA (phosphatidylinositol-4, 5-biphosphate 3-kinase, catalytic subunit alpha) (eg, colorectal and breast cancer) gene analysis, targeted sequence analysis (eg, exons 7, 9, 20)

Other HCPCS codes related to the CPB:


Alpelizib (Piqray), Inavolisib (Itovebi) - no specific code:


ICD-10 codes covered if selection criteria are met:

C50.011 - C50.929	Malignant neoplasm of breast

AKT1 activating mutation:


CPT codes covered if selection criteria are met:

AKT1 activating mutation – no specific code

ICD-10 codes covered if selection criteria are met:

C50.011 - C50.929	Malignant neoplasm of breast [HER2-negative]

Mesenchymal-epithelial transition (MET) testing:


CPT codes covered if selection criteria are met:


Mesenchymal-epithelial transition (MET) testing - no specific code:

0037U	Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden

Other HCPCS codes related to the CPB:


Capmatinib (Tabrecta), osimertinib (Tagrisso), erlotinib (Tarceva) - no specific code:

J8565	Gefitinib, oral, 250 mg

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer] [not covered for osimertinib, erlotinib, gefitinib]

EZH2 mutation testing:


CPT codes covered if selection criteria are met:

81236	EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, myelodysplastic syndrome, myeloproliferative neoplasms) gene analysis, full gene sequence
81237	EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, diffuse large B-cell lymphoma) gene analysis, common variant(s) (eg, codon 646)

Other HCPCS codes related to the CPB:


Tazemetostat (Tazverik) - no specific code:


ICD-10 codes covered if selection criteria are met:

C82.00 - C82.99	Follicular lymphoma

RET testing:


CPT codes covered if selection criteria are met:

RET fusion testing, RET mutation testing- no specific code

Other HCPCS codes related to the CPB:

Pralsentinib, Selpercatinib (Retevmo) -no specific code

ICD-10 codes covered if selection criteria are met:

C73	Malignant neoplasm of thyroid gland

ERBB2 (HER2) amplification testing:


CPT codes covered if selection criteria are met:

0009U	Oncology (breast cancer), ERBB2 (HER2) copy number by FISH, tumor cells from formalin fixed paraffin embedded tissue isolated using image-based dielectrophoresis (DEP) sorting, reported as ERBB2 gene amplified or non amplified

Other HCPCS codes related to the CPB:

J9306	Injection, pertuzumab, 1 mg
J9354	Injection, ado-trastuzumab emtansine, 1 mg
J9355	Trastuzumab, 10 mg
J9356	Injection, trastuzumab, 10 mg and Hyaluronidase-oysk (Herceptin Hylecta)
J9358	Injection, fam-trastuzumab deruxtecan-nxki, 1 mg
Q5112	Injection, trastuzumab-dttb, biosimilar, (Ontruzant), 10 mg
Q5113	Injection, trastuzumab-pkrb, biosimilar, (Herzuma), 10 mg
Q5114	Injection, Trastuzumab-dkst, biosimilar, (Ogivri), 10 mg
Q5116	Injection, trastuzumab-qyyp, biosimilar, (Trazimera), 10 mg
Q5117	Injection, trastuzumab-anns, biosimilar, 10 mg (Kanjinti)
Q5146	Injection, trastuzumab-strf (hercessi), biosimilar, 10 mg

ICD-10 codes covered if selection criteria are met:

C08.0 - C08.9	Malignant neoplasm of other and unspecified major salivary glands
C15.3 - C15.9	Malignant neoplasm of esophagus
C16.0 - C16.9	Malignant neoplasm of stomach
C18.0 - C18.9	Malignant neoplasm of colon
C19 - C21.8	Malignant neoplasm of rectosigmoid junction, rectum, anus and anal canal
C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer]
C50.011 - C50.929	Malignant neoplasm of breast
C53.0 - C53.9	Malignant neoplasm of cervix uteri
C54.0 - C54.9	Malignant neoplasm of corpus uteri

Mass spectrometry and tandem mass spectrometry:


CPT codes covered if selection criteria are met:

83789	Mass spectrometry and tandem mass spectrometry (eg, MS, MS/MS, MALDI, MS-TOF, QTOF), non-drug analyte(s) not elsewhere specified, qualitative or quantitative, each specimen

Other HCPCS codes related to the CPB:

J0594	Injection, busulfan, 1 mg

ICD-10 codes covered if selection criteria are met:

C81.10 - C81.99	Hodgkin lymphoma
C90.00 – C90.02	Multiple myeloma

Measurement of microsatellite instability and mismatch repair:


CPT codes covered if selection criteria are met:

81301	Microsatellite instability analysis (eg, hereditary non-polyposis colorectal cancer, Lynch syndrome) of markers for mismatch repair deficiency (eg, BAT25, BAT26), includes comparison of neoplastic and normal tissue, if performed
88341	Immunohistochemistry or immunocytochemistry, per specimen; each additional single antibody stain procedure (List separately in addition to code for primary procedure)
88342	Immunohistochemistry or immunocytochemistry, per specimen; initial single antibody stain procedure [mismatch repair]

CPT codes not covered for indications listed in the CPB:

0037U	Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden

Other HCPCS codes related to the CPB:

J9228	Injection, ipilimumab, 1 mg
J9271	Injection, pembrolizumab, 1 mg
J9272	Injection, dostarlimab-gxly, 10 mg
J9299	Injection, nivolumab, 1 mg

ICD-10 codes covered if selection criteria are met:

C00.0 - C14.8	Malignant neoplasm of lip, oral cavity and pharynx
C15.3 - C26.9	Malignant neoplasm of digestive organs
C30.0 - C39.9	Malignant neoplasm of respiratory and intrathoracic organs
C40.00 - C41.9	Malignant neoplasm of bone and articular cartilage
C43.111 – C4A.122	Melanoma and other malignant neoplasms of skin
C45.0 – C48.8	Malignant neoplasms of mesothelial and soft tissue
C49.0 - C49.9	Malignant neoplasm of other connective and soft tissue
C50.011 – C50.929	Malignant neoplasm of breast
C51.0 – C58	Malignant neoplasm of female genital organs
C60.0 – C63.9	Malignant neoplasms of male genital organs
C64.1 – C68.9	Malignant neoplasm of urinary tract
C69.00 – C72.9	Malignant neoplasms of eye, brain and other parts of central nervous system
C73 – C75.9	Malignant neoplasm of thyroid and other endocrine glands
C76.40 – C80.2	Malignant neoplasms of ill-defined, other secondary and unspecified sites
C7A.00 - C7A.8	Malignant neuroendocrine tumors
C7B.00 – C7B.8	Secondary neuroendocrine tumors
C81.00 – C88.91	Hodgkin lymphoma, follicular lymphoma, non- follicular lymphoma, mature T/NK-cell lymphomas, other specified and unspecified types of non-Hodgkin lymphoma, other specified types of T/NK-cell lymphoma, and malignant immunoproliferative diseases and certain other B-cell lymphomas
C90.00 – C90.02	Multiple myeloma
C90.20 – C90.32	Extramedullary plasmacytoma and solitary plasmacytoma
C96.0 – C96.9	Other and unspecified malignant neoplasms of lymphoid, hematopoietic and related tissue
D00.00 - D09.9	Carcinoma in situ

Praxis Extended RAS Panel :


CPT codes covered if selection criteria are met:

0111U	Oncology (colon cancer), targeted KRAS (codons 12, 13, and 61) and NRAS (codons 12, 13, and 61) gene analysis utilizing formalin-fixed paraffin-embedded tissue

ICD-10 codes covered if selection criteria are met:

C18.0 - C18.9	Malignant neoplasm of colon

Anaplastic lymphoma kinase (ALK) fusion gene testing, (Vysis ALK Break Apart FISH Probe Kit; Ventana ALK (D5F3) CDx Assay):


CPT codes covered if selection criteria are met:

81401	Molecular pathology procedure, Level 2 translocation analysis, qualitative, and quantitative, if performed EML4/ALK (inv (2)) (eg, non-small cell lung cancer)

Other HCPCS codes related to the CPB:


Alectinib (Alecensa) - no specific code:

J9271	Injection, pembrolizumab, 1 mg

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung carcinoma]

Genotyping for IL28B polymorphism:


CPT codes not covered for indications listed in the CPB:

Genotyping for IL28B polymorphism – No specific code

ICD-10 codes not covered for indications listed in the CPB:

B19.20	Unspecified viral hepatitis C without hepatic coma

rs3798220 allele testing:


CPT codes not covered for indications listed in the CPB:

rs3798220 allele testing - No specific code

ICD-10 codes not covered for indications listed in the CPB:

Z79.82	Long-term (current) use of aspirin

Claudin 18 isoform 2 (CLDN18.2) testing (e.g., VENTANA CLDN18 (43-14A) RxDx Assay):


CPT codes covered if selection criteria are met:

Claudin 18 isoform 2 (CLDN18.2) testing (e.g., VENTANA CLDN18 (43-14A) RxDx Assay) – no specific code

Other HCPCS codes related to the CPB:

J1326	Injection, zolbetuximab-clzb, 2 mg

ICD-10 covered for indications listed in the CPB:

C16.0 – C16.9	Malignant neoplasm of stomach

G551D, G1244E, G1349D, G178R, G551S, R117H, S1251N, S1255P, S549N, and S549R mutation in the CFTR testing:


CPT codes covered if selection criteria are met:

G551D, G1244E, G1349D, G178R, G551S, R117H, S1251N, S1255P, S549N, and S549R mutation in the CFTR testing - No specific codes

Other HCPCS codes related to the CPB:

Ivacaftor (Kalydeco) – no specific code

ICD-10 covered if selection criteria are met:

E84.0 - E84.9	Cystic fibrosis [who are being considered for treatment with ivacaftor (Kalydeco)]

F508del mutation in the CFTR testing:


CPT codes covered if selection criteria are met:

81222	CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene analysis; duplication/deletion variants

Other HCPCS codes related to the CPB:

lumacaftor/ivacaftor (Orkambi) – no specific code

ICD-10 codes covered if selection criteria are met:

E84.0-E84.9	Cystic fibrosis [who are being considered for treatment with lumacaftor/ivacaftor (Orkambi)]

GeneSightRx testing (GeneSight ADHD, GeneSight Analgesic, GeneSight MTHFR and GeneSight Psychotropic):


CPT codes not covered for indications listed in the CPB:

GeneSight ADHD, GeneSight Analgesic, GeneSight MTHFR- no specific code
0345U	Psychiatry (eg, depression, anxiety, attention deficit hyperactivity disorder [ADHD]), genomic analysis panel, variant analysis of 15 genes, including deletion/duplication analysis of CYP2D6

ICD-10 codes not covered for indications listed in the CPB:

F30.10 - F39	Mood [affective] disorders

Platelet reactivity/function testing (VerifyNow P2Y12 Assay):


CPT codes not covered for indications listed in the CPB:

Platelet reactivity/function testing (VerifyNow P2Y12 Assay) - No specific code

ICD-10 codes not covered for indications listed in the CPB:

Z98.61	Coronary angioplasty status

OnDose lab test:


HCPCS codes not covered for indications listed in the CPB:

S3722	Dose optimization by area under the curve (AUC) analysis, for infusional 5-Fluorouracil

MGMT (O(6)-methylguanine-DNA methyltransferase) gene methylation Assay:


CPT codes covered if selection criteria are met:

81287	MGMT (O-6-methylguanine-DNA methyltransferase) (eg, glioblastoma multiforme), methylation analysis

Other HCPCS codes related to the CPB:

J8700	Temozolomide, oral, 5 mg
J9328	Injection, temozolomide, 1 mg

ICD-10 covered for if selection criteria are met:

C71.0 - C71.9	Malignant neoplasm of brain [glioblastoma]

Genecept Assay, Psych HealthPGx Panel:


CPT codes not covered for indications listed in the CPB:

0173U	Psychiatry (ie, depression, anxiety), genomic analysis panel, includes variant analysis of 14 genes
0175U	Psychiatry (eg, depression, anxiety), genomic analysis panel, variant analysis of 15 genes
0423U	Psychiatry (eg, depression, anxiety), genomic analysis panel, including variant analysis of 26 genes, buccal swab, report including metabolizer status and risk of drug toxicity by condition

ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

F20.0 - F20.9	Schizophrenia
F30.10 - F39	Mood [affective] disorders
F32.0 - F32.A	Major depressive disorders
F34.0 - F34.9	Persistent mood [affective] disorders
F40.00 - F48.9	Anxiety, dissociative, stress-related, somatoform and other nonpsychotic mental disorders
F90.1 - F90.9	Attention-deficit hyperactivity disorder

Multigene pharmacogenetics panels: Right Med Comprehensive Test, Right Med Gene Report, Right Med PGx16 Test:


CPT codes not covered for indications listed in the CPB:

Aura Genetics Pharmacogenomics (PGx) Test, EffectiveRX Comprehensive Panel - no specific code
0175U	Psychiatry (eg, depression, anxiety), genomic analysis panel, variant analysis of 15 gene
0289U	Neurology(Alzheimer disease),mRNA, gene expression profiling by RNA sequencing of 24 genes,wholeblood, algorithm reported as predictive risks core
0294U	Longevity and mortality risk,mRNA,gene expression profiling by RNA sequencing of 18 genes,wholeblood,algorithm reported as predictive risks core
0347U	Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA analysis, 16 gene report, with variant analysis and reported phenotypes
0348U	Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA analysis, 25 gene report, with variant analysis and reported phenotypes
0349U	Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA analysis, 27 gene report, with variant analysis, including reported phenotypes and impacted gene-drug interactions
0350U	Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA analysis, 27 gene report, with variant analysis and reported phenotypes
0419U	Neuropsychiatry (eg, depression, anxiety), genomic sequence analysis panel, variant analysis of 13 genes, saliva or buccal swab, report of each gene phenotype
0423U	Psychiatry (eg, depression, anxiety), genomic analysis panel, including variant analysis of 26 genes, buccal swab, report including metabolizer status and risk of drug toxicity by condition
0434U	Drug metabolism (adverse drug reactions and drug response), genomic analysis panel, variant analysis of 25 genes with reported phenotypes
0437U	Psychiatry (anxiety disorders), mRNA, gene expression profiling by RNA sequencing of 15 biomarkers, whole blood, algorithm reported as predictive risk score
0438U	Drug metabolism (adverse drug reactions and drug response), buccal specimen, gene-drug interactions, variant analysis of 33 genes, including deletion/duplication analysis of CYP2D6, including reported phenotypes and impacted gene- drug interactions
0460U	Oncology, whole blood or buccal, DNA single-nucleotide polymorphism (SNP) genotyping by real-time PCR of 24 genes, with variant analysis and reported phenotypes
0461U	Oncology, pharmacogenomic analysis of single-nucleotide polymorphism (SNP) genotyping by real-time PCR of 24 genes, whole blood or buccal swab, with variant analysis, including impacted gene-drug interactions and reported phenotypes

Beta adrenergic receptor genotyping:


CPT codes not covered for indications listed in the CPB:

81401	VKORC1 (vitamin K epoxide reductase complex, subunit 1) (eg, warfarin metabolism), gene analysis, common variant(s) (eg, -1639G>A, c.173+1000C>T)

ICD-10 codes not covered for indications listed in the CPB:

J45.20 - J45.998	Asthma [treatment resistant]

Methotrexate polyglutamates (Avise PG test):


CPT codes not covered for indications listed in the CPB:

Methotrexate polyglutamates (Avise PG test) - No specific code

Other HCPCS codes related to the CPB:

J8610	Methotrexate; oral, 2.5 mg
J8611	Methotrexate (jylamvo), oral, 2.5 mg
J8612	Methotrexate (xatmep), oral, 2.5 mg
J9250	Methotrexate sodium, 5 mg
J9255	Injection, methotrexate (accord) not therapeutically equivalent to j9250 or j9260, 50 mg
J9260	Methotrexate sodium, 50 mg

ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

M05.00 - M14.89	Inflammatory polyarthropathies

Millennium PGT:


CPT codes not covered for indications listed in the CPB:

Millennium PGT - No specific code

ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

G89.21 - G89.29	Chronic pain, not elsewhere classified

GENETWORx Neuropsychiatric Panel:


CPT codes not covered for indications listed in the CPB:

0392U	Drug metabolism (depression, anxiety, attention deficit hyperactivity disorder [ADHD]), gene-drug interactions, variant analysis of 16 genes, including deletion/duplication analysis of CYP2D6, reported as impact of gene-drug interaction for each drug

PersonalisedRx:


CPT codes not covered for indications listed in the CPB:

0380U	Drug metabolism (adverse drug reactions and drug response), targeted sequence analysis, 20 gene variants and CYP2D6 deletion or duplication analysis with reported genotype and phenotype
0434U	Drug metabolism (adverse drug reactions and drug response), genomic analysis panel, variant analysis of 25 genes with reported phenotypes

PersonaGene Genetic Panels:


CPT codes not covered for indications listed in the CPB:

81225	CYP2C19 (cytochrome P450, family 2, subfamily C, polypeptide 19) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *8, *17)
81226	CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *4, *5, *6, *9, *10, *17, *19, *29, *35, *41, *1XN, *2XN, *4XN)
81227	CYP2C9 (cytochrome P450, family 2, subfamily C, polypeptide 9) (eg, drug metabolism), gene analysis, common variants (eg, *2, *3, *5, *6)
81240	F2 (prothrombin, coagulation factor II) (eg, hereditary hypercoagulability) gene analysis, 20210G>A variant
81241	F5 (coagulation factor V) (eg, hereditary hypercoagulability) gene analysis, Leiden variant
81291	MTHFR (5,10-methylenetetrahydrofolate reductase) (eg, hereditary hypercoagulability) gene analysis, common variants (eg, 677T, 1298C)
81355	VKORC1 (vitamin K epoxide reductase complex, subunit 1) (eg, warfarin metabolism), gene analysis, common variant(s) (eg, -1639G>A, c.173+1000C>T)
81401	VKORC1 (vitamin K epoxide reductase complex, subunit 1) (eg, warfarin metabolism), gene analysis, common variant(s) (eg, -1639G>A, c.173+1000C>T)
81418	Drug metabolism (eg, pharmacogenomics) genomic sequence analysis panel, must include testing of at least 6 genes, including CYP2C19, CYP2D6, and CYP2D6 duplication/deletion analysis

ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

Z51.81	Encounter for therapeutic drug level monitoring

HLA-B 58:01:


CPT codes covered if selection criteria are met:

HLA-B 58:01 -No specific code

Aegis Drug-Drug Interaction Test:


CPT codes not covered for indications listed in the CPB:

0006U	Prescription drug monitoring, 120 or more drugs and substances, definitive tandem mass spectrometry with chromatography, urine, qualitative report of presence (including quantitative levels, when detected) or absence of each drug or substance with description and severity of potential interactions, with identified substances, per date of service

OneOme RightMed Pharmacogenomic Test:


CPT codes not covered for indications listed in the CPB:

0015U	Drug metabolism (adverse drug reactions), DNA, 22 drug metabolism and transporter genes, real-time PCR, blood or buccal swab, genotype and metabolizer status for therapeutic decision support

Genotyping of interferon-lambda 3 (IFNL3) for prediction of virological response to pegylated-interferon-alpha and ribavirin combination therapy:


CPT codes not covered for indications listed in the CPB:

81283	IFNL3 (interferon, lambda 3) (eg, drug response), gene analysis, rs12979860 variant

PGxOnePlus genetic testing - no specific code:


ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

F40.00 - F48.9	Anxiety, dissociative, stress-related, somatoform and other nonpsychotic mental disorders

K21.00, K21.9
	Gastro-esophageal reflux disease with or without esophagitis

UrSure tenofovir quantification test:


CPT codes not covered for indications listed in the CPB:

0025U	Tenofovir, by liquid chromatography with tandem mass spectrometry (LC-MS/MS), urine, quantitative

ICD-10 codes not covered for indications listed in the CPB (not all inclusive):

Z20.6	Contact with and (suspected) exposure to human immunodeficiency virus [HIV]
Z20.828	Contact with and (suspected) exposure to other viral communicable diseases

INFINITI Neural Response Panel:


CPT codes not covered for indications listed in the CPB:

0078U	Pain management (opioid-use disorder) genotyping panel, 16 common variants (ie, ABCB1, COMT, DAT1, DBH, DOR, DRD1, DRD2, DRD4, GABA, GAL, HTR2A, HTTLPR, MTHFR, MUOR, OPRK1, OPRM1), buccal swab or other germline tissue sample, algorithm reported as positive or negative risk of opioid-use disorder

ComplyRX urine test:


CPT codes not covered for indications listed in the CPB:

0093U	Prescription drug monitoring, evaluation of 65 common drugs by LC-MS/MS, urine, each drug reported detected or not detected

Oncolytic Assurance RX:


CPT codes not covered for indications listed in the CPB:

0110U	Prescription drug monitoring, one or more oral oncology drug(s) and substances, definitive tandem mass spectrometry with chromatography, serum or plasma from capillary blood or venous blood, quantitative report with steady-state range for the prescribed drug(s) when detected

Snapshot Oral Fluid Compliance test:


CPT codes not covered for indications listed in the CPB:

0116U	Prescription drug monitoring, enzyme immunoassay of 35 or more drugs confirmed with LC-MS/MS, oral fluid, algorithm results reported as a patient-compliance measurement with risk of drug to drug interactions for prescribed medications

ICD-10 codes not covered for indications listed in the CPB:

Z51.81	Encounter for therapeutic drug level monitoring

KRAS G12C mutation testing:


CPT codes covered if selection criteria are met:

81275	KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; variants in exon 2 (eg, codons 12 and 13)
81276	KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; additional variant(s) (eg, codon 61, codon 146)

Other HCPCS codes related to the CPB:

Sotorasib (Lumakras), Adagrasib (Krazati), Erlotinib (Tarceva) - no specific code

ICD-10 codes covered if selection criteria are met:

C34.00 - C34.92	Malignant neoplasm of bronchus and lung [non-small cell lung cancer (NSCLC)]

Experimental and Investigation Treatment - no specific codes:

A1555G, Amerigene PGT pharmacogenetic testing panel, Proove Profiles (including Proove Opioid Risk Panel, Proove Opioid Response Profile, Proove Non-opioid Response Profile, Proove Opioid-Induced Side Effects Profile, Proove NSAID Risk Profile, Proove Fibromyalgia Profile, Proove Epidural with Fentanyl Response Profile, Proove MAT (Medically Assisted Treatment) Response Profile, Proove TMD Profile, Proove Psychiatric Risk and Response Profile, Proove Addiction Profile, Proove Narcotic Risk Panel, Proove Drug Metabolism Panel, rxSEEK Epilepsy Drug Metabolism Test (Courtagen LIfe Sciences), CQuentia pharmacogenetic comprehensive panel and pharmacogenetic studies for prediction of response to medication for chronic kidney disease

CPT codes not covered for indications listed in the CPB:

0032U	COMT (catechol-O-methyltransferase)(drug metabolism)gene analysis, c.472G>A (rs4680) variant
0033U	HTR2A (5-hydroxytryptamine receptor 2A), HTR2C(5-hydroxytryptamine receptor 2C) (eg, citalopram metabolism) gene analysis, common variants (ie, HTR2Ars7997012 [c.614-2211T>C], HTR2Crs3813929 [c.-759C>T] and rs1414334 [c.551-3008C>G])
0332U	Oncology (pan-tumor), genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative polymerase chain reaction (qPCR), whole blood, reported as a high or low probability of responding to immune checkpoint–inhibitor therapy
0411U	Psychiatry (eg, depression, anxiety, attention deficit hyperactivity disorder [ADHD]), genomic analysis panel, variant analysis of 15 genes, including deletion/duplication analysis of CYP2D6
0476U	Drug metabolism, psychiatry (eg, major depressive disorder, general anxiety disorder, attention deficit hyperactivity disorder [ADHD], schizophrenia), whole blood, buccal swab, and pharmacogenomic genotyping of 14 genes and CYP2D6 copy number variant analysis and reported phenotypes [RightMed Mental health gene report]
0477U	Drug metabolism, psychiatry (eg, major depressive disorder, general anxiety disorder, attention deficit hyperactivity disorder [ADHD], schizophrenia), whole blood, buccal swab, and pharmacogenomic genotyping of 14 genes and CYP2D6 copy number variant analysis, including impacted gene-drug interactions and reported phenotypes [RightMed Mental health Medication report]
0516U	Drug metabolism, whole blood, pharmacogenomic genotyping of 40 genes and CYP2D6 copy number variant analysis, reported as metabolizer status [MyGenvar Pharmacogenomics test]
Background

Adverse drug reactions (ADRs) are responsible for many debilitating side effects and are a major cause of death following drug therapy.  It is now clear that a significant portion of these ADRs as well as therapeutic failures are caused by genetic polymorphism and  genetically based inter-individual differences in drug absorption, disposition, excretion or metabolism.  

Pharmacogenomics testing is laboratory testing which has the potential to determine how an individual’s genetic factors may affect the safety and effectiveness of that individual’s response to a specific medication. The goal of pharmacogenomics testing is to reduce the incidence of adverse medication reactions while improving an individual’s positive response to the medication. Additionally, some tests may help provide information on how well a specific treatment may work for an individual.

Genotyping for Cytochrome P450 Polymorphisms

Cytochrome P450 enzymes are a group of enzymes that account for approximately 75 percent of drug metabolism in the human body. Enzymes encoded by the P450 genes (e.g., CYP2C9, CYP2C19, CYP2D6, CYP3A4, CYP3A5 etc.) are found primarily in the liver. The action of the P450 enzymes affects the blood levels of many drugs. Genotyping for cytochrome P450 has been proposed for possible use in medical management of drug therapies including, but not limited to, antidepressants, antiepileptic, antipsychotics, barbiturates, clopidogrel, opioid analgesics, proton pump inhibitors, tamoxifen or warfarin. 

Recent advances in molecular biology have improved the understanding of genetic factors underlying many ADRs.  Until recently, investigations in this field have generally centered on gene coding for drug-metabolizing enzymes.  Inactivating mutations have been found in gene coding for enzymes belonging to the cytochrome P450 (CYP) system, which is important in the hepatic metabolism of many drugs.  Individuals with a lack of functional activity in these enzymes should be treated with very low doses of drugs metabolized by the same enzyme to avoid excessive drug levels and thus toxic effects.  In recent years, research has been focused on gene coding for drug targets.  As a result, most studies have been performed on single genes known to be or assumed to be functionally related to a given ADR.  An alternative method is testing for complex single nucleotide polymorphisms that may be associated with ADRs, although the functional relationship between them may be completely unknown.  As a consequence of influence from non-genetic factors in the development of ADRs, the association between a specific genotype and an ADR will always be less than 100 %.  Thus, there is a need for well-designed clinical trials to ascertain the extent of environmental influences on the ADRs for which a genetic basis has been implicated (Guzey and Spigset, 2004).  This is in agreement with the observation of Pirmohamed and Park (2003) who stated that “ADRs are common and many are suggested to have a genetic predisposition.  There has been intense research in the role of CYP enzyme gene polymorphisms in the cause of ADRs.  The major impact to date of polymorphic CYP expression has been on pre-clinical drug development.  The direct clinical impact of CYP polymorphisms on prediction of ADRs, however, has been limited mainly because published reports have been small and retrospective and their findings conflicting.  Moreover, the clinical- and cost-effectiveness of pre-prescription genotyping for CYP polymorphisms has not been established.  More investigation is needed before prospective CYP genotyping can become routine clinical practice”.

Kirchheiner and Brockmoller (2005) stated that the genetic coding for the CYP enzyme 2C9 (CYP2C9) carries many inherited polymorphisms.  Those coding for R144C (*2) and I359L (*3) amino acid substitutions have both significant functional effects and appreciable high population frequencies, and their in vivo consequences have been examined in humans in relation to drug metabolism -- pharmacokinetics, drug responses as well as outcomes of clinical trials in subjects with different CYP2C9 genotypes.  Tentative estimates of how CYP2C9 genotyping might be applied to dose adjustments in clinical therapy were based on dose-related pharmacokinetic parameters such as clearance or trough drug concentrations.  Mean clearances in homozygous carriers of the *3 allele were below 25 % of that of the wild type for S-warfarin, tolbutamide, glipizide, celecoxib, and fluvastatin.  In the more frequent heterozygous carriers (genotype *1/*3), the clearances were between 40 and 75 %.  In these cases in which individual dosages are derived from clinical drug effects, such as for oral anticoagulants, the pharmacogenetics-based dose adjustments showed a good correlation with the genotype-specific empirically derived doses.  In addition to its role in pharmacokinetics, CYP2C9 contributes to the metabolism of fatty acids, prostanoids, and steroid hormones, and it may catalyze potentially toxic bioactivation reactions.  However, the current understanding of the role of CYP2C9 in biotransformation of endogenous signaling molecules and in drug toxicity is relatively meager.  These investigators concluded that the concept of therapy based on genotyping for CYP polymorphisms has not been assessed in prospective, randomized, controlled studies in which one group is dosed according to genotype while another group is dosed in a usual manner.  It is unlikely that CYP2C9 genotyping will become routine practice unless its clinical value is supported by such rigorous assessments.

The following is a list
Footnotes for list
* of some CYP2C9-metabolized drugs with narrow therapeutic ratios (drugs with a narrow difference between therapeutic and toxic concentrations) with serious toxic effects:

Angiotensin II blockers (e.g., irbesartan, losartan)
NSAIDS (e.g., diclofenac, ibuprofen, naproxen, piroxicam)
Oral hypoglycemic agents (e.g., glipizide, tolbutamide)
Sulfonylureas (e.g., amitriptyline, celecoxib, fluoxetine, fluvastatin, glipizide, rosiglitazone, tamoxifen, tolbutamide, S-warfarin).

Diagnostic genotyping tests for some CYP enzymes are now available commercially.  The AmpliChip (Roche Diagnostics, Basel, Switzerland), cleared by the United States Food and Drug Administration (FDA) through the 510(k) process, is a microarray consisting of many DNA sequences complementary to 2 CYP genes applied in microscopic quantities at ordered locations on a solid surface (chip).  The AmpliChip tests the DNA from a patient's leukocytes collected in a standard anti-coagulated blood sample for 29 polymorphisms and mutations for the CYP2D6 gene and 2 polymorphisms for the CYP2C19 gene.  CYP2D6 metabolizes about 25 % of all clinically used medications (e.g., antiarrhythmics, antidepressants, beta-blockers, dextromethorphan and morphine derivatives), while CYP2C19 metabolizes several important types of drugs including anti-epileptics and proton-pump inhibitors.  The AmpliChip was cleared for marketing for CYP2D6 testing in December 2004, and for CYP2C19 testing in January 2005. 

The AmpliChip is marketed for use in screening of patients who are to be treated with drugs metabolized by the CYP system that have a narrow therapeutic ratio with serious toxic effects. 

The following is a list
Footnotes for list
* of CYP2D6-metabolized drugs with narrow therapeutic ratios (not an all inclusive list):

Anti-depressants (e.g., amitriptyline, clomipramine, desipramine, imipramine, paroxetine)
Anti-psychotics (e.g., chlorpromazine, haloperidol, perphenazine, risperidone, thioridazine)
Beta blockers (e.g., carvedilol, propafenone, timolol).

The following is a list
Footnotes for list
* of CYP2C19-metabolized drugs with narrow therapeutic ratios (not an all inclusive list):

Anti-epileptics (e.g., amitriptyline, clomipramine, cyclophosphamide, diazepam, imipramine, moclobemide, phenytoin, phenobarbitone, primidone, R-warfarin)
Proton pump inhibitors (e.g., lansoprazole, omeprazole, pantoprazole)
Others (e.g., clopidogrel).

Footnotes for list
* These lists are based on information excerpted from the “Cytochrome P450 Drug Interaction Table”, Indiana University School of Medicine, 2009. 

Randomized controlled trials are needed to ascertain if the AmpliChip will lower the incidence of ADRs by detecting patients with CYP2D6 and CYP2C19 mutations.  The effectiveness of the AmpliChip in reducing toxic effects and improving health outcomes would need to be compared with standard methods of therapeutic drug monitoring (e.g., by monitoring clinical response or by measuring serum drug concentrations).  Moreover, it is still unclear that the AmpliChip would eliminate the need for simultaneous use of other methods of therapeutic drug monitoring (including serum measurements) because factors other than genetic polymorphisms such as those described earlier have a significant effect on drug pharmacokinetics and pharmacodynamics. 

Juran and colleagues (2006) stated that "[e]ven though the AmpliChip CYP450 has been approved by the FDA, its practical clinical utility has not yet been determined, and there is a paucity of data related to gastrointestinal and liver diseases.  An understanding of the principles and opportunities provided by this new category of diagnostic test is key before planning the necessary studies to evaluate the usefulness of AmpliChip CYP450 in gastroenterologic clinical practice".

A special report (BlueCross BlueShield Technology Evaluation Center, 2004) on genotyping for CYP polymorphisms to determine drug-metabolizer status stated that diagnostic tests to identify specific polymorphisms that may be linked to increased/reduced effect or serious adverse effects are now available.  Whether such testing, and for which drugs, improves patient outcomes is not yet known.  While genotyping for the CYP enzymes would only need to be performed once per patient and the results could be used to consider other drugs metabolized by the same enzymes, whether genotyping is clinically useful would need to be determined for each drug.  Even drugs of the same class may variably rely on specific CYP enzymes.  For example, the plasma level of the selective serotonin reuptake inhibitor (SSRI) fluoxetine is significantly affected by CYPP2D6 polymorphisms, whereas the SSRI sertraline appears to be little affected and may depend more upon CYP2C19 for metabolism.  The report stated that while the potential of pharmacogenetic studies is intriguing for many clinical applications, it is still unclear which are most likely to provide clinical benefit in the near future.

Ingelman-Sundberg (2005) stated that the polymorphism of CYP2D6 significantly affects the pharmacokinetics of approximately half of the drugs in clinical use, which are CYP2D6 substrates.  The consequences of the polymorphism at ordinary drug doses can be either ADRs or no drug response.  Predictive CYP2D6 genotyping is estimated to be beneficial for treatment of about 30 to 40 % of CYP2D6 drug substrates representing about 7 to 10 % of all clinically used drugs, although prospective clinical studies are needed to determine the exact benefit of drug selection and dosage based on the CYP2D6 genotype.  Furthermore, Sanderson et al (2005) assessed the strength and quality of existing evidence about CYP2C9 gene variants and clinical outcomes in warfarin-treated patients in a meta-analysis (11 studies with a total of 2,775 patients).  These investigators concluded that patients with CYP2C9*2 and CYP2C9*3 alleles have lower mean daily warfarin doses and a greater risk of bleeding.  The authors also stated that while testing for gene variants could potentially alter clinical management in patients commencing warfarin, evidence for the clinical utility and cost-effectiveness of genotyping is necessary before routine testing can be recommended.

In a review article on “Drug Metabolism and Variability among Patients in Drug Response” published in the New England Journal of Medicine, Wilkinson (2005) stated that “there have not yet been prospective clinical trials showing that knowledge of a patient’s genotypic profile before prescribing drugs either increases drug efficacy, prevents or reduces adverse drug reactions, or lower the overall costs of therapy and associate sequelae …. For now, however, the individual patient is probably best served by an alert physician aware of the possibility that a genetic polymorphism in drug metabolism may be a potential factor in unexpected drug response”.

In a multi-center 6-week study, Fux et al (2005) examined the impact of CYP2D6 polymorphism on the tolerability of metoprolol in a primary care setting.  The adverse effects studied comprised effects related to the central nervous system, cardiovascular effects, and sexual dysfunction.  The dosage of metoprolol was determined on an individual basis and could be adjusted on clinical grounds.  The indication for treatment was hypertension in about 90 % of cases.  CYP2D6 genotyping covered alleles *3 to *10 and *41 and the duplications.  Possible ADRs of metoprolol were systematically assessed over the study period using standardized rating scales and questionnaires.  The final study population comprised 121 evaluable patients: 5 ultra-rapid metabolizers (UMs) (4.1 %), 91 extensive metabolizers (EMs) (75 %), 21 intermediate metabolizers (IMs) (17 %), and 4 poor metabolizers (PMs) (3.3 %).  Plasma metoprolol concentrations normalized for the daily dose and metoprolol/alpha-hydroxymetoprolol ratios at steady state were markedly influenced by CYP2D6 genotype and displayed a gene-dose effect.  The median of the dose-normalized metoprolol concentration was 0.0088 ng/ml, 0.047 ng/ml, 0.34 ng/ml, and 1.34 ng/ml among UMs, EMs, IMs, and PMs, respectively (p < 0.0001).  There was no significant association between CYP2D6 genotype-derived phenotype (EMs and UMs combined versus PMs and IMs combined) and ADRs during treatment with metoprolol.  There was a tendency toward a more frequent occurrence of cold extremities in the PM plus IM group as compared with the EM plus UM group (16.0 % versus 4.2 %, p = 0.056; relative risk, 3.8 [95 % confidence interval [CI]: 1.03 to 14.3]).  These investigators concluded that CYP2D6 genotype-derived phenotype was not significantly associated with a propensity for ADRs to develop during treatment with metoprolol.  However, the results concerning tolerability of metoprolol in PMs were inconclusive because of the small number of PMs enrolled.

The mechanisms of variable response to tamoxifen have been the subject of much scrutiny in the published literature. Early studies attempting to link a clinical response to tamoxifen therapy with plasma tamoxifen concentrations reported no statistically significant differences in outcomes between women who received 20 mg of tamoxifen daily and those who received 40 mg of tamoxifen daily, even though women in the 40 mg tamoxifen group had higher plasma tamoxifen concentrations than those in the 20 mg tamoxifen group. These results have been reported as evidence that plasma tamoxifen concentration is not a predictor of clinical outcome.  Because there is evidence that tamoxifen is converted to anti-estrogenic metabolites, one hypothesis is that altered patterns of metabolism of tamoxifen might contribute to inter-individual variability in effects (Jin et al, 2005).  Plasma concentrations of the active tamoxifen metabolite endoxifen are associated with the cytochrome P450 (CYP) 2D6 genotype.

Goetz et al (2005) stated that polymorphisms in tamoxifen metabolizing genes affect the plasma concentration of tamoxifen metabolites, but their effect on clinical outcome is unknown.  These investigators determined cytochrome P450 (CYP)2D6 (*4 and *6) and CYP3A5 (*3) genotype from paraffin-embedded tumor samples and buccal cells (living patients) in tamoxifen-treated women enrolled onto a North Central Cancer Treatment Group adjuvant breast cancer trial.  The relationship between genotype and disease outcome was determined using the log-rank test and Cox proportional hazards modeling.  Paraffin blocks were obtained from 223 of 256 eligible patients, and buccal cells were obtained from 17 living women.  CYP2D6 (*4 and *6) and CYP3A5 (*3) genotypes were determined from 190, 194, and 205 patient samples and in 17 living women.  The concordance rate between buccal and tumor genotype was 100 %.  Women with the CYP2D6 *4/*4 genotype had worse relapse-free time (RF-time; p = 0.023) and disease-free survival (DFS; p = 0.012), but not overall survival (p = 0.169) and did not experience moderate to severe hot flashes relative to women heterozygous or homozygous for the wild-type allele.  In the multi-variate analysis, women with the CYP2D6 *4/*4 genotype still tended to have worse RFS (hazard ratio [HR], 1.85; p = 0.176) and DFS (HR, 1.86; p = 0.089).  The CYP3A5*3 variant was not associated with any of these clinical outcomes.  The authors concluded that in tamoxifen-treated patients, women with the CYP2D6 *4/*4 genotype tend to have a higher risk of disease relapse and a lower incidence of hot flashes, which is consistent with their previous observation that CYP2D6 is responsible for the metabolic activation of tamoxifen to endoxifen.  They noted that "[t]hese findings have the potential to improve the ability of physicians to select the optimal hormonal therapy for the treatment of ER-positive breast cancer.  Further studies are needed in women receiving tamoxifen to fully define the effect of CYP2D6 genetic polymorphisms and medications that inhibit CYP2D6 on tamoxifen response."

An assessment of CYP2D6 pharmacogenomics of tamoxifen treatment conducted by the BlueCross BlueShield Association Technology Evaluation Center (2008) evaluated the evidence for CYP2D6 genotyping and tamoxifen treatment efficacy.  The hypothesis examined by the assessment is that CYP2D6 poor metabolizers, whether by genotype or by co-administration of CYP2D6 inhibitory medication, have reduced tamoxifen metabolism and lower endoxifen levels compared to better metabolizers, and as a result have poorer clinical outcomes.  The reviewers stated that this hypothesis is based on the assumption, not yet supported by evidence, that some level of endoxifen is necessary for tamoxifen efficacy and that this level is not achieved in genotypic and functional CYP2D6 poor metabolizers and possibly not in some intermediate metabolizers.  However, the reviewers found no scientific evidence for a significant association between endoxifen and clinical outcomes.  In addition, they reported several limitations on the association of genotype with clinical outcomes.  The reviewers stated, "Because tamoxifen metabolism is complex and CYP2D6 does not appear to account for all variability in endoxifen levels, it is conceivable that polymorphisms in other tamoxifen metabolic pathway enzymes may affect active metabolite levels, and direct measurement of the metabolite(s) itself may be the better predictor of benefit from tamoxifen treatment.  However, since it takes 8 weeks for tamoxifen metabolites to reach steady-state concentrations, measuring metabolite levels is not practical for clinical applications outside of a retrospective study."  Furthermore, the reviewers noted that multiple enzyme genotypes may be needed to confidently predict tamoxifen versus aromatase inhibitors treatment benefit; however, there are little data at present to recommend any genotype combinations.  The assessment concluded, "There is insufficient evidence to permit conclusions regarding the use of CYP2D6 genotyping for directing endocrine therapy regimen selection for women at high risk for or with breast cancer."

Visvanathan et al (2009) updated the 2002 American Society of Clinical Oncology guideline on pharmacologic interventions for breast cancer risk reduction.  The cytochrome P450 2D6 gene (CYP2D6) encodes the enzyme responsible for catalyzing the conversion of tamoxifen to endoxifen, an active metabolite of tamoxifen.  Functional allelic variants (*4 most common in whites and *10 most common in Asians), have been identified in approximately 7 % of the population.  Lower levels of endoxifen have been observed in women taking tamoxifen who are heterozygous and homozygotes for variant alleles in CYP2D6 in a dose-dependent manner, or in women treated with concomitant medications that block CYP2D6, including certain selective serotonin reuptake inhibitors such as paroxetine.  Further, in a small nested case-control study of women who took part in the Italian prevention trial, a higher prevalence of the CYP2D6 *4/*4 phenotype was observed among women with breast cancer who took tamoxifen compared with controls.  The authors concluded that confirmation of these results in larger studies is needed.  Given the limited evidence, CYP2D6 testing is currently not recommended in the preventive setting.

The BlueCross BlueShield Association's technology assessment on CYP2D6 pharmacogenomics of tamoxifen treatment (2011) stated that studies showing no evidence of association between CYP2D6 genotype and either tamoxifen- or aromatase inhibitor-treated patient outcomes has suggested that using the results of CYP2D6 genetic testing to influence decisions about tamoxifen treatment is not currently warranted.

A BlueCross BlueShield Association Technology Evaluation Center (TEC, 2013) assessment of CYP2D6 pharmacogenetics of tamoxifen treatment concluded that CYP2D6 genotyping does not meet the TEC criteria for directing endocrine therapy regimen selection for women at high risk for primary breast cancer or breast cancer recurrence. The assessment stated that there are several limitations to the overall body of evidence, but the largest, most well-designed studies do not support clinical validity of the test.  In the absence of evidence for clinical validity, evidence to support clinical utility is lacking.

Regan et al (2012) noted that adjuvant tamoxifen therapy is effective for post-menopausal women with endocrine-responsive breast cancer.  Cytochrome P450 2D6 (CYP2D6) enzyme metabolizes tamoxifen to clinically active metabolites, and CYP2D6 polymorphisms may adversely affect tamoxifen efficacy.  In this study, these researchers investigated the clinical relevance of CYP2D6 polymorphisms.  They obtained tumor tissues and isolated DNA from 4,861 of 8,010 post-menopausal women with hormone receptor-positive breast cancer who enrolled in the randomized, phase III double-blind Breast International Group (BIG) 1-98 trial between March 1998 and May 2003 and received tamoxifen and/or letrozole treatment.  Extracted DNA was used for genotyping nine CYP2D6 single-nucleotide polymorphisms using polymerase chain reaction-based methods.  Genotype combinations were used to categorize CYP2D6 metabolism phenotypes as poor, intermediate, and extensive metabolizers (PM, IM, and EM, respectively; n = 4,393 patients).  Associations of CYP2D6 metabolism phenotypes with breast cancer-free interval (referred to as recurrence) and treatment-induced hot flushes according to randomized endocrine treatment and previous chemotherapy were assessed.  Cox proportional hazards models were used to calculate hazard ratios (HRs) and 95 % confidence intervals (CIs).  All statistical tests were two-sided.  No association between CYP2D6 metabolism phenotypes and breast cancer-free interval was observed among patients who received tamoxifen monotherapy without previous chemotherapy (p = 0.35).  PM or IM phenotype had a non-statistically significantly reduced risk of breast cancer recurrence compared with EM phenotype (PM or IM versus EM, HR of recurrence = 0.86, 95 % CI: 0.60 to 1.24).  CYP2D6 metabolism phenotype was associated with tamoxifen-induced hot flushes (p = 0.020).  Both PM and IM phenotypes had an increased risk of tamoxifen-induced hot flushes compared with EM phenotype (PM versus EM, HR of hot flushes = 1.24, 95 % CI: 0.96 to 1.59; IM versus EM, HR of hot flushes = 1.23, 95 % CI: 1.05 to 1.43).  The authors concluded that CYP2D6 phenotypes of reduced enzyme activity were not associated with worse disease control but were associated with increased hot flushes, contrary to the hypothesis.  The results of this study do not support using the presence or absence of hot flushes or the pharmacogenetic testing of CYP2D6 to determine whether to treat post-menopausal breast cancer patients with tamoxifen. 

Rae et al (2012) stated that adjuvant tamoxifen therapy substantially decreases the risk of recurrence and mortality in women with hormone (estrogen and/or progesterone) receptor-positive breast cancer.  Previous studies have suggested that metabolic conversion of tamoxifen to endoxifen by cytochrome P450 2D6 (CYP2D6) is required for patient benefit from tamoxifen therapy.  Tumor specimens from a subset of post-menopausal patients with hormone receptor-positive early-stage (stages I, II, and IIIA) breast cancer, who were enrolled in the randomized double-blind Arimidex, Tamoxifen, Alone or in Combination (ATAC) clinical trial, were genotyped for variants in CYP2D6 (n = 1,203 patients: anastrozole [trade name: Arimidex] group, n = 615 patients; tamoxifen group, n = 588 patients) and UDP-glucuronosyltransferase-2B7 (UGT2B7), whose gene product inactivates endoxifen (n = 1,209 patients; anastrozole group, n = 606 patients; tamoxifen group, n = 603 patients).  Genotyping was performed using polymerase chain reaction-based TaqMan assays.  Based on the genotypes for CYP2D6, patients were classified as poor metabolizer (PM), intermediate metabolizer (IM), or extensive metabolizer (EM) phenotypes.  These investigators evaluated the association of CYP2D6 and UGT2B7 genotype with distant recurrence (primary endpoint) and any recurrence (secondary endpoint) by estimating the hazard ratios (HRs) and corresponding 95 % CIs using Cox proportional hazards models.  All statistical tests were 2-sided.  After a median follow-up of 10 years, no statistically significant associations were observed between CYP2D6 genotype and recurrence in tamoxifen-treated patients (PM versus EM: HR for distant recurrence = 1.25, 95 % CI: 0.55 to 3.15, p = o.64; HR for any recurrence = 0.99, 95 % CI: 0.48 to 2.08, p = 0.99).  A near-null association was observed between UGT2B7 genotype and recurrence in tamoxifen-treated patients.  No associations were observed between CYP2D6 and UGT2B7 genotypes and recurrence in anastrozole-treated patients.  The authors concluded that these findings do not support the hypothesis that CYP2D6 genotype predicts clinical benefit of adjuvant tamoxifen treatment among post-menopausal breast cancer patients.

In a review on pharmacogenomics and individualized drug therapy, Eichelbaum et al (2006) stated that "[t]here is also a growing list of genetic polymorphisms in drug targets that have been shown to influence drug response.  A major limitation that has heretofore moderated the use of pharmacogenetic testing in the clinical setting is the lack of prospective clinical trials demonstrating that such testing can improve the benefit/risk ratio of drug therapy".  Furthermore, Humphries and Hingorani (2006) noted that the full potential of the field of pharmacogenetics will only be realized with much further work.

An assessment of CYP450 genetic testing by the Canadian Coordinating Office for Health Technology Assessment (Palylyk-Colwell, 2006) concluded: "No published studies show that patient outcomes can be predicted or altered by knowledge of DME status in the absence of other confounding variables.  Prospective studies are needed to assess the benefits and potential risks of this technology in guiding drug selection and dose adjustment."

Since their introduction in the late 1980s, SSRIs such as citalopram, fluoxetine, paroxetine, and sertraline have become the most commonly prescribed class of drugs for treating depression.  However, the likelihood that a patient will experience relief from all symptoms of depression after 1 year of treatment is approximately 40 %, and adverse events cause 12 to 15 % of patients who start treatment to stop taking the drug.  Following the recent FDA approval of a test to predict differences in the CYP450 gene, physicians and patients must decide if using such tests to choose a type or dose of an SSRI might improve the patient's response to treatment.

Kirchheiner and colleagues (2003) noted that antidepressants are characterized by a high rate of drug failure.  There is evidence that genetic factors are contributing to the inter-individual variability in response to these medications.  Genetic polymorphisms in drug metabolizing enzymes are well established and have significant effects on oral clearances or elimination half-lives of antidepressants.  These differences can be compensated by adapting the individual dose to genotype in addition to other factors such as age, gender, weight, drug interactions, diseases (cardiovascular, hepatic, renal and respiratory) as well as environmental influences on drug metabolism (e.g., diet and smoking).  Genetic variability is found in molecular structures of antidepressant effects.  Furthermore, Kirchheiner et al (2003) noted that studies on response of antidepressants have revealed influences of polymorphisms in neurotransmitter receptors and transporters altering sensitivity of patients to treatment with antidepressants; however, results were often contradictory.

The Agency for Healthcare Research and Quality (AHRQ, 2006) released an evidence report that found there is insufficient evidence to determine if current gene-based tests intended to personalize the dose of SSRIs improve patient outcomes or aid physicians or patients in making treatment decisions.  The available studies indicated that the tests are largely accurate at evaluating differences in genes belonging to the CYP450 family that affect the rate at which a person metabolizes SSRIs.  However, additional well-designed studies are needed to determine the usefulness of test results in the clinical setting.  This report is the first step in the 2-step process of Centers for Disease Control and Prevention's Evaluation of Genomic Applications in Practice and Prevention (EGAPP) pilot project to evaluate and make recommendations regarding the use of gene-based tests.

The Evaluation of Genomic Applications in Practice and Prevention Working Group (EGAPP, 2007) found insufficient evidence to support a recommendation for or against use of cytochrome P450 (CYP450) testing in adults beginning SSRI treatment for non-psychotic depression.  In the absence of supporting evidence, and with consideration of other contextual issues, EGAPP discourages use of CYP450 testing for patients beginning SSRI treatment until further clinical trials are completed. EGAPP found that use of genetic testing for CYP450 polymorphisms and impact on physician decision-making with regard to use of SSRIs is not known.  EGAPP noted that, in the absence of evidence supporting clinical utility, widespread use of CYP450 genetic testing is potentially costly and may not lead to changes in treatment that improve patient outcomes.

The eradication rates of Helicobacter pylori by triple therapy consisting of 1 proton pump inhibitor (PPI) and 2 antimicrobial agents are mainly influenced by bacterial susceptibility to anti-microbial agents and the magnitude of acid inhibition during treatment with a PPI (e.g., omeprazole, lansoprazole, rabeprazole).  Tailored therapy using CYP2C19 pharmacogenomics with a PPI has been proposed as a method to help improve the efficacy of H. pylori eradication rates. 

A review on the use of pharmacogenomics-based treatment of H. pylori infection conducted by the BlueCross BlueShield Association Technology Evaluation Center (2008) examined the scientific evidence of a pharmacogenomics-based treatment regimen for the eradication of H. pylori.  The reviewers found 1 randomized, controlled study that met their inclusion criteria.  The reviewers reported that this study found higher eradication rates after first-line treatment for the pharmacogenomics group compared with the standard treatment group, however, because of numerous variations in treatment protocol within the pharmacogenomics group, it was not possible to determine whether the improvement resulted from the tailored PPI dosages according to CYP2C19 genetic status, or if it was due to other variations in the treatment protocol unrelated to CYP2C19 status.  Furthermore, the review noted that it was possible other clinical factors, such as clarithromycin resistance, or other treatment factors, such as length of antibiotic treatment, influenced eradication rates.  In addition, the study was performed in a Japanese population and did not employ a diagnostic approach or a treatment regimen that is standard care in the United States.  The review concluded, "The scientific evidence does not permit conclusions on whether the use of a pharmacogenomics-based treatment regimen for H. pylori improves eradication rates."

Clopidogrel bisulfate (Plavix) (Bristol-Myers Squibb/Sanofi Pharmaceuticals, Bridgewater, NJ) is an orally administered antiplatelet drug that is used to prevent blood clots that could lead to heart attacks or strokes in individuals with established cardiovascular atherosclerotic disease.  The degree of platelet inhibition seen following use of the second-generation thienopyridine clopidogrel varies from patient to patient in a normal or bell-shaped distribution.  The variability in non-response is such that, when laboratory measurements of platelet aggregation are performed, 4 to 30 % of patients treated with clopidogrel do not have an adequate anti-platelet response.

Studies have linked reduced response to clopidogrel to variants in the gene CYP2C19.  Mega et al (2009) reported that in healthy subjects who were treated with clopidogrel, carriers of at least one CYP2C19 reduced-function allele had a relative reduction of 32.4 % in plasma exposure to the active metabolite of clopidogrel, as compared with non-carriers.  The CYP2C19*1 allele corresponds to fully functional metabolism while the CYP2C19*2 and *3 alleles are non-functional.  CYP2C19*2 and *3 account for the majority of reduced function alleles in white (85 %) and Asian (99 %) poor metabolizers.  Other alleles associated with absent or reduced metabolism are less frequent, and include, but are not limited to, CYP2C19*4, *5, *6, *7, and *8.  A patient with poor metabolizer status will possess 2 loss-of-function (LOF) alleles as defined above.  Published frequencies for poor CYP2C19 metabolizer genotypes are approximately 2 % for whites, 4 % for blacks and 14 % for Chinese.  Tests are available to determine a patient's CYP2C19 genotype.

The results of this research are now included in the prescribing label for clopidogrel.  The new label includes that reduced CYP2C19 metabolism in intermediate and poor metabolizers is associated with diminished response to clopidogrel and that pharmacogenetic testing can identify genotypes associated with variability in CYP2C19 activity.  However, the information in the new labeling does not include a recommendation for genetic testing prior to administration of the drug and does not provide information on how to determine dosing once a patient’s individual genotype has been established. 

On March 12, 2010, the labeling of Plavix was updated to include a boxed warning about the diminished effectiveness of clopidogrelin poor metabolizers.  The boxed warning states that the effectiveness of Plavix is dependent on its activation to an active metabolite by the cytochrome P450 (CYP) system, principally CYP2C19.  The labeling states that Plavix at recommended doses forms less of that metabolite and has a smaller effect on platelet function in patients who are CYP2C19 poor metabolizers.  Citing cohort studies and retrospective analyses of clinical trials, the labeling states that poor metabolizers with acute coronary syndrome or undergoing percutaneous coronary intervention (PCI) treated with Plavix at recommended doses exhibit higher cardiovascular event rates than do patients with normal CYP2C19 function.  The labeling states that tests are available to identify a patient's CYP2C19 genotype, and that these tests can be used as an aid in determining therapeutic strategy.  The labeling states that clinicians should consider alternative treatment or treatment strategies in patients identified as CYP2C19 poor metabolizers.

The new label notes that reduced CYP2C19 metabolism in intermediate and poor metabolizers is associated with diminished response to clopidogrel and that pharmacogenetic testing can identify genotypes associated with variability in CYP2C19 activity.  However, the information in the new labeling does not include a recommendation for genetic testing prior to administration of the drug.  The label states, "although a higher dose regimen (600 mg loading dose followed by 150 mg once-daily) in poor metabolizers increases antiplatelet response, an appropriate dose regimen for this patient population has not been established in clinical outcome trials."  In addition, the label states that there may be genetic variants of other CYP450 enzymes with effects on the ability to form clopidogrel's active metabolite.

A Clinical Alert from the American College of Cardiology and the American Heart Association (Holmes et al, 2010) stated that the clopidogrel boxed warning leaves the issue of whether to perform CYP2C19 testing up to the individual physician.  It does not specifically require genetic testing or other changes in evaluation or treatment and does not imply that there are solid evidence-based reasons for such actions.  Rather, it serves to make clinicians aware of the imperfect, but significant, knowledge that we have about genetic variations in response to clopidogrel and to emphasize that clinicians should use this knowledge to make decisions about how to treat individual patients.  The Clinical Alert concluded that the evidence base is insufficient to recommend either routine genetic or platelet function testing at the present time.  There is no information that routine testing improves outcome in large subgroups of patients.  In addition, the clinical course of the majority of patients treated with clopidogrel without either genetic testing or functional testing is excellent.  The Clinical Alert stated that clinical judgment is required to assess clinical risk and variability in patients considered to be at increased risk.  The Clinical Alert stated that genetic testing to determine if a patient is predisposed to poor clopidogrel metabolism ("poor metabolizers") may be considered before starting clopidogrel therapy in patients believed to be at moderate or high risk for poor outcomes.  This might include, among others, patients undergoing elective high-risk PCI procedures (e.g., treatment of extensive and/or very complex disease).  The Clinical Alert stated that, if such testing identifies a potential poor metabolizer, other therapies, particularly prasugrel for coronary patients, should be considered.  With these other therapies, the balance of potential ischemic benefit with the known increased risk of bleeding should be considered either with alternative clopidogrel dosing or newer agents such as prasugrel.

Guidelines issued in December 2009 from the American College of Cardiology, the American Heart Association, and the Society for Cardiac Angiography and Interventions stated that, although clopidogrel in combination with ASA has been shown to reduce recurrent coronary events in the post-hospitalized acute coronary syndrome (ACS) population, the response to clopidogrel varies among patients, and clopidogrel resistance has been observed (Kushner et al, 2009).  The guidelines noted that information is accumulating about the variations in the anti-platelet effect of clopidogrel in patients with LOF alleles in the gene encoding CYP450 2C19.  These patients form a subgroup in which failure of clopidogrel effectiveness has been linked to adverse clinical outcomes.  The guidelines stated, accordingly, that the effective clopidogrel dose for an individual undergoing PCI for STEMI may not be known.  The guidelines stated that a large randomized trial is attempting to determine whether adjustment of clopidogrel therapy on the basis of platelet function testing with a point-of-care assay safely improves outcomes after PCI with drug-eluting stent (DES).  The guidelines stated that the current recommended loading dose for clopidogrel is uncertain.  In addition, a period of several hours is required to metabolize clopidogrel to its active metabolite, which leaves a window of time during which there is a reduced level of effectiveness even in responders.

The guidelines observed that thienopyridine prasugrel has a higher level of inhibition of platelet aggregation than clopidogrel and a more rapid onset of action (Kushner et al, 2009).  The guidelines stated that its metabolism is not affected by the 2C19 allele variant.  However, the guidelines do not endorse explicitly one of the thienopyridines over the other.  Furthermore, there are some emerging studies that suggest there may be some patients who are resistant to clopidogrel, but there is little information about the use of strategies to select patients who might do better with prasugrel.  There is not yet experience with the use of prasugrel in routine community practice.  As a result, the guidelines state that there is some uncertainty regarding the net benefit and risks of one drug over another for a given patient.

More recently published evidence casts doubt over the role of genetic variation in CYP2C19 and clopidogrel treatment effects.  A recent study published in the New England Journal of Medicine (Pare et al, 2010) found that, among patients with ACS or atrial fibrillation, the effect of clopidogrel as compared with placebo is consistent, irrespective of CYP2C19 genotype.  Investigators genotyped patients from 2 large, randomized, placebo-controlled trials of the effect of clopidogrel on the rate of cardiovascular events among patients with ACS and among patients with atrial fibrillation.  Investigators found that, in both studies, clopidogrel's superiority to placebo was similar, regardless of patients' genotype.  In addition, CYP2C19 genotype was not associated with adverse events during treatment.

A randomized controlled clinical study published in the Lancet (Wallentin et al, 2010), comparing clopidogrel to ticagrelor in subjects with ACS, found no significant effect difference of clopidogrel by CYP2C19 genotype.  In this study, ticagrelor was found to be more effective than clopidogrel in reducing the risk of cardiovascular events, but the effect difference did not vary significantly by CYP2C19 genotype.  In addition, CYP2C19 genotype was not associated with adverse events during treatment.

Tetrabenazine (Xenazine) is a monoamine depletor for oral administration, and is indicated for the treatment of chorea associated with Huntington’s disease.  The precise mechanism by which tetrabenazine exerts its anti-chorea effects is unknown, but is believed to be related to its effect as a reversible depletor of monoamines (such as dopamine, serotonin, norepinephrine, and histamine) from nerve terminals.  Tetrabenazine is extensively metabolized in the liver to metabolites alpha-HTBZ and beta-HTBZ.  Alpha-HTBZ and beta-HTBZ are metabolized by CYP450 enzymes, principally CYP2D6, to another major circulating metabolite, O-dealkylated-HTBZ.  These metabolites are primarily renally excreted. The US Food and Drug Administration (FDA) approved Xenazine (tetrabenazine) for the treatment of chorea in people with Huntington’s disease. The FDA recommends genotyping for CYP2D6 prior to treatment for daily doses over 50mg. Dosage recommendations are indicated based on test results. The labeling of Xenazine states that, although the pharmacokinetics of tetrabenazine and its metabolites in subjects who do not express the drug metabolizing enzyme CYP2D6 (poor metabolizers) have not been systematically evaluated, it is likely that the exposure to beta-HTBZ would be increased compared to subjects who express the enzyme (extensive metabolizers), with an increase similar to that observed in patients taking strong CYP2D6 inhibitors.  The labeling of Xenazine states that "[p]atients should be genotyped for CYP2D6 prior to treatment with daily doses of tetrabenazine over 50 mg", and patients who are poor metabolizers should not be given daily doses greater than 50 mg.

Knowing how an individual will respond to warfarin would help in tailoring the dose needed to maintain appropriate anticoagulation.  Toward that end, researchers studied variability in the recently discovered gene for the warfarin target, vitamin K epoxide reductase complex 1 (VKORC1).  VKORC1 is the key enzyme of the vitamin K cycle and the molecular target of coumarins, which represent the most commonly prescribed drugs for therapy and prevention of thromboembolic conditions.  Recent studies have identified variants of the VKORC1 gene as responsible for about 25 % of the inter-individual response to warfarin, and for significant inter-ethnic response variability.  Whether or not this is sufficient to successfully direct initial dosing, achieve a shorter time to stable dose, and reduce bleeding events has yet to be shown in a prospective trial.  There are several such prospective clinical studies that are currently ongoing both in the United States and Europe.

Wadelius and Pirmohamed (2007) explained that the most important genes affecting the pharmacokinetic and pharmacodynamic parameters of warfarin are CYP2C9 and VKORC1.  These 2 genes, together with environmental factors, partly explain the inter-individual variation in warfarin dose requirements.

Although studies have shown that genetic polymorphisms in CYP2C9 and VKORC1 affect warfarin dosing, no randomized controlled trials have linked the use of pharmacogenomic testing to improvements in clinical outcomes.  An assessment by the Canadian Agency for Drugs and Technologies in Health (Ndegwa, 2007) found that most of the studies performed to date have been of retrospective or cross-sectional design.  Consequently, individuals who stop warfarin early because of adverse effects or those who have difficulty attaining a therapeutic maintenance dose may have been excluded.  Furthermore, many studies were underpowered to investigate the risk of bleeding.

Rieder et al (2005) analyzed genetic data from 186 American patients of European descent who were recruited from anticoagulation clinics and were receiving long-term warfarin therapy.  They identified 10 single-nucleotide polymorphisms of VKORC1 that showed significant associations with warfarin maintenance doses and that had an overall frequency of more than 5 % in this cohort.  From these 10 single-nucleotide polymorphisms, the researchers inferred five common VKORC1 haplotypes (a haplotype is a set of closely linked genetic markers present on one chromosome that tend to be inherited together).  Four of the haplotypes had independent associations with warfarin maintenance doses, 2 with a low-dose requirement and 2 with a high-dose requirement.  Ultimately, subjects were linked with 1 of 3 haplotype groupings; the mean daily warfarin maintenance dose differed significantly across these 3 subgroups (2.7 mg, 4.9 mg, and 6.2 mg, respectively).  The authors report that VKORC1 haplotype explained 25 % of the variance in warfarin dose, and they replicated these findings in a larger European American population.  The researchers also examined VKORC1 haplotype frequencies in Asian American and African American populations and found significant variability by race.

The FDA has cleared for marketing the Verigene warfarin metabolism test, manufactured by Nanosphere, which detects variants of two genes, CYP2C9 and VKORC1 that can contribute to changes in warfarin metabolism (FDA, 2007).

Large ongoing studies of genes involved in the actions of warfarin, together with prospective assessment of environmental factors, will increase the capacity to accurately predict warfarin dose.  Kamali (2006) stated that prospective studies that incorporate both CYP2C9 and VKORC1 genes and environmental factors in warfarin dose calculation will be needed to demonstrate the safety, cost-effectiveness, and feasibility of individualized dosing regimens.

ECRI Institute's Health Technology Trends (2007) reported that "[a]lthough genetic testing can currently identify who has these variants [CYP2C9 and VKORC1], more studies are needed to explore the precise starting doses for these patients".

Anderson et al (2007) stated that pharmacogenetic-guided dosing of warfarin is a promising application of "personalized medicine" but has not been adequately examined in randomized studies.  In a randomized trial, these investigators assessed genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation (n = 206).  Buccal swab DNA was genotyped for CYP2C9 *2 and CYP2C9 *3 and VKORC1C1173T with a rapid assay.  Standard dosing followed an empirical protocol, whereas pharmacogenetic-guided dosing followed a regression equation including the 3 genetic variants as well as age, sex, and weight.  Prothrombin time INR was measured routinely on days 0, 3, 5, 8, 21, 60, and 90.  A research pharmacist un-blinded to treatment strategy managed dose adjustments.  Patients were followed-up for up to 3 months.  Pharmacogenetic-guided predicted doses more accurately approximated stable doses (p < 0.001), resulting in smaller (p = 0.002) and fewer (p = 0.03) dosing changes and INRs (p = 0.06).  However, percent out-of-range INRs (pharmacogenetic = 30.7 %, standard = 33.1 %), the primary end point, did not differ significantly between arms.  Despite this, when restricted to wild-type patients (who required larger doses; p = 0.001) and multiple variant carriers (who required smaller doses; p < 0.001) in exploratory analyses, results (pharmacogenetic = 29 %, standard = 39 %) achieved nominal significance (p = 0.03).  Multiple variant allele carriers were at increased risk of an INR of greater than or equal to 4 (p = 0.03).  The authors concluded that an algorithm guided by pharmacogenetic and clinical factors improved the accuracy and efficiency of warfarin dose initiation.  Despite this, the primary end point of a reduction in out-of-range INRs was not achieved. In subset analyses, pharmacogenetic guidance showed promise for wild-type and multiple variant genotypes.

A review by Hynicka et al (2007) concluded that the use of pharmacogenomic testing in the initiation of warfarin therapy does not show improved outcomes in either safety or efficacy with warfarin therapy.  Studies of pharmacogenomic testing to improve outcomes with initiation of warfarin therapy were eligible for inclusion.  All patients were adults; however, the included studies varied in their population and treatment regimens.  The genotypes included CYP2C9 and VKORC1 wild-types and variants.  Four studies met the inclusion criteria: 2 randomized controlled trials (n = 238) and 2 prospective cohort studies (n = 345).  None of the studies reported a significant difference in efficacy between dosing strategies, although the prospective cohort studies reported a tendency towards fewer adverse events in patients with pharmacogenomic-guided dosing.

An assessment published by the Canadian Agency for Drugs and Technology in Health (Ndegwa, 2007) on "Pharmacogenomics and Warfarin Therapy" concluded that "prospective studies are needed to determine whether pharmacogenomic testing improves patient outcomes, identify which subgroups of patients may benefit, and clarify the risks and costs associated with the use of these tests.  Several randomized controlled trials are currently evaluating the impact of pharmacogenomics on dosing accuracy, time to achieve and maintain target international normalized ratio (INR), incidence of bleeding or thromboembolic events, and monitoring requirements".

In August 2007, the FDA updated the product label for warfarin to include information on the impact of genetic variations in CYP2C9 and VKORC1 on warfarin metabolism; however the FDA does not require the use of these genetic tests in dosing individual patients initiating warfarin therapy.  On January 28, 2008, the FDA cleared the Infiniti 2C9-VKORC1 Multiplex Assay (AutoGenomic, Inc., Carlsbad, CA) for detection of Warfarin sensitivity.  Guidelines for pharmacogenomics-based warfarin dosing are under development.

A technology assessment by the California Technology Assessment Forum (CTAF, 2008) reviewed the scientific evidence for the use of genetic testing to guide the initial dosing of warfarin when initiating therapy for anticoagulation.  The assessment stated that genotyping studies of patients at a stable, therapeutic dose of warfarin demonstrated that patients who have CYP2C9*2 and CYP2C9*3  require lower doses of warfarin on average than patients with the more common CYP2C9*1 allele; in addition, patients with the A haplotype of VKORC1 require lower doses of warfarin on average than patients with the B haplotype.  The assessment reported that these genetic variations have been shown to predict an increased risk of excessive anticoagulation and major bleeding among patients prescribed warfarin and that statistical models have been developed in an attempt to predict the dose of warfarin needed to achieve stable anticoagulation.  The assessment examined the results of a small case series and 3 small, randomized clinical trials and reported that the model used by the case series did reasonably well at predicting the warfarin dose, but that patients with the CYP2C9*2 and CYP2C9*3 alleles were still at more than a 4-fold risk of excessive anticoagulation; only 1 of the randomized trials used a model incorporating genotyping information from both CYP2C9 and VKORC1 and that study found almost no difference in outcomes between patients receiving warfarin using a pharmacogenetic model and those treated according to a standard approach.  The assessment stated, "Significant uncertainty remains in the field.  There is no widely accepted, standard pharmacogenetic model to determine the starting dose of warfarin and new models are being developed in 2008.  Current models only explain 50 % to 60 % of the variability in warfarin dosing and the remaining variability is unexplained.  Only 1 of the randomized trials used a model incorporating information from both genes with variants known to influence warfarin dosing.  Furthermore, all of the trials to date have been under-powered to meaningfully evaluate the effect of genotyping on major bleeding, the most important clinical outcome."  The assessment concluded that the use of genetic testing to guide initial warfarin dosing does not meet Technology Assessment Criteria 3 through 5 for safety, effectiveness and improvement in health outcomes.  Several large clinical trials are ongoing in both the United Stated and Europe to clarify the role of genetic testing in warfarin management.

The Centers for Medicare & Medicaid Services (CMS, 2013) concluded that the available evidence does not demonstrate that pharmacogenomic testing of CYP2C9 or VKORC1alleles to predict warfarin responsiveness improves health outcomes in Medicare beneficiaries. Therefore, CMS determined that pharmacogenomic testing of CYP2C9 or VKORC1 alleles to predict warfarin responsiveness is not reasonable and necessary.

A randomized controlled clinical trial found that genotype-guided dosing of warfarin did not improve anticoagulation control during the first four weeks of therapy. Kimmel, et al. (2013) randomly assigned 1015 patients to receive doses of warfarin during the first 5 days of therapy that were determined according to a dosing algorithm that included both clinical variables and genotype data or to one that included clinical variables only. All patients and clinicians were unaware of the dose of warfarin during the first 4 weeks of therapy. The primary outcome was the percentage of time that the international normalized ratio (INR) was in the therapeutic range from day 4 or 5 through day 28 of therapy. At 4 weeks, the mean percentage of time in the therapeutic range was 45.2% in the genotype-guided group and 45.4% in the clinically guided group (adjusted mean difference, [genotype-guided group minus clinically guided group], -0.2; 95% confidence interval, -3.4 to 3.1; P=0.91). There also was no significant between-group difference among patients with a predicted dose difference between the two algorithms of 1 mg per day or more. There was, however, a significant interaction between dosing strategy and race (P=0.003). Among black patients, the mean percentage of time in the therapeutic range was less in the genotype-guided group than in the clinically guided group. The rates of the combined outcome of any INR of 4 or more, major bleeding, or thromboembolism did not differ significantly according to dosing strategy.

EGFR

An assessment by the BlueCross BlueShield Association Technology Evaluation Center (2010) concluded that tumor-cell epidermal growth factor receptor (EGFR) mutation analysis to predict response to erlotinib in patients with advanced non-small cell lung cancer (NSCLC) meets the Blue Cross and Blue Shield Association Technology Evaluation Center (TEC) criteria.  Furthermore, guidelines from the National Comprehensive Cancer Network (NCCN, 2024) recommend EGFR testing for the following histologic subtypes of NSCLC:

adenocarcinoma,
large cell, 
NSCLC not otherwise specified, and
squamous cell carcinoma.

The Alberta Provincial Thoracic Tumor Team’s clinical practice guideline on "Non-small cell lung cancer stage IV" (2011) stated that "First-line monotherapy with the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor gefitinib is recommended for patients with EGFR mutation-positive NSCLC.  Testing for EGFR mutations should take place for all eligible patients with advanced NSCLC and adenocarcinoma histology who are being considered for first-line therapy with gefitinib, irrespective of their gender, ethnicity, and smoking status".

Gao et al (2012) stated that gefiinib and erlotinib are 2 similar small molecules of selective and reversible epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs), which have been approved for second-line or third-line indication in previously treated advanced NSCLC patients.  The results of comparing the EGFR-TKI with standard platinum-based doublet chemotherapy as the first-line treatment in advanced NSCLC patients with activated EGFR mutation were still controversial.  A meta-analysis was performed to derive a more precise estimation of these regimens.  Finally, 6 eligible trials involved 1,021 patients were identified.  The patients receiving EGFR-TKI as front-line therapy had a significantly longer PFS than patients treated with chemotherapy [median PFS was 9.5 versus 5.9 months; HR = 0.37; 95 % CI: 0.27 to 0.52; p < 0.001].  The overall response rate (ORR) of EGFR-TKI was 66.60 %, whereas the ORR of chemotherapy regimen was 30.62 %, which was also a statistically significant favor for EGFR-TKI [relative risk (RR) = 5.68; 95 % CI: 3.17 to 10.18; p < 0.001].  The OS was numerically longer in the patients received EGFR-TKI than patients treated by chemotherapy, although the difference did not reach a statistical significance (median OS was 30.5 versus 23.6 months; HR = 0.94; 95 % CI: 0.77 to 1.15; p = 0.57).  Comparing with first-line chemotherapy, treatment of EGFR-TKI achieved a statistical significantly longer PFS, higher ORR and numerically longer OS in the advanced NSCLC patients harboring activated EGFR mutations, thus, it should be the first choice in the previously untreated NSCLC patients with activated EGFR mutation.

NCCN non-small cell lung cancer (NSCLC) guidelines (2024) state that EGFR should be conducted as part of a multiplex/next generation sequencing. The presence of EGFR exon 19 deletions or EGFR exon 21 L858R mutations is predictive of treatment benefit from EGFR tyrosine kinase inhibitor therapy. The NCCN NSCLC Guidelines Panel "strongly advises broader molecular profiling with the goal of identifying rare driver mutations for which effective drugs may already be available, or to appropriately counsel patients regarding the availability of clinical trials". 

Guidelines from the American Society for Clinical Oncology (2016) state: "The clinician should not use HER1/epidermal growth factor receptor expression by IHC to guide adjuvant chemotherapy selection" in breast cancer.

KRAS

The ras proto-oncogenes are normal cellular components, which are thought to be important for transduction of signals required for proliferation and differentiation.   The ras oncogene family has 3 members: H-ras, K-ras, and N-ras.  Ras gene mutations can be found in a variety of tumor types, although the incidence varies greatly. The highest incidences are found in adenocarcinomas of the pancreas (90 %), colon (50 %), and lung (30 %); thyroid tumors (50 %), and myeloid leukemia (30 %).

Investigators have established an association between some genotypes of K-ras (KRAS) oncogenes and response to treatment with cetuximab or panitumumab (Lievre et al, 2006 and 2008; Di Fiore et al, 2007; Gonçalves et al, 2008; De Roock et al, 2008).  Patients whose tumors express specific forms of the KRAS gene exhibit considerably decreased responses to cetuximab and panitumumab.  It has been theorized that cetuximab and panitumumab do not target epidermal growth factor receptor (EGFR) associated with these specific KRAS mutations and thus are unable to block their activation.  It has been suggested that KRAS genotype be considered as a selection factor for cancer patients who are candidates for treatment with cetuximab or panitumumab.

Karapetis and colleagues (2008) stated that treatment with cetuximab improves overall survival (OS) and progression-free survival (PFS) and preserves the quality of life in patients with colorectal cancer that has not responded to chemotherapy. The mutation status of the K-ras gene in the tumor may affect the response to cetuximab and have treatment-independent prognostic value.  These investigators analyzed tumor samples, obtained from 394 of 572 patients (68.9 %) with colorectal cancer who were randomly assigned to receive cetuximab plus best supportive care or best supportive care alone, to look for activating mutations in exon 2 of the K-ras gene.  They evaluated if the mutation status of the K-ras gene was associated with survival in the cetuximab and supportive-care groups.  Of the tumors evaluated for K-ras mutations, 42.3 % had at least one mutation in exon 2 of the gene.  The effectiveness of cetuximab was significantly associated with K-ras mutation status (p = 0.01 and p < 0.001 for the interaction of K-ras mutation status with OS and PFS, respectively).  In patients with wild-type K-ras tumors, treatment with cetuximab as compared with supportive care alone significantly improved OS (median of 9.5 versus 4.8 months; hazard ratio for death, 0.55; 95 % confidence interval [CI], 0.41 to 0.74; p < 0.001) and PFS (median of 3.7 months versus 1.9 months; hazard ratio for progression or death, 0.40; 95 % CI, 0.30 to 0.54; p < 0.001).  Among patients with mutated K-ras tumors, there was no significant difference between those who were treated with cetuximab and those who received supportive care alone with respect to OS (hazard ratio, 0.98; p = 0.89) or PFS (hazard ratio, 0.99; p = 0.96).  In the group of patients receiving best supportive care alone, the mutation status of the K-ras gene was not significantly associated with OS (hazard ratio for death, 1.01; p = 0.97).  The authors concluded that patients with a colorectal tumor bearing mutated K-ras did not benefit from cetuximab, whereas patients with a tumor bearing wild-type K-ras did benefit from cetuximab.  The mutation status of the K-ras gene had no influence on survival among patients treated with best supportive care alone.

The ASCO's provisional clinical opinion on testing for KRAS gene mutations in patients with metastatic colorectal carcinoma to predict response to anti-EGFR monoclonal antibody therapy (Allegra et al, 2009) stated that based on systematic reviews of the relevant literature, all patients with metastatic colorectal carcinoma who are candidates for anti-EGFR antibody therapy should have their tumor tested for KRAS mutations in a CLIA-accredited laboratory.  If KRAS mutation in codon 12 or 13 is detected, then patients with metastatic colorectal carcinoma should not receive anti-EGFR antibody therapy as part of their treatment.

The KRAS oncogene mutation tests are intended to aid in the formulation of treatment decisions for patients who may be candidates for treatment of metastatic epithelial cancers with anti-EGFR therapies such as cetuximab or panitumumab.  Several tests for KRAS mutation are currently available in the United States; however, at this time, no KRAS genotype test kits have been approved by the FDA.

At the 2008 Annual Meeting of the American Society of Clinical Oncology (ASCO), data on 540 patients with metastatic colorectal cancer in the randomized, phase III CRYSTAL trial were presented.  Among 192 patients with KRAS mutations, there was no improvement in overall responses or PFS from the addition of cetuximab to standard chemotherapy.  In the patients with normal KRAS, the 1-year PFS rate was 43 % for patients receiving cetuximab versus 25 % for those receiving only standard chemotherapy, and the overall response rate was 59 % versus 43 %, respectively (van Cutsem, 2008).  Also at the 2008 ASCO meeting, data from 233 metastatic colorectal cancer patients were presented that confirmed the correlation of KRAS status with patient response to anti-EGFR therapy.  No benefit was found after addition of cetuximab to standard chemotherapy with FOLFOX (the combination of fluorouracil, leucovorin, and oxaliplatin) in patients with a mutated KRAS; however, addition of cetuximab to FOLFOX increased both response rate and PFS in patients with a wild-type (i.e., un-mutated) KRAS gene (Bokemeyer, 2008).  Response to panitumumab was correlated to KRAS status in a published phase III trial.  A total of 427 patients with metastatic colorectal cancer received either panitumumab or best supportive care.  Panitumumab exhibited a 17% response rate among patients with normal KRAS, but 0% response among patients with KRAS mutations (Amado, 2008).

A meta-analysis of results from 8 studies involving 817 patients with colorectal cancer found that the presence of KRAS mutation predicted lack of response to treatment with anti-EGFR monoclonal antibodies (e.g., panitumumab or cetuximab), whether as stand-alone therapy or in combination with chemotherapy (Linardou et al, 2008).  This analysis also provided empirical evidence that k-RAS mutations are highly specific negative predictors of response (de-novo resistance) to single-agent EGFR tyrosine-kinase inhibitors in advanced non-small cell lung cancer; and similarly to anti-EGFR monoclonal antibodies alone or in combination with chemotherapy in patients with metastatic colorectal cancer.

The Blue Cross and Blue Shield Association (BCBSA, 2008) Technology Evaluation Center Medical Advisory Panel concluded that use of KRAS mutation analysis meets TEC criteria to predict non-response to anti-EGFR monoclonal antibodies cetuximab and panitumumab to treat metastatic colorectal cancer. The TEC assessment found that the evidence is sufficient to conclude that patients with mutated KRAS tumors in the setting of metastatic colorectal cancer do not respond to anti-EGFR monoclonal antibody therapy. The assessment explained that the data show that the clinical benefit of using EGFR inhibitors in treating metastatic colorectal cancer, either as monotherapy or in combination with other treatment regimens, is not seen in patients with KRAS-mutated tumors.  The assessment found: "This data supports knowing a patient's tumor mutation status before consideration of use of an EGFR inhibitor in the treatment regimen.  Identifying patients whose tumors express mutated KRAS will avoid exposing patients to ineffective drugs, avoid exposure to unnecessary drug toxicities, and expedite the use of the best available alternative therapy."

Colorectal cancer guidelines from the National Comprehensive Cancer Network (NCCN, 2010) recommend consideration of reflex BRAF testing in patients with wild type KRAS. The NCCN guidelines explain that several small studies suggest that patients with wild-type KRAS and a BRAF mutation are unlikely to respond to anti-EGFR therapies such as cetuximab and panitumumab. The guidelines explain that patients with a known BRAF mutation are unlikely to respond to anti-EGFR antibodies, although the data are somewhat inconsistent. Studies demonstrate that in patients with metastatic colorectal cancer, about 8 percent have mutations in the BRAF gene. Testing for the BRAF V600E mutation is performed by PCR amplification and direct DNA sequence analysis.

Ratner et al (2010) stated that ovarian cancer (OC) is the single most deadly form of women's cancer, typically presenting as an advanced disease at diagnosis in part due to a lack of known risk factors or genetic markers of risk.  The KRAS oncogene and altered levels of the microRNA (miRNA) let-7 are associated with an increased risk of developing solid tumors.  In this study, these researchers investigated a hypothesized association between an increased risk of OC and a variant allele of KRAS at rs61764370, referred to as the KRAS-variant, which disrupts a let-7 miRNA binding site in this oncogene.  Specimens obtained were tested for the presence of the KRAS-variant from non-selected OC patients in 3 independent cohorts, 2 independent ovarian case-control studies, and OC patients with hereditary breast and ovarian cancer syndrome (HBOC) as well as their family members.  The results indicated that the KRAS-variant is associated with more than 25 % of non-selected OC cases.  Furthermore, these researchers found that it is a marker for a significant increased risk of developing OC, as confirmed by 2 independent case-control analyses.  Lastly, they determined that the KRAS-variant was present in 61 % of HBOC patients without BRCA1 or BRCA2 mutations, previously considered uninformative, as well as in their family members with cancer.  These findings supported the hypothesis that the KRAS-variant is a genetic marker for increased risk of developing OC, and they suggested that the KRAS-variant may be a new genetic marker of cancer risk for HBOC families without other known genetic abnormalities.

Hollestelle et al (2011) noted that recently, a variant allele in the 3'UTR of the KRAS gene (rs61764370 T>G) was shown to be associated with an increased risk for developing non-small cell lung cancer, as well as OC, and was most enriched in OC patients from HBOC families.  This functional variant has been shown to disrupt a let-7 miRNA binding site leading to increased expression of KRAS in vitro.  In the current study, these investigators genotyped this KRAS-variant in breast cancer index cases from 268 BRCA1 families, 89 BRCA2 families, 685 non-BRCA1/BRCA2 families, and 797 geographically matched controls.  The allele frequency of the KRAS-variant was found to be increased among patients with breast cancer from BRCA1, but not BRCA2 or non-BRCA1/BRCA2 families as compared to controls.  As BRCA1 carriers mostly develop ER-negative breast cancers, these researchers also examined the variant allele frequency among indexes from non-BRCA1/BRCA2 families with ER-negative breast cancer.  The prevalence of the KRAS-variant was, however, not significantly increased as compared to controls, suggesting that the variant allele not just simply associates with ER-negative breast cancer.  Subsequent expansion of the number of BRCA1 carriers with breast cancer by including other family members in addition to the index cases resulted in loss of significance for the association between the variant allele and mutant BRCA1 breast cancer.  In this same cohort, the KRAS-variant did not appear to modify breast cancer risk for BRCA1 carriers.  More importantly, results from the current study suggested that KRAS-variant frequencies might be increased among BRCA1 carriers, but solid proof requires confirmation in a larger cohort of BRCA1 carriers. 

Therascreen KRAS RGQ PCR Kit (Qiagen) is intended to detect 7 mutations in codons 12 and 13 of the KRAS gene (Raman, et al., 2013). The kit utilizes two technologies — ARMS and Scorpions — for detection of mutations in real-time PCR. The therascreen KRAS RGQ PCR kit is being developed as a companion diagnostic to aid clinicians, through detection of KRAS mutations, in the identification of patients with metastatic colorectal cancer (mCRC) who are more likely to benefit from cetuximab.

PreOvar™ tests (Mira Dx) for the KRAS-variant, and will help identify ovarian cancer patients whose female relatives should also be evaluated for the KRAS-variant (Raman, et al., 2013). PreOvar™ may also help assess the relative risk of developing ovarian cancer for women who have a family history of ovarian cancer without a living proband (ancestor with the disease). The KRAS-Variant is present in 6-10% of the general population and 25% of non-selected women with epithelial ovarian cancer. Additionally, the KRAS-variant was identified in over 60% of Hereditary Breast and Ovarian Cancer (HBOC) patients that were previously classified as "uninformative," or negative for other known genetic markers of ovarian cancer risk. The test determines if KRAS-variant may put someone at increased risk for developing ovarian cancer.

The Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group (EWG) (2013) found that, for patients with metastatic colorectal cancer (mCRC) who are being considered for treatment with cetuximab or panitumumab, there is convincing evidence to recommend clinical use of KRAS mutation analysis to determine which patients are KRAS mutation positive and therefore unlikely to benefit from these agents before initiation of therapy. The level of certainty of the evidence was deemed high, and the magnitude of net health benefit from avoiding potentially ineffective and harmful treatment, along with promoting more immediate access to what could be the next most effective treatment, is at least moderate.

The EWG found insufficient evidence to recommend for or against BRAF V600E testing for the same clinical scenario (EGAPP, 2013). The level of certainty for BRAF V600E testing to guide anti-epidermal growth factor receptor (EGFR) therapy was deemed low. The EWG encourages further studies of the potential value of testing in patients with mCRC who were found to have tumors that are wild type (mutation negative) for KRAS to predict responsiveness to therapy.

UGT1A1 Molecular Assay

Colorectal cancer (CRC) is one of the most common malignancies in western countries showing an increasing incidence, and has been associated with genetic as well as lifestyle factors.  About 150,000 new cases of CRC are diagnosed each year in the United States, 40 to 50 % of which are metastatic.  Irinotecan (Camptosar) is a chemotherapeutic agent approved as a combination therapy with 5-FU/leucovorin for the treatment of advanced CRC.  The response to irinotecan is variable, possibly because of individuals' variation in the expression of the enzymes that metabolize irinotecan.  Although multiple genes may play a role in irinotecan activity, the uridine diphosphate glucuronosyltransferase 1 family, polypeptide A1 (UGT1A1) enzyme has been strongly associated with irinotecan-related toxicity.  The UGT1A1 gene is responsible for glucuronidation of the active metabolite of irinotecan.  A common di-nucleotide repeat polymorphism in the UGT1A1 promoter region (UGT1A1*28) has been correlated with toxicity in cancer patients receiving irinotecan-containing therapy. 

Lentz et al (2005) stated that hepatic metastases occur in about 50 % of patients with CRC.  Since hepatic metastases are often inaccessible for surgery, chemotherapy of metastases is important.  The most commonly used chemotherapeutic agents for hepatic metastases are fluorouracil, irinotecan, and oxaliplatin.  Several enzymes are known to be involved in the metabolism of these drugs, and the activity of these enzymes varies greatly between individuals.  The causes of this variation include genetic polymorphisms, different regulation between normal and cancer tissue, and the influence of chemotherapy on enzyme expression.  The varying enzyme activity may have an important effect on the outcome of chemotherapy.  Lentz et al (2005) reported that several studies confirm the influence of the activity of thymidylate synthase, thymidine phosphorylase and dihydropyrimidine dehydrogenase on the outcome of fluorouracil therapy for CRC, with higher enzyme activities predicting lower treatment efficacy.  Although fewer studies are available regarding therapy of hepatic metastases, the same relationship between thymidylate synthase activity and outcome of fluorouracil therapy observed for primary CRC was found.  For the other 2 enzymes, only a few studies are available, but the results indicate similarly that higher enzyme activity seems to be disadvantageous.  Enzymes that are responsible for the activation, metabolism, and mechanism of action of irinotecan (e.g., CYP 3A4 and UGT1A1) also exhibit variable inter-individual activity.  The authors concluded therefore, that there may be an association between enzyme activity and response to therapy.  For example, in patients with CRC, higher enzyme activity of topoisomerase-I seems to be predictive of a better response to irinotecan.  CYP3A4 and UGT1A1 activity levels might be predictive of irinotecan toxicity rather than efficacy.  These authors stated that available data indicate the importance of the different enzyme activities on the outcome of chemotherapy of hepatic metastases in CRC.  The authors noted that more information is needed, especially for the newer drugs irinotecan and oxaliplatin.  However, the existing data are very promising in respect to the potential to guide dose and drug selection for more efficient and less toxic chemotherapy of hepatic metastases from CRC.

The Invader UGT1A1 molecular assay (Genzyme Corporation, Cambridge, MA) was cleared by the FDA on August 22, 2005.  The test can be performed before starting irinotecan therapy and is designed to identify patients who may be at risk for ADRs to the chemotherapeutic agent by detecting a genetic variation in the UGT1A1 gene. 

A clinical study (Innocenti et al, 2004) indicated that patients with one of these variations in the UGT1A1 gene have a significantly greater risk of experiencing drug-related toxicity from irinotecan than those without it.  The product labeling for irinotecan (Camptosar) was updated to recommend that a reduced initial dose should be considered for patients homozygous for UGT1A1*28 allele, although the precise dose reduction in this group of patients is unknown (Waknine, 2005).  The product labeling, however, does not include a recommendation for assessment of UGT1A1 status prior to initiation of irinotecan therapy.

In a prospective study, Innocenti and colleagues (2004) assessed the association between the prevalence of severe toxicity and UGT1A1 genetic variation.  A total of 66 cancer patients with advanced disease refractory to other treatments received irinotecan 350 mg/m2 every 3 weeks.  Toxicity and pharmacokinetic data were measured during cycle 1.  UGT1A1 variants (-3279G>T, -3156G>A, promoter TA indel, 211G>A, 686C>A) were genotyped.  The prevalence of grade 4 neutropenia was 9.5 %.  Grade 4 neutropenia was much more common in patients with the TA indel 7/7 genotype (UGT1A1(*)28 homozygous) (3 of 6 patients; 50 %) compared with 6/7 (3 of 24 patients; 12.5 %) and 6/6 (0 of 29 patients; 0 %) (p = 0.001).  The TA indel genotype was significantly associated with the absolute neutrophil count nadir (7/7 less than 6/7 less than 6/6, p = 0.02).  The relative risk of grade 4 neutropenia was 9.3 (95 %) for the 7/7 patients versus the rest of the patients.  Pre-treatment total bilirubin levels were significantly higher in patients with grade 4 neutropenia (0.83 +/- 0.08 mg/dL) compared to those without grade 4 neutropenia (0.47 +/- 0.03 mg/dL; p < 0.001).  The -3156G>A variant seemed to distinguish different phenotypes of total bilirubin within the TA indel genotypes.  The -3156 genotype and the SN-38 area under the concentration versus time curve were significant predictors of absolute neutrophil count nadir (r2 = 0.51).  These investigators concluded that UGT1A1 genotype and total bilirubin levels are strongly associated with severe neutropenia, and could be used to identify cancer patients predisposed to the severe toxicity of irinotecan.  Furthermore, the hypothesis that the -3156G>A variant is a better predictor of UGT1A1 status than the previously reported TA indel requires further testing.

In an accompanying editorial, McLeod and Watters (2004) raised questions regarding the findings of Innocenti et al (2004): is the relative risk of grade 4 neutropenia in a patient with the UGT1A1(*)28 homozygous genotype the same after 300 to 350 mg/m2 every 3 weeks (9.3-fold risk) as after the 100 to 125 mg/m2 weekly regimen? and does this marker retain its predictive power when irinotecan is used as part of a combined treatment?  
is the relative risk of grade 4 neutropenia in a patient with the UGT1A1(*)28 homozygous genotype the same after 300 to 350 mg/m2 every 3 weeks (9.3-fold risk) as after the 100 to 125 mg/m2 weekly regimen? and
does this marker retain its predictive power when irinotecan is used as part of a combined treatment? 

These researchers also noted that it would be difficult to perform randomized controlled trials to ascertain if there is a predictable, safe, and effective dose of irinotecan that can be given to patients with the UGT1A1(*)28 homozygous genotype, or if physicians should choose a non-irinotecan-containing regimen, since only 10 % of all patients have this genotype.  Moreover, studies are planned to determine the impact of dosage on the safety of irinotecan in patients with either the UGT1A1 6/6/ or 6/7 genotype, which are the 2 most common genotypes in the general patient population.

While several investigators (And et al, 2000; Iyer et al, 2002, and Innocenti et al, 2004) reported that testing of patients carrying the UGT1A1*28 polymorphism may detect their susceptibility to irinotecan-related toxicity, others (Marcuello et al, 2004, Carlini et al, 2005, Dervieux et al, 2005, and Eichelbaum et al, 2005) have questioned its clinical value.  In a clinical trial, Marcuello et al (2004) examined the influence of the UGT1A1 gene promoter polymorphism in the toxicity profile, in the response rate and in the overall survival (OS) in 95 patients with metastatic CRC treated with an irinotecan-containing chemotherapy.  Genotypes were determined by analyzing the sequence of TATA box of UGT1A1 of genomic DNA from the patients.  Clinical parameters and genotypes were compared by uni-variate and multi-variate statistical methods.  The more frequent ADRs were asthenia (n = 34), diarrhea (n = 29) and neutropenia (n = 20).  Severe diarrhea was observed in 7/10 (70 %) homozygous and 15/45 (33 %) heterozygous in comparison to 7/40 (17 %) wild-type patients (p = 0.005).  These results maintained the statistical significance in logistic regression analysis (p = 0.01) after adjustment for other clinical relevant variables.  The presence of severe hematological toxicity increased from wild-type patients to UGT1A1(*)28 homozygotes, but without achieving statistical significance.  No relationship was found between the UGT1A1(*)28 genotypes and infection, nausea or mucositis.  In uni-variate studies, patients with the UGT1A1(*)28 polymorphism showed a trend to a poorer OS (p = 0.09).  In the multi-variate analysis, the genotype was not related to clinical response or to OS.  These investigators stated that the role of the UGT1A1 genotype as a predictor of toxicity in patients with CRC receiving irinotecan demands the performance of randomized controlled studies to determine if genotype-adjusted dosages of the drug can help to establish safe and effective doses not only for patients with the UGT1A1(*)28 homozygous genotype, but also for those with the most common UGT1A1 6/6 or 6/7 genotype.

In a phase II clinical study, Carlini and co-workers (2005) examined whether germ-line polymorphisms within genes related to drug target (thymidylate synthase) or metabolizing enzymes (UGT isozymes) would alter response and toxicity to the combination of capecitabine plus irinotecan.  A total of 67 patients with measurable CRC were treated with intravenous irinotecan (100 or 125 mg/m2) on days 1 and 8 and capecitabine orally (900 or 1,000 mg/m2, twice daily) on days 2 through 15 of each 3-week cycle.  Genomic DNA was obtained from peripheral blood and genotyped using Pyrosequencing, GeneScan, and direct sequencing technologies.  The overall objective response rate was 45 % with 21 patients (31 %) exhibiting grade 3 or 4 diarrhea and 3 patients (4.5 %) demonstrating grade 3 or 4 neutropenia in the first 2 cycles.  Low enzyme activity UGT1A7 genotypes, UGT1A7*2/*2 (6 patients) and UGT1A7*3/*3 (7 patients), were significantly associated with anti-tumor response (p = 0.013) and lack of severe gastrointestinal toxicity (p = 0.003).  In addition, the UGT1A9 -118 (dT)(9/9) genotype was significantly associated with reduced toxicity (p = 0.002) and increased response (p = 0.047).  There were no statistically significant associations between UGT1A1, UGT1A6, or thymidylate synthase genotypes and toxicity or tumor response.  The authors concluded that these data strongly suggest that UGT1A7 and/or UGT1A9 genotypes may be predictors of response and toxicity in CRC patients treated with capecitabine plus irinotecan.  Specifically, patients with genotypes conferring low UGT1A7 activity and/or the UGT1A9 (dT)(9/9) genotype may be particularly likely to exhibit greater anti-tumor response with little toxicity.  However, it is interesting to note that the allele frequencies of UGT1A7 gene in Taiwan Chinese are different from those in Caucasians and Japanese (Huang et al, 2005).

In a review, Dervieux et al (2005) stated that several proofs of principle have established that pharmacogenetic testing for mutations altering expression and functions of genes associated with drug disposition and response can reduce the "trial-and-error" dosing and decrease the risk of ADRs.  These proofs of principle include UGT1A1 and irinotecan therapy, as well as CYP450 2C9 and S-warfarin therapy.  These evidences advocate for the prospective identification of mutations associated with drug response, serious ADRs and treatment failure.  The authors stated that with the convergence of rising drug costs and evidence supporting the clinical benefits of pharmacogenetic testing, it will be important to demonstrate the improved net health outcomes attributed to the additional costs for this testing.

This in agreement with the observation of Eichelbaum et al (2005) who noted that there is also a growing list of genetic polymorphisms in drug targets that have been demonstrated to influence drug response.  A major limitation that has moderated the use of pharmacogenetic testing in the clinical setting is the lack of prospective clinical trials showing that such testing can improve the benefit/risk ratio of drug therapy.  Moreover, Gardiner and Begg (2005) stated that currently pharmacogenetic tests for drug metabolizing enzymes are rarely carried out in clinical practice, despite repeated claims that they may benefit patient care.  They noted that "the only tests performed with any regularity in Australasia are for thiopurine methyltransferase and pseudocholinesterase, and CYP2D6 phenotyping in 1 center for patients on perhexiline.  The low clinical utilization reflects a poor evidence base, un-established clinical relevance and, in the few cases with the strongest rationale, a slow translation to the clinical setting".

Han et al (2006) determined if uridine diphosphate-glucuronosyltransferase 1A1, UGT1A7, and UGT1A9 polymorphisms affect the pharmacokinetics (PK) of irinotecan and treatment outcome of Korean patients with advanced non-small-cell lung cancer (NSCLC).  A total of 81 patients with advanced NSCLC were treated with irinotecan (80 mg/m2) on day 1 and 8 and cisplatin (60 mg/m2) on day 1 intravenously of each 3-week cycle.  Genomic DNA was extracted from peripheral blood and genotyped using direct sequencing.  These researchers analyzed the association of UGT1A genotypes with irinotecan PK and clinical outcomes.  All statistical tests were two-sided.  In genotype-PK association analysis, UGT1A1*6/*6 (n = 6), UGT1A7*3/*3 (n = 6), and UGT1A9-118(dT)9/9 (n = 11) were associated with significantly lower area under the time-concentration curve (AUC) SN-38G to SN-38 (AUC(SN-38G)/AUC(SN-38)) ratio (p = 0.002, p =0 .009, and p = 0.001, respectively).  In linkage disequilibrium analysis, the UGT1A7 variants were highly linked with the UGT1A1*6 (D' = 0.85, r2 = 0.63) and UGT1A9*22 (D'= 0.95, r2 = 0.88), which was substantiated in haplotype analysis.  Patients with UGT1A1*6/*6 had lower tumor response and higher incidence of severe neutropenia.  UGT1A9-118(dT)9/9 also showed a trend for high incidence of severe diarrhea, but not tumor response.  In survival analysis, patients with UGT1A1*6/*6 had significantly shorter progression-free survival (p = 0.001) and overall survival (p = 0.017).  The authors concluded that UGT1A1*6 and UGT1A9*22 genotypes may be important for SN-38 glucuronidation and associate with irinotecan-related severe toxicity.  Specifically, UGT1A1*6 might be useful for predicting tumor response and survival outcome of Korean patients with NSCLC treated with irinotecan-based chemotherapy.  These investigators also stated that "[a]lthough it is still hypothetical, we suggest that UGT1A1*6 and/or UGT1A9*22 genotypes might be important for predicting severe toxicity and treatment outcome after irinotecan-based chemotherapy.  To confirm the data observed in this study, further larger studies are needed in an independent data set, preferably in a group of patients of similar ethnicity".

In an editorial that accompanied the study by Han et al (2006), Innocenti and associates (2006) stated that "[t]he study by Han et al provides evidence that the UGT1A1*6 polymorphism can be considered a biomarker of severe toxicity of irinotecan in Asians.  The impact of this variant on efficacy in irinotecan-containing regimens should be prospectively investigated in patients of Asian descent ….".

In a review on genetic polymorphisms of drug-metabolizing enzymes and drug transporters in the chemotherapeutic treatment of cancer, Bosch et al (2006) focused on the clinical significance of polymorphisms in drug-metabolizing enzymes (CYP2C8, CYP2C9, CYP2C19, CYP2D6, CYP3A4, CYP3A5, dihydropyrimidine dehydrogenase, UGT1A1, glutathione S-transferase, sulfotransferase [SULT] 1A1, N-acetyltransferase [NAT], thiopurine methyltransferase [TPMT]) and drug transporters (P-glycoprotein [multi-drug resistance 1], multi-drug resistance protein 2 [MRP2], breast cancer resistance protein [BCRP]) in influencing toxicity and effectiveness of chemotherapy.  The authors stated that the clinical application of pharmacogenetics in cancer treatment will require more detailed information of the different polymorphisms in drug-metabolizing enzymes and drug transporters; and that larger studies, in different ethnic populations, and extended with haplotype and linkage disequilibrium analysis, are needed for each anti-cancer drug separately.

While initial reports described above suggested that UGT1A1*28 homozygotes were at high risk for worse irinotecan-related hematologic and gastrointestinal toxicity, more recent reports suggest that the magnitude of the problem (particularly the association with worse diarrhea) is not as great as was initially suspected.  In a prospective study of 250 patients with metastatic colorectal cancer starting irinotecan, fluorouracil and leucovorin, the relative risk for grade 3 or 4 hematologic toxicity was significantly higher among UGT1A1*28 homozygotes (odds ratio 8.63, 95 % CI: 1.31 to 56.55) (Toffoli et al, 2006).  However, the absolute magnitude of risk was relatively low (13.6 % versus 1.7 % for those with the wild-type alleles), and relevant for the first cycle only.  Furthermore, there was no significant association between the presence of a UGT1A1*28 polymorphism and severity of diarrhea, or the need for irinotecan dose reduction.

One study found higher rates of neutropenia in persons homozygous for the UGT1A1*28 allele, regardless of whether the combination chemotherapy regimen included irinotecan (McLeod et al, 2006).  In a preliminary analysis of data from 520 patients with colorectal cancer enrolled in the United States Intergroup (INT) 9741 trial, which compared a variety of first-line oxaliplatin and irinotecan-containing chemotherapy regimens, the risk of grade 3 or 4 neutropenia was significantly higher for homozygotes (but not heterozygotes) regardless of whether they received irinotecan or oxaliplatin-based chemotherapy (36.2 % versus 18.2 % and 14.8 % for homozygotes, heterozygotes, and wild-type alleles, respectively).  Similar to the study by Toffoli et al (2006) described above, the investigators found no association between inheritance of UGT1A1*28 alleles and treatment-related diarrhea.  UGT1A1*28 was also not a predictor of tumor response, time to progression, or overall survival.

The Evaluation of Genomic Applications in Practice and Prevention Working Group (EGAPP, 2009) found that the evidence is currently insufficient to recommend for or against the routine use of UGT1A1 genotyping in patients with metastatic colorectal cancer who are to be treated with irinotecan, with the intent of modifying the dose as a way to avoid adverse drug reactions (severe neutropenia).  The EGAPP Working Group found no evidence to support clinical utility.  Preliminary modeling suggests that, even if targeted dosing were to be highly effective, it is not clear that benefits (reduced adverse drug events) outweigh harms (unresponsive tumors).

In summary, although it has been advocated that pharmacogenetic testing of patients with CRC before chemotherapy with irinotecan may reduce the frequency of severe toxicities by allowing alternate therapy selections for patients carrying the UGT1A1*28 polymorphism, the clinical value of this testing (i.e., whether testing will lead to better health outcomes) has yet to be established by prospective, randomized, controlled trials.  Only about 1 in 10 patients will be identified as being homozygous, and the excess risk of severe neutropenia that is attributable to the inheritance of this polymorphism appears to be small.  There is a lack of consensus on whether initial dose reduction is needed for UGT1A1*28 homozygotes, and the precise dose reduction that is warranted in this patient population has not been determined.

Genotyping of HLA Class 1 for Hyperuricemia Prior to Initiation of Allopurinol

The American College of Rheumatology (ACR)’s guidelines on “Management of gout” (Khanna et al, 2012) recommended that “Prior to initiation, consider HLA-B*5801 testing in selected patients, specifically in higher risk sub-populations for severe allopurinol hypersensitivity reaction (e.g., Koreans with stage 3 or worse CKD; Han Chinese and Thai irrespective of renal function)” (Evidence A).  Level A grading was assigned to recommendations supported by multiple (i.e., more than 1) randomized clinical trials or meta-analyses.

Hersfield et al (2013) noted that allopurinol is the most commonly used drug for the treatment of hyperuricemia and gout; however, allopurinol is also one of the most common causes of severe cutaneous adverse reactions (SCARs), which include drug hypersensitivity syndrome, Stevens–Johnson syndrome, and toxic epidermal necrolysis.  A variant allele of the human leukocyte antigen (HLA)-B, HLA-B*58:01, associates strongly with allopurinol-induced SCAR.  These investigators summarized the evidence from the published literature and developed peer-reviewed guidelines for allopurinol use based on HLA-B genotype.  They noted that the Taiwan Department of Health has updated the labeling for allopurinol to include information on HLA-B*58:01 (http://www.doh.gov.tw/EN2006/Newsroom/Press_list.aspx?year=2009&doc_no=72847).  The updated label describes the strong association between HLA-B*58:01 and allopurinol-induced SJS/TEN in the Han-Chinese population and recommended testing for the allele before the use of allopurinol.  The updated label does not recommend such testing for patients who have had no AEs after prolonged use of allopurinol.  At the time of writing this article, the FDA had not updated the labeling for allopurinol; however, given the high specificity for allopurinol-induced SCAR, allopurinol should not be prescribed to patients who have tested positive for HLA-B*58:01.  Alternative medication should be considered for these patients to avoid the risk of developing SCAR.  For patients who have tested negative, allopurinol may be prescribed as usual.  However, testing negative for HLA-B*58:01 does not totally eliminate the possibility of developing SCAR, especially in the European population.

Furthermore, an UpToDate review on “Drug hypersensitivity: Classification and clinical features” (Pichler, 2022) states that “Some DiHS/DRESS reactions occur more frequently in patients with certain human leukocyte antigen (HLA) types, since the drugs have been shown to bind to the certain HLA-allele itself predominantly (e.g., allopurinol/oxypurinol to the HLA-B*58:01) or exclusively (abacavir and HLA-B*57:01), a phenomenon that is also seen in SJS and TEN.  Specific examples include: DiHS/DRESS and SJS/TEN to allopurinol is associated with HLA-B*58:01.  This B*58:01 association is less stringent in Caucasians (approximately 60 %), in whom other alleles are also involved in SJS/TEN due to allopurinol … Once a patient has been identified as having a high-risk HLA profile, family members of that patient should also be advised to avoid the relevant drug, as familial occurrence of such hypersensitivity reactions has been noted.  Recommendations have been made for screening patients for specific alleles prior to administration of carbamazepine, oxcarbazepine, abacavir, and allopurinol”.

Genotyping for HLA-B*5801

Human leukocyte antigen-B (HLA-B) gene variations are associated with adverse reactions to some medications. Before taking allopurinol, HLA-B*5801 testing may be done for individuals of Korean descent with stage 3 or worse chronic kidney disease or of Han-Chinese or Thai descent. 

Genotyping for HLA-B*1502

Carbamazepine (brand names Carbatrol, Equetro and Tegretol) is used for the treatment of patients with epilepsy, bipolar disorder, and neuropathic pain.  The use of carbamazepine is associated with rare but severe and sometimes life-threatening skin reactions, which includes toxic epidermal necrolysis and Stevens-Johnson syndrome, characterized by multiple skin lesions, blisters, fever, itching and other symptoms.  The risk of these reactions is estimated to be about 1 to 6 per 10,000 new users of the drug in countries with mainly white populations.  However, the risk is estimated to be about 10 times higher in some Asian countries.  Studies have demonstrated a strong association between certain serious skin reactions and an inherited variant of an immune system gene, HLA-B*1502, found almost exclusively among individuals with Asian ancestry (Hung et al, 2006; Chung et al, 2007). 

Before taking carbamazepine, HLA-B*1502 testing may be used in individuals of Asian ancestry to identify an increased risk of developing severe skin disorders (eg, Stevens-Johnson syndrome (SJS) and toxic epidermal necrolysis [TEN]). Additionally, individuals who have tested positive for HLA-B 1502 may be directed to a medication other than phenytoin. 

In December 2007, the FDA announced that manufacturers of drugs containing carbamazepine have agreed to add to the drugs' labeling a recommendation that, before starting therapy with the drugs patients with Asian ancestry should receive genotyping of HLA-B*1502.

HLA-B*5701 Screening

HLA-B*5701 screening is indicated prior to initiation of an abacavir-containing regimen to reduce the risk of a hypersensitivity reaction in HIV individuals.

Abacavir (Ziagen) is a nucleoside analogue reverse transcriptase inhibitor indicated for use in combination with other antiretroviral drugs for the treatment of HIV-1 infection.  Review of reports of hypersensitivity in patients receiving abacavir (Glaxo Wellcome Inc.,  Research Triangle Park, NC) indicated that respiratory symptoms (including cough, dyspnea, and pharyngitis) have occurred in approximately 20 % of patients who have had hypersensitivity reactions.  The frequency of the HLA-B*5701 allele varies in different populations, occurring in whites 5 to 8 %, Hispanics 4 to 7 %, Asians less than 1 %, Spaniards 1 to 4 %, and rarely in Sub-Saharan Africans.  A delay in diagnosis of hypersensitivity can result in abacavir being continued or re-introduced, leading to more severe hypersensitivity reactions, including life-threatening hypotension and death.

In a double-blind, prospective, randomized study, Mallal et al (2008) examined whether HLA-B*5701 screening could prevent hypersensitivity reaction to abacavir.  Patients who were infected with HIV-1 infection (n = 1956) who had not previously received abacavir were randomly assigned to undergo prospective HLA-B*5701 screening, with exclusion of HLA-B*5701 positive patients from abacavir treatment (prospective screening group), or to undergo a standard of care approach of abacavir use without prospective HLA-B*5701 screening (control group).  All patients who started abacavir were observed for 6 weeks.  Epi-cutaneous patch testing with abacavir was performed to immunologically confirm and enhance the specificity of the clinical diagnosis of hypersensitivity reaction to abacavir.  The prevalence of HLA-B*5701 in this predominantly white study population was 5.6 %.  Hypersensitivity reaction was diagnosed in 93 patients with a significantly lower incidence in the prospective-screening group (3.4 %) than in the control group (7.8 %).  Of the patients receiving abacavir, 72 % were men, 84 % were white, and 18 % had not previously received anti-retroviral therapy.  Screening eliminated immunologically confirmed hypersensitivity reaction (0 % in the prospective-screening group versus. 2.7 % in the control group), with a negative predictive value of 100 % and a positive predictive value of 47.9 %.  The authors concluded that HLA-B*5701 screening reduced the risk of hypersensitivity reaction to abacavir.  Although the population in Mallal's study was predominantly white, other investigators have reported comparable sensitivity results of HLA-B*5701 for abacavir hypersensitivity in different ethnic groups, including blacks (Saag M et al, 2007) and Spaniards (Rodríguez-Nóvoa et al, 2007).  In addition, Guidelines for the Use of Antiretroviral Agents in HIV-1-Infected Adults and Adolescents (2008) developed by the Department of Health and Human Services recommends:

Screening for HLA-B*5701 before starting patients on an abacavir-containing regimen to reduce the risk of hypersensitivity reaction (strength of recommendation: Strong evidence with at least 1 randomized trial with clinical results);
HLA-B*5701-positive patients should not be prescribed abacavir (strength of recommendation: Strong evidence with at least 1 randomized trial with clinical results);
The positive status should be recorded as an abacavir allergy in the patient’s medical record (strength of recommendation: Strong evidence with clinical trials with laboratory results);
When HLA-B*5701 screening is not readily available, it remains reasonable to initiate abacavir with appropriate clinical counseling and monitoring for any signs of hypersensitivity reaction (strength of recommendation: Optional with expert opinion).
Genotyping for Apolipoprotein E (Apo E)

Apolipoprotein E (Apo E), a member of the apolipoprotein gene family, is essential in the formation of very low density lipoprotein (VLDL) and chylomicrons.  Among the variants of this gene, alleles e2, e3, and e4 are the common polymorphism found in most populations.  Of these variants, Apo e3 is the most frequent (greater than 60 %) in all populations studied.  The  polymorphism has functional effects on lipoprotein metabolism mediated through the hepatic binding, uptake, and catabolism of chylomicrons, chylomicron remnants, VLDL, and high density lipoprotein subspecies.  Apolipoprotein E is the primary ligand for 2 receptors, the low density lipoprotein (LDL) receptor (also known as the B/E receptor) found on the liver and other tissues and an Apo E-specific receptor found in the liver.  The interactions of these lipoprotein complexes with their receptors form the basis for the metabolic regulation of cholesterol.  Allelic variation in apolipoprotein is consistently associated with plasma concentrations of total cholesterol, LDL cholesterol, and Apo B (the major protein of LDL, VLDL, and chylomicrons).  Apolipoprotein has been studied in disorders associated with elevated cholesterol levels or lipid derangements (i.e., hyperlipoproteinemia type III, coronary heart disease, strokes, peripheral artery disease, and diabetes mellitus).  The apolipoprotein genotype yields poor predictive values when screening for clinically defined atherosclerosis despite positive, but modest associations with plaque and coronary heart disease outcomes.  In addition to genotype-phenotype associations with vascular disease, the alleles and isoforms of apolipoprotein have been related to dementias, most commonly Alzheimer's disease.

Several studies have been published assessing the interactions between Apo E and cholesterol in response to lipid-lowering drugs.

Gerdes et al (2000) examined whether the beneficial effects of simvastatin treatment differed by apolipoprotein genotype.  After providing dietary advice, they randomized men and women aged 35 to 70 years with a history of myocardial infarction or angina, serum total cholesterol concentrations in the range of 5.5 to 8.0 mmol/L, and serum triglyceride levels of less than 2.5 mmol/L to placebo or simvastatin groups.  Simvastatin treatment reduced the mortality risk more in e4 carriers than in other patients, although the difference was not statistically significant for the treatment by genotype interaction. 

At least 2 other studies have examined the influence of the Apo E polymorphism in response to lipid-lowering drug treatments in patients with combined hyper-lipoproteinemia and familial hypercholesterolemia. 

Knijff et al (1990) examined the influence of the Apo E polymorphism on pre-treatment plasma lipid levels and on the response to simvastatin treatment in a sample of 120 Dutch patients with heterozygous familial hypercholesterolemia.  They found that differences in pre-treatment lipid levels were not related to the Apo E polymorphism in these patients.  With respect to the effect of 12 weeks of simvastatin treatment, a reduction of 33 %, 38 %, and 19 % (on average) was found in the plasma levels of total cholesterol, LDL cholesterol, and triglycerides, respectively.  Inter-individual variation in response to simvastatin treatment was not related to the Apo E polymorphism.

Nestel et al (1997) conducted a cross-over, randomized trial to examine the efficacy of simvastatin and gemfibrozil in patients with combined hyperlipoproteinemia.  Efficacy was noted after 6 and 12 weeks on each treatment for the 66 subjects enrolled.  The lipid-lowering responsiveness was greatest in those with the Apo E2 isoform with both medications.

An Agency for Healthcare Research and Quality (AHRQ, 2008) technology assessment on pharmacogenetic testing reviewed the available evidence of Apo E genotype (e2, e3, and e4) and statin treatment and found that genotyping for Apo E has not been shown to help select patients for treatment.  "No studies addressed the effects of therapeutic choice: there were no data on the benefits, harms, or adverse effects on patients from subsequent therapeutic management after pharmacogenetic testing for the three Apo E genotypes."The AHRQ assessment found that the pooled reduction in total and LDL cholesterol from baseline values was lower for all 3 genotypes but did not differ significantly among them.

The AHRQ also found significant between-study heterogeneity.  "Although few studies included certain subgroups, factors that may affect the associations between all three Apo E genotypes and response to statin therapy were ethnicity, sex, familial hyperlipidemia, the type of statin used, and possibly the presence of diabetes."

In addition, there are no prospective data showing improved clinical management of hypercholesterolemia patients as a result of genotyping for Apo E.

Genotyping for Methylenetetrahydrofolate Reductase (MTHFR)

Methylenetetrahydrofolate reductase (MTHFR) is a key enzyme regulating intracellular folate levels, which in turn affects DNA synthesis and methylation.  Two MTHFR gene polymorphisms, C677T and A1298C, influence the metabolism of folates and could modify the pharmacodynamics of antifolates and many other drugs whose metabolism, biochemical effects, or target structures require methylation reactions.  Several studies have shown these two polymorphisms may reduce cancer susceptibility and increase drug-related toxicity when folate antagonists (e.g., methotrexate, fluorouracil) are utilized, but data are inconsistent and contradictory.  According to the National Cancer Institute, 5 of 6 patients who experienced grade-4 toxicity in their first cycle of adjuvant chemotherapy with cyclophosphamide, methotrexate and 5-FU for early breast cancer had the variant C677T MTHFR genotype. 

An Agency for Healthcare Research and Quality (AHRQ, 2008) technology assessment on pharmacogenetic testing reviewed the available evidence on MTHFR gene polymorphisms for their associations with patient's response to therapy with antifolate chemotherapy and found that the evidence indicates that MTHFR gene polymorphisms do not predict response to chemotherapy. 

AHRQ found limited data on MTHFR gene testing and therapeutic choice which preclude making meaningful inferences about the relationship between common variants in MTHFR and chemotherapy of the folate metabolic pathway.

AHRQ also found considerable variation in study designs, study populations, medication dosages, and the type of medications.

Studies have shown that MTHFR polymorphisms may affect the sensitivity to antifolate chemotherapy, however, there is insufficient evidence of its clinical effectiveness.

Measurement of Thromboxane Metabolites in Urine

The U.S. Preventive Services Task Force (USPSTF, 2009) has developed guidelines for chronic aspirin therapy based upon a person's age and Framingham risk score. Long-term aspirin administration has clear benefits for the secondary prevention of cardiovascular diseases with a significant 21 % reduction in the risk of cardiovascular events over 2 years (Berger et al, 2008).  However, this indicates that not all individuals respond equally to aspirin therapy and cardiovascular events may occur during aspirin therapy.  This is often described as "clinical aspirin resistance".  A systematic review and meta-analysis on aspirin resistance indicated that patients who are resistant to aspirin are at a greater risk (odds ratio [OR]: 3.85) of clinically important cardiovascular morbidity than patients who are sensitive to aspirin (Krasopoulos et al, 2008).  The effect of aspirin administration varies considerably among patients at high risk for cardiovascular events.  Gum and co-workers (2001) found insufficient inhibition of platelet aggregation by aspirin in 6 to 24 % of patients with stable coronary artery disease, while other estimates range from 5 to 60 % (Martin and Talbert, 2005).

Many authors believe that aspirin resistance can be detected by biochemical tests and several commercially available products are being marketed for this purpose.  Tests used in research laboratories are aggregometry (turbidometric and impedance), tests based on activation-dependent changes in platelet surface, and tests based on activation-dependent release from platelets.  Point-of-care tests include PFA-100, IMPACT, and VerifyNow, which can detect platelet dysfunction that may be due to aspirin effect. 

It has been proposed that aspirin resistance can also be detected by thromboxane metabolites in urine.  Aspirin inhibits platelet activation through the permanent inactivation of the cyclooxygenase (COX) activity of prostaglandin H synthase-1 (referred to as COX-1), and consequently inhibits the biosynthesis of thromboxane A2(TXA2), a platelet agonist.  The urinary concentrations of the metabolite 11-dehydrothromboxane B2(11 dhTxB2) indicate the level of TXA2 generation.

Eikelboom et al (2002) studied whether aspirin resistance, defined as failure of suppression of thromboxane generation, increases the risk of cardiovascular events in a high-risk population.  Baseline urine samples were obtained from 5,529 Canadian patients enrolled in the Heart Outcomes Prevention Evaluation (HOPE) Study.  Using a nested case-control design, the investigators measured urinary 11 dhTxB2 levels, a marker of in vivo thromboxane generation, in 488 cases treated with aspirin who had myocardial infarction, stroke, or cardiovascular death during 5 years of follow-up and in 488 sex- and age-matched control subjects also receiving aspirin who did not have an event.  After adjustment for baseline differences, the odds for the composite outcome of myocardial infarction, stroke, or cardiovascular death increased with each increasing quartile of 11 dhTxB2, with patients in the upper quartile having a 1.8-times-higher risk than those in the lower quartile (OR, 1.8; 95 % CI: 1.2 to 2.7; p = 0.009).  Those in the upper quartile had a 2-times-higher risk of myocardial infarction (OR, 2.0; 95 % CI: 1.2 to 3.4; p = 0.006) and a 3.5-times-higher risk of cardiovascular death (OR, 3.5; 95 % CI: 1.7 to 7.4; p < 0.001) than those in the lower quartile.  The authors concluded that in aspirin-treated patients, urinary concentrations of 11 dhTxB2 predict the future risk of myocardial infarction or cardiovascular death and that these findings raise the possibility that elevated urinary 11 dhTxB2 levels identify patients who are relatively resistant to aspirin and who may benefit from additional anti-platelet therapies or treatments that more effectively block in vivo thromboxane production or activity.  However, Altman et al (2004) reviewed this study and stated that the authors support the view that failure to suppress thromboxane generation defines aspirin resistance and that this hypothesis assumes a direct association between the rise of urinary 11 dhTxB2 levels and increment of vascular events (e.g., myocardial infarction, stroke and cardiovascular death).  Altman and colleagues explained that failure of aspirin to produce the expected inhibition of platelet function might be attributed to several mechanisms and that it can not be defined by the level of serum thromboxane or its urinary metabolites because these measurements do not correlate with the reduction of inhibition of platelet aggregation in response to multiple stimuli, and also because although most of the thromboxane is believed to come from the platelets, there are additional cellular origins (e.g., monocytes/macrophages are also a rich source of TXA2), unlike the platelet, the macrophage is capable of synthesizing new COX-2 after aspirin has inhibited it; COX-2 is the enzyme responsible for most of the metabolism of arachidonic acid in the macrophage, and low dose aspirin is not sufficient to inhibit COX-2 maximally, macrophages in atheromata may contribute significantly to the pool of TXA2, and aspirin only inhibits monocyte PGHS-2, which is inducible by inflammatory stimuli, transiently at very high concentrations.
although most of the thromboxane is believed to come from the platelets, there are additional cellular origins (e.g., monocytes/macrophages are also a rich source of TXA2),
unlike the platelet, the macrophage is capable of synthesizing new COX-2 after aspirin has inhibited it; COX-2 is the enzyme responsible for most of the metabolism of arachidonic acid in the macrophage, and low dose aspirin is not sufficient to inhibit COX-2 maximally,
macrophages in atheromata may contribute significantly to the pool of TXA2, and
aspirin only inhibits monocyte PGHS-2, which is inducible by inflammatory stimuli, transiently at very high concentrations.

The AspirinWorks Test Kit (Corgenix Medical Corp; Broomfield, CO) is an enzyme-linked immunoassay test that can be used to determine levels of 11 dhTxB2 in human urine.  AspirinWorks received 510(k) marketing clearance from the FDA in May, 2007 and is intended to aid in the qualitative detection of aspirin in apparently healthy individuals post ingestion. 

The AspirinWorks Test Kit was compared to the Accurnetrics VerifyNow Aspirin Assay as the predicate device.  The manual AspirinWorks Test Kit measures urinary 11 dhTxB2, while the automated Accumetrics VerifyNow Aspirin Assay is a turbidimetric-based optical detection system, which measures platelet-induced aggregation in whole blood.  The two devices have similar intended uses in that they both measure aspirin effect.  The AspirinWorks kit detects a metabolite of TxA2, a direct inducer of platelet aggregation, while the Accumetrics kit measures ex vivo platelet aggregation caused by TxA2 by artificially inducing aggregation and measuring an optical signal.  Ultimately, both are analyzing aspirin's effect through the reduction of TxA2 production or the resulting inhibition of platelet aggregation.

According to the FDA, 2 different clinical studies were employed for the evaluation of the AspirinWorks Test Kit.  Results from these studies established a cutoff for aspirin effect at less than 1500 pg 11d hTxB2/mg creatinine.  Further analysis revealed that 180/204 (88.2 %) of samples from individuals not taking aspirin were above the cut-off value.  Analysis of samples from individuals taking various doses of aspirin revealed that 7/163 (4.3 %) of 81 mg/day aspirin users indicated a lack of aspirin effect (greater than 1500 pg 1 ldhTxB2/mg creatinine) and 4/38 (10.5 %) of the 325 mg/day aspirin users indicated a lack of aspirin effect.  In total, 11/201 (5.5 %) of all aspirin users tested indicated a lack of aspirin effect.  These percentages are consistent with those in published literature for aspirin non-responsiveness or lack of aspirin effect.

Lordkipanidze et al (2007) compared the results obtained from 6 major platelet function tests in the assessment of the prevalence of aspirin resistance in patients with stable coronary artery disease.  Patients with stable coronary artery disease (n = 201) receiving daily aspirin therapy (80 mg or more) were recruited.  Platelet aggregation was measured by: light transmission aggregometry (LTA) after stimulation with 1.6 mM of arachidonic acid (AA), LTA after adenosine diphosphate (ADP) (5, 10, and 20 microM) stimulation, whole blood aggregometry, PFA-100, and VerifyNow Aspirin; urinary 11 dhTxB2 concentrations were also measured.  
light transmission aggregometry (LTA) after stimulation with 1.6 mM of arachidonic acid (AA),
LTA after adenosine diphosphate (ADP) (5, 10, and 20 microM) stimulation,
whole blood aggregometry,
PFA-100, and
VerifyNow Aspirin; urinary 11 dhTxB2 concentrations were also measured. 

A total of 8 patients (4 %, 95 % CI: 0.01 to 0.07) were deemed resistant to aspirin by LTA and AA.  The prevalence of aspirin resistance varied according to the assay used: 10.3 to 51.7 % for LTA using ADP as the agonist, 18.0 % for whole blood aggregometry, 59.5 % for PFA-100, 6.7 % for VerifyNow Aspirin, and finally, 22.9 % by measuring urinary 11 dhTxB2  concentrations.  Results from these tests showed poor correlation and agreement between themselves.  The authors concluded that platelet function tests are not equally effective in measuring aspirin's anti-platelet effect and correlate poorly amongst themselves and that the clinical usefulness of the different assays to classify correctly patients as aspirin resistant remains undetermined.

Hedegaard et al (2009) assessed the use of optical platelet aggregation versus thromboxane metabolites in healthy individuals and patients with stable coronary artery disease after low-dose aspirin administration.  The authors investigated whether 75 mg of daily non-enteric coated aspirin would completely inhibit the platelet cyclooxygenase-1 activity to a comparable extent in healthy individuals and stable coronary artery disease (CAD) patients.  Serum thromboxane B2 (S-TxB2), urinary 11 dhTxB2 (U-TxM) and arachidonic acid-induced optical platelet aggregometry (OPA) were compared in 44 coronary artery disease (CAD) patients on aspirin and in 22 healthy individuals before and after aspirin.  Optical platelet aggregometry was performed in duplicate for 4 consecutive days during aspirin treatment after 1 week of treatment.  Compliance was optimized by face-to-face interviews and pill counting and confirmed by S-TxB2 measurements.  The authors found that aspirin inhibited S-TxB2 in healthy individuals (greater than 99 %; median 1.1 ng/mL, inter-quartile range [IQR] = 0.8;1.9 after aspirin) and in patients, S-TxB2 was reduced to a similar level (0.9 ng/mL (0.7;1.5)).  Healthy individuals had a median U-TxM of 278.5 pg/mg creatinine (229.5;380.0) before aspirin and 68.5 pg/mg creatinine (59.0;99.7) on aspirin corresponding to an average 74 % inhibition of the endogenous TxA2 biosynthesis.  In patients median U-TxM was 67.5 pg/mg creatinine (54.0;85.5).  Seven study participants (11 %) were aspirin low-responders according to OPA, but none had S-TxB2 in the highest quartile.  The authors concluded that low-dose aspirin suppressed S-TxB2 to comparable levels in CAD patients and healthy individuals.  The authors found that despite an almost complete inhibition of S-TxB2, some participants were low-responders according to OPA.  The authors concluded that thorough compliance control and use of thromboxane-specific assays are important when measuring platelet response to aspirin.

While some investigators believe that aspirin resistance can be detected by thromboxane metabolites in urine, other investigators support the view that aspirin resistance can not be defined by the level of serum thromboxane or its urinary metabolites because these measurements do not correlate with the reduction of inhibition of platelet aggregation in response to multiple stimuli as well as various other factors.  Investigators have found a number of variables that may impact an individual's response to aspirin, including patient's compliance, dose, smoking, hyperlipidemia, hyperglycemia, acute coronary syndrome, percutaneous revascularization, recent stroke, extracorporeal circulation, heart failure, exercise, circadian rhythm, absorption, concomitant medications, and polymorphisms.

Many issues are yet to be resolved in order to apply the concept of "aspirin resistance" to actual clinical practice.  The clinical usefulness of a test that measures thromboxane metabolites in urine has yet to be determined.  The relevance of the various ex vivo functional indexes of platelet capacity to in vivo platelet activation and the precise mechanisms underlying aspirin resistance are still largely unknown.  Further investigation is needed regarding strategies to identify and treat patients resistant to aspirin.

Genetic Testing for rs3798220 Allele (LPA-Aspirin Check)

LP(a) aspirin genotype testing (eg, Cardio IQ LPA Aspirin Genotype Test and LPA-Aspirin Genotype Test) has been proposed to identify individuals at risk of cardiovascular disease (CVD) which may respond to aspirin therapy.

Investigators have described a non-synonymous single nucleotide polymorphism (SNP rs3798220) in apo(a) gene (LPA) that has been associated with increased plasma levels of Lp(a) and advanced coronary artery disease.  Observational studies have also associated that this polymorphism may identify a subgroup of patients who benefit from chronic aspirin therapy in regard to a reduction in clinical coronary heart disease events, and a subgroup which reveals no benefit and thus an increased risk for gastrointestinal bleeding.  A genetic testing for the rs3798220 allele (LPA-Aspirin Check, Celera, Alameda, CA) is commercially available.  This SNP, encoding an isoleucine to methionine substitution in the protease-like domain of apo(a) at amino acid 4399 (I4399M), has been associated with both elevated levels of Lp(a) and cardiovascular disease.  In the Women's Health Study, carriers of this apolipoprotein(a) variant had elevated Lp(a), doubled cardiovascular risk, and appeared to benefit more from aspirin than non-carriers (Chasman et al, 2009).  Chasman and colleagues (2009) investigated whether this allele was associated with elevated Lp(a) and cardiovascular risk in a post hoc analysis of the Women's Health Study, a randomized trial of low-dose aspirin, and whether aspirin reduced cardiovascular risk in minor allele carriers.  The investigators determined genotypes of rs3798220 for 25,131 initially healthy Caucasian participants.  Among women with successful genotyping, the minor allele of rs3798220 in the LPA gene was carried by 906 (3.6 %) heterozygotes and 15 (0.06 %) homozygotes for a minor allele frequency of 1.9 %.  Median Lp(a) levels at baseline were 10.0, 79.5, and 153.9 mg/dL for major allele homozygotes, heterozygotes, and minor allele homozygotes, respectively (p < 0.0001).  During the 9.9 years of follow-up, minor allele carriers (3.7 %) in the placebo group had 2-fold higher risk of major cardiovascular events than non-carriers (age-adjusted hazard ratio (HR) = 2.21, 95 % CI: 1.39 to 3.52).  The investigators found that, among carriers, risk was reduced more than 2-fold by aspirin: for aspirin compared with placebo the age-adjusted HR was 0.44 (95 % CI: 0.20 to 0.94); risk was not significantly reduced among non-carriers (age-adjusted HR = 0.91, 95 % CI: 0.77 to 1.08).  The investigators reported that this interaction between carrier status and aspirin allocation was significant (p = 0.048).  Shiffman and colleagues (2009) reported data on the interaction of the LPA rs3798220 variant and aspirin use from the Atherosclerosis Risk in Communities (ARIC) study, a prospective cohort study of risk factors for coronary artery disease in 15,792 individuals.  The LPA genetic substudy of ARIC included 6752 individuals with data available for LPA genotype and aspirin use, including 221 individuals with the LPA rs3798220 genotype.  Among carriers of rs3798220, the risk of cardiovascular events was compared in aspirin users and non-users.  The HR for non-aspirin users (n = 168) was elevated at 1.57 but did not reach statistical significance (95 % CI: 0.92 to 2.69), while the HR for users of aspirin was not elevated at 0.86 (95 % CI 0.38 to 1.95).  Prospective clinical studies are necessary to determine whether selection of persons for chronic aspirin therapy on the basis of rs2798220 status results in improved clinical outcomes.

Area Under the Curve (AUC)-Targeted 5-Fluorouracil Dosing

Diagnostic tests (e.g., Myriad Genetics OnDose) have been developed to measure colorectal cancer patients’ exposure to 5-fluorouracil (5‑FU), to help oncologists adjust and optimize 5‑FU dosing.  This testing is based upon the belief that that 5‑FU chemotherapy dosing can be improved by modulating dose to plasma concentration, specifically an AUC target value, as treatment is delivered.  An assessment by the BlueCross BlueShield Association Technology Evaluation Center (TEC, 2010) stated that, given the limitations of the existing evidence, the evidence is insufficient to draw conclusions about the impact of 5‑FU exposure measurement and AUC-targeted dose adjustment on outcomes of patients administered current chemotherapy regimens for colorectal or head and neck cancer.

To determine if there is adequate evidence that AUC-targeted dose adjustment of 5-FU improves clinical outcomes, the TEC assessment (BCBSA, 2010) focused primarily on the results of 2 randomized, controlled trials, 1 enrolling patients with colorectal cancer (Gamelin et al, 2008) and the other patients with head and neck cancer (Fety et al, 1998).

The randomized controlled trial of AUC-targeted dosing of 5-FU for colorectal cancer reported significantly improved tumor response and a trend toward improved survival using AUC-targeted dosing compared to fixed dosing (Gamelin et al, 2008).  The authors of the clinical trial also reported 18 % grade 3 to 4 diarrhea in the fixed-dose arm, which the TEC assessment noted is higher than reported historically, where the rates of grade 3 to 4 diarrhea have ranged from approximately 5 to 7 %.  However, the TEC assessment cited an editorial accompanying the study (Wako and McLeod, 2008) that noted that the fixed-dose administration schedule used in this study is “rarely used in current practice in most countries"; the TEC assessment also noted that this administration schedule is absent from current guidelines.

The AUC dose modulation trial in head and neck cancer patients reported overall 5‑FU exposures that were significantly reduced after dose adjustment compared to the fixed-dose arm (Fety et al, 1998).  This resulted in reduced toxicity, but no improvement in clinical response.  The TEC assessment noted that the dose adjustment method in this trial may have been too complex, as the 12 protocol violations in this treatment arm (of 61 enrolled) were all related to 5‑FU dose adjustment mis-calculations (BCBSA, 2010).  The TEC assessment stated: "Because patients with protocol violations were not evaluated in an intention-to-treat analysis, the results do not reflect the 'real world' experience of this study."  The TEC assessment also noted that the study used a 2-drug induction regimen, which has now been replaced by a 3-drug induction regimen.

Pharmacogenetic Testing for 5-Fluorouracil Toxicity

Genetic polymorphisms in the genes coding for dihydropyrimidine dehydrogenase (DPYD) and thymidylate synthase (TYMS), key enzymes in 5-FU metabolism, may result in enzyme products with different activity levels, resulting in 5-FU excess, the accumulation of 5-FU anabolic products, and severe toxicity.  TheraGuide 5-FU (Myriad laboratories) is a commercially available test that analyzes DNA from peripheral blood cells to fully sequence the DPYD gene (for the 3 most common DYPD polymorphisms and other rare variants) and detect TYMS variants that increase risk for 5-FU toxicity.  An assessment by the BlueCross BlueShield Association Technology Evaluation Center (TEC, 2010) found, however, that testing for genetic variants of the genes coding for DPYD and TYMS enzymes has poor predictive value for 5-FU toxicity and no studies have shown that it is useful in directing 5-FU dose alterations to reduce toxicity without adversely impacting tumor response.

DPYD Gene Variant Testing for Dihydropyrimidine Dehydrogenase (DPD) Protein Deficiency

Dihydropyrimidine dehydrogenase (DPD) deficiency is a rare autosomal recessive disorder of the pyrimidine degradation pathway and can manifest as intellectual disability, motor retardation, and seizures (van Kuilenburg et al., 2019). Dihydropyrimidine dehydrogenase (DPYD) is the enzyme responsible of for breaking down fluoropyrimidines (FDA, 2025). The DPYD gene encodes the DPYD enzyme and variants of this gene in cancer patients receiving fluoropyrimidine therapy have an increased predictive risk for serious and life-threatening toxicities (FDA, 2025; van Kuilenburg et al., 2019). Complete or near complete absence of DPD activity (compete DPD deficiency) can place patients at risk for acute early-onset toxicity and serious, including fatal, adverse reactions (e.g., mucositis, diarrhea, neutropenia, and neurotoxicity). Patients with partial DPD activity (partial DPD deficiency) may also be at increased risk of serious, including fatal, adverse reactions (FDA, 2025).

The FDA Oncology Center of Excellence, under Project Renewal, updated the Xeloda (capecitabine tablets) product labeling to include additional information on DPD deficiency with an emphasis of discussing DPD deficiency with patients. A safety labeling update to 5-fluorouracil (FU) included the same DPD deficiency information as in Xeloda. The product labeling for both drugs include the following (FDA, 2025):

Capecitabine or fluorouracil are not recommended for use in patients known to have certain homozygous or compound heterozygous DPYD variants that result in complete DPD deficiency. No dose has been proven safe for patients with complete DPD deficiency.
There is insufficient data to recommend a specific dose in patients with partial DPD deficiency.
All healthcare providers should inform patients of the potential for serious and life-threatening adverse reactions due to DPD deficiency. Providers should discuss with their patient whether the patient should be tested for genetic variants that are associated with an increased risk of serious adverse reactions from the use of capecitabine or 5-FU.
Providers should consider testing prior to initiating capecitabine or fluorouracil to reduce the risk of serious adverse reactions.
Providers should be aware of the signs and symptoms associated with adverse reactions due to DPD deficiency and advise patients to immediately contact their provider if these occur, including severe mucositis, diarrhea, neutropenia, and neurotoxicity.
Providers should withhold or permanently discontinue capecitabine or 5-FU based on clinical assessment of the onset, duration, and severity of the observed adverse reactions in patients with evidence of acute early-onset or unusually severe reactions, which may indicate complete DPD deficiency.
Four DPYD variants have been associated with impaired DPD activity in White populations, and one variant has been associated with impaired activity in individuals of African ancestry.

Both NCCN Guidelines Version 3.2025 Colon Cancer and NCCN Guidelines Version 2.2025 Rectal Cancer recommend that testing for DPYD genetic variants should be considered prior to fluoropyrimidine therapy.

Mutation Testing in the BRAF V600 Gene

B-RAF(V600E) (also known as BRAF) kinase mutations occur in about 8 % of all solid tumors and approximately 50 % to 60 % of melanomas; these mutations are found at exon 15, at a single amino acid residue, usually a substitution for valine by glutamic acid, V599E, now referred to as V600E (Nazarian et al, 2010; Dienstmann and Tabernero, 2011).  The quantification of BRAF-mutated alleles in plasma may represent a useful biomarker for non-invasive diagnosis and prediction of response to therapy.  The development of efficient methods for its detection in free circulating DNA of patients may lead to the improvement of diagnostic and prognostic tools.

Daniotti and colleagues (2007) examined if BRAFV600E represents a detectable marker in the plasma/serum from patients with melanoma.  Circulating cell-free DNA was extracted from the serum or plasma of 15 healthy donors and 41 melanoma patients at different clinical stages and obtained either pre-surgery or post-surgery during follow-up.  Quantitative analysis showed higher levels of circulating free DNA in patients compared to controls, with the highest levels detected in samples obtained pre-surgery and at stage IV.  Four different PCR methods were compared for their capacity to amplify a few copies of BRAFV600E in wild-type DNA.  BRAFV600E was detectable in circulating DNA of 12 patients and in none of the controls; only 1 PCR method reproducibly amplified BRAFV600E.  Positive samples were obtained from 8/13 patients at stage IV and from 4/24 patients at stage III, but not in 4 patients at stage I to II; 50 % of the positives were obtained pre-surgery and 50 % at follow-up.  Correspondence between circulating DNA and related tumors were examined for 20 patients, and a correlation was found for stage IV patients.  The authors concluded that this method can be utilized for monitoring the disease in patients with stage IV melanoma, but it appears unsatisfactory for the early detection of melanoma.

Panka et al (2010) stated that the BRAFV600E mutation has been detected in patients with metastatic melanoma, colon, thyroid and other cancers.  Recent studies suggested that tumors with this mutation are especially sensitive to BRAF inhibitors, hence the need to reliably determine the BRAF status of tumor specimens.  The present technologies used to screen for this mutation fail to address the problems associated with infiltrating stromal and immune cells bearing wild-type BRAF alleles and thus may fail to detect the presence of mutant BRAFV600E tumors.  These investigators developed a rapid, inexpensive method that reduces the contamination of wild-type BRAF sequences from tumor biopsies.  The protocol involves a series of PCR amplifications and restriction digestions that take advantage of unique features of both wild-type and mutant BRAF RNA at position 600.  Using this protocol, mutant BRAF can be detected in RNA from mixed populations with as few as 0.1 % BRAFV600E mutant cells.

Pinzani and co-workers (2010) proposed an assay based on the use of a locked nucleic acid probe and an allele specific primer to measure plasma-circulating BRAFV600E concentration in patients affected by cutaneous melanoma (n = 55) and non-melanoma skin cancers (n = 13) as well as 18 healthy subjects.  The assay is highly sensitive and accurate in detecting down to 0.3 % of mutated allele in plasma.  A significant difference between the control group and invasive melanomas (p < 0.01) was evidenced in BRAFV600E concentration, either as relative percentage or absolute values.  Receiver operating characteristic curve indicated that BRAFV600E absolute concentration has the maximal diagnostic relevance with 97 % sensitivity and 83 % specificity.  Comparison of the results obtained in plasma with those found in the corresponding tissues indicated an 80 % concordance.  The authors concluded that the allele specific Taqman-based real-time PCR assay allows the sensitive, accurate and reliable measurement of BRAFV600E mutated DNA in plasma.

Pinzani et al (2011) investigated COLD-PCR (co-amplification at lower denaturation temperature-PCR) as a new approach for the pre-analytical enrichment of the BRAFV600E variant in formalin fixed paraffin embedded (FFPE) melanoma tissues.  COLD-PCR was used to selectively amplify BRAFV600E minority alleles from mixtures of wild-type and mutated sequences, and from biological samples.  The method showed higher specificity than other conventional PCR-based methods in detecting somatic mutations.  These investigators used COLD-PCR to increase the theoretical sensitivity of 3 different post-PCR methods: sequencing, (pyro-sequencing, and (HRMA.  
sequencing,
(pyro-sequencing, and
(HRMA. 

The gain in sensitivity seems to be more evident for HRMA, which allows the detection of 3.1 % mutated alleles.  More than 20 % of patients initially classified negative for BRAFV600E were found positive after COLD-PCR.  The authors concluded that COLD-PCR was confirmed as a suitable method for the enrichment of mutated alleles, particularly for samples in which the percentage of tumor cells is very low.

On August 17, 2011, the FDA approved vemurafenib (Zelboraf) for the treatment of patients with unresectable or metastatic melanoma with the BRAFV600E mutation as detected by an FDA-approved test. The approval was based on a randomized, open-label trial in patients with previously untreated metastatic or unresectable melanoma with the BRAFV600E mutation as detected by the cobas 4800 BRAF V600 Mutation Test (Roche Molecular Systems, Inc.).  This companion diagnostic test was approved by the FDA concurrently with vemurafenib’s approval.  The cobas 4800 BRAF V600 test is a PCR-based test.  It has several advantages over the commonly used Sanger sequencing, including greater sensitivity and reliability for detecting mutations and quicker results.

On May 29, 2013, the FDA approved 2 new drugs -- dabrafenib (Tafinlar) and trametinib (Mekinist), for patients with metastatic or unresectable melanoma.  Tafinlar, a BRAF inhibitor, is approved to treat patients with melanoma whose tumors express the BRAF V600E gene mutation.  Mekinist, a MEK inhibitor, is approved to treat patients whose tumors express the BRAF V600E or V600K gene mutations.  Approximately 50 % of melanomas arising in the skin have a BRAF gene mutation.  Tafinlar and Mekinist are being approved as single agents, not as a combination treatment.  Furthermore, the FDA approved Tafinlar and Mekinist with a genetic test called the THxID BRAF test, a companion diagnostic that will help determine if a patient’s melanoma cells have the V600E or V600K mutation in the BRAF gene.

On September 4, 2014, the FDA approved pembrolizumab (Keytruda) for the treatment of patients with unresectable or metastatic melanoma and disease progression following ipilimumab and, if BRAF V600 mutation positive, a BRAF inhibitor (e.g., dabrafenib, trametinib, or vemurafenib).

On June 22, 2017, the FDA approved dabrafenib (Tafinlar) and trametinib (Mekinist) administered in combination for patients with metastatic non-small cell lung cancer (NSCLC) with BRAF V600E mutation as detected by an FDA-approved test. These are the first FDA approvals specifically for treatment of patients with BRAF V600E mutation-positive metastatic NSCLC. Furthermore, the FDA approved a next generation sequencing (NGS) test called Oncomine™ Dx Target Test, a companion diagnostic that will help detect multiple gene mutations for lung cancer in a single test from a single tissue specimen. This test detects the presence of BRAF, ROS1, and EGFR gene mutations or alterations in tumor tissue of patients with NSCLC. This test can be used to select patients with NSCLC with the BRAF V600E mutation for treatment with the combination of dabrafenib and trametinib. This is the first NGS oncology panel test approved by the FDA for multiple companion diagnostic indications.

The approvals are based on Study BRF113928 (NCT01336634), an international, multicenter, three-cohort, non-randomized, non-comparative, open-label, trial in patients with locally confirmed BRAF V600E mutation-positive metastatic NSCLC. Ninety-three patients were treated with the combination of dabrafenib (150 mg orally twice daily) and trametinib (2 mg orally once daily). Of these 93 patients, 36 had received no prior systemic therapy for metastatic NSCLC and 57 received at least one platinum-based chemotherapy regimen with demonstrated disease progression. Seventy-eight patients with previously treated BRAF V600E mutation-positive NSCLC received single-agent dabrafenib. In the previously treated group, the overall response rate (ORR) for the combination based on independent radiology review committee assessment per RECIST 1.1 was 63% (95% CI: 49%, 76%) with a median duration of response (DoR) of 12.6 months (95% CI: 5.8, not estimable [NE]). In the treatment-naive group, the ORR for the combination was 61% (95% CI: 44%, 77%) and median DoR was not estimable (95% CI: 6.9, NE); however, 59% of responders had response durations greater than six months. The ORR for patients who received single-agent dabrafenib was 27% (95% CI: 18%, 38%) and the median DoR was 9.9 months. The incidence and severity of adverse reactions occurring in patients with NSCLC were generally similar to those reported in prior approvals for patients with melanoma. The most common adverse reactions (≥20%) were pyrexia, fatigue, nausea, vomiting, diarrhea, dry skin, decreased appetite, edema, rash, chills, hemorrhage, cough, and dyspnea. The most common Grade 3-4 adverse reactions were pyrexia, fatigue, dyspnea, vomiting, rash, hemorrhage, and diarrhea. The majority of laboratory abnormalities were Grade 1-2. The most common (≥5%) Grade 3-4 laboratory abnormalities were hyponatremia, lymphopenia, anemia, hyperglycemia, neutropenia, leukopenia, hypophosphatemia, and increased alanine aminotransferase. Dabrafenib and trametinib were discontinued for adverse reactions in 18% and 19% of patients, respectively. The recommended doses are dabrafenib 150 mg orally twice daily, approximately 12 hours apart, with trametinib 2 mg orally once daily. The presence of BRAF V600E mutation in tumor specimen should be confirmed by an FDA-approved test prior to initiation of therapy.

On May 4, 2018, the FDA approved Tafinlar (dabrafenib) and Mekinist (trametinib), administered together, for the treatment of anaplastic thyroid cancer (ATC) that cannot be removed by surgery or has spread to other parts of the body (metastatic), and has a type of abnormal gene, BRAF V600E (BRAF V600E mutation-positive).

The efficacy of Tafinlar and Mekinist in treating ATC was shown in an open-label clinical trial of patients with rare cancers with the BRAF V600E mutation. Data from trials in BRAF V600E mutation-positive, metastatic melanoma or lung cancer and results in other BRAF V600E mutation-positive rare cancers provided confidence in the results seen in patients with ATC. The trial measured the percent of patients with a complete or partial reduction in tumor size (overall response rate). Of 23 evaluable patients, 57 percent experienced a partial response and 4 percent experienced a complete response; in nine (64 percent) of the 14 patients with responses, there were no significant tumor growths for six months or longer. The side effects of Tafinlar and Mekinist in patients with ATC are consistent with those seen in other cancers when the two drugs are used together. Common side effects include fever (pyrexia), rash, chills, headache, joint pain (arthralgia), cough, fatigue, nausea, vomiting, diarrhea, myalgia (muscle pain), dry skin, decreased appetite, edema, hemorrhage, high blood pressure (hypertension) and difficulty breathing (dyspnea). Severe side effects of Tafinlar include the development of new cancers, growth of tumors in patients with BRAF wild-type tumors, serious bleeding problems, heart problems, severe eye problems, fever that may be severe, serious skin reactions, high blood sugar or worsening diabetes, and serious anemia. Severe side effects of Mekinist include the development of new cancers; serious bleeding problems; inflammation of intestines and perforation of the intestines; blood clots in the arms, legs or lungs; heart problems; severe eye problems; lung or breathing problems; fever that may be severe; serious skin reactions; and high blood sugar or worsening diabetes. Both Tafinlar and Mekinist can cause harm to a developing fetus; women should be advised of the potential risk to the fetus and to use effective contraception.

On June 27, 2018, the FDA approved encorafenib and encorafenib (Braftovi) in combination with binimetinib (Mektovi) for patients with unresectable or metastatic melanoma with a BRAF V600E or V600K mutation, as detected by an FDA-approved test. The recommended doses are binimetinib 45 mg orally twice daily and encorafenib 450 mg orally once daily. Furthermore, the FDA also granted approval of the THxID BRAF Kit (bioMérieux) as a companion diagnostic for this therapy.

Approval for combination therapy with Braftovi and Mektovi was based on a randomized, active-controlled, open-label, multicenter trial (COLUMBUS; NCT01909453) in 577 patients with BRAF V600E or V600K mutation-positive unresectable or metastatic melanoma. Patients were randomized (1:1:1) to receive binimetinib 45 mg twice daily plus encorafenib 450 mg once daily, encorafenib 300 mg once daily, or vemurafenib 960 mg twice daily. Treatment continued until disease progression or unacceptable toxicity. The major efficacy measure was progression-free survival (PFS) using RECIST 1.1 response criteria and assessed by blinded independent central review. The median PFS was 14.9 months for patients receiving binimetinib plus encorafenib, and 7.3 months for the vemurafenib monotherapy arm (hazard ratio 0.54, 95% CI: 0.41, 0.71, p<0.0001). Overall response rates assessed by central review were 63% and 40%, respectively. Median response duration was 16.6 months vs. 12.3 months, respectively. The most common (≥25%) adverse reactions in patients receiving the combination were fatigue, nausea, diarrhea, vomiting, abdominal pain, and arthralgia. Discontinuation of therapy due to adverse reactions occurred in 5% of patients receiving the combination; the most common reasons were hemorrhage and headache.

IL28B Polymorphism Genotyping for Interferon Therapy for Hepatitis C

The IL28B gene is involved with viral resistance and is up-regulated by interferons.  IL28B polymorphisms appear to be a strong independent predictor of viral responsiveness.  Thus, it is likely that testing for IL28B polymorphisms will be included in making treatment decisions and potentially guiding therapy.  Currently, there are no studies evaluating the impact of ILB28B on treatment decisions.  Well-designed studies are needed to clarify the role, if any, of IL28B polymorphism genotyping for interferon therapy for hepatitis C.

CYP2D6 Polymorphism and Alzheimer's Disease

In a multi-center, prospective cohort study, Pilotto et al (2009) evaluated the influence of the single nucleotide polymorphism rs1080985 in the cytochrome P450 2D6 (CYP2D6) gene on the efficacy of donepezil in patients with mild to moderate Alzheimer's disease (AD).  A total of 127 white patients with AD according to the National Institute of Neurological and Communicative Disorders and Stroke-Alzheimer's Disease and Related Disorders Association Work Group criteria were included in this study.  Patients were treated with donepezil 5 to 10 mg/daily for 6 months.  Cognitive and functional statuses were evaluated at baseline and at 6-month follow-up.  Response to therapy was defined according to the National Institute for Health and Clinical Excellence criteria.  Compliance and drug-related adverse events were also evaluated.  The analyses identifying the CYP2D6 and APOE polymorphisms were performed in blinded fashion.  At 6-month follow-up, 69 of 115 patients (60 %) were responders and 46 patients (40 %) were non-responders to donepezil treatment.  A significantly higher frequency of patients with the G allele of rs1080985 was found in non-responders than in responders (58.7 % versus 34.8 %, p = 0.013).  Logistic regression analysis adjusted for age, sex, Mini-Mental State Examination score at baseline, and APOE demonstrated that patients with the G allele had a significantly higher risk of poor response to donepezil treatment (odds ratio 3.431, 95 % CI: 1.490 to 7.901).  The authors concluded that the single nucleotide polymorphism rs1080985 in the CYP2D6 gene may influence the clinical efficacy of donepezil in patients with mild to moderate AD.  The analysis of CYP2D6 genotypes may be useful in identifying subgroups of patients with AD who have different clinical responses to donepezil.

In a multi-center prospective cohort study, Seripa et al (2011) evaluated the effect of 16 functional polymorphisms in the CYP2D6 gene on the clinical response to donepezil treatment in patients with mild-to-moderate AD.  These researchers evaluated 57 unrelated Caucasians clinically diagnosed as AD according to the National Institute of Neurological and Communicative Disorders and Stroke-Alzheimer's Disease and Related Disorders Association Work Group criteria.  Patients were treated with donepezil (5 to 10 mg/daily) for 6 months.  The response to donepezil treatment was evaluated at 6-month follow-up according to the National Institute for Health and Clinical Excellence requirements.  The identification of 16 clinically relevant CYP2D6 gene variants was performed by a high-throughput genetic analysis.  Thirty-eight of 57 patients (67 %) were responders and 19 patients (33 %) were non-responders to donepezil treatment.  A significantly higher frequency of gene variants conferring decreased or absent enzyme activity was observed in responder than in non-responder patients (73.68 % versus 36.84 %; p = 0.005).  The presence of gene variants conferring decreased or absent activity of the CYP2D6 enzyme was significantly associated with a clinical response to donepezil treatment (odds ratio = 6.286; 95 % CI: 1.828 to 21.667).  The authors concluded that functional polymorphisms in the CYP2D6 gene can influence the clinical efficacy of donepezil.  The analysis of CYP2D6 genotypes may be useful in identifying subgroups of AD patients with different clinical response to donepezil treatment.

The Vysis ALK Break Apart FISH Probe Kit

Crizotinib (Xalkori) is a kinase inhibitor which has recently received accelerated FDA approval for use in locally advanced or metastatic NSCLC that is ALK-positive as detected by an FDA-approved test.  The FDA approved the Vysis ALK Break Apart FISH Probe Kit concurrently with crizotinib as a companion diagnostic test designed to detect rearrangements of the ALK gene in NSCLC.  FISH refers to fluorescence in-situ hybridization, which has numerous uses, including identify whether too many, or too few, copies of a particular gene are present in the body’s cells or whether certain genes have rearrangements that play an active role in disease progression (Abbott Press Release, 2011).

Oncogenic fusion genes consisting of EML4 and anaplastic lymphoma kinase (ALK) are present in a subgroup of non-small-cell lung cancers, representing 2 to 7 % of such tumors (Kwak et al, 2010).  Rodig et al (2010) noted that “crizotinib has been particularly effective against anaplastic large cell lymphoma and non-small cell lung cancer (NSCLC) cell lines that harbor ALK translocations resulting in expression of oncogenic ALK fusion proteins”.

Kwak et al (2011) screened tumor samples from approximately 1500 NSCLC patients for the presence of ALK rearrangements and found 82 patients with advanced ALK-positive disease in their study population.  These patients were subsequently enrolled in a clinical trial of crizotinib therapy.  At a mean treatment duration of 6.4 months, the overall response rate was 57 %, with an estimated probability of 6-month progression free survival was estimated at study completion to be 72 %.  The investigators also noted patients with ALK rearrangements tended to be younger than those without, and most of these patients were light or non-smokers.

The 2010 American Society of Clinical Oncology Annual Meeting included a presentation on crizotinib for the treatment of advanced NSCLC.  Stinchcombe et al (2011) summarized the evidence presented and stated that the results of crizotinib in molecularly selected patients with advanced NSCLC whose tumor cells had a novel fusion protein involving ALK reinforces the importance of understanding molecular heterogeneity in this patient population.

EpiSwitch CiRT (Checkpoint-Inhibitor Response Test)

The EpiSwitch CiRT (Checkpoint-inhibitor Response Test ) (Oxford BioDynamics, PLC) is a blood test that is used to predict a patient's therapeutic response to checkpoint inhibitor immunotherapy. The test consists of genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative polymerase chain reaction (qPCR), from whole blood, and is reported as a high or low probability of responding to immune checkpoint–inhibitor therapy.

GeneSightRx Testing

GeneSightRx, developed by AssureRx Health (Mason, OH), is a technology that measures and analyzes genomic variants affecting the metabolism and response to health medications in patients.  These genomic tests were developed to serve as a clinical treatment support tool, supposedly providing objective genetic-based patient information in advance of making a medication decision.  Knowing a patient's genetic profile may aid in understanding which medications the patient would metabolize properly and help inform treatment choices unique to each patient.  GeneSightRx Psychotropic analyzes 6 genes that may affect a patient’s response to anti-depressant and anti-psychotic medications.  Four pharmacokinetic genes from the cytochrome P450 family and 2 pharmacodynamic genes related specifically to the serotonin system are genotyped.

Hall-Flavin et al (2013) reported on an open-label study to evaluate the potential benefit of GeneSight for the management of psychotropic medications used to treat major depression in an outpatient psychiatric practice. The open-label study was divided into two groups. In the first (unguided) group (n = 113), pharmacogenomic information was not shared until all participants completed the study. In the second (guided) group (n = 114), the pharmacogenomic report was provided to physicians for clinical use. Three depression ratings, the 17-item Hamilton Rating Scale for Depression (HAMD-17), the Quick Inventory of Depressive Symptomatology - Clinician Rated (QIDS-C16), and the Patient Health Questionnaire (PHQ-9), were collected at baseline, and at 2, 4, and 8 weeks. The guided group experienced greater percent improvement in depression scores from baseline on all three depression instruments (HAMD-17, P < 0.0001; QIDS-C16, P < 0.0001; PHQ-9, P < 0.0001) compared with the unguided group. Eight-week response rates were higher in the guided group than in the unguided group on all three measurements (HAMD-17, P = 0.03; QIDS-C16, P = 0.005; PHQ-9, P = 0.01). Eight-week QIDS-C16 remission rates were higher in the guided group (P = 0.03). Participants in the unguided group who at baseline were prescribed a medication that was most discordant with their genotype experienced the least improvement compared with other unguided participants (HAMD-17, P = 0.007). Participants in the guided group and on a baseline medication most discordant with their genotype showed the greatest improvement compared with the unguided cohort participants (HAMD-17, P = 0.01).

Winner et al (2013) reported on a prospective double-blind randomized control trial (RCT) to evaluate the benefit of a combinatorial, five gene pharmacogenomic test and interpretive report (GeneSight) for the management of psychotropic medications used in the treatment of major depression in an outpatient psychiatric practice. Depressed adult outpatients were randomized to a treatment as usual (TAU, n = 25) arm or a pharmacogenomic-informed GeneSight (n = 26) arm. Subjects were blinded to their treatment group and depression severity was assessed by blinded study raters. Within two days of enrollment, clinicians of subjects in the guided group received the GeneSight report that categorized each of 26 psychotropic medications within a green, yellow, or red "bin" based on the relationship of each medication to a subject's pharmacokinetic and pharmacodynamic combinatorial gene variant profile. Antidepressant medication changes began within 2 weeks after baseline assessments. Depression severity was assessed by blinded study raters using the HAMD-17, PHQ-9, QIDS-SR, and QIDS-CR administered 4, 6, and 10 weeks after baseline assessment. Between-group nonsignificant trends were observed with greater than double the likelihood of response and remission in the GeneSight group measured by HAMD-17 at week 10. Mean percent improvement in depressive symptoms on HAMD-17 was higher for the GeneSight group over TAU (30.8% vs 20.7%; p = 0.28), although the difference was not statistically significant. TAU subjects who had been prescribed medications at baseline that were contraindicated based on the individual subject's genotype (i.e., red bin) had less improvement (0.8%) in depressive symptoms measured by HAMD-17 at week 10 than the pharmacogenomics guided subjects who started on a red bin medication (33.1%, p = 0.06) and the improvement in GeneSight subjects overall (26.4%, p=0.08), although differences were not statistically significant.

Winner et al (2013) reported on a 1 year blinded and retrospective study that evaluated eight direct or indirect health care utilization measures for 96 patients with a DSM-IV-TR diagnosis of depressive or anxiety disorder. The eight measures were evaluated in relation Genesight, an interpretive pharmacogenomic test and reporting system, designed to predict antidepressant responses based on DNA variations in cytochrome P450 genes (CYP2D6, CYP2C19, CYP2C9 and CYP1A2), the serotonin transporter gene (SLC6A4) and the serotonin 2A receptor gene (5HTR2A). All subjects had been prescribed at least one of 26 commonly prescribed antidepressant or antipsychotic medications.  Subjects whose medication regimen included a medication identified by the gene-based interpretive report as most problematic for that patient and are in the 'red bin' (medication status of 'use with caution and frequent monitoring'), had 69% more total health care visits, 67% more general medical visits, greater than 3-fold more medical absence days, and greater than four-fold more disability claims than subjects taking drugs categorized by the report as in the green bin ('use as directed') or yellow bin ('use with caution'). There were no correlations between the number of medications taken and any of the eight healthcare utilization measures.

Kirchheiner et al (2010) stated that more than 50 years of pharmacogenetic research have produced many examples of the impact of inherited variability in the response to psychotropic drugs.  These successes, however, have as yet failed to translate into broadly applicable strategies for the improvement of individual drug treatment in psychiatry.  One important argument against the widespread adoption of pharmacogenetics as a clinical tool is the lack of evidence showing its impact on medical decision making and on risk benefit ratio for the patients.  The individual drug metabolizing capacity is assessed by genotyping drug metabolizing enzymes.  The potential implications of information gained from genotyping are dose adjustments according to genotype.  However, even when the consequences of genotype on pharmacokinetics are significant and well known, as in the case of many tricyclic anti-depressants and several selective serotonin reuptake inhibitors (SSRIs), there is still considerable controversy on whether adjustment of dosage driven by genetic information may improve therapeutic efficacy, and/or adverse events is prevented, to an extent of any practical importance in clinical practice.  Different types of pharmacogenetic studies may improve the understanding of the functional consequence of a genetic variant in the clinical setting.  The use of intermediate phenotypes instead of broad outcome parameters such as drug response or remission might improve the knowledge on what exactly happens if an individual with a specific genotype takes a certain drug.

Fleeman et al (2010) examined if testing for cytochrome P450 (CYP) polymorphisms in adults entering anti-psychotic treatment for schizophrenia leads to improvement in outcomes, is useful in medical, personal or public health decision-making, and is a cost-effective use of health-care resources.  The following electronic databases were searched for relevant published literature: Cochrane Controlled Trials Register, Cochrane Database of Systematic Reviews, Database of Abstracts of Reviews of Effectiveness, EMBASE, Health Technology Assessment database, ISI Web of Knowledge, MEDLINE, PsycINFO, NHS Economic Evaluation Database, Health Economic Evaluation Database, Cost-effectiveness Analysis (CEA) Registry and the Centre for Health Economics website.  In addition, publicly available information on various genotyping tests was sought from the internet and advisory panel members.  A systematic review of analytical validity, clinical validity and clinical utility of CYP testing was undertaken.  Data were extracted into structured tables and narratively discussed, and meta-analysis was undertaken when possible.  A review of economic evaluations of CYP testing in psychiatry and a review of economic models related to schizophrenia were also carried out.  For analytical validity, 46 studies of a range of different genotyping tests for 11 different CYP polymorphisms (most commonly CYP2D6) were included.  Sensitivity and specificity were high (99 to 100 %).  For clinical validity, 51 studies were found.  In patients tested for CYP2D6, an association between genotype and tardive dyskinesia (including Abnormal Involuntary Movement Scale scores) was found.  The only other significant finding linked the CYP2D6 genotype to parkinsonism.  One small unpublished study met the inclusion criteria for clinical utility.  One economic evaluation assessing the costs and benefits of CYP testing for prescribing anti-depressants and 28 economic models of schizophrenia were identified; none was suitable for developing a model to examine the cost-effectiveness of CYP testing.  The authors concluded that tests for determining genotypes appear to be accurate although not all aspects of analytical validity were reported.  Given the absence of convincing evidence from clinical validity studies, the lack of clinical utility and economic studies, and the unsuitability of published schizophrenia models, no model was developed; instead key features and data requirements for economic modeling were presented.  Recommendations for future research cover both aspects of research quality and data that will be required to inform the development of future economic models.

Lohoff and Ferraro (2010) noted that despite continued efforts to optimize pharmacological treatment for individuals with psychiatric disorders, efficacy and tolerability of medication remains highly variable.  In addition to clinical heterogeneity, diagnostic uncertainty, environmental, social factors, and genetic factors have been identified as playing an important role in the inter-individual differences in therapeutic and toxic drug effects.  This article described recent developments in the field of psychiatric pharmacogenetics and focused on anti-depressant, anti-psychotic, anti-convulsant, and mood-stabilizing drugs.  Recent findings from pharmacogenetic clinical trials were reviewed and clinical implications were discussed.  The authors concluded that although current data are too sparse to allow the development of guidelines for using pharmacogenetic testing in routine clinical practice, the field of psychiatric pharmacogenetics is rapidly developing and identification of genetic biomarkers that predict therapeutic response and risk of side effects will ultimately help the practitioner to choose effective and safe treatment for patients suffering from psychiatric disorders.

Plesnicar (2010) stated that anti-psychotics are the lodestar in the treatment of schizophrenia despite the variability of the therapeutic response and drug-induced adverse effects (especially extrapyramidal symptoms, gain weight, and metabolic disturbances).  More and more data are supporting the notion that genetic factors -- as well as often over-looked personal and environmental factors -- that define the inter-individual differences in pharmacokinetic and pharmacodynamic treatment response.  At present, there are no practical pharmakogenetic tests that could be used in everyday clinical practice; however, in the field of psychiatry they are expected within a few years.  Pharmacogenetic tests will indubitably become an important tool for personalized prescription.

Gelenberg (2010) noted that  variety of American and European guidelines are available for clinicians treating major depressive disorder and depressive subtypes.  Major Western guidelines published since 2000 made similar recommendations for all stages of treatment for depression, including a reliance on measurement-based care.  First-line treatment is usually a SSRI, psychotherapy, or a combination of pharmacotherapy and psychotherapy.  Next-step treatment recommendations are switching or augmentation, depending on patient response to the initial treatment.  Maintenance therapy continues the approach that led to remission.  The American Psychiatric Association will release a new treatment guideline to offer information on developments made since the last guidelines were published in 2000.  Despite progress made during the last decade, no major breakthroughs in the treatment of depression have occurred, and genetic testing developments allowing for personalized care remain the goal of research.

Lewis et al (2011) tested the hypothesis that patients homozygous for the long (insertion) polymorphism of the serotonin transporter (5-HTTLPR) have an increased response to SSRI antidepressants but not to noradrenaline reuptake inhibitors (NARIs) antidepressants.  In an individually randomized, parallel-group controlled trial, people meeting criteria for a depressive episode who were referred by their general practitioner were randomized to receive either citalopram (an SSRI) or reboxetine (an NARI).  Randomization was by means of a remote automated system accessed by telephone.  The main outcome was depressive symptoms, measured by Beck Depression Inventory (BDI) total score 6 weeks after randomization.  A total of 298 participants were randomized to receive citalopram and 303 were randomized to reboxetine.  At 6 weeks follow-up, complete data were available for 258 participants taking citalopram and 262 taking reboxetine.  These investigators found no evidence to support an influence of 5-HTTLPR on outcome following antidepressant treatment.  The interaction term for BDI score at 6 weeks was 0.50 (95 % CI: -2.04 to 3.03, p = 0.70), which indicated that responses to the SSRI and NARI were similar irrespective of 5-HTTLPR genotype.  The authors concluded that it is unlikely that the 5-HTTLPR polymorphism alone will be clinically useful in predicting response to antidepressants in people with depression.

In a review and meta-analysis, Biernacka et al (2012) evaluated the evidence for association between the serotonin transporter gene promoter polymorphism (5HTTLPR) and antidepressant induced mania (AIM).  Medline up to November 2009 was searched for key words bipolar, antidepressant, serotonin transporter, SLC6A4, switch, and mania.  A total of 5 studies have evaluated the SLC6A4 promoter polymorphism and AIM in adults (total n = 340 AIM+ cases, n = 543 AIM- controls).  Although a random effects meta-analysis showed weak evidence of association of the S allele with AIM+ status, a test of heterogeneity indicated significant differences in estimated genetic effects between studies.  A similar weak association was observed in a meta-analysis based on a subset of 3 studies that excluded patients on mood stabilizers; however the result was again not statistically significant.  The authors concluded that there is insufficient published data to confirm an association between 5HTTLPR and antidepressant induced mania.  Pharmacogenomic studies of antidepressant induced mania have high potential clinical impact provided future studies are of adequate sample size and include rigorously assessed patient characteristics (e.g., ancestry, rapid cycling, concurrent mood stabilization, and length of antidepressant exposure).

Vetti et al (2010) systematically categorized the experience from routine CYP2D6 genotyping in a diagnostic laboratory.  All samples submitted to the authors' laboratory for CYP2D6 genotyping in the period of June 29, 1998 to December 28, 2009 were examined retrospectively.  The samples were classified into 3 indication groups based on clinical information given in the request form.  All samples, and a control group consisting of 100 healthy blood donors, were tested for the 4 most prevalent non-functional CYP2D6 alleles in the European population, and for ultra-rapid metabolizer-associated duplications of the gene.  A total of 325 samples were included.  The proportion of ultra-rapid metabolizers was significantly higher in the patient group (4.0 %, p = 0.045) than in the control group (0 %), with the highest proportion among those patients that used a known CYP2D6 substrate.  The percentage of poor metabolizers was not significantly higher in the patient group (8.3 %) than in the control group (6.0 %) (p = 0.528).  The authors concluded that the CYP2D6 analysis could rarely explain the patients' side effects or lack of drug response, even though the study group was selected because of clinical problems due to drugs they were using.  Two explanations may be that the indication(s) for genetic testing is not clearly defined and that the CYP2D6 genotype is only one of many factors that determine individual drug response.

Jurgens et al (2012) examined the clinical impact of CYP2D6 genotype in patients with a diagnosis within the schizophrenic spectrum using medication pattern as proxy for therapeutic and side effect.  The study was conducted in patients genotyped during an inpatient stay (n = 576).  Continuous antipsychotic, adjuvant, and anticholinergic drug regimens were registered retrospectively in a cross-sectional manner before genotyping.  Antipsychotics were divided into CYP2D6 dependent and independent, and dose equivalents were calculated as chlorpromazine equivalents (CPZEq).  Poor metabolizers and ultra-rapid metabolizers were treated with significantly higher median CPZEq doses (625.8; inter-quartile range [IQR] 460.4 to 926.7; and 550; IQR, 199.8 to 1,049) than extensive metabolizers (EMs) and intermediate metabolizers (IMs) (384; IQR, 150 to 698; and 446; IQR, 150 to 800) (p = 0.018).  Logistic regression showed no association between anticholinergic treatment and CYP2D6 genotype or concomitant treatment with CYP2D6 inhibitors (p = 0.79 and p = 0.46, respectively).  The authors concluded that these findings indicated that CYP2D6 genotype has no sufficient clinical impact that poor metabolizers and ultra-rapid metabolizers are easily clinically identified with.

Altar et al (2015) collected DNA of 258 patients with treatment-resistant depression in three 8-10 week, two-arm, prospective clinical trials. Forty-four allelic variations were measured in genes for the cytochrome P450 (CYP) enzymes CYP2D6, CYPC19, and CYP1A2, the serotonin transporter (SLC6A4), and the 5-HT2A receptor (HTR2A). The combinatorial pharmacogenomic (CPGx™) GeneSight test results were provided to clinicians to support medication changes from baseline (guided arm), or they were provided at the end of each study to clinicians of unguided patients who were treated as usual (TAU). TAU subjects who at baseline were prescribed medications genetically discordant for them showed only a 12% symptom improvement, far less than the 32.5% or 28.5% improvements of the TAU subjects on yellow-category ('use with caution'; p = 0.002) or green-category medications ('use as recommended'; p = 0.02), respectively. The odds of a clinical response were increased 2.3-fold among all GeneSight-guided compared to all TAU subjects (p = 0.004), and overall, the guided group had a 53% greater improvement in depressive symptoms (p = 0.0002), a 1.7-fold relative improvement in response (p = 0.01), and a number needed to treat for one clinical response above that seen in the TAU group of 6.07. However, lack of blinding in two of the three studies limits the validity of the meta-analysis.Greden et al (2019) reported on the outcomes of the GUIDED study, where outpatients (N = 1167) diagnosed with MDD and with a patient- or clinician-reported inadequate response to at least one antidepressant were enrolled in the Genomics Used to Improve DEpression Decisions (GUIDED) trial – a rater- and patient-blind randomized controlled trial. Patients were randomized to treatment as usual (TAU) or a pharmacogenomics-guided intervention arm in which clinicians had access to a pharmacogenomic test report to inform medication selections (guided-care). Medications were considered congruent (‘use as directed’ or ‘use with caution’ test categories) or incongruent (‘use with increased caution and with more frequent monitoring’ test category) with test results. Un-blinding occurred after week 8. Primary outcome was symptom improvement [change in 17-item Hamilton Depression Rating Scale (HAM-D17)] at week 8; secondary outcomes were response (≥50% decrease in HAM-D17) and remission (HAM-D17 ≤ 7) at week 8. At week 8, symptom improvement for guided-care was not significantly different than TAU (27.2% versus 24.4%, p = 0.107); however, improvements in response (26.0% versus 19.9%, p = 0.013) and remission (15.3% versus 10.1%, p = 0.007) were statistically significant. Patients taking incongruent medications prior to baseline who switched to congruent medications by week 8 experienced greater symptom improvement (33.5% versus 21.1%, p = 0.002), response (28.5% versus 16.7%, p = 0.036), and remission (21.5% versus 8.5%, p = 0.007) compared to those remaining incongruent. The investigators concluded that pharmacogenomic testing did not significantly improve mean symptoms but did significantly improve response and remission rates for difficult-to-treat depression patients over standard of care.The GUIDED study failed to achieve statistical significance in the prespecified primary outcome, change in 17-item Hamilton Depression Rating Scale (HAM-D17)] at week 8. Some secondary endpoints achieved statistical significance, specifically response (≥50% decrease in HAM-D17) and remission (HAM-D17 ≤ 7) rates at week 8. However, the study had 25 secondary outcome measures and 3 “other” outcome measures. Although it is possible to design a trial to have more than one primary endpoint, those primary endpoints would need to be prespecified, and the statistical analysis adjusted to account for multiple primary endpoints (multiplicity). In a post hoc reanalysis of data, statistical significance in the primary HAM-D17 endpoint was achieved if the analysis only included participants who were taking medications subject to gene-drug interactions at baseline. Although it is possible to design a study that focused the analysis on these subjects who were taking medications subject to gene-drug interactions,  this would have to be prespecified in order to avoid the potential bias from after-the-fact data reanalysis.There are other concerns about the GUIDED study. The study compared GeneSight guided therapy to “treatment as usual.” It is not known whether similar improvements in outcomes could be achieved by providing the clinicians with information about known drug-drug interactions and side-effect profile of the various antidepressants, as well as other information relevant to selection of antidepressants (minus the pharmacogenetic test results). This is especially relevant given that the study was not limited to psychiatrists, but included primary care physicians. Another issue relates to the magnitude of the benefit from testing; a letter to the editor questioned the clinical significance of the reported improvements in the secondary outcomes.

Khani et al (2024) stated that pharmacogenomics could optimize anti-psychotic treatment by preventing adverse drug reactions, improving treatment effectiveness or relieving the cost burden on the healthcare system.  In a systematic review, these investigators examined if pharmacogenetic testing in individuals undergoing anti-psychotic treatment would influence clinical or economic outcomes.  On January 12, 2024, these investigators searched Medline, Embase, PsycINFO and Cochrane Centrale Register of Controlled Trials.  The findings were summarized using a narrative approach and summary tables.  A total of 13 studies were eligible for inclusion in the systematic review.  The current evidence base is either in favor of pharmacogenetics-guided prescribing or showed no difference between pharmacogenetics and treatment as usual for clinical and economic outcomes.  The authors concluded that RCTs with sufficient sample sizes are needed to provide recommendations for patients who receive anti-psychotics based on a broad, multi-gene panel, with consistent and comparable clinical outcomes.

The authors stated that this study had several drawbacks.  First, the scope of this review was wide due to the scarcity of the data.  This meant that there was heterogeneity among the studies due to differences in study design (RCTs and non-RCTs with multiple different comparators) and outcomes measured, especially for clinical outcomes that were evaluated using many different clinical scales.  Second, the search retrieved very few studies from outside of Europe and North America, indicating limited clinical generalizability of the findings; thus, highlighting an important gap in the literature that should be addressed in future research.  This is important because the prevalence of schizophrenia is high in East and South Asia, with a patient population of approximately 7.2 and 4.0 million, respectively.  Third, compared to Caucasian cohorts, these populations have different frequencies of variants for CYP450 enzymes.  For example, while CYP2D6*10 is the most abundant allele in East Asian populations (minor allele frequency 58.7 %), this allele is considerably less common in Europeans (minor allele frequency 0.2 %).  Fourth, not all anti-psychotics have pharmacogenetic recommendations, which would further reduce the ability to detect differences.

GeneCept

Brennan et al (2015) examined the effectiveness of genetic testing in a real-world setting and to assess its impact on clinician treatment decisions. This was a naturalistic, unblinded, prospective analysis of psychiatric patients and clinicians who utilized a commercially available genetic test (between April and October of 2013), which incorporates 10 genes related to pharmacokinetics and pharmacodynamics of psychiatric medications. Each patient's genetic results were provided to participating clinicians, who completed a baseline survey including patient medications, history, and severity of illness. Clinicians were prompted to complete surveys within 1 week of receiving the genetic results and again 3 months later. Patients likewise completed assessments of depression, anxiety, medication side effects, and quality of life at baseline, 1 month, and 3 months. Data from 685 patients were collected. Approximately 70% and 29% of patients had primary diagnoses of either a mood or anxiety disorder, respectively. Clinician-reported data, as measured by the Clinical Global Impressions-Improvement scale, indicated that 87% of patients showed clinically measurable improvement (rated as very much improved, much improved, or minimally improved), with 62% demonstrating clinically significant improvement. When analysis was restricted to the 69% of individuals with ≥ 2 prior treatment failures, 91% showed clinically measurable improvement. Patients also reported significant decreases in depression (P < .001), anxiety (P < .001), and medication side effects (P < .001) and increases in quality of life (P < .001). The authors concluded that these results suggest that a substantial proportion of individuals receiving pharmacogenetic testing showed clinically significant improvements on multiple measures of symptoms, adverse effects, and quality of life over 3 months. In the absence of randomization, blinding, and a control group, the study does not provide a rigorous assessment of the GeneCept assay’s ability to improve symptom outcomes.

Perlis et al (2018) stated that naturalistic and small randomized trials have suggested that pharmacogenetic testing may improve treatment outcomes in depression; however, its cost-effectiveness is not known.  There is growing enthusiasm for personalized medicine, relying on genetic variation as a contributor to heterogeneity of treatment effects.  In a case-control study, these researchers examined the relationship between a commercial pharmacogenetic test (the GeneCept Assay) for psychotropic medications and 6-month cost of care and utilization in a large commercial health plan.  They carried out a propensity-score matched analysis of longitudinal health claims data from a large U.S. insurer.  Individuals with a mood or anxiety disorder diagnosis (n = 817) who received genetic testing for pharmacokinetic and pharmacodynamic variation were matched to 2,745 individuals who did not receive such testing.  Outcomes included number of outpatient visits, inpatient hospitalizations, emergency room (ER) visits, and prescriptions, as well as associated costs over 6 months.  On average, individuals who underwent testing experienced 40 % fewer all-cause ER visits (mean difference [MD] of 0.13 visits; p < 0.0001) and 58 % fewer inpatient all-cause hospitalizations (MD of 0.10 visits; p < 0.0001) than individuals in the control group.  The 2 groups did not differ significantly in number of psychotropic medications prescribed or mood-disorder related hospitalizations.  Overall, 6-month costs were estimated to be $1,948 (SE 611) lower in the tested group.  The authors concluded that pharmacogenetic testing represents a promising strategy to reduce costs and utilization among patients with mood and anxiety disorders.  Moreover, these researchers stated that additional study will be useful in further defining the optimal application of such testing in clinical practice, and the impact of testing on quality as well as quantity of care.

Virelli et al (2021) stated that pharmacogenomics (PGx) is the study of genetic influences on an individual's response to medications.  Improvements in the quality and quantity of PGx research over the past 20 years have enabled the establishment of commercial markets for PGx tests.  Nevertheless, PGx testing has yet to be adopted as a routine practice in clinical care.  Accordingly, policy regulating the commercialization and reimbursement of PGx testing is in its infancy.  Several studies have been published on the topic of challenges, or “barriers” to clinical adoption of this healthcare innovation.  However, many do not include recent evidence from randomized controlled trials (RCTs), economic utility studies, and qualitative assessments of stakeholder opinions.  The authors revisited the most cited barriers to adoption of PGx testing: evidence for clinical utility, evidence for economic effectiveness, and stakeholder awareness.  They considered these barriers in the context of reviewing PGx literature published over the past 20 years and emphasized data from commercial PGx testing companies, since they have published the largest datasets.  The authors concluded that there appeared to be great promise in the application of PGx testing in psychiatric care.  Progress in addressing barriers to clinical adoption, identified in this review, should aid in actualizing the benefits of this innovation and advancing the potential of personalized medicine to improve healthcare systems worldwide.

Bousman et al (2021) noted that the implementation of PGx testing in psychiatry remains modest, in part due to divergent perceptions of the quality and completeness of the evidence base and diverse perspectives on the clinical utility of PGx testing among psychiatrists and other healthcare providers.  Recognizing the current lack of consensus within the field, the International Society of Psychiatric Genetics (ISPG) assembled a group of experts to conduct a narrative synthesis of the PGx literature, prescribing guidelines, and product labels related to psychotropic medications as well as the key considerations and limitations related to the use of PGx testing in psychiatry.  The group concluded that to inform medication selection and dosing of several commonly used anti-depressant and anti-psychotic medications, current published evidence, prescribing guidelines, and product labels support the use of PGx testing for 2 cytochrome P450 genes (CYP2D6, CYP2C19).  Furthermore, the evidence supports testing for human leukocyte antigen (HLA) genes when using the mood stabilizers carbamazepine (HLA-A and HLA-B), oxcarbazepine (HLA-B), and phenytoin (CYP2C9, HLA-B).  For valproate, screening for variants in certain genes (POLG, OTC, CSP1) is recommended when a mitochondrial disorder or a urea cycle disorder is suspected.  The authors concluded that although barriers to implementing PGx testing remain to be fully resolved, the current trajectory of discovery and innovation in the field suggested these barriers will be overcome and testing will become an important tool in psychiatry.

Moreover, these researchers stated that PGx testing is currently not regulated, and many of the available tests include genes that have little to no support for clinical implementation.  Recommendations produced by these tests could lead to inappropriate medication selection and dosing decisions.  Various resources to assist in the interpretation and implementation of test results exist, but these resources do not supplant clinical judgement.  They noted that a number of larger PGx studies, such as the Ubiquitous Pharmacogenomics Project in Europe and the Precision Medicine in Mental Health Care Study in the U.S. are underway.  These investigators expected that with the completion of these studies and others that the PGx evidence will continue to evolve, barriers to testing will be cleared, and the uptake of genome sequencing and population-level precision medicine initiatives will increase.

Kumar and Kearney (2021) stated that psychiatric PGx testing is commonly used by providers in primary care and mental health settings.  These investigators described the extent to which psychiatric PGx testing supports clinical practice.  HLA-A and HLA-B should be tested before initiating carbamazepine and oxcarbazepine due to risk of serious skin reactions.  For psychotropic medications metabolized through the liver, limited evidence suggested testing for variation in metabolism through CYP2D6 and CYP2D19.  For specific medication and genotype-phenotype variations, guidance through the Clinical Pharmacogenetics Implementation Consortium (CPIC) or the ISPG should be reviewed.  Commercial tests interpret this information differently and should not be used for broad guidance.  Clinicians should follow current guidelines from professional bodies such as CPIC or ISPG and test for HLA-A or HLA-B before initiating carbamazepine or oxcarbazepine.  Evidence is limited for psychiatric pharmacogenetic testing.

de Lara et al (2021) stated that PGx testing is available to healthcare professionals to guide drug selection and prevent adverse events (AEs); however, its implementation in the clinical practice of psychiatry/neurology still has barriers, mainly due to a lack of evidence.  These investigators carried out a literature search on Cochrane Library, Embase and PubMed, from their inception to June 18, 2020.  They included 16 published systematic reviews.  The most studied drug categories were anticonvulsants and selective serotonin reuptake inhibitors (SSRIs) associated with human leukocyte antigen and cytochrome P450 genes (HLA-A, HLA-B, CYP2C9, CYP2D6, CYP2C19), classified as critically low quality/low quality.  The authors concluded that there is a need for more robust studies with adequate design to examine the potential benefits of adopting pharmacogenetics in health systems and services.

Vasiliu (2023) noted that examining the possible correlations between gene variations and the clinical effects of the new-generation anti-psychotics is considered essential in the framework of personalized medicine.  It is expected that pharmacogenetic data will be useful for increasing the treatment efficacy, tolerability, therapeutic adherence, functional recovery, and QOL in patients with severe psychiatric disorders (SPD).  In a scoping review, this investigator examined the available evidence regarding the pharmacokinetics, pharmacodynamics, and pharmacogenetics of 5 new-generation anti-psychotics, i.e., cariprazine, brexpiprazole, aripiprazole, lumateperone, and pimavanserin.  Based on the analysis of 25 primary and secondary sources and the review of these agents' summaries of product characteristics, aripiprazole benefited from the most relevant data about the impact of gene variability on its pharmacokinetics and pharmacodynamics, with significant consequences on this anti-psychotic's efficacy and tolerability.  The determination of the CYP2D6 metabolizer status is important when administering aripiprazole, either as monotherapy or associated with other pharmacological agents.  Allelic variability in genes encoding dopamine D2, D3, and serotonin, 5HT2A, 5HT2C receptors, COMT, BDNF, and dopamine transporter DAT1 was also associated with different AEs or variations in the clinical efficacy of aripiprazole.  Brexpiprazole also benefited from specific recommendations regarding the CYP2D6 metabolizer status and the risks of associating this anti-psychotic with strong/moderate CYP2D6 or CYP3A4 inhibitors.  U.S. Food and Drug Administration (FDA) and European Medicines Agency (EMA) recommendations regarding cariprazine referred to possible pharmacokinetic interactions with strong CYP3A4 inhibitors or inducers.  Pharmacogenetic data regarding cariprazine is sparse, and relevant information regarding gene-drug interactions for lumateperone and pimavanserin is yet lacking.  The authors concluded that more studies are needed to detect the influence of gene variations on the pharmacokinetics and pharmacodynamics of new-generation antipsychotics.  This type of research could increase the ability of clinicians to predict favorable responses to specific anti-psychotics and to improve the tolerability of the treatment regimen in patients with SPD.

Korchia et al (2023) stated that in clinical practice, pharmacogenetic variations may not explain all of the variance in drug metabolism, and it is important that clinicians consider sex, age, concomitant medications, renal/hepatic status, lifestyle factors such as smoking, and weight when making anti-psychotic prescribing decisions.  Furthermore, dosing and monitoring drug concentration may be more readily available and directly inform dose adjustments, complementing pharmacogenetic testing.  Nevertheless, studies of real-world utility of PGx testing indicated that certain subgroups, such as the most severely ill or those who are non-adherent to treatment, may have substantially greater odds of improvement when prescribing follows pharmacogenetics-based recommendations.  Regarding anti-psychotics, prescribing guidelines have been developed only for CYP2D62 and CYP3A4, and the evidence for testing other variants has not yet been deemed sufficient to make more general recommendations.  The authors concluded that increasing prescribers’ use of PGx testing in psychosis programs, especially when facing inadequate treatment response, has the potential to improve remission rates and reduce overall adverse effects associated with anti-psychotics in clinical practice.

IDgenetix / NeuroIDgenetix Test (Pharmacogenetic-Guided Treatment for Anxiety and Depression)

Olson et al (2017) stated pharmacogenetic testing holds promise as a personalized medicine tool by permitting individualization of pharmacotherapy in accordance with genes influencing therapeutic response, side effects, and adverse events. The authors evaluated the effect on outcomes for patients diagnosed with neuropsychiatric disorders of pharmacogenetics (PGx)-guided treatment compared to usual standard of care. This was a prospective, randomized study of 237 patients at an outpatient community-based psychiatric practice conducted between April 2015 and October 2015. Baseline patient assessments and a buccal swab were collected for pharmacogenetic testing at study initiation. For the experimental group, PGx results were provided to the clinicians as guides to treatment. Control subjects were treated according to the usual standard of care with no clinician reference to their PGx results. Neuropsychiatric Questionnaire (NPQ) and Symbol Digit Coding Test (SDC) scores and adverse drug events, hospitalizations, and medication information were collected at 30, 60, and 90 days. More than half (53%) of patients in the control group reported at least 1 adverse drug event compared to 28% of patients with PGx-guided medication management (P = .001). NPQ and SDC scores improved for both groups, but no statistical difference in efficacy as measured by these assessments was observed within the 90-day observation period. The authors concluded that pharmacogenetic testing may facilitate psychiatric drug therapy with greater tolerability and similar efficacy compared to standard of care.

Sugarman et al (2016) stated among long-term care facility residents, polypharmacy is common, and often appropriate, given the need to treat multiple, complex, chronic conditions. Polypharmacy has, however, been associated with increased healthcare costs, adverse drug events, and drug interactions. The current study evaluates the potential medication cost savings of adding personalized pharmacogenetic information to traditional medication management strategies. One hundred and twelve long-term care residents completed pharmacogenetic testing for targeted variants in the following genes: CYP1A2, CYP2C9, CYP2C19, CYP2D6, CYP3A4/CYP3A5, HTR2A, HTR2C, SLC6A4, SLC6A2 COMT, OPRM1, SLCO1B1, VKORC1 and MTHFR. Following reporting of the IDgenetix Polypharmacy® test results, an internal medication management assessment was performed by a licensed clinical pharmacist to identify potential opportunities for regimen optimization through medication changes or discontinuations. The medication cost differences before and after the pharmacogenetic-guided review were assessed. Medication review following pharmacogenetic result reporting identified 54 patients (48.2%) with a total of 132 drug change recommendations (45 reductions; 87 replacements) and an average of 2.4 proposed medication changes (range 1-6) per patient. Medication cost savings related to the identified reduction and replacement opportunities exceeded the cost of testing and are estimated to be US$ 1300 (year 2016 cost) per patient annually assuming full implementation. The authors concluded that compared with traditional medication review, pharmacogenetic testing resulted in a 38% increase in the number of patients with current medication change opportunities and also offered valuable genetic information that could be referenced to personalize future prescribing decisions for all patients.

The NeuroIDgenetix test is a pharmacogenetic test employed to predict individual patient’s response to medications used in treatment of anxiety and depression.

In a prospective, randomized, subject- and rater-blinded study, Bradley et al (2018) examined the effect of pharmacogenetics-guided treatment (PGT) on patients diagnosed with depression and/or anxiety, in a diverse set of clinical settings, as compared to the standard of care (SOC).  This trial enrolled a total of 685 patients from clinical providers specializing in Psychiatry, Internal Medicine, Obstetrics & Gynecology, and Family Medicine.  The NeuroIDgenetix test uses a genetic variant panel of 10 genes, along with concomitant medications, to make medication management recommendations based on gene-drug and drug-drug interactions for over 40 medications used in the treatment of depression and anxiety.  Pharmacogenetic testing was carried out at the initial screening visit and baseline patient assessments were determined using the 17-item Hamilton Rating Scale for Depression (HAM-D17) and the Hamilton Rating Scale for Anxiety (HAM-A).  Following enrollment and randomization, pharmacogenetic results for subjects assigned to the experimental group were provided to physicians to guide treatment selection, while control subjects were treated according to the usual SOC.  HAM-D17 and HAM-A assessments were collected at 4 weeks, 8 weeks, and 12 weeks after baseline to examine the effectiveness of therapeutic selection.  In patients diagnosed with depression, response rates (p = 0.001; OR: 4.72 [1.93 to 11.52]) and remission rates (p = 0.02; OR: 3.54 [1.27 to 9.88]) were significantly higher in the PGT group as compared to the control group at 12 weeks.  Furthermore, patients in the PGT group diagnosed with anxiety showed a meaningful improvement in HAM-A scores at both 8 and 12 weeks (p = 0.02 and 0.02, respectively), along with higher response rates (p = 0.04; OR: 1.76 [1.03 to 2.99]).  The authors concluded that the findings of this study showed the clinical validity and utility of PGT for depression and anxiety.  These researchers stated that while medication selection based on pharmacogenetic testing continues to become closer to routine clinical practice, test implementation will result from pharmacogenetic algorithms that provide clinically validated information in an easily actionable format.  These investigators believed this study to be the 1st report of using pharmacogenetics to improve outcomes for patients diagnosed with anxiety.  While statistical significance was found in the percent reduction of HAM-A scores of the experimental group comprised of “anxiety or both depression/anxiety” subjects, “anxiety only” subjects appeared to benefit even more from pharmacogenetic-guidance.  While further investigation may be needed to explain this difference, this outcome may reflect a unique set of pharmacotherapy requirements for treating anxiety in co-morbid patients or it may represent a clinical prioritization of managing depression symptoms ahead of anxiety.

Maciel et al (2018) noted that the burden of depression significantly impacts the patient, the healthcare system, and society, at large.  Medication management guided by pharmacogenetics has been shown to increase therapeutic effectiveness and improve symptoms in patients diagnosed with depression; however, limited data are available on the cost-savings of PGT outside of psychiatric clinical specialties.  These researchers used published healthcare costs and clinical patient outcome data to model the economic impact of PGT for depression in a variety of clinical settings.  Assuming a test cost of USD$2,000 for pharmacogenetic testing, the model predicts a savings of USD$3,962 annually per patient with PGT.  The authors stated that this study is a 1st step in modeling the potential cost savings associated with the implementation of pharmacogenetics in patients with depression and/or anxiety from a societal perspective.  This study has potential limitations that are important to note and should be addressed in future studies.  First, this model utilizes data published in peer-reviewed journals to calculate annual cost-savings associated with pharmacogenetic testing.  As such, clinical efficacy and cost data are derived from 2 different sources.  While not ideal, the cost-savings modeled in this study are in line with previous studies, providing support to the findings that PGT of patients with depression and anxiety can produce significant cost-savings.  Second, this model does not incorporate quality of life (QOL) or willingness to pay thresholds in their calculations.  These researchers stated that future studies could develop cost-effectiveness models with varying time horizons and cost perspectives (payer, patient) to better approximate the costs and savings associated with pharmacogenetic testing.  Models such as Markov or Discrete Event Simulation would allow for sensitivity analyses to examine the robustness of the model results and provide more reliable data.

Roberts et al (2021) stated that there is limited evidence to support pharmacogenetic (PGx) testing in children.  These investigators carried out a retrospective review of PGx testing among 452 patients at an academic children's hospital to determine the potential use of PGx in diseases of childhood and to identify targets for future pediatric pharmacogenetic research.  An actionable gene-drug pair associated with the 28 genes tested (Clinical Pharmacogenetics Implementation Consortium (CPIC) level A or B, Pharmacogenomics Knowledge Base (PharmGKB) level 1A or B, or FDA recommendation and a PharmGKB level) was present in 98.7 % of patients.  These researchers identified 203 actionable gene-drug-diagnosis groups based on the indications for each actionable drug listed in Lexicomp.  Among patients with an actionable gene-drug-diagnosis group, 49.3 % had a diagnosis where the drug was a therapeutic option and PGx could be used to guide treatment selection.  Among patients with an associated diagnosis, 30.9 % had a prescription for the actionable drug allowing PGx guided dosing.  Three genes (CYP2C19, CYP2D6, and CYP3A5) accounted for all the gene-drug-diagnosis groups with matching diagnoses and prescriptions.  The most common gene-drug-diagnosis groups with matching diagnoses and prescriptions were CYP2C19-citalopram-escitalopram-depression 3.3 % of patients tested; CYP2C19-dexlansoprazole-gastritis-esophagitis 3.1 %; CYP2C19-omeprazole-gastritis-esophagitis 2.4 %; CYP2D6-atomoxetine-attention deficit hyperactivity disorder 2.2 %; and CYP2C19-citalopram-escitalopram-obsessive-compulsive disorder 1.5 %.  PGx could be used to guide selection of current therapeutic options or medication dosing in almost half (48.7 %) of pediatric patients tested.  The authors concluded that mood disorders and gastritis/esophagitis are promising targets for future study of PGx testing because of the high prevalence of these diagnoses and associated actionable gene-drug pairs in the pediatric population.  Moreover, these investigators stated that considerations for future work should include the development of targeted pediatric PGx panels for dissemination in primary care that eliminate genes without any evidenced‐based guidelines or recommendations for drug management (e.g., COMT, DRD2, GRIK4, HTR2A, HTR2C, IL28B, OPRM1, and SLC6A4), or are associated with drugs rarely used in children and adolescents (such as IFNL4‐peginterferon alfa‐2b).  Limiting the number of genes tested may reduce complexity for the general practitioner in interpreting/returning results to patients and families.  Development of a targeted pediatric PGx panel should ideally be informed by a cost/benefit analysis when determining which evidence‐based genes to omit (e.g., cost of repeat PGx testing at a later date and parental anxiety) balanced with the benefits of pre‐emptive testing to assist with diseases that might occur 40 to 50 years in the future.  Through future study of the impact of PGx testing on patient outcomes and the optimal delivery of PGx findings to patients and families, clinicians will learn how best to use this important tool to implement and practice precision medicine in pediatric patients.

The authors stated that this study had several drawbacks.  These researchers obtained their sample from a group of patients referred to an academic children’s hospital that may have a higher burden of disease; therefore, this may not represent the larger population of patients in the community.  However, the prevalence of actionable genetic variants in their population mirrored that of adult literature, decreasing this concern regarding their sample.  The findings of this study may not apply to different pediatric healthcare systems with different specialty clinics or referral patterns.  These investigators only examined guidance that contained genotypes included on the commercial PGx panel used at their center.  A panel that evaluated different genes or a larger variety of genotypes would produce different results and might demonstrate a greater benefit of PGx in pediatric patients.  The findings regarding diagnoses and prescriptions were limited to those contained within the authors hospital’s electronic health records (EHRs).  Diagnoses and prescriptions not documented in the EHR would not be captured, and neither would medications for indications that are not FDA‐approved or present in the Lexicomp; thus, these findings may under-estimate the prevalence of relevant diagnoses and medication use in this population.  Furthermore, these researchers did not include medications used to treat the symptoms of acute pain and severe nausea (oxycodone, codeine, and ondansetron) in their analytic strategy.  However, these investigators estimated how many patients may benefit per year from PGx before using these medicines by examining the number of prescriptions for these medications and rates of actionable gene‐drug pairs in their sample.  It remains unclear though how many of these prescriptions were 1st‐time prescriptions and how many were follow‐ups of previous prescriptions.  Patients who had taken these medicines previously and found them effective and without side effects would be at much lower risk of having an underlying actionable PGx variant; thus, their method of estimation may over-estimate the benefit of PGx testing associated with these medications.  Given the retrospective nature of this study and limitations related to available clinical documentation within the EHR, these researchers were unable to determine the impact of PGx testing on provider and family decision-making.  The authors stated that future studies in pediatrics may consider examining patients’ healthcare and medication use before and after testing, association of test results with patients’ experience of treatment effectiveness and side effects, providers’ understanding of the PGx test results, and the influence of dosing guidelines on provider dosing practices.

Bousman et al (2021) noted that the implementation of PGx testing in psychiatry remains modest, in part due to divergent perceptions of the quality and completeness of the evidence base and diverse perspectives on the clinical utility of PGx testing among psychiatrists and other healthcare providers.  Recognizing the current lack of consensus within the field, the International Society of Psychiatric Genetics assembled a group of experts to conduct a narrative synthesis of the PGx literature, prescribing guidelines, and product labels related to psychotropic medications as well as the key considerations and limitations related to the use of PGx testing in psychiatry.  The group concluded that to inform medication selection and dosing of several commonly used anti-depressant and anti-psychotic medications, current published evidence, prescribing guidelines, and product labels support the use of PGx testing for 2 cytochrome P450 genes (CYP2D6, CYP2C19).  Furthermore, the evidence supports testing for human leukocyte antigen genes when using the mood stabilizers carbamazepine (HLA-A and HLA-B), oxcarbazepine (HLA-B), and phenytoin (CYP2C9, HLA-B).  For valproate, screening for variants in certain genes (POLG, OTC, CSP1) is recommended when a mitochondrial disorder or a urea cycle disorder is suspected.  Moreover, these researchers stated that a number of larger PGx studies, such as the Ubiquitous Pharmacogenomics Project in Europe, as well as the Precision Medicine in Mental Health Care Study in the United States (NCT03170362) are currently underway.  These investigators expect with the completion of these studies and others that the PGx evidence will continue to evolve, barriers to testing will be cleared, and the uptake of genome sequencing and population-level precision medicine initiatives will increase.

Furthermore, a recent Ontario Health’s health technology assessment on “Multi-gene pharmacogenomic testing that includes decision-support tools to guide medication selection for major depression” (2021) stated that “Multi-gene pharmacogenomic testing that includes decision-support tools to guide medication selection for depression varies widely.  Differences between individual tests must be considered, as clinical utility observed with one test might not apply to other tests.  Overall, effectiveness was inconsistent among the six multi-gene pharmacogenomic tests we identified.  Multi-gene pharmacogenomic tests may result in little or no difference in improvement in depression scores compared with treatment as usual, but some tests may improve response to treatment or remission from depression.  The impact on adverse events is uncertain.  The evidence, however, is uncertain, and therefore our confidence that these observed effects reflect the true effects is low to very low”.

Brown et al (2022) stated that pharmacogenomic (PGx) testing has emerged as a compelling strategy that clinicians can use to inform anti-depressant medication selection and dosing; however, the effectiveness of this strategy has been questioned.  In a systematic review and meta-analysis, these researchers examined the association between the use of PGx-guided anti-depressant therapy and depressive symptom remission in patients with major depressive disorder (MDD).  They included prospective, controlled clinical trials published in English up to July 12, 2022.  Data extraction and synthesis adhered to the 2020 Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines.  Each trial was evaluated for risk of bias; and a random-effects model was employed to estimate pooled risk ratios.  A total of 13 studies comprising 4,767 patients were analyzed, including 10 randomized controlled trials (RCTs), and 3 open-label trials.  Across all included trials, those that received PGx-guided anti-depressant therapy (n = 2,395) were 1.41 (95 % CI: 1.15 to 1.74, p = 0.001) more likely to achieve remission compared with those that received un-guided anti-depressant therapy (n = 2,372).  Pooled risk ratios for RCTs and open-label trials were 1.46 (95 % CI: 1.13 to 1.88) and 1.26 (95 % CI: 0.84 to 1.88), respectively.  The authors concluded that the findings of this systematic review/meta-analysis suggested that PGx-guided anti-depressant therapy was associated with a modest but significant increase in depressive symptom remission in adults with MDD.  Moreover, these researchers stated that efforts to address the heterogeneity in PGx test composition (i.e., genes and alleles tested) and accompanying prescribing recommendations across trials will likely reduce the uncertainty regarding the effectiveness of PGx-guided anti-depressant therapy in the literature.

The authors stated that several drawbacks should be considered when interpreting the findings of this study.  First, the clinical generalizability of the findings was limited.  Although 2 of the included studies exclusively enrolled individuals of Chinese ancestry and 2 studies were predominantly male subjects, most of the individuals included across the 13 studies were female adults aged 40 to 50 years with a European background.  As such, uncertainty remains regarding the effectiveness of PGx‐guided anti-depressant treatment in diverse settings and emphasized the critical need for trial evidence that is more representative of real‐world clinical populations.  Second, prescribing physicians in all the included studies were not blinded to intervention allocation, which may have introduced a high risk of performance bias as well as attention and ancillary treatment biases.  Innovative trial designs are needed to address these biases.  One example is being used in an ongoing Australian PGx trial, in which prescribing recommendations are provided to all healthcare providers in an identical format, without reference to the participant's genetic results.  Recommendations for participants allocated to the PGx‐guided group are tailored to their CYP2C19 and CYP2D6 genotypes, whereas the treatment as usual group receives recommendations based on current Australian anti-depressant prescribing guidelines.  Third, these researchers were unable to determine which gene or combination of genes are driving the observed effect.  Although all the included studies tested for genetic variants in CYP2C19 and CYP2D6, they also tested for variants in several other genes (e.g., SLC6A4 and HTR2A) that are often used by commercial laboratories to derive anti-depressant prescribing recommendations, despite the absence of dosing guidelines for these genes.  Complicating this matter further is that many of the PGx tests used in the included studies used proprietary combinatorial or “black box” algorithms that could result in discordant recommendations among tests and with current PGx‐based dosing guidelines.  As a consequence, one could argue that the true impact of PGx‐guided anti-depressant treatment on depressive symptom remission was likely being attenuated by such heterogeneity.  As standardization efforts, such as the Standardizing Laboratory Practices in Pharmacogenomics initiative, develop and regulations related to PGx testing tighten, these investigators expected this heterogeneity to lessen.  Lastly, not all studies reported side effects in an evaluable way, precluding these researchers from evaluating the pooled effect of PGx‐guided anti-depressant treatment on side effects in this population.

Oslin et al (2022) noted that selecting effective anti-depressants for the treatment of MDD is an imprecise practice, with remission rates of approximately 30 % at the initial treatment.  In a randomized clinical trial, these investigators examined if PGx testing would affect anti-depressant medication selection and whether such testing would result in better clinical outcomes versus usual care.  Subjects included 676 clinicians and 1,944 patients.  Subjects were enrolled from 22 Department of Veterans Affairs (VA) medical centers from July 2017 through February 2021, with follow-up ending November 2021.  Eligible patients were those with MDD who were initiating or switching treatment with a single anti-depressant.  Exclusion criteria included an active substance use disorder, mania, psychosis, or concurrent treatment with a specified list of medications.  Results from a commercial PGx test were given to clinicians in the PGx-guided group (n = 966).  The comparison group received usual care and access to pharmacogenomic results after 24 weeks (n = 978).  The co-primary outcomes were the proportion of prescriptions with a predicted drug-gene interaction written in the 30 days following randomization and remission of depressive symptoms as measured by the Patient Health Questionnaire-9 (PHQ-9) (remission was defined as PHQ-9 of 5 or less).  Remission was analyzed as a repeated measure across 24 weeks by blinded raters.  Among 1,944 patients who were randomized (mean age of 48 years; 491 women [25 %]), 1,541 (79 %) completed the 24-week assessment.  The estimated risks for receiving an anti-depressant with none, moderate, and substantial drug-gene interactions for the PGx-guided group were 59.3 %, 30.0 %, and 10.7 % compared with 25.7 %, 54.6 %, and 19.7 % in the usual care group.  The PGx-guided group was more likely to receive a medication with a lower potential drug-gene interaction for no drug-gene versus moderate/substantial interaction (OR of 4.32 [95 % CI: 3.47 to 5.39]; p < 0.001) and no/moderate versus substantial interaction (OR of 2.08 [95 % CI: 1.52 to 2.84]; p = 0.005) (p < 0.001 for overall comparison).  Remission rates over 24 weeks were higher among patients whose care was guided by PGx testing than those in usual care (OR of 1.28 [95 % CI: 1.05 to 1.57]; p = 0.02; risk difference, 2.8 % [95 % CI: 0.6 % to 5.1 %]); but were not significantly higher at week 24 when 130 patients in the PGx-guided group and 126 patients in the usual care group were in remission (estimated risk difference, 1.5 % [95 % CI: -2.4 % to 5.3 %]; p = 0.45).  The authors concluded that among patients with MDD, provision of PGx testing for drug-gene interactions reduced prescription of medications with predicted drug-gene interactions compared with usual care.  Furthermore, provision of test results had small non-persistent effects on symptom remission.

The authors stated that this study had several drawbacks.  First, there was no attempt to blind either the clinician or patient in the study; therefore, the modest effects in the PGx-guided group could be a placebo-type effect.  Second, this study was not powered to examine outcomes such as the effect of changes in dosing in the PGx-guided group among patients with predicted drug-gene interactions, the presence of adverse drug reactions, the effect of medication adherence by patients, or the effect of anti-depressant switches following randomization.  Third, because this study employed a proprietary PGx test; therefore, results may not translate to other commercial products.  The proprietary algorithm used to make recommendations regarding drug-gene interactions may not align with recommendations from groups such as the Clinical Pharmacogenetics Implementation Consortium, a non-profit organization that provides recommendations for drug-gene interaction.  Fourth, many patients had a delay of unclear meaning and importance in initiating a new episode of treatment though post-hoc analysis showed the delay decreased rapidly over the trial.  None of the co-variates were associated with this delay.

Wang et al (2023) stated that PGx testing-guided treatment have been developed to guide drug selection or conversion in patients with MDD.  Whether patients benefit from PGx testing remains unclear.  In a systematic review and meta-analysis, these investigators examined the effect of PGx testing guiding on clinical outcomes of patients with MDD.  PubMed, Embase, and Cochrane Library of Clinical Trials were searched from inception until August 2022.  Key terms included pharmacogenomic and anti-depressive.  Odds ratios (RR) with 95 % CIs were calculated using fixed-effects model for low or moderate heterogeneity or random-effects model for high heterogeneity.  A total of 11 studies (5,347 patients) were included.  Compared with usual group, PGx testing-guided group was associated with an increased response rate at week 8 (OR 1.32, 95 % CI: 1.15 to 1.53, 8 studies, 4,328 participants) and week 12 (OR 1.36, 95 % CI: 1.15 to 1.62, 4 studies, 2,814 participants).  Similarly, guided group was associated with an increased rate of remission at week 8 (OR 1.58, 95 % CI: 1.31 to 1.92, 8 studies, 3,971 participants) and week 12 (OR 2.23, 95 % CI: 1.23 to 4.04, 5 studies, 2,664 participants).  However, no significant differences were observed between the 2 groups in response rate at week 4 (OR 1.12, 95 % CI: 0.89 to 1.41, 2 studies, 2,261 participants) and week 24 (OR 1.16, 95 % CI: 0.96 to 1.41, 2 studies, 2,252 participants), and remission rate at week 4 (OR 1.26, 95 % CI: 0.93 to 1.72, 2 studies, 2,261 participants) and week 24 (OR 1.06, 95 % CI: 0.83 to 1.34, 2 studies, 2,252 participants).  Medication congruence in 30 days was significantly reduced in the PGx-guided group compared with the usual care group (OR 2.07, 95 % CI: 1.69 to 2.54, 3 studies, 2,862 participants).  These investigators found significant differences between subgroups of target population in response and remission rate.  The authors concluded that patients with MDD may benefit from PGx testing-guided treatment by achieving target response and remission rates more quickly.  Moreover, these researchers stated that future well-designed, multi-ethnic studies are needed to confirm the benefit of PGx testing guidance in different populations.

The authors stated that although this study revealed the effect of PGx testing-guided treatment on MDD medications and indicated its striking value in Asian MDD patients, there are still several methodological limitations that may lead to interference and bias that emerged from this study.  The reliability of the inclusion of unblinded clinicians was reduced in most of the reported pooled effect studies.  Furthermore, patients were unblinded in the largest scale research included in this current analysis, which may also lead to detection bias of outcomes.  Another significant limitation was that this study was based on data in per-protocol analysis rather than intention-to-treat analysis, which may lead to over-estimating the outcomes of PGx testing-guided treatment on MDD medications.  Nevertheless, this meta-analysis was a re-analysis based on current research so a few genotypes were involved.  These investigators noted that previous reports focused on the effects of CYP2C19 and CYP2D6 on antidepressants; therefore, this research mostly mentioned the 2 variants.  In the future, more genotypes would be involved in their prospective research.

Castle Biosciences, Inc. (Freindswood, TX) offers IDgenetix, a pharmacogenomics (PGx) test that uses a prospectively designed algorithm to integrate a 15-gene variant panel testing results with drug-drug interaction data and lifestyle factors. IDgenetix is intended to provide medication recommendations for patients diagnosed with major depressive disorder (MDD), anxiety, or other mental illness (Castle Biosciences, 2024).

ABCB1 gene

In a small pilot study, Breitenstein et al (2014) stated the gene product of the ABCB1 gene, the P-glycoprotein, functions as a custodian molecule in the blood-brain barrier and regulates the access of most antidepressants into the brain. Previous studies showed that ABCB1 polymorphisms predicted the response to antidepressants that are substrates of the P-gp, while the response to non-substrates was not influenced by ABCB1 polymorphisms. The aim of the present study was to evaluate the clinical application of ABCB1 genotyping in antidepressant pharmacotherapy. Data came from 58 depressed inpatients participating in the Munich Antidepressant Response Signature (MARS) project, whose ABCB1 gene test results were implemented into the clinical decision making process. Hamilton Depression Rating Scale (HAM-D) scores, remission rates, and duration of hospital stay were documented with dose and kind of antidepressant treatment. Patients who received ABCB1 genotyping had higher remission rates [χ2(1) = 6.596, p = 0.005, 1-sided] and lower Hamilton sores [t(111) = 2.091, p = 0.0195, 1-sided] at the time of discharge from hospital as compared to patients without ABCB1 testing. Among major allele homozygotes for ABCB1 single nucleotide polymorphisms (SNPs) rs2032583 and rs2235015 (TT/GG genotype), an increase in dose was associated with a shorter duration of hospital stay [rho(28) = -0.441, p = 0.009, 1-sided], whereas other treatment strategies (e.g., switching to a non-substrate) showed no significant associations with better treatment outcome. Discussion The implementation of ABCB1 genotyping as a diagnostic tool influenced clinical decisions and led to an improvement of treatment outcome. Patients carrying the TT/GG genotype seemed to benefit from an increase in P-gp substrate dose. The authors concluded that results suggest that antidepressant treatment of depression can be optimized by the clinical application of ABCB1 genotyping.

BRCA Testing for Olaparib (Lynparza)

The FDA approved BRACAnalysis CDx (Myriad Genetics) as a companion diagnostic for olaparib (Lynparza) (AstraZeneca), a drug for that has been approved by the FDA for women with advanced ovarian cancer associated with BRCA genes.

BRACAnalysis CDx is an in vitro diagnostic device intended for the qualitative detection and classification of variants in the protein coding regions and intron/exon boundaries of the BRCA1 and BRCA2 genes using genomic DNA obtained from whole blood specimens collected in EDTA (FDA, 2014). Single nucleotide variants and small insertions and deletions (indels) are identified by polymerase chain reaction (PCR) and Sanger sequencing. Large deletions and duplications in BRCA1 and BRCA2 are detected using multiplex PCR. Results of the test are used as an aid in identifying ovarian cancer patients with deleterious or suspected deleterious germline BRCA variants eligible for treatment with olaparib (Lynparza)). This assay is only performed at Myriad Genetic Laboratories, a single laboratory site located in Salt Lake City, UT.

In clinical studies, BRACAnalysis CDx was proven to effectively identify patients with BRCA mutations who would be candidates for Lynparza (Myriad, 2014). The FDA concurrently approved olaparib (Lynparza), a poly ADP-ribose polymerase (PARP) inhibitor that blocks enzymes involved in repairing damaged DNA. Olaparib is approved by the FDA as monotherapy in patients with deleterious or suspected deleterious germline BRCA mutated (as detected by an FDA-approved test) advanced ovarian cancer who have been treated with three or more prior lines of chemotherapy. 

The FDA's approval of the companion diagnostic is based on data from the clinical study used to support approval of the new drug. Blood samples from clinical trial participants were tested to validate the test's use for detecting BRCA mutations in this population. The agency evaluated BRACAnalysis CDx's safety and efficacy under its premarket approval pathway used for high-risk medical devices. Myriad had been marketing this test without FDA approval as a laboratory developed test, although not specifically for use as a companion diagnostic.

On January 12, 2018, the FDA expanded the indication for BRACAnalysis CDx for use as a companion diagnostic to identify patients with HER2-negative metastatic breast cancer who have a germline BRCA mutation and are candidates for treatment with the PARP inhibitor Lynparza (olaparib) (FDA, 2018).

Robson et al. (2017) conducted a randomized, open-label, phase 3 trial in which olaparib monotherapy was compared with standard therapy in patients with a germline BRCA mutation and HER2-negative metastatic breast cancer who had received no more than two previous chemotherapy regimens for metastatic disease. Patients (n=302) were randomly assigned, in a 2:1 ratio, to receive olaparib tablets (n=205; 300 mg twice daily) or standard therapy (n=97) with single-agent chemotherapy of the physician's choice (capecitabine, eribulin, or vinorelbine in 21-day cycles). The primary end point was progression-free survival, which was assessed by blinded independent central review and was analyzed on an intention-to-treat basis. Median progression-free survival was significantly longer in the olaparib group than in the standard-therapy group (7.0 months vs. 4.2 months; P<0.001). The response rate was 59.9% in the olaparib group and 28.8% in the standard-therapy group. The rate of grade 3 or higher adverse events was 36.6% in the olaparib group and 50.5% in the standard-therapy group, and the rate of treatment discontinuation due to toxic effects was 4.9% and 7.7%, respectively. Clinical outcoms has shown median progression-free survival 2.8 months longer, and the risk of disease progression or death 42% lower with olaparib monotherapy than with standard therapy.The authors concluded that olaparib monotherapy provided a significant benefit over standard therapy among patients with HER2-negative metastatic breast cancer and a germline BRCA mutation.

Platelet Reactivity/Function Testing (VerifyNow P2Y12 Assay)
Clinical evidence has been controversial regarding the influence of clopidogrel on treatment platelet reactivity and ischemic outcomes.  Brar et al (2011) systematically evaluated the significance of platelet reactivity on clopidogrel treatment on adverse cardiovascular events using a collaborative meta-analysis using patient-level data for the VerifyNow P2Y12 assay (Accumetrics, San Diego, CA).  Medline, Scopus, and the Cochrane library databases were searched through January 2010.  A database containing individual patient-level time-to-event data was generated from identified studies.  The primary outcome of interest was a composite of death, myocardial infarction (MI), or stent thrombosis (STh).  Secondary outcomes included the incidence of: death; MI; and STh.  
death;
MI; and
STh. 

A total of 6 studies with 3,059 patients were included.  In each study, clopidogrel responsiveness was assessed using the same point-of-care assay after percutaneous coronary intervention (PCI).  The primary endpoint occurred more frequently in higher quartiles of P2Y(12) reaction unit (PRU) values: quartile I, 5.8 %; quartile II, 6.9 %; quartile III, 10.9 %; quartile IV, 15.8 % (p < 0.001).  Taking quartile I as referent, the hazard ratios (HRs) for the primary endpoint were as follows: quartile II, HR: 1.13 (95 % CI: 0.72 to 1.78; p = 0.60); quartile III, HR: 1.82 (95 % CI: 1.20 to 2.75; p = 0.005); quartile IV, HR: 2.62 (95 % CI: 1.78 to 3.87; p < 0.001).  On a continuous scale, every 10-U increase in PRU was associated with a significantly higher rate of the primary endpoint (HR: 1.04; 95 % CI: 1.03 to 1.06; p < 0.0001).  According to receiver-operating characteristic (ROC) curve analysis, a PRU value of 230 appeared to best predict death, MI, or STh (p < 0.001).  A PRU value greater than or equal to 230 was associated with a higher rate of the composite primary endpoint (HR: 2.10; 95 % CI: 1.62 to 2.73; p < 0.0001), as well as the individual endpoints of death (HR: 1.66; 95 % CI: 1.04 to 2.68; p = 0.04), MI (HR: 2.04; 95 % CI: 1.51 to 2.76; p < 0.001), and STh (HR: 3.11; 95 % CI: 1.50 to 6.46; p = 0.002).   The authors concluded that in this collaborative meta-analysis, the level of on-treatment platelet reactivity according to the P2Y(12) assay is associated with long-term cardiovascular events after PCI, including death, MI, and STh.

Gurbel and Tantry (2011) noted that platelet-mediated thrombosis is a dreaded clinical event and is the primary cause of acute coronary syndromes (ACS) and post-PCI ischemic events.  There has been a long-standing interest in the ex-vivo quantification of platelet reactivity to assess the risk of thrombosis.  Early studies demonstrated platelet activation and heightened platelet reactivity in ACS and after PCI.  However, a demonstration that heightened reactivity actually precipitated the ischemic event was lacking.  Knowledge of platelet receptor physiology and the advent of novel inhibitors have significantly advanced the field.  The P2Y12 receptor has been shown to play a pivotal role in the amplification of platelet activation by multiple agonists and its inhibition has resulted in improved clinical outcomes.  The most widely used drug to block P2Y12 receptor, clopidogrel, is associated with resistance in selected patients and these patients have been shown to be at increased risk for post-PCI ischemic event occurrence in multiple studies.  Importantly, a threshold of high platelet reactivity has been demonstrated, and beyond this threshold ischemic events occur precipitously.  Based on the current evidence, it is rational to quantify the intensity of the ADP-P2Y12 interaction in the patient at the greatest risk for thrombosis -- the PCI patient.  However, there is only evidence from small clinical trials demonstrating the clinical efficacy of changing an anti-platelet regimen based on an ex-vivo platelet function measurement.  Moreover, there are numerous patients with vulnerable coronary anatomy that have not yet experienced plaque rupture; the prognostic role of a measurement of platelet reactivity in the latter group has never been studied.  The authors stated that large-scale trials are ongoing that will investigate the role of personalized anti-platelet therapy in the PCI patient.

Holmes et al (2011) appraised evidence on the association of CYP2C19 genotype and clopidogrel response through systematic review and meta-analysis.  PubMed and EMBASE from their inception to October 2011 were searched.  Studies that reported clopidogrel metabolism, platelet reactivity or clinically relevant outcomes (cardiovascular disease [CVD] events and bleeding), and information on CYP2C19 genotype were included.  These researchers extracted information on study design, genotyping, and disease outcomes and investigated sources of bias.  They retrieved 32 studies of 42,016 patients reporting 3,545 CVD events, 579 STh, and 1,413 bleeding events.  Six studies were randomized trials ("effect-modification" design) and the remaining 26 reported individuals exposed to clopidogrel ("treatment-only" design).  In treatment-only analysis, individuals with 1 or more CYP2C19 alleles associated with lower enzyme activity had lower levels of active clopidogrel metabolites, less platelet inhibition, lower risk of bleeding (relative risk [RR], 0.84; 95 % CI: 0.75 to 0.94; absolute risk reduction of 5 to 8 events per 1,000 individuals), and higher risk of CVD events (RR, 1.18; 95 % CI: 1.09 to 1.28; absolute risk increase of 8 to 12 events per 1,000 individuals).  However, there was evidence of small-study bias (Harbord test p = 0.001).  When analyses were restricted to studies with 200 or more events, the point estimate was attenuated (RR, 0.97; 95 % CI: 0.86 to 1.09).  In effect-modification studies, CYP2C19 genotype was not associated with modification of the effect of clopidogrel on CVD end points or bleeding (p > 0.05 for interaction for both).  Other limitations included selective outcome reporting and potential for genotype mis-classification due to problems with the * allele nomenclature for cytochrome enzymes.  The authors concluded that although there was an association between the CYP2C19 genotype and clopidogrel responsiveness, overall there was no significant association of genotype with cardiovascular events.

Varenhorst et al (2011) noted that insufficient platelet inhibition is a major determinant of STh, although the etiology is multi-factorial.  These investigators examined on-clopidogrel platelet reactivity in patients with previous angiographically confirmed STh, MI, and controls.  Using the Swedish Coronary Angiography and Angioplasty Registry, these researchers identified patients with angiographically confirmed STh (n = 48) or MI (n = 30) while on dual anti-platelet therapy within 6 months of PCI and matched control patients (n = 78).  On-clopidogrel platelet reactivity was measured with VerifyNow P2Y12 and vasodilator-stimulated phosphoprotein (VASP) phosphorylation assay.  The mean PRU was higher (246.8 +/- 75.9 versus 200.0 +/- 82.7, p = 0.001) in STh patients compared with controls.  The optimal cut-off for STh was 222 PRU or higher (area under the curve 0.69, p < 0.0001) in a ROC analysis.  The cut-off level resulted in 70.2 % sensitivity and 67.3 % specificity.  There was no significant difference in mean PRU but a higher device-reported percent inhibition (45.1 +/- 23.8 versus 32.1 +/- 23.2, p = 0.04) in patients with MI compared with controls.  Results with the VASP phosphorylation assay were not related to the occurrence of STh or MI.  The authors concluded that STh was associated with high on-clopidogrel platelet reactivity measured with VerifyNow (cut-off level of PRU greater than or equal to 222), but spontaneous MI in stented patients on clopidogrel treatment was not.  There was, however, a substantial overlap in on-clopidogrel platelet reactivity between patients with and without on-treatment STh questioning the clinical use of platelet function testing to identify patients at high-risk for STh.

In a randomized, double-blind, active-control trial (Gauging Responsiveness with A VerifyNow assay-Impact on Thrombosis And Safety [GRAVITAS]), Price et al (2011) evaluated the effect of high-dose compared with standard-dose clopidogrel in patients with high on-treatment platelet reactivity after PCI.  A total of 2,214 patients with high on-treatment reactivity 12 to 24 hours after PCI with drug-eluting stents at 83 centers in North America between July 2008 and April 2010 were included in this study.  Patients received high-dose clopidogrel (600-mg initial dose, 150 mg daily thereafter) or standard-dose clopidogrel (no additional loading dose, 75 mg daily) for 6 months.  The primary end point was the 6-month incidence of death from cardiovascular causes, non-fatal MI, or STh.  The key safety end point was severe or moderate bleeding according to the Global Utilization of Streptokinase and t-PA for Occluded Coronary Arteries (GUSTO) definition.  A key pharmacodynamic end point was the rate of persistently high on-treatment reactivity at 30 days.  At 6 months, the primary end point had occurred in 25 of 1,109 patients (2.3 %) receiving high-dose clopidogrel compared with 25 of 1,105 patients (2.3 %) receiving standard-dose clopidogrel (hazard ratio [HR], 1.01; 95 % CI: 0.58 to 1.76; p = 0.97).  Severe or moderate bleeding was not increased with the high-dose regimen (15 [1.4 %] versus 25 [2.3 %], HR, 0.59; 95 % CI: 0.31 to 1.11; p = 0.10).  Compared with standard-dose clopidogrel, high-dose clopidogrel provided a 22 % (95 % CI: 18 % to 26 %) absolute reduction in the rate of high on-treatment reactivity at 30 days (62 %; 95 % CI: 59 % to 65 % versus 40 %; 95 % CI: 37 % to 43 %; p < 0.001).  The authors concluded that among patients with high on-treatment reactivity after PCI with drug-eluting stents, the use of high-dose clopidogrel compared with standard-dose clopidogrel did not reduce the incidence of death from cardiovascular causes, non-fatal MI, or STh.

Sharma et al (2012) noted that the substantial reduction in ischemic events provided by the dual anti-platelet regimen with aspirin and clopidogrel is well-documented in patients with ACS and patients undergoing PCI.  Recently the variable response to the anti-platelet agents has received considerable attention after several "boxed warnings" on clopidogrel.  This led to intense controversy on pharmacokinetic, pharmacodynamic, and pharmacogenomic issues of anti-platelet drugs, especially clopidogrel.  Research use of platelet function testing has been successfully validated in identifying new anti-platelet drugs like prasugrel and ticagrelor.  These platelet function assays are no longer regarded just as a laboratory phenomenon but rather as tools that have been shown to predict mortality in several clinical trials.  It is believed that suboptimal response to an anti-platelet regimen (pharmacodynamic effect) may be associated with cardiovascular, cerebrovascular, and peripheral arterial events.  There has been intense controversy about this variable response of anti-platelet drugs and the role of platelet function testing to guide anti-platelet therapy.  While the importance of routine platelet function testing may be uncertain, it may be useful in high-risk patients such as those with diabetes mellitus, diffuse 3-vessel coronary artery disease, left main stenosis, diffuse atherosclerotic disease, and those with chronic renal failure undergoing PCI.  It could also be useful in patients with suspected pharmacodynamic interaction with other drugs to assure the adequacy of platelet inhibition.  The authors stated that "While we wait for definitive trials, a predictive prognostic algorithm is necessary to individualize antiplatelet therapy with P2Y12 inhibitors based on platelet function assays and genetic testing".

Yu et al (2012) confirmed the predictive cut-off values for PRU and aspirin reaction units (ARU) and evaluated the clinical impact of VerifyNow® assays.  From November 2007 to October 2009, a total of 186 eligible patients were prospectively recruited.  Post-treatment platelet reactivity was measured by VerifyNow® assays within 12 to 24 hours after intervention, followed by standard dual maintenance dose therapy for 1 year.  All patients had scheduled clinical follow-ups at 1, 3, 6, and 12 months.  The rate of low- responders to clopidogrel, aspirin, and both drugs were 41.4 %, 10.2 %, and 3.8 %, respectively.  The predictive factors for low-responsiveness to clopidogrel (PRU greater than or equal to 240) were female sex, age, and non-use of cilostazol medication in uni-variate analysis and age greater than or equal to 65 years and non-use cilostazol in the multi-variate analysis.  The predictors of low-responsiveness to aspirin (ARU greater than or equal to 550) were male sex and age in both uni-variate and multi-variate analyses.  There was no significant difference in the clinical event rate with a cut-off value of PRU greater than or equal to 240 or ARU greater than or equal to 550 for 30 days and 1-year (p > 0.05).  The authors concluded that hypo-responsiveness to anti-platelet agents (namely aspirin and clopidogrel) was identified in about 50 % of the patients.  The cut-off point of PRU greater than or equal to 240 or ARU greater than or equal to 550 did not confer predictive value for 30-day or 1-year clinical event rates in patients who had undergone PCI with drug-eluting stents.

Kim et al (2012) stated that although adjunctive cilostazol to dual anti-platelet therapy can reduce the risks of clinical events after PCI, whether genetic polymorphism can influence the pharmacodynamics of this regimen has not been evaluated.  In this study, a total of 127 patients treated with PCI and taking triple anti-platelet therapy (greater than or equal to 1 month) were enrolled.  Platelet reactivity was assessed by conventional aggregometry and the VerifyNow P2Y12 assay.  High on-treatment platelet reactivity (HPR) was defined as 5 µm ADP-induced maximal platelet reactivity (Agg(max)) greater than 46 %.  CYP3A5*3, CYP2C19*2/*3 and ABCB1 3435C > T were genotyped.  CYP3A5*3 and ABCB1 3435C > T variants did not affect the anti-platelet effect of triple anti-platelet therapy.  For non-carriers, 1 and 2 carriers of the CYP2C19 loss-of-function (LOF) allele, Agg(max) consecutively increased after the addition of 5 µm [mean (95 % CI): 24.6 % (20.8 to 28.5 %) versus 28.7 % (25.4 to 32.0 %) versus 32.3 % (25.8 to 38.7 %), p = 0.062, respectively] and 20 µm ADP [34.2 % (29.3 to 39.0 %) versus 41.7 % (37.8 to 45.6 %) versus 44.9 % (37.9 to 51.9 %), p = 0.007, respectively].  Likewise, late platelet reactivity and PRU proportionally changed according to the number of CYP2C19 LOF alleles.  High on-treatment platelet reactivities were observed in 9.2 % of subjects: 6.3 %, 7.4 % and 20.0 % with 0, 1 and 2 carriers of CYP2C19 LOF allele(s) (p = 0.099).  In multi-variate analysis, carriage of 2 CYP2C19 LOF alleles was a significant predictor for the prevalence of HPR (odds ratio 5.78, 95 % CI: 1.21 to 27.78, p = 0.028).  The authors concluded that among PCI-treated patients, the effect of triple anti-platelet therapy is influenced by the CYP2C19 LOF allele.  They stated that its clinical benefit needs to be validated according to the CYP2C19 metabolic phenotype in future clinical trials.

O'Connor et al (2012) noted that clopidogrel used in conjunction with aspirin has a central role in the treatment of patients with an ACS and/or undergoing PCI.  The pharmacokinetic and pharmacodynamic responses to this drug are highly variable leaving up to 1/3 of patients with inadequate platelet inhibition or HPR, and subsequent increased ischemic cardiovascular events.  Genetic variability in drug absorption and metabolism is a key factor responsible for the inefficient generation of the active drug metabolite.  The 2-step hepatic cytochrome P450 (CYP)-dependent oxidative metabolism of the pro-drug appears to be of particular importance.  Pharmacogenomic analyses have identified LOF variant alleles of CYP 2C19 and specifically the 2C19*2 allele, to be the predominant genetic mediators of the anti-platelet effect of clopidogrel.  Carriers were have been shown to have lower active metabolite levels of clopidogrel, higher platelet reactivity and associated poorer outcomes.  Rapid and accurate point-of-care genetic tests to identify these alleles are currently in development but several questions about the role of such testing remain such as patient selection and whether personalized treatment based on genotype has a positive impact on clinical outcome.  At present, genetic testing can not be recommended in routine clinical practice due to insufficient prospective data.  However, the significant body of research published to date suggests a likely role when used in combination with platelet function analysis in ACS patients undergoing stenting who have other known risk factors for recurrent ischemic events.

Mallouk et al (2012) performed a systematic review to estimate the prevalence of poor biological response to clopidogrel and investigated the factors known to modulate this.  An exhaustive search was performed.  A total of 171 publications were identified, providing data for 45,664 subjects.  The estimated prevalence of poor biological response to clopidogrel ranged from 15.9 % to 49.5 % according to the platelet function assay employed.  The assays most frequently used were light transmittance aggregometry (LTA), the vasodilator-stimulated phosphoprotein (VASP) assay and the Verifynow® assay.  For all these assays, higher cut-off values were associated with a lower prevalence of poor biological response to clopidogrel.  However, when choosing a fixed cut-off point for each assay, the prevalence of poor biological response to clopidogrel was highly variable suggesting that other factors could modulate poor biological response to clopidogrel.  Finally, none of the studied factors could apparently explain the variability of poor biological response to clopidogrel.  This meta-analysis showed that the prevalence of poor biological response depends on the assay employed, the cut-off value and on various unidentified additional factors.

A draft AHRQ report (2012) found evidence that high on-clopidogrel platelet reactivity is associated with an increased risk of adverse cardiovascular outcomes for at least some of the available assays.  The report stated that the strength of evidence regarding these prognostic effects is low because of concerns regarding selective outcome reporting and the relatively small number of studies reporting clinical outcomes.  The report concluded that strength of evidence regarding the use of platelet reactivity testing to guide anti-platelet treatment selection is insufficient, because studies reporting on clinical outcomes are few, have diverse designs, and included heterogeneous populations.

In summary, there is currently insufficient evidence to support the use of platelet reactivity/function testing for patients who have undergone after PCI.  The incorporation of platelet reactivity/function testing into clinical practice awaits the results of ongoing clinical studies where treatment is changed based on platelet reactivity/function testing data.

Beta Adrenergic Receptor Genotyping (ADRB2) for Treatment-Resistant Asthma

ADRB2 (beta adrenergic receptor genotyping) is a genetic test used for asthma patients with poor symptom control. The target for beta2-agonist asthma medication is the B2-adrenergic receptor. The gene for the B2-adrengergic receptor is ADRB2. It is believed that a variation at one location of this gene may predict therapeutic responses to beta2-agonists. The Arg/Arg homozygous genotype at amino acid position 15 may indicate a need for a change in medication.

The textbook Cleveland Clinic: Current Clinical Medicine (2010) states that, in the presence of a polymorphism, the acute bronchodilator response to a β agonist, or protection from a bronchoconstrictor, may be affected. Studies indicate that in patients with Arg16Arg variant, the resulting β2-adrenergic receptor is resistant to endogenous circulating catecholamines (i.e., receptor density and integrity are preserved), with a subsequent ability to produce an acute bronchodilator response to an agonist). There are conflicting data regarding whether Arg/Arg homozygotes are prone to experience reflex morbidity with inhaled LABA, but the weight of evidence, particularly from more-recent studies, indicates that response to LABA when used in combination with ICS does not vary based on β2-adrenergic genotypes at codon 16. 

In 2010, Bleecker and colleagues examined whether the response to salmeterol alone or in combination with an inhaled corticosteroid is influenced by beta- receptor polymorphisms. Subjects using only as-needed albuterol were screened and completed two sequential open-label run-in periods (8 wk on as-needed albuterol; 8 wk on as-needed ipratropium). Five hundred forty-four subjects were randomized by Arg16Gly genotype to salmeterol alone or with fluticasone propionate for 16 weeks. Change from baseline in morning peak expiratory flow was the primary endpoint. Lung function responses were sustained over treatment and no statistically significant changes from baseline between genotypes within treatments were observed. Overall mean changes in morning peak flow for salmeterol with fluticasone propionate were 32.6 L/min (Arg/Arg vs. Gly/Gly, 95 % CI, -6.3, 22.1), 25.9 L/min (Arg/Arg vs. Arg/Gly, 95% CI, -7.1, 21.3), and 24.9 L/min (Arg/Gly vs. Gly/Gly, 95 % CI, -13.0, 14.6), and for salmeterol alone were 19.4 L/min (Arg/Arg vs. Gly/Gly, 95 % CI, -1.7, 21.4), 24.6 L/min (Arg/Arg vs. Arg/Gly, 95 % CI, -13.0, 10.6), and 12.4 L/min (Arg/Gly vs. Gly/Gly, 95 % CI, -0.2, 22.3) for Arg/Arg, Arg/Gly, and Gly/Gly genotypes, respectively. Other measures of asthma control showed similar responses. The results showed no evidence of a pharmacogenetic effect of beta-receptor variation on salmeterol response. 

Basu and colleagues (2009) investigated whether the presence of Arg16 allele of the adrenergic beta(2)-receptor agonist gene (ADRB2) predisposed to exacerbations in young asthmatic patients taking regular salmeterol. Arg/Gly status at position 16 of ADRB2 was assessed in 1182 asthmatic patients. Asthma exacerbations, use of beta-agonists and other medications over the previous 6 months, and lung function were also studied. An increased risk of exacerbations per copy of the Arg16 allele was observed in asthmatic patients, regardless of treatment regimen (odds ratio [OR], 1.30; 95% CI, 1.09-1.55; P = .003). This appeared to be largely due to exposure to beta(2)-agonists because the risk of exacerbations observed in patients with the Arg16 allele was only observed in those receiving daily inhaled long- or short-acting beta(2)-agonist treatment (OR, 1.64; 95% CI, 1.22-2.20; P = .001). In contrast, there was no genotypic risk for exacerbations in patients using inhaled beta(2)-agonists less than once a day (OR, 1.08; 95% CI, 0.85-1.36; P = .525). The Arg16 genotype-associated risk for exacerbations was significantly different in those exposed to beta(2)-agonists daily versus those that were not (test for interaction, P = .022). The authors concluded that Arg16 genotype of ADRB2 was associated with exacerbations in asthmatic children and young adults exposed daily to beta(2)-agonists, regardless of whether the exposure is to albuterol or long-acting agonists, such as salmeterol. 

In a 2008 article, Martin and associates evaluated the influence of single nucleotide polymorphisms in the beta(2)-adrenoceptor gene, on the response to inhaled beta(2)-agonists in children with acute asthma. One hundred and forty-eight children with acute asthma were recruited and genotyped for beta(2)Arg16Gly and beta(2)Gln27Glu. For Gln27Glu, individuals Gln27Gln took longest to stretch out to 1, 2 and 4 hourly beta(2)-agonists, followed by heterozygotes who were intermediate and Glu27Glu who responded most rapidly (1 hourly: 2.6 hr vs. 2.0 vs. 1.4, p = 0.02; 2 hourly: 10.6 hr vs. 10.7 vs. 6.8, p = 0.07; 4 hourly: 29.8 hours vs. 28.5 vs. 24.3, p = 0.30). The authors reported that the ability to prospectively identify children who respond less effectively to beta (2)-agonists during an acute asthma attack has the potential to allow the generation of genotype-specific treatment pathways. 

In 2008, Giubergia et al assessed the frequency of beta2-adrenergic receptor (beta2-AR) polymorphisms in asthmatic children from Argentina, and evaluated their influence on bronchodilator desensitization to albuterol over a 4-week treatment. beta2-AR genotypes were determined in 117 children with asthma and 101 of them were under 4 weeks treatment with albuterol. Spirometric changes in FEV(1) were recorded at the beginning (day 1) and at the end of the study (day 30) and compared to genotypes at position 16 and 27 of the receptor. The frequency of the polymorphisms was calculated in all population. The presence of glutamine at position 27 (Gln27) was significantly more frequent in this Argentinean study population than in other Caucasian populations. The homozygosity for Gln27 polymorphism was associated to a desensitization of the receptor with a decline in the bronchodilator response to albuterol after chronic use.

In 2007, Bleeker et al investigated whether beta2-adrenergic receptor (ADRB2) polymorphisms affect response to long acting beta2-agonists in combination with inhaled corticosteroids. Asthmatics were stratified by ADRB2 genotype in two studies to assess the effects of inhaled corticosteroids plus long acting beta2-agonists on asthma exacerbations. In study 1 (double-blind), 2250 asthmatics were randomly assigned to budesonide plus formoterol maintenance and reliever therapy, fixed-dose budesonide plus formoterol, or fixed-dose fluticasone plus salmeterol for 6 months. Study 2 (open-label) consisted of 405 asthmatics and compared an adjustable regimen of budesonide plus formoterol with fixed-dose budesonide plus formoterol and fixed-dose fluticasone plus salmeterol for 7 months. The relation between ADRB2 polymorphism, severe asthma exacerbations, and other asthma outcomes was analyzed. Primary endpoints for studies 1 and 2 were severe asthma exacerbation and asthma control as assessed by measures of exacerbations, respectively. In study 1, Gly16Arg genotype had no effect on the percentage of participants with severe exacerbations across all treatment groups (99 [12%] of 833 Gly/Gly, 110 [11%] of 1028 Gly/Arg, and 32 [9%] of 361 Arg/Arg participants). Secondary endpoints, including forced expiratory volume in 1 s, peak expiratory flow, use of as-needed medication, and number of nights with awakenings were similar between genotype groups. No relation was recorded between ADRB2 haplotype and primary and secondary endpoints. In study 2, the frequency of asthma exacerbations (15 [9%] of 168 Gly/Gly, 13 [8%] of 169 Gly/Arg, and 6 [9%] of 67 Arg/Arg participants) and other study endpoints were closely similar for all ADRB2 genotypes. 

Hawkins and colleagues (2006) sought to identify ADRbeta2 polymorphisms and haplotype structure in white and African American subjects and to test for genotype and haplotype association with asthma phenotypes. A 5.3-kb region of ADRbeta2 was re-sequenced in 669 individuals from 429 whites and 240 African Americans. A total of 12 polymorphisms, representing an optimal haplotype tagging set, were genotyped in whites (338 patients and 326 control subjects) and African Americans (222 patients and 299 control subjects). A total of 49 polymorphisms were identified, 21 of which are novel; 31 polymorphisms (frequency > 0.03) were used to identify 24 haplotypes (frequency > 0.01) and assess linkage disequilibrium. Association with ratio (FEV1/FVC)2 for single-nucleotide polymorphism +79 (p < 0.05) was observed in African Americans. Significant haplotype association for (FEV1/FVC)2 was also observed in African Americans. The authors concluded, “these data suggest that the length of a poly-C repeat (+1269) in the 3' untranslated region of ADRbeta2 may influence lung function, and may be important in delineating variation in beta-agonist responses, especially in African Americans.” 

Litonjua (2006) writes the gene that encodes the beta2-adrenergic receptor (ADRB2) is one of the most studied candidate genes in asthma. Candidate gene association studies of ADRB2 and asthma have been dominated by analyses of the two common non-synonymous coding single nucleotide polymorphisms, Arg16Gly and Glu27Gly. Published studies have yielded inconsistent results. Three recent meta-analyses on the effects of these two polymorphisms have found no associations with asthma, although there were suggestions of associations with other asthma-related phenotypes, such as nocturnal asthma and asthma severity. Other recent studies have investigated other single nucleotide polymorphisms in this gene (i.e. single nucleotide polymorphisms in the promoter region and other single nucleotide polymorphisms in the coding region). These analyses have investigated the association between these single nucleotide polymorphisms (and haplotypes of these polymorphisms) and asthma-related phenotypes such as lung function, airways hyperresponsiveness, and response to a bronchodilator, and have suggested that certain regions of the gene may be associated with different phenotypes. Results from these studies, however, have also been inconsistent. Polymorphisms of ADRB2 are not major risk factors for the development of asthma. These polymorphisms are likely to be important, however, in determining drug response. Future studies need to fully characterize all of the variation in the gene and perform comprehensive association studies. Finally, interactions between ADRB2 and other genes in the beta-agonist pathway are an important and active area of research that will shed more light on inter-individual differences in drug response.

In 2005, Taylor et al measured bronchodilator response in patients with asthma stratified by ADRB2 haplotype after eliminating the confounding effect of prior drug treatment with inhaled beta2-agonists and corticosteroids. ADRB2 haplotype was determined in 176 patients with asthma, of whom 161 harbored the six most common combinations. There were no significant differences in bronchodilator response (% improvement in FEV(1)) with respect to any of the major ADRB2 haplotypes or genotypes. The authors concluded, “genetic variation of the ADRB2 does not influence the immediate response to inhaled beta2-agonist. The confounding effect of tolerance resulting from regular beta2-agonist use must be controlled when assessing the pharmacogenetic influences on clinical outcomes with beta2-agonists.” 

According to Johnson (2006), the presence of a particular form of the B2-receptor, which might influence clinical efficacy of regular B2-agonists, is of increasing interest in predicting good and poor responders to therapy.

MGMT Gene Methylation Assay for Gliomas

Glioblastomas are among the most aggressive of all known human tumors.  The median survival times remain in the 12 to15-month range despite aggressive surgery, radiation, and chemotherapy.  Through molecular and genetic profiling efforts, underlying mechanisms of resistance to these therapies are becoming better understood.  Resistance to alkylating agents via direct DNA repair by O(6)-methylguanine methyltransferase (MGMT) has been investigated as a barrier to successful treatment.  Assessment of MGMT status could help identify gliomas patients more likely to respond to chemotherapy or to benefit from MGMT depletion strategies.  Strategies to overcome MGMT-mediated chemo-resistance are being actively investigated.

Current guidelines from the National Comprehensive Cancer Network (NCCN, 2012) include recommendations for the use of the MGMT gene methylation assay in assessing likelihood of response to temozolomide in glioblastoma. The guidelines recommend use of temozolomide if methylguanine methyl-transferase [MGMT] promoter methylation is positive in persons with glioblastoma.

In a review, Hegi et al (2008) stated that one strategy to overcome MGMT-mediated chemo-resistance includes treatment with nontoxic pseudo-substrate inhibitors of MGMT, such as O(6)-benzylguanine, or RNA interference-mediated gene silencing of MGMT.  However, the author reported that systemic application of MGMT inhibitors is limited by an increase in hematologic toxicity.  Another strategy, the author explained, is to deplete MGMT activity in tumor tissue using a dose-dense temozolomide schedule.  The author stated that these alternative schedules are well tolerated; however, it remains unclear whether they are more effective than the standard dosing regimen or whether they effectively deplete MGMT activity in tumor tissue.  The author noted that not all patients with glioblastoma having MGMT promoter methylation respond to alkylating agents, and even those who respond will inevitably experience relapse.

Data from a retrospective study reported by Cancer Care Ontario (2006) suggested that newly diagnosed malignant glioma patients who had undergone surgery and external beam radiotherapy had a greater benefit from temozolomide treatment if their tumors had MGMT promoter methylation versus patients with unmethylated MGMT tumors.  However, it should be noted that these studies were performed retrospectively and prospective validation is required before MGMT methylation can be used for clinical stratification purposes. 

Idbaih et al (2007) reviewed recent studies on molecular markers including MGMT in the treatment of glioma and found evidence that MGMT inactivation is a prognostic marker and predictor of chemo-sensitivity in gliomas.  The author stated, "Although such markers remain to be formally validated by ongoing and planned prospective trials, it is likely that they will soon become essential for optimizing treatment decisions."

Millennium PGT

According to the Millennium Laboratories, Millennium PGT - Pharmacogenetic Testing is saliva-based testing designed to detect genetic variations in enzymes associated with the metabolism of medications commonly prescribed to patients suffering from chronic pain and pain-related effects.  Millennium PGT is designed to help clinicians more effectively personalize treatment by identifying patients who may benefit from modifying the drug selection or dosing of certain prescribed opioids, including codeine, hydrocodone, oxycodone, tramadol, and methadone; and benzodiazepines; tricyclic antidepressants (TCAs); selective serotonin reuptake inhibitors (SSRIs); and serotonin norepinephrine reuptake inhibitors (SNRIs). There is a lack of peer-reviewed publications on the Millennium PGT.

Vuilleumier et al (2012) stated that translating pharmacogenetics to clinical practice has been particularly challenging in the context of pain, due to the complexity of this multi-faceted phenotype and the overall subjective nature of pain perception and response to analgesia.  Overall, numerous genes involved with the pharmacokinetics and dynamics of opioids response are candidate genes in the context of opioid analgesia.  The clinical relevance of CYP2D6 genotyping to predict analgesic outcomes is still relatively unknown; the 2 extremes in CYP2D6 genotype (ultra-rapid and poor metabolism) seem to predict pain response and/or adverse effects.  Overall, the level of evidence linking genetic variability (CYP2D6 and CYP3A4) to oxycodone response and phenotype (altered biotransformation of oxycodone into oxymorphone and overall clearance of oxycodone and oxymorphone) is strong; however, there has been no randomized clinical trial on the benefits of genetic testing prior to oxycodone therapy.  On the other hand, predicting the analgesic response to morphine based on pharmacogenetic testing is more complex; though there was hope that simple genetic testing would allow tailoring morphine doses to provide optimal analgesia, this is unlikely to occur.  A variety of polymorphisms clearly influence pain perception and behavior in response to pain.  However, the response to analgesics also differs depending on the pain modality and the potential for repeated noxious stimuli, the opioid prescribed, and even its route of administration. 

Svetlík et al (2013) reviewed the impact of genetic variability of drug metabolizing enzymes, transporters, receptors, and pathways involved in chronic pain perception on the efficacy and safety of analgesics and other drugs used for chronic pain treatment.  Several candidate genes have been identified in the literature, while there is usually only limited clinical evidence substantiating for the penetration of the testing for these candidate biomarkers into the clinical practice.  Moreover, the authors noted that the pain-perception regulation and modulation are still not fully understood, and thus more complex knowledge of genetic and epigenetic background for analgesia will be needed prior to the clinical use of the candidate genetic biomarker.

Thiopurine S-Methyltransferase (TPMT)

Thiopurine s-methyltransferase (TPMT) is an enzyme that is involved in the metabolism of medications called thiopurines that are used in the treatment of inflammatory bowel disease (IBD). Genotyping tests (eg, PRO-PredictRx) and phenotyping tests (eg, PRO-Predict EnzAct) for TPMT enzyme activity can be used to help make treatment decisions involving thiopurine medications such as azathioprine and 6-mercaptopurine in IBD.

Coenen et al (2015) performed a prospective study to determine whether genotype analysis of thiopurine S-methyltransferase (TPMT) before thiopurine treatment, and dose selection based on the results, affected the outcomes of patients with inflammatory bowel disease (IBD). In a study performed at 30 Dutch hospitals, patients were assigned randomly to groups that received standard treatment (control) or pre-treatment screening (intervention) for 3 common variants of TPMT (TPMT*2, TPMT*3A, and TPMT*3C).  Patients in the intervention group found to be heterozygous carriers of a variant received 50 % of the standard dose of thiopurine (azathioprine or 6-mercaptopurine), and patients homozygous for a variant received 0 % to 10 % of the standard dose.  These researchers compared, in an intention-to-treat analysis, outcomes of the intervention (n = 405) and control groups (n = 378) after 20 weeks of treatment.  Primary outcomes were the occurrence of hematologic adverse drug reactions (ADRs) (leukocyte count of less than 3.0*10(9)/L or reduced platelet count of less than 100*10(9)/L) and disease activity (based on the Harvey-Bradshaw Index for Crohn's disease [n = 356] or the partial Mayo score for ulcerative colitis [n = 253]).  Similar proportions of patients in the intervention and control groups developed a hematologic ADR (7.4 % versus 7.9 %; RR, 0.93; 95 % CI: 0.57 to 1.52) in the 20 weeks of follow-up evaluation; the groups also had similar mean levels of disease activity (p = 0.18 for Crohn's disease and p = 0.14 for ulcerative colitis).  However, a significantly smaller proportion of carriers of the TPMT variants in the intervention group (2.6 %) developed hematologic ADRs compared with patients in the control group (22.9 %) (RR, 0.11; 95 % CI: 0.01 to 0.85).  The authors concluded that screening for variants in TPMT did not reduce the proportions of patients with hematologic ADRs during thiopurine treatment for IBD.  However, there was a 10-fold reduction in hematologic ADRs among variant carriers who were identified and received a dose reduction, compared with variant carriers who did not, without differences in treatment efficacy.

Plumpton and colleagues (2016) stated that pharmacogenetics offers the potential to improve health outcomes by identifying individuals who are at greater risk of harm from certain medicines. Routine adoption of pharmacogenetic tests requires evidence of their cost-effectiveness.  These researchers conducted a systematic literature review of economic evaluations of pharmacogenetic tests aimed to reduce the incidence of ADRs.  Literature was searched using Embase, Medline and the NHS Economic Evaluation Database with search terms relating to pharmacogenetic testing, adverse drug reactions, economic evaluations and pharmaceuticals.  Titles were screened independently by 2 reviewers.  Articles deemed to meet the inclusion criteria were screened independently on abstract, and full texts reviewed.  These investigators identified 852 articles, of which 47 met the inclusion criteria. 

There was evidence supporting the cost-effectiveness of testing for the following:

HLA-B*57:01 (prior to abacavir)
HLA-B*15:02 and HLA-A*31:01 (prior to carbamazepine)
HLA-B*58:01 (prior to allopurinol)
CYP2C19 (prior to clopidogrel treatment)

Economic evidence was inconclusive with respect to the following:

TPMT (prior to 6-mercaptoputine, azathioprine and cisplatin therapy)
CYP2C9 and VKORC1 (to inform genotype-guided dosing of coumarin derivatives)
MTHFR (prior to methotrexate treatment) and factor V Leiden testing (prior to oral contraception)

Testing for A1555G is not cost-effective before prescribing aminoglycosides.

The authors concluded that systematic review identified robust evidence of the cost-effectiveness of genotyping prior to treatment with a number of common drugs. However, further analyses and (or) availability of robust clinical evidence is necessary to make recommendations for others.

PROOVE Narcotic Risk Test

The PROOVE narcotic risk test is comprised of 12 panels designed to predict risk of narcotic addiction.  These 12 genetic assessment tests include serotonin 2A receptor, serotonin transporter, catechol-o-methyl transferase, dopamine D2 receptor dopamine D1 receptor, dopamine D4 receptor, dopamine transporter, dopamine beta hydroxylase, methylene tetrahydrofolate reductase, kappa opioid receptor, gamma-aminobutyric acid, and mu opioid receptor.  The clinical utility of these tests used prior to onset of narcotic pain relief has not been established in the peer-reviewed literature.

Aegis Drug-Drug Interaction Test

Aegis Drug-Drug Interaction Test is urinary test that is used for drug-monitoring (120 or more drugs and substances) by means of tandem mass spectrometry with liquid chromatography.  It provides a qualitative report of presence (including quantitative levels, when detected) or absence of each drug or substance with description and severity of potential interactions, with identified substances.

Chen and colleagues (2016) noted that amino acids (AAs), neurotransmitters, purines, and pyrimidines are bioactive molecules that play fundamental roles in maintaining various physiological functions.  Their metabolism is closely related to the health, growth, development, reproduction, and homeostasis of organisms.  Most recently, comprehensive measurements of these metabolites have shown their potential as innovative approaches in disease surveillance or drug intervention.  However, simultaneous measurement of these metabolites presents great difficulties.  These researchers reported a novel quantitative method that uses hydrophilic interaction ultra-high-performance liquid chromatography-tandem mass spectrometry (HILIC-UPLC-MS/MS), which is highly selective, high throughput, and exhibits better chromatographic behavior than existing methods.  The developed method enabled the rapid quantification of 37 metabolites, spanning AAs, neurotransmitters, purines, and pyrimidines pathways, within 6.5 minutes.  The compounds were separated on an ACQUITY UPLC BEH amide column.  Serum and brain homogenate were extracted by protein precipitation.  The intra- and inter-day precision of all of the analytes was less than 11.34 %, and the accuracy was between -11.74 and 11.51 % for all quality control (QC) levels.  The extraction recoveries of serum ranged from 84.58 % to 116.43 % and those of brain samples from 80.80 % to 119.39 %, while the relative standard deviation (RSD) was 14.61 % or less for all recoveries.  This method was used to successfully characterize alterations in the rat brain and, in particular, their dynamics in serum.  The following study was performed to simultaneously test global changes of these metabolites in a serotonin antagonist p-chlorophenylalanine (PCPA)-induced anxiety and insomnia rat model to understand the effect and mechanism of PCPA.  Taken together, these results showed that the method is able to simultaneously monitor a large panel of metabolites and that this protocol may represent a metabolomic method to diagnose toxicological and pathophysiological states.

Signatera

Signatera is a custom-built circulating tumor DNA (ctDNA) test for molecular treatment monitoring and molecular residual disease (MRD) assessment.  It can be used for individuals with stage II/III colorectal cancer who are considering adjuvant chemotherapy (ACT) and/or who are being monitored for relapse post-treatment.  However, there is currently insufficient evidence to support  its clinical effectiveness.

Reinert and colleagues (2019) stated that novel sensitive methods for detection and monitoring of residual disease could improve post-operative risk stratification with implications for patient selection for adjuvant chemotherapy (ACT), ACT duration, intensity of radiologic surveillance, and, ultimately, outcome for patients with colorectal cancer (CRC).  In a prospective, multi-center cohort study, these investigators examined the association of circulating tumor DNA (ctDNA) with recurrence using longitudinal data from ultra-deep sequencing of plasma cell-free DNA in patients with CRC before and after surgery, during and after ACT, and during surveillance.  ctDNA was quantified in the pre-operative and post-operative settings of stages I to III CRC by personalized multiplex, PCR-based, next-generation sequencing.  The study enrolled 130 patients at the surgical departments of Aarhus University Hospital, Randers Hospital, and Herning Hospital in Denmark from May 1, 2014, to January 31, 2017.  Plasma samples (n = 829) were collected before surgery, post-operatively at day 30, and every 3rd month for up to 3 years.  Main outcomes were ctDNA measurement, clinical recurrence, and RFS.  A total of 130 patients with stages I to III CRC (mean [SD] age of 67.9 [10.1] years; 74 [56.9 %] men) were enrolled in the study; 5 patients discontinued participation, leaving 125 patients for analysis.  Pre-operatively, ctDNA was detectable in 84 of 94 patients (89.4 %).  After definitive treatment, longitudinal ctDNA analysis identified 14 of 16 relapses (87.5 %).  At post-operative day 30, ctDNA-positive patients were 7 times more likely to relapse than ctDNA-negative patients (HR, 7.2; 95 % CI: 2.7 to 19.0; p < 0.001).  Similarly, shortly after ACT ctDNA-positive patients were 17 times (HR, 17.5; 95 % CI: 5.4 to 56.5; p < 0.001) more likely to relapse.  All 7 patients who were ctDNA positive after ACT experienced relapse.  Monitoring during and after ACT indicated that 3 of the 10 ctDNA-positive patients (30.0 %) were cleared by ACT.  During surveillance after definitive therapy, ctDNA-positive patients were more than 40 times more likely to experience disease recurrence than ctDNA-negative patients (HR, 43.5; 95 % CI: 9.8 to 193.5; p < 0.001).  In all multi-variate analyses, ctDNA status was independently associated with relapse after adjusting for known clinicopathologic risk factors.  Serial ctDNA analyses revealed disease recurrence up to 16.5 months ahead of standard-of-care radiologic imaging (mean of 8.7 months; range of 0.8 to 16.5 months).  Actionable mutations were identified in 81.8 % of the ctDNA-positive relapse samples.  The authors concluded that circulating tumor DNA analysis could potentially change the post-operative management of CRC by enabling risk stratification, ACT monitoring, and early relapse detection.  They stated that there are potential limitations to this study, including the modest sample size of patients with recurrent CRC and the analysis of multiple patient subsets.  Moreover, these researchers stated that these findings suggested many potentially paradigm-changing clinical applications of ctDNA in CRC and provide a framework for future clinical trials to examine the clinical benefits of ctDNA-guided disease management. 

Tie and associates (2019) noted that ACT in patients with stage III colon cancer prevents recurrence by eradicating MRD.  However, which patients remain at high risk of recurrence after completing standard adjuvant treatment cannot currently be determined.  Post-surgical ctDNA analysis can detect MRD and is associated with recurrence in CRCs.  In a multi-center, population-based, cohort biomarker study, these investigator examined if serial post-surgical and post-chemotherapy ctDNA analysis could provide a real-time indication of adjuvant therapy efficacy in stage III colon cancer.  This trial recruited 100 consecutive patients with newly diagnosed stage III colon cancer planned for 24 weeks of ACT from November 1, 2014, through May 31, 2017.  Patients with another malignant neoplasm diagnosed within the last 3 years were excluded.  Median duration of follow-up was 28.9 months (range of 11.6 to 46.4 months).  Physicians were blinded to ctDNA results.  Data were analyzed from December 10, 2018, through June 23, 2019.  Serial plasma samples were collected after surgery and after chemotherapy.  Somatic mutations in individual patients' tumors were identified via massively parallel sequencing of 15 genes commonly mutated in CRC; personalized assays were designed to quantify ctDNA.  Main outcome measures included detection of ctDNA and recurrence-free interval (RFI).  After 4 exclusions, 96 eligible patients were eligible; median patient age was 64 years (range of 26 to 82); 49 (51 %) were men.  At least 1 somatic mutation was identified in the tumor tissue of all 96 evaluable patients.  Circulating tumor DNA was detectable in 20 of 96 (21 %) post-surgical samples and was associated with inferior RFS (HR, 3.8; 95 % CI: 2.4 to 21.0; p < 0.001); ctDNA was detectable in 15 of 88 (17 %) post-chemotherapy samples.  The estimated 3-year RFI was 30 % when ctDNA was detectable after chemotherapy and 77 % when ctDNA was undetectable (HR, 6.8; 95 % CI: 11.0 to 157.0; p < 0.001).  Post-surgical ctDNA status remained independently associated with RFI after adjusting for known clinicopathologic risk factors (HR, 7.5; 95 % CI: 3.5 to 16.1; p < 0.001).  The authors concluded that the findings of this study suggested that ctDNA analysis after surgery is a promising prognostic marker in stage III colon cancer.  Post-chemotherapy ctDNA analysis may define a patient subset that remains at high risk of recurrence despite completing standard adjuvant treatment.

Wang and colleagues (2019) stated that for patients with resected, non-metastatic CRC, the optimal surveillance protocol remains unclear.  These researchers examined if serial ctDNA levels detected disease recurrence earlier, compared with conventional post-operative surveillance, in patients with resected CRC.  This study included patients (n = 58) with stage I, II, or III CRC who underwent radical surgical resection at 4 Swedish hospitals from February 2, 2007, to May 8, 2013; 18 patients received ACT at the discretion of their clinicians, who were blinded to the ctDNA results.  Blood samples were collected at 1 month after the surgical procedure and every 3 to 6 months thereafter for ctDNA analysis.  Patients were followed-up until metachronous metastases were detected, or for a median of 49 months.  Data analysis was performed from March 1, 2009 to June 23, 2018.  Sensitivity and timing of ctDNA positivity were compared with those of conventional surveillance modalities (CT scans and serum CEA tests) for the detection of disease recurrence.  This study included 319 blood samples from 58 patients, with a median (range) age of 69 (47 to 83) years and 34 men (59 %).  The recurrence rate among patients with positive ctDNA levels was 77 % (10 of 13 patients).  Positive ctDNA preceded radiologic and clinical evidence of recurrence by a median of 3 months.  Of the 45 patients with negative ctDNA throughout follow-up, none (0 %; 95 % CI: 0 % to 7.9 %) experienced a relapse, with a median follow-up of 49 months.  However, 3 (6 %; 95 % CI: 1.3 % to 17 %) of the 48 patients without relapse had a positive ctDNA result, which subsequently fell to undetectable levels during follow-up.  The authors concluded that although these findings needed to be validated in a larger, prospective trial, they suggested that ctDNA analysis could complement conventional surveillance strategies as a triage test to stratify patients with resected CRC on the basis of risk of disease recurrence.

The authors stated that this study was limited by the sample size, involving only 319 blood samples from 58 patients.  Nevertheless, it was encouraging that the 10 patients who relapsed clinically had positive ctDNA levels that preceded radiographic evidence of recurrence.  Although ctDNA positivity preceded recurrence by a median of 4 months in patients who did not receive adjuvant chemotherapy, the blood samples did not become ctDNA positive until a median of 9 months after surgical resection.  This lead time would not be early enough to affect the decision on ACT; however, it might still be sufficient to allow for earlier implementation of other curative or palliative strategies.  These investigators suspected that the shorter lead time may, in part, be associated with the higher frequency of imaging than recommended by many guidelines for stage II or III disease; 2/3 of the patients in the study underwent CT imaging every 6 months, and the remaining 1/3 was imaged every 12 months.

Reece and associates (2019) noted that CRC is one of the most common cancers worldwide and has a high mortality rate following disease recurrence.  Treatment efficacy is maximized by providing tailored cancer treatment, ideally involving surgical resection and personalized neoadjuvant and adjuvant therapies, including chemotherapy, radiotherapy and increasingly, targeted therapy.  Early detection of recurrence or disease progression results in more treatable disease and is essential to improving survival outcomes.  Recent advances in the understanding of tumor genetics have resulted in the discovery of ctDNA.  These investigators carried out a literature search in PubMed to identify all original articles preceding April 2019 that employ ctDNA for the purpose of monitoring response to CRC treatment.  A total of 92 studies were included.  These studies showed that ctDNA is a reliable measure of tumor burden; they demonstrated the utility of ctDNA in assessing the adequacy of surgical tumor clearance and changes in ctDNA levels reflected response to systemic treatments.  ctDNA could be used in the selection of targeted treatments.  The re-appearance or increase in ctDNA, as well as the emergence of new mutations, correlated with disease recurrence, progression, and resistance to therapy, with ctDNA measurement allowing more sensitive monitoring than currently used clinical tools.  The authors concluded that ctDNA showed enormous promise as a sensitive biomarker for monitoring response to many treatment modalities and for targeting therapy; therefore, it is emerging as a new way for guiding treatment decisions-initiating, altering, and ceasing treatments, or prompting investigation into the potential for residual disease.  However, many potentially useful ctDNA markers are available and more work is needed to determine which are best suited for specific purposes and for improving specific outcomes.

The authors stated that 1 drawback of this review was the lack of standardization of ctDNA detection methods between studies.  Assay methods include PCR, droplet digital PCR, BEAMing, and NGS.  These methods can vary greatly in cost, and not all technologies are available in standard laboratories.  For standardization it is also important to establish quality controls to account for the inter-laboratory variability.  A recent study showed that through sending ctDNA samples to 32 laboratories across Europe for mutation testing, and using 6 different cell-free DNA extraction methods and 5 different analysis methods, it led to a percentage of errors that could have had implications for clinical decision-making around therapy of 20.1 %.  It has also been shown that artefactual KRAS mutations may occur, depending on the method applied.  Setting a threshold for the mutant allele frequency may reduce the incidence of false results.  Similarly, ultra-sensitive techniques should also be applied, otherwise the absence of detectable ctDNA may indicate a poor quality sample or analysis, rather than the absence of disease.  A further drawback of the studies was that very few established whether ctDNA was an independent predictor for treatment response.  Of 20 studies that calculated a HR of ctDNA for assessing either PFS or OS, only 5 studies performed multi-variate analysis, correcting for other important variables that may affect survival.

Wang and co-workers (2020) noted that surgical resection is the primary treatment for patients with non-metastatic CRC.  However, even after undergoing radical resection procedure, 30 to 50 % of patients will still experience relapse.  Circulation tumor DNA, deriving from tumor cells, is shed into the bloodstream and is a potential predictive biomarker of recurrence in CRC.  In a meta-analysis, these researchers examined the clinical value of ctDNA in predicting the recurrence of CRC patients in post-operative.  PubMed, Embase, the Cochrane Library, and Web of Science were searched to identify the studies that reported the function of ctDNA for predicting recurrence in CRC patients.  The eligible studies were pooled to calculate the relative risk (RR) of recurrence in ctDNA positive and negative groups.  The data of ctDNA on RFS were extracted and computed in HR and 95 % CI; subgroup analyses were also performed.  A total of 7 studies including 424 patients were included and analyzed in this meta-analysis.  The results showed that pooled RR was 4.65 (95 % CI: 2.68 to 8.08, p < 0.05), indicating positive ctDNA could predict the recurrence of CRC after curative surgical.  The pooled HR demonstrated strong connection between ctDNA positive and RFS in patients with CRC (HR = 9.14, 95 % CI: 4.02 to 20.75, p < 0.05).  The authors concluded that evidence from the meta-analysis suggested that ctDNA is a promising potential biomarker for predicting post-operative recurrence of CRC.  Moreover, these researchers stated that given the inherent limitations of this study, they look forward to more well-designed clinical studies to validate and update this analysis in the future.

Furthermore, National Comprehensive Cancer Network’s clinical practice guidelines on "Colon cancer" (Version 4.2020) and "Rectal cancer" (Version 6.2020) do not mention measurement of cell-free DNA as a management tool.

Bratman et al (2020) noted that immune checkpoint blockade (ICB) provides clinical benefit to a subset of patients with cancer; however, existing biomarkers do not reliably predict treatment response across diverse cancer types.  Limited data exist to show how serial circulating tumor DNA (ctDNA) testing may perform as a predictive biomarker in patients receiving ICB.  In a prospective, phase-II clinical trial, these researchers examined ctDNA in 5 distinct cohorts of patients with advanced solid tumors treated with pembrolizumab.  They employed bespoke ctDNA assays to 316 serial plasma samples obtained at baseline and every 3 cycles from 94 patients.  Baseline ctDNA concentration correlated with progression-free survival (PFS), overall survival (OS), clinical response and clinical benefit.  This association became stronger when considering ctDNA kinetics during treatment.  All 12 patients with ctDNA clearance during treatment were alive with median 25 months follow-up.  The authors concluded that the findings of this phase-II clinical trial suggested broad clinical use for ctDNA-based surveillance in patients treated with ICB.  This is a non-invasive strategy to predict clinical benefit and long-term survival that could be generalizable across cancer types.  They stated that future interventional studies are needed to enable clinical decisions using ctDNA levels to guide ICB treatment as well as surveillance in patients treated with ICB.

Loupakis et al (2021) stated that more than 50 % of patients with stage IV colorectal cancer (metastatic colorectal cancer [mCRC]) relapse post-resection.  The effectiveness of post-operative systemic treatment is limited in this setting; thus, these patients would greatly benefit from the use of a reliable prognostic biomarker, such as ctDNA to identify minimal or molecular residual disease (MRD).  In a retrospective study, these researchers analyzed a cohort of 112 patients with mCRC who had undergone metastatic resection with curative intent as part of the PREDATOR clinical trial.  The study examined the prognostic value of ctDNA, correlating MRD status post-surgery with clinical outcomes by using a personalized and tumor-informed ctDNA assay (bespoke multiple PCR, next-generation sequencing [NGS] assay).  Post-resection, systemic therapy was administered to 39.2 % of the patients at the discretion of the treating physician.  Post-surgical, MRD positivity was observed in 54.4 % (61 of 112) of patients, of which 96.7 % (59 of 61) progressed at the time of data cut-off (hazard ratio [HR]: 5.8; 95 % CI: 3.5 to 9.7; p < 0.001).  MRD-positive status was also associated with an inferior OS: HR: 16.0; 95 % CI: 3.9 to 68.0; p < 0.001.  At the time of analyses, 96 % (49 of 51) of patients were alive in the MRD-negative arm compared with 52.4 % (32 of 61) in the MRD-positive arm.  Patients who did not receive systemic therapy and were MRD-negative in the combined ctDNA analysis at 2 time-points had an OS of 100 %.  In the multi-variate analysis, ctDNA-based MRD status was the most significant prognostic factor associated with disease-free survival (DFS; HR: 5.78; 95 % CI: 3.34 to 10.0; p < 0.001).  The authors concluded that the findings of this study suggested that clinical trials centered on patients with mCRC could benefit from the implementation of ctDNA testing in their design.  For example, clinical trials could benefit from patient stratification on the basis of their post-surgical MRD status before randomly assigning patients into a treatment versus placebo arm.  Furthermore, MRD-guided trials could also benefit by enriching patients with high-risk of relapse, leading to significant reductions in trial sample size and unnecessary treatment cost.  In addition, the use of ctDNA as a surrogate endpoint for treatment response monitoring is being actively investigated, wherein an early indication of treatment efficacy (ctDNA clearance) relative to conventional strategies may lead to expedited approval of new therapies.  The results of this trial support the continuous expansion of the number of clinical studies in patients with mCRC using personalized ctDNA-based MRD analysis and provides direct evidence of the predictive and prognostic value of ctDNA, which could help clinicians and researchers with real numbers to design their clinical studies and support therapeutic decisions in the adjuvant setting.  These researchers stated that post-operative MRD analysis could be a strong prognostic biomarker for patients with mCRC undergoing resection of metastases; it holds promises for being implemented in clinical decision-making, informing clinical trial design, and further translational research.

The authors stated that drawbacks of this study included a relatively small sample size (n = 112) and the retrospective design with the use of archived samples, because of which these researchers observed a high degree of necrosis and low tumor cellularity in the tissue samples, resulting in a WES QC failure rate of 16.9 %.  This was because some patients received systemic treatment before liver or met resection, resulting in deteriorated sample quality.  However, in real-world, prospective studies with analysis performed on primary or untreated samples, these researchers had observed a WES QC failure rate of less than 3 %, which was compatible with clinical practice.  Another drawback of this study was that these investigators only tested 2 time-points (post-surgical) in this setting.  They stated that their future studies will include monitoring of ctDNA dynamics using serial testing at regular intervals within a subgroup of patients from the present analysis.  Overall, these investigators believed that the use of serial testing could allow for tailoring of treatment regimens, with treatment escalation in patients with progressive disease and the opportunity of early therapeutic interventions with more aggressive follow-up in patients who were ctDNA-positive but have not yet progressed.  Furthermore, previous studies had shown ctDNA clearance to be a proxy of treatment efficacy, especially in the adjuvant setting where post-operative ctDNA negativity showed better outcomes compared with ctDNA-positive patients.  The authors acknowledged the limitation of their small dataset to establish this evidence.  However, they did see a clear pattern, wherein 100 % of the patients who remained positive or turned positive at the 2nd time-point progressed as compared with the ctDNA-negative patients.

Powles et al (2021) noted that minimally invasive approaches to detect residual disease following surgery are needed to identify patients with cancer who are at risk for metastatic relapse.  Circulating tumor DNA (ctDNA) holds promise as a biomarker for molecular residual disease and relapse.  These researchers examined outcomes in 581 patients who had undergone surgery and were evaluable for ctDNA from a randomized phase-III clinical trial of adjuvant atezolizumab versus observation in operable urothelial cancer.  This trial did not reach its efficacy endpoint in the intention-to-treat (ITT) population.  These investigators showed that ctDNA testing at the start of therapy (cycle 1 day 1) identified 214 (37 %) patients who were positive for ctDNA and who had poor prognosis (observation arm hazard ratio [HR] = 6.3 (95 % confidence interval [CI]: 4.45 to 8.92); p < 0.0001).  Notably, patients who were positive for ctDNA had improved disease-free survival (DFS) and overall survival (OS) in the atezolizumab arm versus the observation arm (DFS HR = 0.58 (95 % CI: 0.43 to 0.79); p = 0.0024, OS (HR = 0.59 (95 % CI: 0.41 to 0.86)).  No difference in DFS or OS between treatment arms was noted for patients who were negative for ctDNA.  The rate of ctDNA clearance at week 6 was higher in the atezolizumab arm (18 %) than in the observation arm (4 %) (p = 0.0204).  Transcriptomic analysis of tumors from patients who were positive for ctDNA revealed higher expression levels of cell-cycle and keratin genes.  For patients who were positive for ctDNA and who were treated with atezolizumab, non-relapse was associated with immune response signatures and basal-squamous gene features, whereas relapse was associated with angiogenesis and fibroblast TGFβ signatures.  The authors concluded that these findings suggested that adjuvant atezolizumab may be associated with improved outcomes compared with observation in patients who were positive for ctDNA and who were at a high risk of relapse.  These findings, if validated in other settings, would shift approaches to post-operative cancer care.

The authors stated that this study had 2 main drawbacks.  First, ctDNA status was a prospective but exploratory endpoint; thus, further studies are needed to validate and expand its clinical utility.  Second, this method of ctDNA analysis required whole-exome sequencing (WES) and took approximately 2 to 3 weeks.  This is clinically applicable in the adjuvant setting where patients require a period of time to recover from surgery before starting therapy.

In an expert commentary regarding the study by Powles et al (2021), Woldu and Lotan (2021) stated that “While this is all quite exciting, the results are far from ideal and further work is required.  As an example, 31 % of patients who were negative for ctDNA in the observation arm in this study experienced relapse.  This lack of sensitivity may be due to technical limitations of the assay, which could reasonably be expected to be mitigated with further advances.  However, there are also likely biologic limitations, such as non-shedding tumors and heterogeneity in tumors that relapse, that will inherently limit the sensitivity of such assays.  In addition, while ctDNA may very well correlate with MRD and have prognostic value, novel biomarkers with predictive capacity that might guide future adjuvant options are needed”.

Kotani et al (2023) stated that despite standard-of-care (SOC) treatment, more than 30 % of patients with resectable colorectal cancer (CRC) relapse; and ctDNA analysis may enable post-surgical risk stratification and adjuvant chemotherapy (ACT) treatment decision-making.  In an observational study, these investigators reported results from GALAXY, which is an observational arm of the ongoing CIRCULATE-Japan study that analyzed pre-surgical and post-surgical ctDNA in patients with stage II-IV resectable CRC (n = 1,039).  In this cohort, with a median follow-up of 16.74 months (range of 0.49 to 24.83 months), post-surgical ctDNA positivity (at 4 weeks after surgery) was associated with higher recurrence risk (hazard ratio (HR) 10.0, p < 0.0001) and was the most significant prognostic factor associated with recurrence risk in patients with stage II or III CRC (HR 10.82, p < 0.001).  Furthermore, post-surgical ctDNA positivity identified patients with stage II or III CRC who derived benefit from ACT (HR 6.59, p < 0.0001).  The authors concluded that in this large, observational study, they showed that post-surgical ctDNA status is a most significant prognostic biomarker than the currently used high-risk clinicopathological features and could potentially be predictive of ACT benefit.  These researchers stated that ongoing prospective, randomized trials will further examine the optimal ctDNA-guided treatment strategy for surgically resectable CRC.

The authors stated that potential limitations of these results included the observational nature of the study and the bias in patient characteristics.  To partly mitigate these potential biases, these investigators carried out a multi-variate analysis, demonstrating the clear benefit of ACT for reduction of recurrence risk in ctDNA-positive patients 4 weeks after surgery.  Another limitation of the study was the non-feasibility of performing a randomized trial of ACT versus the observation arm in post-surgical ctDNA-positive patients in Japan.  However, these findings were supportive of the benefit of ACT and may need further investigation in a prospective randomized trial with adequate follow-up to evaluate the non-inferiority of the observation arm versus ACT in ctDNA-negative patients.  These data are supported by the results of the recently published prospective, randomized DYNAMIC trial.  Of 455 patients with stage II colon cancer, 294 underwent ctDNA-guided adjuvant therapy, and 147 underwent standard management (median follow-up of 37 months).  ctDNA-guided management reduced the proportion of patients receiving adjuvant therapy (15 % in the ctDNA-guided arm versus 28 % in the standard management arm), without compromising 2-year recurrence-free survival (RFS; 93.5 % in the ctDNA-guided arm versus 92.4 % in the standard management arm), implying that ctDNA-guided ACT is not inferior to standard management.  Interestingly, patients who were ctDNA-negative were not treated, and the 3-year RFS for ctDNA-negative patients was 92.5 % compared with 86.4 % for ctDNA-positive patients.

An assessment of Signatera by the National Institute for Health and Clinical Excellence (2022) stated that "[k]ey uncertainties around the evidence or technology are that there is no prospective evidence on using Signatera in clinical practice or its effect on treatment decisions or clinical outcomes.. . . Experts advised that there is not enough evidence to support routine use of the technology in the NHS. This is in line with recommendations from the European Society for Medical Oncology on the use of ctDNA. But there are several ongoing trials that may address gaps in the evidence."

Natera's Signatera Molecular Monitoring (MRD) for Breast Cancer

Natera’s Signatera (RUO) is the first ctDNA assay custom-built for treatment monitoring and molecular residual disease assessment.  The Signatera (RUO) methodology differs from currently available liquid biopsy assays, which test for a panel of genes independent of an individual's tumor.

Oellerich et al (2017) stated that high-quality genomic analysis is critical for personalized pharmacotherapy in patients with cancer.  Tumor-specific genomic alterations can be identified in cell-free DNA (cfDNA) from patient blood samples and can complement biopsies for real-time molecular monitoring of treatment, detection of recurrence, and tracking resistance.  cfDNA can be especially useful when tumor tissue is unavailable or insufficient for testing.  For blood-based genomic profiling, next-generation sequencing (NGS) and droplet digital PCR (ddPCR) have been successfully applied.  The Food and Drug Administration (FDA) recently approved the first such "liquid biopsy" test for EGFR mutations in patients with non-small cell lung cancer (NSCLC).  Such non-invasive methods allow for the identification of specific resistance mutations selected by treatment, such as EGFR T790M, in patients with NSCLC treated with gefitinib.  Chromosomal aberration pattern analysis by low coverage whole genome sequencing is a more universal approach based on genomic instability.  Gains and losses of chromosomal regions have been detected in plasma tumor-specific cfDNA as copy number aberrations and can be used to compute a genomic copy number instability (CNI) score of cfDNA.  A specific CNI index obtained by massive parallel sequencing discriminated those patients with prostate cancer from both healthy controls and men with benign prostatic disease.  Furthermore, androgen receptor gene aberrations in cfDNA were associated with therapeutic resistance in metastatic castration resistant prostate cancer.  Change in CNI score has been shown to serve as an early predictor of response to standard chemotherapy for various other cancer types (e.g. NSCLC, colorectal cancer, pancreatic ductal adenocarcinomas).  CNI scores have also been shown to predict therapeutic responses to immunotherapy.  Serial genomic profiling can detect resistance mutations up to 16 weeks before radiographic progression.  There is a potential for cost savings when ineffective use of expensive new anticancer drugs is avoided or halted.  Challenges for routine implementation of liquid biopsy tests include the necessity of specialized personnel, instrumentation, and software, as well as further development of quality management (e.g., external quality control).  Validation of blood-based tumor genomic profiling in additional multi-center outcome studies is needed; however, cfDNA monitoring can provide clinically important actionable information for precision oncology approaches.

Volckmar et al (2018) noted that recently, many genome-wide profiling studies provided insights into the molecular make-up of major cancer types.  The deeper understanding of these genetic alterations and their functional consequences led to the discovery of novel therapeutic opportunities improving clinical management of cancer patients.  While tissue-based molecular patient stratification is the gold standard for precision medicine, it has certain limitations:  Tissue biopsies are invasive sampling procedures carrying the risk of complications and may not represent the entire tumor due to underlying genetic heterogeneity.  In this context, complementary characterization of genetic information in the blood of cancer patients can serve as minimal-invasive 'liquid biopsy'.  Fragments of cfDNA are released from tissues of healthy individuals as well as cancer patients.  The fraction of cfDNA that is released from primary tumors or metastases (i.e., circulating tumor DNA, ctDNA) represents genetic aberrations in cancer cells, which are a potential source for diagnostic, prognostic, and predictive biomarkers.  Recent studies have demonstrated technical feasibility and clinical applications including detection of drug targets and resistance mutations as well as longitudinal monitoring of tumors under therapy.  To this end, a variety of pre-analytical procedures for blood processing, isolation and quantification of cfDNA are being employed and several analytical methods and technologies ranging from PCR-based single locus assays to genome-wide approaches are available, which considerably differ in sensitivity, specificity, and throughput.  However, broad implementation of ctDNA analysis in daily clinical practice requires a thorough understanding of theoretical, technical, and biological concepts and necessitates standardization and validation of pre-analytical and analytical procedures across different technologies.

Oellerich et al (2019) stated that genomic analyses in oncologic care allow for the development of more precise clinical laboratory tests that will be critical for personalized pharmacotherapy.  Traditional biopsy-based approaches are limited by the availability of sequential tissue specimens to detect resistance.  Blood-based genomic profiling ("liquid biopsy") is useful for longitudinal monitoring of tumor genomes and can complement biopsies.  Tumor-associated mutations can be identified in ctDNA from patient blood samples and used for monitoring disease activity.  The FDA approved a liquid biopsy test for EGFR-activating mutations in patients with NSCLC as a companion diagnostic for therapy selection.  ctDNA also allows for the identification of mutations selected by treatment such as EGFR T790M in NSCLC.  ctDNA can also detect mutations such as KRAS G12V in colorectal cancer and BRAF V600E/V600K in melanoma.  Chromosomal aberration pattern analysis by low-coverage whole genome sequencing is a new, broader approach.  Genomic imbalances detected in cfDNA can be used to compute a CNI score.  In clinical studies, it was demonstrated that the change in CNI score can serve as an early predictor of therapeutic response to chemotherapy/immunotherapy of many cancer types.  In multi-variable models, it could be shown that the CNI score was superior to clinical parameters for prediction of overall survival in patients with head and neck cancer.  There is emerging evidence for the clinical validity of ctDNA testing regarding identification of candidates for targeted therapies, prediction of therapeutic response, early detection of recurrence, resistance mutation detection, measuring genetic heterogeneity, tumor burden monitoring, and risk stratification.  Improvement of sensitivity to detect tumors at very early stages is difficult due to insufficient mutant DNA fraction of less than or equal to 0.01 %.  The authors stated that further developments will include validation in prospective multi-center interventional outcome studies and the development of digital platforms to integrate diagnostic data.

Coombes and colleagues (2019) stated that up to 30 % of patients with breast cancer relapse after primary treatment.  There are no sensitive and reliable tests to monitor these patients and detect distant metastases before overt recurrence.  These researchers reported on the use of personalized circulating tumor DNA (ctDNA) profiling for detection of recurrence in breast cancer.  A total of 49 primary patients with breast cancer were recruited following surgery and adjuvant therapy.  Plasma samples (n = 208) were collected every 6 months for up to 4 years.  Personalized assays targeting 16 variants selected from primary tumor whole-exome data were tested in serial plasma for the presence of ctDNA by ultra-deep sequencing (average greater than 100,000X).  Plasma ctDNA was detected ahead of clinical or radiologic relapse in 16 of the 18 relapsed patients (sensitivity of 89 %); metastatic relapse was predicted with a lead time of up to 2 years (median of 8.9 months; range of 0.5 to 24.0 months).  None of the 31 non-relapsing patients was ctDNA-positive at any time-point across 156 plasma samples (specificity of 100 %).  Of the 2 relapsed patients who were not detected in the study, the 1st had only a local recurrence, whereas the 2nd patient had bone recurrence and had completed chemotherapy just 13 days prior to blood sampling.  The authors concluded that they presented a sensitive and specific clinical test that could be used to identify pre-clinical metastases and follow all patients with breast cancer following therapy irrespective of molecular subtype.  It out-performs conventional means of monitoring and showed promise as a tool for guiding future precision medicine.  Moreover, these researchers stated that future studies will address the issue of the effects of therapy on ctDNA levels in patients with breast cancer.

The authors stated that this study had several drawbacks.  The test is not suitable for detecting a 2nd primary breast cancer unless it recurred from the original tumor; this was exemplified by a patient, where a 2nd contralateral primary cancer was detected.  Second, relying on ctDNA required that sufficient molecules were present in the plasma at the time of collection, which might not have been the case in patients with smaller and less aggressive breast cancers.  This was exemplified by a patient who relapsed with local resectable disease but was ctDNA-negative.

Hufnagl et al (2020) noted that the clinical decisions made when treating patients with metastatic cancer require knowledge of the current tumor extent and response to therapy.  For the majority of solid tumors, a response assessment, which is based on imaging, is used to guide these decisions.  However, measuring serum protein biomarkers (i.e., tumor markers) may be of additional use.  Furthermore, tumor markers exhibit variable specificity and sensitivity and cannot therefore be solely relied upon when making decisions regarding cancer treatment.  Thus, there is a clinical requirement for the identification of specific, sensitive and quantitative biomarkers.  In recent years, circulating cfDNA and mutation-specific circulating cell-free tumor DNA (cftDNA) have been identified as novel potential biomarkers.  In the current study, cfDNA and cftDNA were compared using imaging-based staging and current tumor markers in 15 patients with metastatic colorectal, pancreatic or breast cancer.  These patients were treated at the Third Medical Department of Paracelsus Medical University Salzburg (Austria).  The results of the current study demonstrated a statistically significant correlation between the concentration changes of cfDNA and cftDNA and response to treatment, which was assessed by imaging.  A correlation was not indicated with current clinically used tumor markers, including carcino-embryonic antigen (CEA), carcinoma antigen (CA) 15-3 and CA 19-9.  The present study also indicated a correlation between cfDNA and cftDNA and the tumor volume of metastatic lesions, which was not observed with the current clinically used tumor markers.  The authors concluded that cfDNA and cftDNA exhibited the potential to become novel biomarkers for the response assessment following cancer treatment, and may serve as a tool for the estimation of tumor volume.  The current study further supports the increasingly important role of cfDNA and cftDNA as new monitoring tools for use during cancer therapy.

Moss et al (2020) stated that tumor-derived cfDNA is present in the plasma of individuals with cancer.  Assays aimed at detecting common cancer mutations in cfDNA are being developed for the detection of several cancer types.  In breast cancer, however, such assays have failed to detect the disease at a sensitivity relevant for clinical use, in part due to the absence of multiple common mutations that can be co-detected in plasma.  Unlike individual mutations that exist only in a subset of tumors, unique DNA methylation patterns are universally present in cells of a common type and therefore may be ideal biomarkers.  These researchers described the detection and quantification of breast-derived cfDNA using a breast-specific DNA methylation signature.  They collected plasma from patients with localized breast cancer before and throughout treatment with neoadjuvant chemotherapy and surgery (n = 235 samples).  Pre-treatment breast cfDNA was detected in patients with localized disease with a sensitivity of 80 % at 97 % specificity.  High breast cfDNA levels were associated with aggressive molecular tumor profiles and metabolic activity of the disease.  During neoadjuvant chemotherapy, breast cfDNA levels decreased dramatically.  Importantly, the presence of breast cfDNA towards the end of the chemotherapy regimen reflected the existence of residual disease.  The authors proposed that breast-specific cfDNA is a universal and powerful marker for the detection and monitoring of breast cancer.

These researchers stated that in this preliminary study, they provided proof of the concept that breast cfDNA quantification opens a window into the dynamics of breast cancer with a potential for early diagnosis, monitoring of treatment response, and detection of recurrence.  It should be noted that this study was based on data obtained from a small number of individuals who underwent neoadjuvant chemotherapy and further validation in another series of patients receiving neoadjuvant chemotherapy is needed.  Additionally, further expansion of the panel of breast DNA markers used may dramatically increase assay sensitivity and clinical utility.  These investigators also underscored the importance of sufficiently powered studies to explain the variance in breast cfDNA concentrations observed among patients and to test the utility of breast cfDNA in identifying recurrence of the disease.  Finally, while in this study the authors had focused on patients with localized disease at stage II to III, future studies, using a maximally sensitive version of the assay, should examine the relevance of this approach to patients at even earlier stages of the disease.

Furthermore, National Comprehensive Cancer Network’s clinical practice guideline on "Breast cancer" (Version 2.2020) does not mention the use of circulating cell-free DNA as a management tool.

Magbanua and associates (2021) noted that pathologic complete response (pCR) to neoadjuvant chemotherapy (NAC) is strongly associated with favorable outcome.  These investigators examined the use of serial ctDNA testing for predicting pCR and risk of metastatic recurrence.  Cell-free DNA was isolated from 291 plasma samples of 84 high-risk early breast cancer patients treated in the neoadjuvant I-SPY 2 TRIAL with standard NAC alone or combined with MK-2206 (AKT inhibitor) treatment.  Blood was collected at pre-treatment (T0), 3 weeks after initiation of paclitaxel (T1), between paclitaxel and anthracycline regimens (T2), or prior to surgery (T3).  A personalized ctDNA test was designed to detect up to 16 patient-specific mutations (from whole-exome sequencing of pre-treatment tumor) in cfDNA by ultra-deep sequencing.  The median follow-up time for survival analysis was 4.8 years.  At T0, 61 of 84 (73 %) patients were ctDNA-positive, which decreased over time (T1: 35 %; T2: 14 %; and T3: 9 %).  Patients who remained ctDNA-positive at T1 were significantly more likely to have residual disease after NAC (83 % non-pCR) compared with those who cleared ctDNA (52 % non-pCR; OR 4.33, p = 0.012).  After NAC, all patients who achieved pCR were ctDNA-negative (n = 17, 100 %).  For those who did not achieve pCR (n = 43), ctDNA-positive patients (14 %) had a significantly increased risk of metastatic recurrence [HR 10.4; 95 % CI: 2.3 to 46.6]; interestingly, patients who did not achieve pCR but were ctDNA-negative (86 %) had excellent outcome, similar to those who achieved pCR (HR 1.4; 95 % CI: 0.15 to 13.5).  The authors concluded that the findings of this study showed promise that early response prediction by highly sensitive ctDNA analysis in high-risk early breast cancer patients may facilitate a timely and judicious change in treatment to improve patients’ chances of achieving favorable long-term outcomes.  The I-SPY 2 TRIAL provided an excellent platform to examine how personalized ctDNA testing could complement imaging and pathologic evaluation of tumor response to fine-tune pCR as a surrogate endpoint for improved survival.  These researchers stated that dynamic monitoring of ctDNA during NAC could facilitate evaluation of new agents by providing an early endpoint of treatment efficacy.  Response over time as measured by imaging and ctDNA in the setting of early (pCR) and late (DRFS) outcomes will provide a robust framework for examining the potential clinical utility of ctDNA in the neoadjuvant setting.

OneOme RightMed Pharmacogenomic Test

The OneOme RightMed Pharmacogenomic Test ascertains how individuals’ genes may affect their response to medications.  The test analyzes a patient’s genetic information and provides a clinical report to aid physicians make more precise prescription decisions.  The OneOme RightMed Pharmacogenomic Test includes 22 genes (CYP1A2, CYP2B6, CYP2C9, CYP2C19, CYP2D6, CYP3A4, CYP3A5, COMT, DPYD, DRD2, F2, F5, GRIK4, HTR2A, HTR2C, IL28B (IFNL3), NUDT15, OPRM1, SLCO1B1, TPMT, UGT1A1, and VKORC1) and entails more than 340 medications and over 20 conditions (e.g., allergies, benign prostatic hyperplasia, cancer, diabetes, gastro-esophageal reflux disease, hypertension, migraine, pain, rheumatoid arthritis, seizure, and sleep disorders).  The genes tested in the OneOme RightMed Pharmacogenomic Test might affect other drugs that are not included in the Test’s pharmacogenomic database that covers approximately 1,000 of the most commonly used medications, including over 340 that are part of the RightMed Pharmacogenomic Test.  The absence of a drug in the OneOme report does not imply that there is no interaction between the drug and the genes analyzed in this test.  Information included in the OneOme report is based on scientific literature and does not take into account other genetic variants as well as environmental or social factors that may affect a patient's response.  Other factors not included in the report include, but are not limited to, environmental factors (e.g., smoking), health factors (e.g., diet), social and familial factors, various medical conditions and drug-drug interactions.Singh et al (2015) stated major depressive disorder (MDD) is projected to be a leading cause of disability globally by 2030. Only a minority of patients remit with antidepressants. If assay of polymorphisms influencing central nervous system (CNS) bioavailability could guide prescribers to more effectively dose patients, remission rates may improve and the burden of disease from MDD reduce. Hepatic and blood brain barrier (BBB) polymorphisms appear to influence antidepressant CNS bioavailability. A 12-week prospective double blind randomized genetically guided versus unguided trial of antidepressant dosing in Caucasian adults with MDD (n=148) was conducted. Subjects receiving genetically guided prescribing had a 2.52-fold greater chance of remission (95% confidence interval [CI]=1.71-3.73, z=4.66, p<0.0001). The number needed to genotype (NNG) was 3 (95% CI=1.7-3.5) to produce an additional remission. The authors concluded that these data suggest that a pharmacogenetic dosing report (CNSDose(®)) improves antidepressant efficacy. The effect size was sufficient that translation to clinical care may arise if results are independently replicated.

Hafner and colleagues (2016) stated that polymorphisms in genes encoding drug targets, drug transport proteins, or drug metabolizing enzymes may be responsible, among other factors, for the observed variation in individuals' responses to medications.  The field of pharmacogenetics aims to identify patients at higher genetically-determined risk of adverse effects (AEs) or poor response to medication.  This information would allow for modification of dosage or substitution with alternative therapy.  However, there is a lack of awareness of pharmacogenetic clinical practice guidelines.  These researchers performed a systematic review using the Medline and PharmGKB databases, which focused on published guidelines for dosage modification or selection of drugs based on germline mutations in genes with pharmacokinetic or pharmacodynamic impact.  Prescribing information from the European Medicines Agency, the German Federal Institute for Drugs and Medical Devices, and the FDA was also screened for pharmacogenetic guidance.  The literature review revealed 20 guideline publications elucidating 43 drugs with recommendations for genotype-guided prescribing.  Moreover, drug labels for 37 drugs also contained genotype-guided prescribing recommendations, some of which were linked to optional or obligatory pre-therapeutic pharmacogenetic testing.  The authors concluded that existing guidelines for genotype-based drug prescribing are rarely derived from prospective, controlled trials; thus, their level of evidence is usually low.  Even with low-quality evidence, strong recommendations can be made in favor of pharmacogenetic modification of prescription, such as for abacavir and the HLA-B genotype, if there is a large and certain difference between the benefits and harms.  For other drug-gene pairs, such as vitamin K antagonists and CYP2C9/VKORC1, the net benefit from the pharmacogenetic-based dosing strategy is small and matter of debate.  They stated that because pharmacogenetics is playing a growing role in drug development and pre-prescription genotyping will become more widespread, specific pharmacogenetic guidance for treating physicians will become increasingly important.

Berm and associates (2016) noted that as a consequence of extended application of pharmacogenetic and pharmacogenomic screening (PGx) tests it is important to examine if they provide good value for money.  This review provided an update of the literature.  These investigators performed a literature search in PubMed and papers published between August 2010 and September 2014, examining the cost-effectiveness of PGx screening tests.  Papers from 2000 until July 2010 were included via 2 previous systematic reviews.  Studies' overall quality was assessed with the Quality of Health Economic Studies (QHES) instrument.  These  researchers found 38 studies, which combined with the previous 42 studies resulted in a total of 80 included studies.  An average QHES score of 76 was found.  Since 2010, more studies were funded by pharmaceutical companies.  Most recent studies performed cost-utility analysis, univariate and probabilistic sensitivity analyses, and discussed limitations of their economic evaluations.  Most studies indicated favorable cost-effectiveness.  Majority of evaluations did not provide information regarding the intrinsic value of the PGx test.  There were considerable differences in the costs for PGx testing.  Reporting of the direction and magnitude of bias on the cost-effectiveness estimates as well as motivation for the chosen economic model and perspective were frequently missing.  The authors concluded that application of PGx tests was mostly found to be a cost-effective or cost-saving strategy, although some studies concluded otherwise which underlined the importance of future studies assessing the cost-effectiveness of PGx tests.  These investigators found that only the minority of recent pharmacoeconomic evaluations assessed the intrinsic value of the PGx tests.  New compounds that are not affected by genetics, are emerging as cost-effective alternatives for pharmacogenetic testing strategies.  Over the past 10 years, there was an increase in the number of studies and in the reporting of quality associated characteristics.  Due to rapid development in analytical techniques, reporting of analytical and clinical validity of the assessed PGx test is recommended for future evaluations.  Furthermore, they stated that robust clinical evidence regarding PGx tests’ effectiveness is needed.

This study had 2 main drawbacks: the authors did not include studies from other databases than PubMed or grey literature and they only included English written studies.  Thus, some studies might have been missed.  In general, studies that are not indexed in Medline or written in English do not have a large impact on reviews’ outcomes.  Nevertheless, these studies are frequently of lower quality, and therefore the average quality of the studies included in this review might have been slightly over-estimated.  However, based on the comparison with other reviews the authors found a slightly lower average quality score, and publication bias can always influence the findings of a review; thus, cost-effectiveness of PGx tests could be over-estimated.
the authors did not include studies from other databases than PubMed or grey literature and they only included English written studies.  Thus, some studies might have been missed.  In general, studies that are not indexed in Medline or written in English do not have a large impact on reviews’ outcomes.  Nevertheless, these studies are frequently of lower quality, and therefore the average quality of the studies included in this review might have been slightly over-estimated.  However, based on the comparison with other reviews the authors found a slightly lower average quality score, and
publication bias can always influence the findings of a review; thus, cost-effectiveness of PGx tests could be over-estimated.

Hinderer and co-workers (2017) noted that pharmacogenomic clinical decision support systems (CDSS) have the potential to help overcome some of the barriers for translating pharmacogenomic knowledge into clinical routine.  Before developing a prototype it is crucial for developers to know which pharmacogenomic CDSS features and user-system interactions have yet been developed, implemented and tested in previous pharmacogenomic CDSS efforts and if they have been successfully applied.  These researchers addressed this issue by providing an overview of the designs of user-system interactions of recently developed pharmacogenomic CDSS.  They searched PubMed for pharmacogenomic CDSS published between January 1, 2012 and November 15, 2016; 32 out of 118 identified articles were summarized and included in the final analysis.  These investigators then compared the designs of user-system interactions of the 20 pharmacogenomic CDSS that they had identified.  Alerts were the most widespread tools for physician-system interactions, but need to be implemented carefully to prevent alert fatigue and avoid liabilities.  Pharmacogenomic test results and override reasons stored in the local electronic health record might help communicate pharmacogenomic information to other internal care providers.  Integrating patients into user-system interactions through patient letters and online portals might be crucial for transferring pharmacogenomic data to external health care providers.  Inbox messages informed physicians about new pharmacogenomic test results and enabled them to request pharmacogenomic consultations.  Search engines enabled physicians to compare medical therapeutic options based on a patient's genotype.  The authors concluded that within the last 5 years, several pharmacogenomic CDSS have been developed.  However, most of the included articles were solely describing prototypes of pharmacogenomic CDSS rather than evaluating them.  They stated that further evaluation efforts are needed to support the development of prototypes; in the future, pharmacogenomic CDSS will likely include prediction models to identify patients who are suitable for preemptive genotyping.

Verbelen and colleagues (2017) stated that PGx has the potential to personalize pharmaceutical treatments.  Many relevant gene-drug associations have been discovered, but PGx-guided treatment needs to be cost-effective as well as clinically beneficial to be incorporated into standard health-care.  These investigators reviewed economic evaluations for PGx associations listed in the FDA Table of Pharmacogenomic Biomarkers in Drug Labeling.  They determined the proportion of evaluations that found PGx-guided treatment to be cost-effective or dominant over the alternative strategies, and estimated the impact on this proportion of removing the cost of genetic testing.  Of the 137 PGx associations in the FDA Table, 44 economic evaluations, relating to 10 drugs, were identified.  Of these evaluations, 57 % drew conclusions in favor of PGx testing, of which 30 % were cost-effective and 27 % were dominant (cost-saving).  If genetic information was freely available, 75 % of economic evaluations would support PGx-guided treatment, of which 25 % would be cost-effective and 50 % would be dominant.  Thus, PGx-guided treatment can be a cost-effective and even a cost-saving strategy.  The authors concluded that having genetic information readily available in the clinical health record is a realistic future prospect, and would make more genetic tests economically worthwhile.

Peterson and associates (2017) performed an evidence review of the effectiveness, harms, and cost-effectiveness of pharmacogenomics-guided anti-depressant treatment for major depressive disorder (MDD).  They searched Medline, the Cochrane Central Registry of Controlled Trials, and PsycINFO through February 2017.  They used pre-specified criteria to select studies, abstract data, and rate internal validity and strength of the evidence (PROSPERO number CRD42016036358).  These researchers included 2 RCTs, 5 controlled cohort studies, and 6 modeling studies of mostly women in their mid-40s with few co-morbidities.  CNSDose (ABCB1, ABCC1, CYP2C19, CYP2D6, UGT1A1) is the only pharmacogenomics test that significantly improved remission (1 additional remitting patient in 12 weeks per 3 genotyped, 95 % CI: 1.7 to 3.5) and reduced intolerability in an RCT.  ABCB1 genotyping leads to 1 additional remitting patient in 5 weeks per 3 genotyped (95 % CI: 3 to 20), but tolerability was not reported.  In an RCT, GeneSight (CYP2D6, CYPC19, CYP1A2, SLC6A4, HTR2A) did not statistically significantly improve remission, and evidence is inconclusive about its tolerability.  Evidence is generally low strength because RCTs were few and under-powered.  Cost-effectiveness is unclear due to lack of directly observed cost-effectiveness outcomes.  These researchers found no studies that evaluated whether pharmacogenomics shortened time to optimal treatment, whether improvements were due to switches to genetically congruent medication, or whether effectiveness varies based on test and patient characteristics.  The authors concluded that certain pharmacogenomics tools showed promise of improving short-term remission rates in women in their mid-40s with few co-morbidities; however, important evidence limitations precluded recommending their widespread use and indicated a need for further research.

Rosenblat and associates (2017) stated that pharmacogenomics has shown promise for predicting anti-depressant response and tolerability in the treatment of MDD.  In theory, pharmacogenomics can improve clinical outcomes by guiding anti-depressant selection and dosing.  In a systematic review, these investigators determined the impact of pharmacogenomic testing on clinical outcomes in MDD, and evaluated its cost-effectiveness.  The Medline/PubMed and Google Scholar databases were systematically searched for relevant articles published prior to October 2015.  Search terms included various combinations of the following: major depressive disorder (MDD), depression, mental illness, mood disorder, antidepressant, response, remission, outcome, pharmacogenetic, pharmacogenomics, pharmacodynamics, pharmacokinetic, genetic testing, genome wide association study (GWAS), CYP450, personalized medicine, cost-effectiveness, and pharmaco-economics.  Of the 66 records identified from the initial search, relevant clinical studies, written in English, assessing the cost-effectiveness and/or efficacy of pharmacogenomic testing for MDD were included.  Each publication was critically examined for relevant data.  A total of 2 non-randomized, open-label, 8-week, prospective studies reported overall greater improvement in depressive symptom severity in the group of MDD subjects receiving psychiatric care guided by results of combinatorial pharmacogenomic testing (GeneSight) when compared to the unguided group.  One industry-sponsored, randomized, double-blind, 10-week prospective study reported a trend for improved outcomes for the GeneSight-guided group; however, the trend did not reach statistical significance.  Another industry-sponsored, randomized, double-blind, 12-week prospective study reported a 2.5-fold increase in remission rates in the CNSDose-guided group (p < 0.0001).  One naturalistic, un-blinded, industry-sponsored study showed clinical improvement when pharmacogenomics testing guided prescribing; however, this study lacked a control group.  A single cost-effectiveness study concluded that single-gene testing was not cost-effective.  Conversely, a separate study reported that combinatorial pharmacogenomic testing is cost-effective.  The authors concluded that a limited number of studies have shown promise for the clinical utility of pharmacogenomic testing; however, cost-effectiveness of pharmacogenomics, as well as demonstration of improved health outcomes, is not yet supported with replicated evidence.

It is also interesting to note that in a systematic review on “Pharmacogenomic testing for psychotropic medication selection”, Health Quality Ontario (2017) reported that there is uncertainty about the use of GeneSight Psychotropic pharmacogenomic genetic panel to guide medication selection.

Bouseman et al (2017) stated pharmacogenetic-based dosing support tools have been developed to personalize antidepressant-prescribing practice. However, the clinical validity of these tools has not been adequately tested, particularly for specific antidepressants. The authors examined the concordance between the actual dose and a polygene pharmacogenetic predicted dose of desvenlafaxine needed to achieve symptom remission. A 10-week, open-label, prospective trial of desvenlafaxine among Caucasian adults with major depressive disorder (n=119) was conducted. Dose was clinically adjusted and at the completion of the trial, the clinical dose needed to achieve remission was compared with the predicted dose needed to achieve remission. Among remitters (n=95), there was a strong concordance (Kendall's τ-b=0.84, P=0.0001; Cohen's κ=0.82, P=0.0001) between the actual and the predicted dose need to achieve symptom remission, showing high sensitivity (≥85%), specificity (≥86%), and accuracy (≥89%) of the tool. The authors concluded that findings provide initial evidence for the clinical validity of a polygene pharmacogenetic-based tool for desvenlafaxine dosing.

Del Casale et al (2022) stated that treatment-resistant depression (TRD; 2 or more recent failed psychopharmacological trials) reduces affected patients' QOL and leads to important social healthcare costs; and PGT may be effective in the cure of TRD.  In a retrospective study, these researchers examined the clinical changes following PGT in patients with TRD affected by bipolar disorder (BD) or MDD compared to a control group with treatment as usual (TAU).  These researchers based the PGT on evaluating different gene polymorphisms involved in the pharmacodynamics and pharmacokinetics of drugs.  They analyzed, with a repeated-measure ANOVA, the changes between the baseline and a 6-month follow-up of the effectiveness index examined via the Clinical Global Impression (CGI) scale, and depressive symptoms via the Hamilton Depression Rating Scale (HDRS).  The PGT sample included 53 patients (26 BD and 27 MDD), and the TAU group included 52 patients (31 BD and 21 MDD).  These investigators found a significant within-subject effect of treatment time on symptoms and effectiveness index for the whole sample, with significant improvements in the effectiveness index (F = 8.544; partial η² = 0.077, p < 0.004) and clinical global impression of severity of illness (F = 6.818; partial η² = 0.062, p < 0.01) in the PGT versus the TAU group.  These investigators also found a significantly better follow-up response (χ² = 5.479; p = 0.019) and remission (χ² = 10.351; p = 0.001) rates in the PGT versus the TAU group.  The authors concluded that PGT may be an important option for the long-term treatment of patients with TRD affected by mood disorders, providing information that could better define drug treatment strategies and increase therapeutic improvement.  Moreover, these researchers stated that the evidence for applying PGT in MDD and BD has been growing, although more studies on drug safety, side effect burden, and treatment adherence are needed.  The authors stated that the main drawback of this study was the small sample size and its retrospective nature, for which the results should be interpreted with caution and need to be replicated in larger, prospective, and multi-center studies.

CYP2C19 Polymorphisms Testing for Individuals on Fluoxetine

Luk et al (2014) stated that diazepam is often used as an adjuvant to pain therapy.  Cytochrome P450 (CYP) 3A4 and 2C19 metabolize diazepam into the active metabolites: nordiazepam, temazepam and oxazepam.  Owing to diazepam's side-effect profile, mortality risk and potential for drug-drug interactions with CYP 3A4 and/or CYP 2C19 inhibitors, urine drug testing (UDT) could be a helpful monitoring tool.  This was a retrospective data analysis that evaluated urine specimens from pain management practices for the distribution of diazepam metabolites with and without CYP 3A4 and 2C19 inhibitors.  Inter-subject nordiazepam, temazepam and oxazepam geometric mean fractions were 0.16, 0.34 and 0.47, respectively.  Intra-subject geometric mean fractions were 0.157, 0.311 and 0.494, respectively.  Sex, but not age or urinary pH, had an effect on metabolite fractions.  Methadone significantly increased temazepam and oxazepam urinary fractions via CYP3A4 inhibition, whereas fluoxetine and esomeprazole increased nordiazepam fractions via CYP2C19 inhibition.  The authors concluded that although more studies are needed, these results suggested the viability of UDT for increased monitoring for therapy and possible drug-drug interactions.

Dinger et al (2016) noted that in-vitro cytochrome P450 (CYP) inhibition assays are common approaches for testing the inhibition potential of drugs for predicting potential interactions.  In contrast to marketed medicaments, drugs of abuse, particularly the so-called novel psychoactive substances, were not tested before distribution and consumption.  Therefore, the inhibition potential of methylenedioxy-derived designer drugs (MDD) of different drug classes such as aminoindanes, amphetamines, benzofurans, cathinones, piperazines, pyrrolidinophenones, and tryptamines should be elucidated.  The FDA-preferred test substrates, split in 2o cocktails, were incubated with pooled human liver microsomes and analyzed after protein precipitation using LC-high-resolution-MS/MS.  IC50 values were determined of MDD showing more than 50 % inhibition in the prescreening.  Values were calculated by plotting the relative metabolite concentration formed over the logarithm of the inhibitor concentration.  All MDD showed inhibition against CYP2D6 activity and most of them in the range of the clinically relevant CYP2D6 inhibitors quinidine and fluoxetine.  In addition, the beta-keto compounds showed inhibition of the activity of CYP2B6, 5,6-MD-DALT of CYP1A2 and CYP3A, and MDAI of CYP2A6, all in the range of clinically relevant inhibitors.  The authors concluded that all MDD showed inhibition of the activity of CYP2D6, 6 of CYP1A2, 3 of CYP2A6, 13 of CYP2B6, 2 of CYP2C9, 6 of CYP2C19, 1 of CYP2E1, and 6 of CYP3A.  They stated that these findings showed that the CYP inhibition by MDD might be clinically relevant, but further studies are needed for final conclusions.

Bykov et al (2017) noted that clopidogrel is a pro-drug that requires activation by the cytochrome P450 (CYP) enzyme system.  Patients receiving clopidogrel are often treated with SSRIs for co-existing depression.  SSRIs that inhibit the CYP2C19 enzyme have the potential to reduce the effectiveness of clopidogrel.  Using 5 US databases (1998 to 2013), these investigators conducted a cohort study of adults who initiated clopidogrel while being treated with either an SSRI that inhibits CYP2C19 (fluoxetine and fluvoxamine) or a non-inhibiting SSRI.  Patients were matched by propensity score and followed for as long as they were exposed to both clopidogrel and the index SSRI group (primary analysis) or for 180 days after clopidogrel initiation (sensitivity analysis).  Outcomes included a composite ischemic event (myocardial infarction, ischemic stroke, or a re-vascularization procedure) and a composite major bleeding event (gastro-intestinal bleed or hemorrhagic stroke).  The final propensity score-matched cohort comprised 9,281 clopidogrel initiators on CYP2C19-inhibiting SSRIs and 44,278 clopidogrel initiators on a non-inhibiting SSRIs.  Compared with those treated with a non-inhibiting SSRI, patients on a CYP2C19-inhibiting SSRI had an increased risk of ischemic events (hazard ratio [HR] 1.12; 95 % confidence interval [CI]: 1.01 to 1.24), which was more pronounced in patients greater than or equal to 65 years (HR 1.22; 95 % CI: 1.00 to 1.48).  The HR for major bleeding was 0.76 (95 % CI: 0.50 to 1.17).  The authors concluded that the findings from this large, population-based study suggested that being treated with a CYP2C19-inhibiting SSRI when initiating clopidogrel may be associated with slight decrease in effectiveness of clopidogrel.

PGxOnePlus

An UpToDate review on “Anxiety disorders in children and adolescents: Epidemiology, pathogenesis, clinical manifestations, and course” (Bennett and Walkup, 2017) states that “Studies suggest that children of parents with an anxiety disorder are at an increased risk of developing an anxiety disorder themselves.  Twin studies have demonstrated significant familial aggregation among GAD, panic disorder, phobias, and obsessive-compulsive disorder.  Most estimates of anxiety trait heritability in children are around 30 %, though as high as 50 to 60 % in some studies.  These findings suggest that genetic factors play a large role in the development of these disorders relative to environmental factors.  A meta-analysis of 13 cohorts (42,585 subjects) with social anxiety showed that genetic and non-shared environmental factors explain most of the variance for social anxiety disorder and social anxiety symptoms across age groups.  Adult cohorts showed a higher contribution of non-shared environment and half the genetic contribution compared with younger patients.  Some research suggests that the contribution of genetics to the development of anxiety disorders may change over the course of an individual’s development.  Supporting a dynamic contribution of genetics, a study of 2,508 twins suggested an effect of some genes that are “turned on” in adolescence and others that are diminished throughout development.  The overall variance accounted for by genetic contributions decreased from 72 % at ages 8 and 9 to 12 % at ages 19 and 20.   It is unlikely that a single gene associated with one specific pathophysiological function is responsible for anxiety.  Family and genetic studies suggest some people may have a genetic vulnerability to develop anxiety disorders generally.  Genome Wide Association Studies (GWAS) have begun to look for specific candidate genes associated with anxiety disorders; however, these studies result in a large number of candidate genes with relatively small odds ratios or lack of statistical significance in genotype/phenotype associations, possibly due to the cumulative effect of many genes on the vulnerability for anxiety disorders.  Genetic findings associated with anxiety disorders or anxiety-like behavior include a gene that promotes a corticotropin releasing hormone and a single nucleotide polymorphism (SNP) in the brain-derived neurotrophic factor (BDNF) gene … Data from a study of 385 monozygotic and 486 dizygotic same-sex twin families assessed the children of twins and found evidence of a significant direct environmental transmission of anxiety from parent to child using structural equation modeling.  There was no evidence of direct genetic transmission”.  There is no mentioning of genetic testing in the “Summary and Recommendations” section.

It is also interesting to note that Health Quality Ontario (2017) states that the Assurex GeneSight Psychotropic test is a pharmacogenomic panel that provides clinicians with a report to guide medication selection that is unique to each patient based on their individual genetic profile.  However, it is uncertain whether guided treatment using GeneSight is effective compared with unguided treatment (usual care).  Health Quality Ontario concluded that there is uncertainty about the use of GeneSight Psychotropic pharmacogenomic genetic panel to guide medication selection.  It was associated with improvements in some patient outcomes, but not others.  As well, the confidence in these findings is low because of limitations in the body of evidence.

An UpToDate review on “Genetic factors in the pathogenesis of hypertension” (Ehret, 2017) states that “The use of genome-wide association studies, which examine hundreds of thousands of single-nucleotide polymorphisms (SNPs) in large cohorts, has improved the understanding of blood pressure genomics and has demonstrated the presence of clearly reproducible blood pressure loci.  However, these loci have so far only explained a small proportion of the total blood pressure heritability.  There are approximately 80 published blood pressure loci”.  There is no mentioning of genetic testing in the “Summary and Recommendations” section.

Bonfiglio et al (2017) stated that gastro-esophageal reflux disease (GERD), the regurgitation of gastric acids often accompanied by heartburn, affects up to 20 % of the general population.  Genetic predisposition is suspected from twin and family studies but gene-hunting efforts have so far been scarce and no conclusive genome-wide study has been reported.  These researchers exploited data available from general population samples, and studied self-reported reflux symptoms in relation to genome-wide single nucleotide polymorphism (SNP) genotypes.  The authors reported a large-scale genetic study of GERD, and highlighted genes and pathways that contribute to further our understanding of its pathogenesis and therapeutic opportunities.

An UpToDate review on “Approach to refractory gastroesophageal reflux disease in adults” (Fass, 2017) states that “Proton pump inhibitors are metabolized through the hepatic cytochrome system (specifically the CYP2C isoenzyme).  As a result, genetically determined variability in the processes underlying drug metabolism may influence their efficacy.  Patients with rapid metabolism of PPIs may have a decreased effect on gastric acidity.  On the other hand, CYP2C is absent in about 3 % of Caucasian patients and in substantially higher numbers of Asians (greater than 10 %), potentially leading to greater suppression of gastric acidity”.

Genotyping of Interferon-Lambda 3 (IFNL3) for Prediction of Virological Response to Pegylated-Interferon-Alpha and Ribavirin Combination Therapy

According to the Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for “IFNL3 (IL28B) genotype and peginterferon alpha based regimens”, Muir and colleagues (2014) stated that Peg-interferon-alpha and ribavirin (Peg-IFNα/RBV)-based regimens are the mainstay for the treatment of hepatitis C virus (HCV) genotype 1 infection.  For patients treated with Peg-IFNα and RBV alone, interferon-lambda 3 (IFNL3) genotype is the strongest pre-treatment predictor of HCV treatment response.  In the intention-to-treat analysis of the original discovery cohort with rs12979860, Caucasian patients with CC genotype were more likely than those with CT or TT genotype to have undetectable serum viral RNA by week 4 (28 % versus 5 % and 5 %, respectively; p < 0.0001) and to achieve sustained virologic response (SVR) (69 % versus 33 % and 27 %, respectively; p < 0.0001).  Similar patterns were observed in Hispanic and African-American patients in this cohort; HCV treatment is associated with significant side effects, and the likelihood of response treatment influenced shared decision-making between clinicians and patients about initiating treatment.

Ishiguro and associates (2015) evaluated IFNL3 polymorphisms in response-guided Peg-IFNα/RBV therapy for genotype 2 (G2) chronic HVC infection.  Between January 2006 and June 2012, a total of 180 patients with chronic infections of G2 HCV were treated with response-guided Peg-IFNα/RBV therapy.  The treatment duration was 24 weeks for patients who achieved rapid virologic response (RVR), and 36 or 48 weeks for patients who did not.  Then, the impact of the IFNL3 single nucleotide polymorphism (SNP) genotype (TT/non-TT at rs8099917) on treatment outcomes was evaluated in the 180 patients, and between patients infected with either HCV sub-genotype 2a or 2b.  Of the 180 patients evaluated, 111 achieved RVR, while the remaining 69 patients did not.  In RVR patients, the SVR rate was 96.4 %, and the IFNL3 genotype did not influence the SVR rate (96.6 % versus 95.8 % in IFNL3 genotype TT versus non-TT).  However, in non-RVR patients, the SVR rate decreased to 72.5 % (p < 0.0001), and this rate was significantly different between the IFNL3 genotype TT and non-TT groups (80.0 % versus 42.9 %, p = 0.0146).  Multi-variate regression analysis in non-RVR patients identified the IFNL3 genotype TT as the only baseline-significant factor associated with SVR (OR = 5.39, 95 % CI: 1.29 to 22.62; p = 0.0189).  In analysis according to HCV sub-genotype, no significant difference in the SVR rate was found between HCV sub-genotypes 2a and 2b.  The authors concluded that in response-guided Peg-IFNα/RBV combination therapy for chronically HCV G2-infected patients, the impact of the IFNL3 genotype on SVR was limited to non-RVR patients.

Haj-Sheykholeslami and co-workers (2015) evaluated the efficacy of Peg-IFN in chronic HCV patients in relation to IFNL polymorphisms.  This study enrolled patients with chronic HCV referred to the Tehran Blood Transfusion Hepatitis Clinic in 2011.  Patients were included in the study if they had no concomitant hepatic illness, were negative for human immunodeficiency virus (HIV) antibodies, and had no prior history of treatment with any type of Peg-IFN.  Patients were treated with 180 μg Peg-IFNα-2a (Pegaferon) weekly and 800 to 1,200 mg ribavirin daily for 24 or 48 weeks depending on weight and HCV genotype.  Blood samples were collected from patients to obtain DNA for determination of IFNL rs12979860 and rs8099917 polymorphisms.  The virologic response in patients was then evaluated and compared between the different IFNL genotypes.  A total of 152 patients with a mean age of 41.9 ± 10.0 years were included in the study, of which 141/152 were men (92.8 %).  The most frequent HCV genotype was type-1, infecting 93/152 (61.2 %) patients; SVR was achieved in 81.9 % of patients with HCV genotype-1 and 91.1 % of patients with HCV genotype-3.  Treatment success was achieved in 91.2 % (52/57) of patients with the IFNL rs12979860 CC genotype and 82.1 % (78/95) in those with other genotypes.  Similar treatment response rates were also observed in patients with rs8099917 TT (39/45; 86.7 %) and non-TT (61/68; 89.7 %) genotypes.  Uni-variate analyses identified the following factors that influenced treatment response for inclusion in a multi-variate analysis: age, HCV RNA level, stage of liver fibrosis, rs12979860 CC genotype, and aspartate transaminase level.  A logistic regression analysis revealed that only the rs12979860 CC genotype was significantly associated with achievement of SVR (OR = 6.2; 95 % CI: 1.2 to 31.9; p = 0.03).  The authors concluded that the rs12979860 CC genotype was associated with SVR in patients receiving Peg-IFNα/RBV therapy, however, the SVR rate in other rs12979860 genotypes was also relatively high.

Nakamoto and colleagues (2017) noted that genetic variation near the IFNL3 is known to be associated with response to Peg-IFNα/RBV therapy in patients with chronic HCV infection, which is often accompanied by hepatic steatosis.  These researchers examined if this genetic variation is associated with host lipids and treatment response.  A total of 101 Japanese patients who had undergone liver biopsy before treatment with Peg-IFNα/RBV therapy for HCV genotype 1b infection were retrospectively analyzed for association between IFNL3 genotypes (rs8099917) and clinical factors including histopathological features of the liver.  The presence of greater than 5 % steatosis in the liver specimen was defined as hepatic steatosis.  A total of 40 patients (40 %) had liver steatosis before therapy.  Patients with IFNL3 minor genotype (non-TT) showed lower low-density lipoprotein cholesterol level (p = 0.0045), higher γ-glutamyl transpeptidase level (p = 0.0003) and higher prevalence of hepatic steatosis (p = 0.0002).  Advanced fibrosis [OR 4.63, p = 0.03] and IFNL3 major genotype (OR 0.13, p = 0.001) were 2 independent factors for determining the presence of hepatic steatosis.  Among the factors associated with SVR, IFNL3 genotype was the most significant predictor, as per multi-variate analysis.  The authors concluded that these findings confirmed that IFNL3 genotype was associated with hepatic steatosis as well as IFN response.

UrSure Tenofovir Quantification Test
Koenig and associates (2017) noted that tenofovir disoproxil fumarate/emtricitabine (TDF/FTC) is approved for pre-exposure prophylaxis (PrEP) against HIV infection.  Adherence is critical for the success of PrEP, but current adherence measurements are inadequate for real-time adherence monitoring.  These researchers developed and validated a urine assay to measure tenofovir (TFV) to objectively monitor adherence to PrEP.  These investigators developed a urine assay using high-performance liquid chromatography (HPLC) coupled to tandem mass spectrometry (MS) with high sensitivity/specificity for TFV that allowed these researchers to determine TFV concentrations in log10 categories between 0 and 10,000 ng/ml.  They validated the assay in 3 cohorts: HIV-positive subjects with undetectable viral loads on a TDF/FTC-based regimen, healthy HIV-negative subjects who received a single dose of TDF/FTC, and HIV-negative subjects receiving daily TDF/FTC as PrEP for 24 weeks.  
HIV-positive subjects with undetectable viral loads on a TDF/FTC-based regimen,
healthy HIV-negative subjects who received a single dose of TDF/FTC, and
HIV-negative subjects receiving daily TDF/FTC as PrEP for 24 weeks. 

The urine assay detected TFV with greater sensitivity than plasma-based measures and with a window of measurements within 7 days of the last TDF/FTC dose.  Based on the urine log-linear clearance after the last dose and its concordance with all detectable plasma levels, a urine TFV concentration of greater than 1,000 ng/ml was identified as highly predictive of the presence of TFV in plasma at greater than 10 ng/ml.  The urine assay was able to distinguish high and low adherence patterns within the last 48 hours (greater than 1,000 ng/ml versus 10 to 1,000 ng/ml), as well as non-adherence (less than 10 ng/ml) extended over at least 1 week prior to measurement.  The authors provided proof of concept that a semi-quantitative urine assay measuring levels of TFV could be further developed into a point-of-care (POC) test and be a useful tool to monitor adherence to PrEP.

 Cressey and colleagues (2017) stated that TDF is key component of PrEP and anti-retroviral therapy (ART) for HIV, but existing tools to monitor drug adherence are often inaccurate.  Detection of TFV in accessible biological samples, such as finger-prick blood, urine or oral fluid samples could be a novel objective measure of recent TDF adherence.  To measure TFV concentrations associated with different levels of TDF adherence, these researchers designed a randomized clinical trial to evaluate the blood, urine and oral fluid concentrations of TFV in adults with perfect, moderate, and low drug adherence.  This is a randomized, open-label, clinical pharmacokinetic study of TFV in healthy adult volunteers without HIV or hepatitis B infection in Thailand.  Consenting, eligible participants are randomized (1:1:1) among 3 groups to receive a controlled number of TDF (300 mg) doses in a combination pill with emtricitabine (FTC, 200 mg) for 6 weeks.  Participants in Group 1 receive a single TDF/FTC tablet once-daily (perfect adherence); Group 2 receive a single TDF/FTC tablet 4 times/week (moderate adherence); and Group 3 receive a single TDF/FTC tablet 2 times/week (low adherence).  Blood, plasma, urine and oral fluid samples are collected for drug measurement during 3 study phases: (initial 6-week treatment phase; intensive 24-hour blood sampling phase after 6 weeks; 4-week washout phase.  
(initial 6-week treatment phase;
intensive 24-hour blood sampling phase after 6 weeks;
4-week washout phase. 

A total of 30 adults with evaluable pharmacokinetic samples (10 per group) will be enrolled [based on ensuring 25 % precision in pharmacokinetic parameter estimates].  Pre-dose drug concentrations during the treatment phase will be descriptive and comparisons between groups performed using a Kruskal-Wallis test.  A non-compartmental pharmacokinetic analysis will be performed on the intensive sampling data at week 7 and the time course of TFV washout in the difference biological matrices will be reported based on the detected concentrations following drug cessation.  The authors concluded  that the findings of this randomized trial will define the target concentration thresholds of TFV in blood, urine and oral fluid that can distinguish between different levels of TDF adherence.  Such adherence “benchmarks” can be applied to real-time drug testing and novel POC tests to identify individuals with poor PrEP or ART adherence.  Moreover, they stated that the introduction of scalable TFV-based POC tests may allow rapid identification of patients struggling with PrEP or ART adherence, in order to develop and implement targeted adherence interventions.  They noted that improving adherence to both PrEP and ART will help reduce and prevent HIV transmission, while also preserving the use an important drug in the efforts to end the HIV/AIDS epidemic.  The enrollment to this clinical trial is by invitation (last updated October 26, 2017).Current guidelines on pre-exposure prophylaxis have no recommendations for urine tenofovir monitoring. Also, no recommendations for urine tenofovir monitoring are included in the PrEP prescribing information. Additional studies demonstrating the clinical utility of urine tenofovir monitoring are needed, as well as comparisons of urine drug monitoring with other established methods of improving PrEP compliance.

INFINITI Neural Response Panel

On June 04, 2018, the FDA granted Breakthrough Device designation to the INFINITI Neural Response Panel by AutoGenomics, Inc., which indicates that the FDA has agreed to prioritize and expedite their review of the panel test. The INFINITI Neural Response Panel is an in vitro, qualitative, automated, microarray–based molecular diagnostic test for the identification of patients who may be at risk for opioid dependency. The panel is designed to identify mutations in 16 genes (ABCB1, COMT, DAT1, DBH, DOR, DRD1, DRD2, DRD4, GABA, GAL, HTR2A, HTTLPR, MTHFR, MUOR, OPRK1, and OPRM1) involved in the brain reward pathways that are associated with increased risk of opioid use disorder, and is intended for use by physicians as an aid for safe and effective pain management.

Two studies suggesting that a new genetic test could potentially help predict which patients are at risk of abusing prescription opioids were first presented at the 69th AACC Annual Scientific Meeting & Clinical Lab Expo in San Diego in 2017. In the first study, Donaldson et al (2017; Prescient Medicine) stated over 116 million people worldwide have chronic pain and prescription dependence. In the US, opioids account for the majority of overdose deaths, and in 2014, almost 2 million Americans abused or were dependent on prescription opioids. Genetic factors may play a key role in opioid prescription addiction. The authors describe genetic variations between opioid addicted and non-addicted populations and derive a predictive model determining risk of opioid addiction. This case cohort study compares the frequency of 16 single nucleotide polymorphisms involved in the brain reward pathways in patients with and without opioid addiction. Data from 37 patients with prescription opioid or heroin addiction and 30 age and gender matched controls were used to design the predictive score. The predictive score was then tested on an additional 138 samples to determine generalizability. Results for Method Derivation of Observed data: ROC statistic=0.92, sensitivity=82% (95% CI: 66-90), specificity=75% (95% CI:56-87). TreeNet "learn" data: ROC statistic=0.92, sensitivity=92%, specificity=90%, precision=92%, and overall correct=91%. Results of Generalizability data: Sensitivity=97% (95% CI: 90 to 100), specificity=87% (95% CI: 86 to 93), positive likelihood ratio=7.3 (95% CI: 4.0 to 13.5), and negative likelihood ratio=0.03 (95% CI: 0.01 to 0.13). This negative likelihood ratio can be used as evidence based measure to exclude patients with a high risk of opioid addiction or substance use disorder. By identifying patients with a lower risk for opioid addiction, our model may inform therapeutic decisions. The authors further state that these types of predictive risk scores may also foster tailored medical detoxification regimens and reduce drug related adverse events for patients. Yet the use of genetic algorithms to determine predictive risk scores is still a relatively new science an prospective, longitudinal studies are needed to better define the breadth of the test’s importance.

In the second clinical study by Chang et al. (2017; AutoGenomics) assessed the risk of developing prescription opioid addiction with a multi-variant addiction panel involved in the mesolimbic dopamine system. The authors genotyped samples for 16 single nucleotide polymorphisms (SNPs) involved in the brain reward pathways from 70 patients diagnosed with prescription opioid/heroin addiction and 68 normal control patients with a multiplexed film-based microarray technology. The addiction panel targets 16 mutations: 5-HTR2A (rs7997012), 5-HTTLPR (rS25531), COMT (rs4680), DRD2 (rs1800497), DRD1 (rs4532), DRD4 (rs3758653), DAT1 (rS6347), DBH (rs1611115), MTHFR (rs1801133), OPRK1 (rs1051660), GABA (rs211014), OPRM1 (rs1799971), MUOR (rs9479757), GAL (rs948854), DOR (rs2236861) and ATP-BCT (rs1045642). The genotyping data were subjected to a class predication model building with 10-fold cross validation for testing. A risk score from 1 to 100 was further computed with a score over 52 representing an elevated risk of addiction. The receiver operating characteristic (ROC) for the model was 0.78 with 76% sensitivity (95% CI: 84 - 85) and 72% specificity (95% CI: 60 - 82). PPV is 74% and NPV is 74%. Fifty-Three of 70 (75.7%) addicts and 19 of 68 (27.9%) normal controls showed an addiction risk score over 52. (X2=31.55;df=1;P<.05). The authors concluded that the prediction algorithm with this multi-variant genetic panel can be used for prescription opioid addiction risk assessment. By identifying patients with high risk to prescription opioid addiction along with mutation status of cytochrome p450 genes involved in therapeutics, it may provide information to physicians to improve therapeutic decisions in pain management and prevent abuse and addiction.

Snapshot Oral Fluid Compliance

Snapshot Oral Fluid Compliance test by Ethos Laboratories is a prescription drug monitoring enzyme immunoassay of oral fluid samples that tests 35 or more drugs with liquid chromatography mass spectrometry (LC-MS/MS) for patient-compliance and drug to drug interactions for prescribed medications. Results are summarized in a specialized report (i.e., Ethos Snapshot).

Richter et al (2019) stated nonadherence to antihypertensive drugs therapy is known to be a serious issue in hypertension treatment. Liquid chromatography (LC) coupled to mass spectrometry (MS) was shown to allow the assessment of such nonadherence in blood and urine sample. However, their sampling may represent a logistical challenge and are often not favored by the patients. The authors questioned whether oral fluid (OF) might be an easier accessible alternative matrix for adherence monitoring of cardiovascular drugs (CD). A qualitative method for adherence monitoring of 78 commonly prescribed cardiovascular drugs in OF using LC high-resolution MS (LC-HRMS/MS) was therefore developed, validated, and used to study the presence of antihypertensive medication in OF. Selectivity, ion suppression and enhancement due coeluting analytes, carry over, limits of detection (LOD), limits of identification (LOI), recovery (RE), matrix effects (ME), and process efficiency (PE) were investigated. For demonstrating applicability, over 50 OF samples were investigated and data were compared to findings in blood and urine. Selectivity in OF was given for all compounds via their MS2 spectra and no total suppression of signals could be observed. Determined LOI in OF for ten analytes was higher than the given therapeutic plasma concentration. Furthermore, RE, ME, and PE were in acceptable ranges for more than 65% of the compounds. In total, 208 prescriptions of CD to 57 patients were analyzed and demonstrated the suitability of for adherence monitoring in principle. OF was comparable to plasma regarding the drug categories and the frequencies of hits, except for acidic compounds but more hits could be found in urine samples. A analytical method using OF as analytical matrix was successfully developed. The authors concluded that application showed that it might be a suitable alternative for adherence monitoring of selected drugs in the future, particularly those having no acidic function.  

OncolyticAssuranceRX 

OncolyticAssuranceRX, by Firstox Laboratories, LLC., is a prescription drug monitoring test that monitors adherence to one or more oral oncology drug(s) and substances through a proprietary liquid chromatography mass spectrometry (LC-MS/MS) test combined with a minimally invasive finger stick collection, and provides a quantitative report with steady-state range for the prescribed drug(s) when detected.

Janssen et al (2019) stated a liquid chromatography-tandem mass spectrometry assay was developed and validated for the nine oral anticancer agents alectinib, cobimetinib, lenvatinib, nintedanib, osimertinib, palbociclib, ribociclib, vismodegib and vorinostat in order to support therapeutic drug monitoring (TDM). The assay was based on reversed-phase chromatography coupled with tandem mass spectrometry operating in the positive ion mode. The assay was validated based on the guidelines on bioanalytical methods by the US Food and Drug Administration and European Medicines Agency. The method was validated over a linear range of 10-200 ng/mL for alectinib, lenvatinib, nintedanib and vismodegib; 50-1000 ng/mL for cobimetinib and palbociclib; 100-2000 ng/mL for osimertinib; 5.00-100 ng/mL for ribociclib; 25-500 ng/mL for vorinostat. Intra-assay and inter-assay bias was within ±20% for all analytes at the lower limit of quantification and within ±15% at remaining concentrations. Stability experiments showed that osimertinib is unstable in the biomatrix and should be shipped on dry-ice and stored at -20 °C until analysis. All other compounds were stable in the biomatrix. The authors concluded that the described TDM method was successfully validated and applied for TDM in patients treated with these KIs.

van Nuland M et al (2019) stated a liquid chromatography-mass spectrometry assay was developed and validated for simultaneous quantification of anti-hormonal compounds abiraterone, anastrozole, bicalutamide, Δ(4)-abiraterone (D4A), N-desmethyl enzalutamide, enzalutamide, Z-endoxifen, exemestane and letrozole for the purpose of therapeutic drug monitoring (TDM). Plasma samples were prepared with protein precipitation. Analyses were performed with a triple quadrupole mass spectrometer operating in the positive and negative ion-mode. The validated assay ranges from 2 to 200 ng/mL for abiraterone, 0.2-20 ng/mL for D4A, 10-200 ng/mL for anastrozole and letrozole, 1-20 ng/mL for Z-endoxifen, 1.88-37.5 ng/mL for exemestane and 1500-30,000 ng/mL for enzalutamide, N-desmethyl enzalutamide and bicalutamide. Due to low sensitivity for exemestane, the final extract of exemestane patient samples should be concentrated prior to injection and a larger sample volume should be prepared for exemestane patient samples and QC samples to obtain adequate sensitivity. Furthermore, we observed a batch-dependent stability for abiraterone in plasma at room temperature and therefore samples should be shipped on ice. The authors concluded that this newly validated method has been successfully applied for routine TDM of anti-hormonal drugs in cancer patients.

Cardoso et al (2018) stated A sensitive and selective method of high performance liquid chromatography (HPLC) coupled to tandem mass spectrometry (MS/MS) has been developed for the simultaneous quantification of six anticancer protein kinase inhibitors (PKIs), dabrafenib, trametinib, vemurafenib, cobimetinib, pazopanib, regorafenib, and two active metabolites (regorafenib-M2 and regorafenib-M5) in human plasma. Plasma protein precipitation with methanol enables the sample extraction of 100 μL aliquot of plasma. Analytes are detected by electrospray triple-stage quadrupole mass spectrometry and quantified using the calibration curves with stable isotope-labeled internal standards. The method was validated based on FDA recommendations, including assessment of extraction yield (74-104%), matrix effects, analytical recovery (94-104%) with low variability (<15%). The method is sensitive (lower limits of quantification within 1 to 200 ng/mL), accurate (intra- and inter-assay bias: -0.3% to +12.7%, and -3.2% to +6.3%, respectively) and precise (intra- and inter-assay CVs within 0.7-7.3% and 2.5-8.0%, respectively) over the clinically relevant concentration range (upper limits of quantification 500 to 100,000 ng/mL). The authors concluded that this method is applied in their laboratory for both clinical research programs and routine therapeutic drug monitoring service of PKIs.  

FGFR Testing for Pemigatinib (Pemazyre) Therapy for Advanced Cholangiocarcinoma

Abou-Alfa et al (2020) stated fibroblast growth factor receptor (FGFR) 2 gene alterations are involved in the pathogenesis of cholangiocarcinoma. Pemigatinib is a selective, potent, oral inhibitor of FGFR1, 2, and 3. This study evaluated the safety and anti-tumor activity of pemigatinib in patients with previously treated, locally advanced or metastatic cholangiocarcinoma with and without FGFR2 fusions or rearrangements. In this multi-center, open-label, single-arm, multi-cohort, phase-2 study (FIGHT-202), patients aged 18 years or older with disease progression following at least one previous treatment and an Eastern Cooperative Oncology Group (ECOG) performance status of 0-2 recruited from 146 academic or community-based sites in the USA, Europe, the Middle East, and Asia were assigned to one of three cohorts: patients with FGFR2 fusions or rearrangements, patients with other FGF/FGFR alterations, or patients with no FGF/FGFR alterations. All enrolled patients received a starting dose of 13·5 mg oral pemigatinib once daily (21-day cycle; 2 weeks on, 1 week off) until disease progression, unacceptable toxicity, withdrawal of consent, or physician decision. The primary endpoint was the proportion of patients who achieved an objective response among those with FGFR2 fusions or rearrangements, assessed centrally in all patients who received at least one dose of pemigatinib. This study is registered with ClinicalTrials.gov, NCT02924376, and enrolment is completed. Between Jan 17, 2017, and March 22, 2019, 146 patients were enrolled: 107 with FGFR2 fusions or rearrangements, 20 with other FGF/FGFR alterations, 18 with no FGF/FGFR alterations, and one with an undetermined FGF/FGFR alteration. The median follow-up was 17·8 months (IQR 11·6-21·3). 38 (35·5% [95% CI 26·5-45·4]) patients with FGFR2 fusions or rearrangements achieved an objective response (three complete responses and 35 partial responses). Overall, hyperphosphataemia was the most common all-grade adverse event irrespective of cause (88 [60%] of 146 patients). 93 (64%) patients had a grade 3 or worse adverse event (irrespective of cause); the most frequent were hypophosphataemia (18 [12%]), arthralgia (nine [6%]), stomatitis (eight [5%]), hyponatraemia (eight [5%]), abdominal pain (seven [5%]), and fatigue (seven [5%]). 65 (45%) patients had serious adverse events; the most frequent were abdominal pain (seven [5%]), pyrexia (seven [5%]), cholangitis (five [3%]), and pleural effusion (five [3%]). Overall, 71 (49%) patients died during the study, most frequently because of disease progression (61 [42%]); no deaths were deemed to be treatment related. The authors concluded that these data support the therapeutic potential of pemigatinib in previously treated patients with cholangiocarcinoma who have FGFR2 fusions or rearrangements.

EZH2 Testing for Tazemetostat (Tazverik) Therapy for Follicular Lymphoma

The efficacy of Tazverik was evaluated in two open-label, single-arm cohorts (cohort 4: EZH2-mutated follicular lymphoma and cohort 5: EZH2 wild-type follicular lymphoma) of a multi-center study (Study E7438-G000-101, NCT01897571) in patients with histologically confirmed follicular lymphoma after at least two prior systemic therapies. Patients were required to have ECOG PS of 0-2 and were enrolled based on EZH2 mutation status. EZH2 mutations were identified prospectively using formalin-fixed, paraffin-embedded tumor samples, which were centrally tested using the cobas EZH2 Mutation Test; the cobas EZH2 Mutation test is designed to detect the following mutations: Y646X [S,H,C], Y646F, Y646N, A682G, and A692V. Patients received TAZVERIK 800 mg orally twice daily until confirmed disease progression or unacceptable toxicity. Tumor response assessments were performed every 8 weeks through Week 24 and then every 12 weeks. The major efficacy outcome measures were overall response rate and duration of response according to the International Working Group Non-Hodgkin Lymphoma (IWG-NHL) criteria1 as assessed by Independent Review Committee. Median duration of follow-up was 22 months (range 3 months to 44 months) for patients with EZH2 MT positive tumors and 36 months (range 32 months to 39 months) for patients whose tumors did not have an EZH2 mutation detected. A total of 99 patients were enrolled, including 45 patients whose tumors had one of these EZH2 mutations (mutant) and 54 patients whose tumors did not have one of these mutations (wild-type). Among the 45 patients with EZH2 mutant follicular lymphoma, median age was 62 years (range 38 to 80), 58% were female, 42% had early progression following front-line therapy (POD24), and all had an ECOG PS of 0 or 1. Race was reported in 84% of patients; of these patients, 82% were White. Based on the cobas EZH2 Mutation test, 36%, 29%, 27%, 11% and 2% of patients had the following mutations: Y646X [S,H,C], Y646F, Y646N, A682G, and A692V, respectively. The median number of lines of prior systemic therapy was 2 (range 1 to 11), with 49% refractory to rituximab, 49% refractory to their last therapy, and 9% had received prior stem cell transplant. Among the 54 patients with EZH2 wild-type follicular lymphoma, median age was 61 years (range 36 to 87), 63% were male, 59% had POD24, and 91% had an ECOG PS of 0 or 1. Race was reported in 57% of patients; of these patients, 48% were White and 3% were Asian. The median number of lines of prior systemic therapy was 3 (range 1 to 8), with 59% refractory to rituximab, 41% refractory to their last therapy, and 39% had received prior stem cell transplant. The approval of Tazverik was based upon the efficacy in 95 patients (42 EZH2 Mutant, 53 EZH2 WildType) who had received at least 2 prior systemic therapies). The overall response rate in 42 patients with EZH2-mutated follicular lymphoma was 69% (95% confidence interval [CI] = 53%–82%), with 12% of patients having a complete response and 57%, a partial response. Median duration of response in these patients was 10.9 months (95% CI = 7.2–not evaluable). The overall response rate in 53 patients with EZH2 wild-type follicular lymphoma was 34% (95% CI = 22%–48%), with 4% of patients having a complete response and 30%, a partial response. Median duration of response was 13 months (95% CI = 5.6–not evaluable). The most commonly reported (≥ 20%) adverse reactions included fatigue, upper respiratory tract infection, musculoskeletal pain, nausea, and abdominal pain. Serious adverse reactions occurred in 30% of patients, most often from infection. Second primary malignancy was the most common reason for treatment discontinuation (2% of patients). The prescribing information includes a warning and precaution for secondary malignancies. The recommended tazemetostat dose is 800 mg taken orally twice daily with or without food (Epizyme 2020).

CQuentia Pharmacogenetic Comprehensive Panel

CQuentia pharmacogenetic comprehensive panel identifies patient's metabolic profiles to give prescribers more information when choosing drug therapy options for their patients.  It is supposedly one of the most comprehensive panels of pharmacogenetic tests available on the market today.  It categorizes a patient into 1 of 4 metabolizer categories for each genetic pathway and interprets that data in the context of the following specialties: cardiology, family practice, functional medication, gastroenterology, geriatrics, internal medicine, long-term care, oncology, pain management, psychiatry, urology and women's health care.  

Zeier and colleagues (2018) stated that the accrual and analysis of genomic sequencing data have identified specific genetic variants that are associated with major depressive disorder.  Moreover, substantial investigations have been devoted to identifying gene-drug interactions that affect the response to anti-depressant medications by modulating their pharmacokinetic or pharmacodynamic properties.  Despite these advances, individual responses to anti-depressants, as well as the unpredictability of adverse side effects, leave clinicians with an imprecise prescribing strategy that often relies on trial and error.  These limitations have spawned several combinatorial pharmacogenetic testing products (e.g., CNSDose, GeneCept, GeneSight, and IDgenetix)  that are marketed to physicians.  Typically, combinatorial pharmacogenetic decision support tools use algorithms to integrate multiple genetic variants and assemble the results into an easily interpretable report to guide prescribing of anti-depressants and other psychotropic medications.  The authors reviewed the evidence base for several combinatorial pharmacogenetic decision support tools whose potential utility has been evaluated in clinical settings.  They found that, at present, there are insufficient data to support the widespread use of combinatorial pharmacogenetic testing in clinical practice, although there are clinical situations in which the technology may be informative, particularly in predicting side effects.

The authors stated that “The ideal combinatorial pharmacogenetics tool would include all variants for which there is a moderate to high level of evidence supporting an interaction with antidepressant medications while excluding spurious variants that are irrelevant to the treatment of major depression or that have little empirical support.  Unfortunately, it is difficult to ascertain how well the available tests conform to this ideal, because some combinatorial pharmacogenetic products do not report the specific gene variants that are interrogated.  Furthermore, evaluating the level of evidence for each variant is a painstaking process, even with access to resources such as the PharmGKB knowledge base.  With over 30 combinatorial pharmacogenetic tools on the market, evaluating the relative clinical value of each variant independently is not practical, and such an approach does not test the proprietary algorithm-based phenotyping that is unique to each combinatorial pharmacogenetic product.  The manner by which combinatorial pharmacogenetic algorithms integrate and weigh the most important genetic variants is not reported by the companies that market them, and the application of information in the combinatorial pharmacogenetic guidance for a given patient may vary significantly from one clinician to the next.  Both of these factors reduce the interpretability of results from observational studies, which comprise the majority of combinatorial pharmacogenetic studies sponsored by the companies that sell them … Although substantial ambiguity remains as to which are the most relevant candidates for further development, we can envision a day when even more comprehensive combinatorial pharmacogenetic tests and more elaborate algorithms are available to predict antidepressant efficacy and tolerability for any patient.  Assuming that the most clinically relevant genotyping is eventually fully identified, a next generation of investigation will be needed to determine whether the available decision support tools effectively convey actionable information in a manner that improves the treatment of major depression by altering drug prescribing.  Clinicians will undoubtedly embrace decision support tools that provide easily consumable pharmacogenetic information, but only if they can be certain that the information is valid and improves the efficacy, tolerability, or affordability of specific pharmacotherapies and that the tool works well in real-life practice, in which patients often have multiple comorbidities and resistance to first-line agents.  Until then, clinicians must evaluate each commercially available combinatorial pharmacogenetic tool according to the results of a few clinical trials in which they were tested and from post hoc retrospective analysis of data from a few flawed trials.  The available literature on combinatorial pharmacogenetic products suffers from publication bias, because some products garner more investment than do others, and questions about scientific integrity are inherent in studies conducted by or reports authored by personnel with significant financial interests in the outcome.  Although some of the preliminary published data sound promising, particularly with regard to theCYP450gene variants and side effect burden, we conclude that there is insufficient evidence to support widespread use of combinatorial pharmacogenetics decision support tools at this point in time”.

CYP2D6 Genotyping for Uses Associated with Opioid Medications

VanderVaart and colleagues (2011) stated that codeine, a common opiate prescribed for pain post-cesarean section (C-section), is bio-transformed by the highly polymorphic cytochrome P450 enzyme 2D6 (CYP2D6).  Ultra-rapid metabolizers (UMs), individuals with multiple active copies of CYP2D6, can bio-transform up to 50 % more codeine into morphine than normal individuals can.  In contrast, poor metabolizers (PMs), individuals who have no active CYP2D6 genes, convert almost no codeine into morphine and as a result may take multiple doses of codeine without attaining analgesia.  In a pilot study, these researchers examined the relationship between CYP2D6 genotype and codeine analgesia among women recovering from C-section.  A total of 45 mothers prescribed codeine provided a blood sample for CYP2D6 genotyping and recorded their pain level 4 times a day for 3 days immediately after a C-section.  Codeine was used on an as-needed basis; doses and times were recorded.  The relationship between CYP2D6 genotype, pain scores, need for codeine, and adverse events was studied.  Theoretical morphine dose, based on CYP2D6 genotype, was estimated.  Women at the genotypic extremes reported codeine effects consistent with their genotype: the 2 PMs of codeine reported no analgesia as a result of taking codeine, whereas 2 of the 3 UMs reported immediate pain relief from codeine but stopped taking it due to dizziness and constipation.  These investigators stated that much larger numbers are needed to study similar correlations among extensive and intermediate metabolizers.  The authors concluded that in this pilot study, the extreme CYP2D6 genotypes (PMs and UMs) seemed to predict pain response and adverse events.  Moreover, they stated that larger sample sizes are needed to correlate the range of genotypes with pain response.

Kummer and associates (2011) noted that the main metabolic pathways of oxycodone, a potent opioid analgesic, are N-demethylation (CYP3A4) to inactive noroxycodone and O-demethylation (CYP2D6) to active oxymorphone.  These investigators performed a 3-way, placebo-controlled, double-blind cross-over study to assess the pharmacokinetic and pharmacodynamic consequences of drug interactions with oxycodone.  The 12 participants (CYP2D6 extensive metabolizers) were pre-treated with placebo, ketoconazole or paroxetine before oral oxycodone ingestion (0.2 mg/kg).  Pre-treatment with ketoconazole increased the AUC for oxycodone 2- to 3-fold compared with placebo or paroxetine.  In combination with placebo, oxycodone induced the expected decrease in pupil diameter.  This decrease was accentuated in the presence of ketoconazole, but blunted by paroxetine.  In comparison to pre-treatment with placebo, ketoconazole increased nausea, drowsiness, and pruritus associated with oxycodone.  In contrast, the effect of pre-treatment with paroxetine on the above-mentioned adverse events was not different from that of placebo.  Ketoconazole increased the analgesic effect of oxycodone, whereas paroxetine was not different from placebo.  The authors concluded that inhibition of CYP3A4 by ketoconazole increases the exposure and some pharmacodynamic effects of oxycodone.  Paroxetine pre-treatment inhibits CYP2D6 without inducing relevant changes in oxycodone exposure, and partially blunts the pharmacodynamic effects of oxycodone due to intrinsic pharmacological activities.  The authors concluded that pharmacodynamic changes associated with CYP3A4 inhibition may be clinically important in patients treated with oxycodone.

Vandenbossche and co-workers (2016) stated that combined analyses from 2 open-label, phase-I studies -- the pharmacokinetic profile of tramadol and its metabolite (M1) following a single oral dose of tramadol extended release (ER) (25 to 100 mg) in children (7 to 11 years old; study 1: n = 37) and adolescents (12 to 17 years old; study 2: n = 38) with painful conditions-were historically compared with that of healthy adults following similar dosing.  The dose-normalized area under the curve (DN AUC0-24h ) and maximum concentration (DN Cmax) of tramadol and of M1 in children and in adolescents were lower than those in adults (children versus adults: tramadol, DN AUC0-24h 82.19 %; DN Cmax 80.38 %, p = 0.0031; M1, DN AUC0-24h 51.19 %, DN Cmax 52.68 %, p < 0.0001; adolescents versus adults: tramadol, DN AUC0-24h 89.56 %, DN Cmax 84.01 %; M1, DN AUC0-24h 85.28 %, DN Cmax 83.03 %, p = 0.0004).  The arithmetic mean terminal elimination t1/2 of tramadol in children and adolescents was comparable to that in adults (children 8.4 hours; adolescents 8.5 hours; adults 7.9 hours).  The most frequently reported (greater than or equal to 5 % of participants) treatment-emergent adverse events (AEs) in children included headache, upper abdominal pain and constipation, and in adolescents were headache, nausea, dizziness, and stomach discomfort.  The authors concluded that multiple factors may have contributed to these observations, including a higher proportion of children (56 %) who may have a lower activity of CYP2D6, resulting in reduced clearance of tramadol.

St Sauver and colleagues (2017) examined the association between CYP2D6 phenotype and poor pain control or other adverse symptoms related to the use of opioids in a sample of primary care patients.  These researchers identified all patients in the Mayo Clinic RIGHT Protocol who were prescribed an opioid medication between July 1, 2013 and June 30, 2015, and categorized patients into 3 phenotypes: poor, intermediate to extensive, or ultra-rapid CYP2D6 metabolizers.  They reviewed the electronic health record of these patients for indications of poor pain control or adverse symptoms related to medication use.  Associations between phenotype and outcomes were assessed using Chi-square tests and logistic regression.  Overall, 257 (25 % of RIGHT Protocol participants) patients received at least 1 opioid prescription; of these, 40 (15 %) were poor metabolizers, 146 (57 %) were intermediate to extensive metabolizers, and 71 (28 %) were ultra-rapid metabolizers.  These investigators removed patients that were prescribed a CYP2D6 inhibitor medication (n = 38).  After adjusting for age and sex, patients with a poor or ultra-rapid phenotype were 2.7 times more likely to experience either poor pain control or an adverse symptom related to the prescription compared to patients with an intermediate to extensive phenotype (odds ratio [OR]: 2.68; 95 % confidence interval [CI]: 1.39 to 5.17; p = 0.003).  The authors concluded that their findings suggested that greater than 30 % of patients with a poor or ultra-rapid CYP2D6 phenotype may experience an adverse outcome after being prescribed codeine, tramadol, oxycodone, or hydrocodone.  These medications are frequently prescribed for pain relief, and approximately 39 % of the US population is expected to carry one of these phenotypes, suggesting that the population-level impact of these gene-drug interactions could be substantial.

The authors stated that significant limitations of this study included a high potential for misclassification.  Specifically, these researchers relied on complete reporting of pain response and adverse drug events through the electronic health records (EHRs).  Such data may be incompletely reported by patients and recorded by the health care practitioners, and the authors may have missed AEs that actually occurred.  If there was differential reporting or recording of adverse outcomes by CYP2D6 phenotype, these findings could be biased.  First, if persons with an extreme phenotype were less likely to have an adverse outcome reported in their record than those with a normal phenotype, these results will under-estimate the true association between extreme phenotype and adverse outcomes.  If persons with an extreme phenotype were more likely to have an adverse outcome reported compared to those with a normal phenotype, these results will over-estimate the true association.  However, both patients and health care providers were unaware of the patient’s phenotype at the time of the prescriptions.  Thus, the authors expected that the most likely scenario is that AEs were consistently under-reported in the medical records regardless of patient phenotype.  If such data were incompletely recorded regardless of phenotype, the result will be an overall reduction in the number of possible outcomes, and these findings would be less likely to reach statistical significance, but would not be biased.  In fact, it is somewhat surprising that the authors observed statistically significant associations.

These researchers also lacked data regarding medication compliance.  If compliance and adverse drug events were reported differentially by CYP2D6 phenotype, these findings may be biased.  The authors expected that persons with a past history of opioid use that they attributed to an AE were less likely to actually take their prescription compared to persons who had no problems with the prescription.  These researchers also expected that persons with an extreme CYP2D6 phenotype were more likely in general to have had a poor experience with opioid medications, and these persons would be less likely to take their prescribed medications.  Failing to take an opioid may result in poor pain control, and this could account for the unexpected finding that persons with an ultra-rapid phenotype seemed to experience poor pain control at about the same rate as poor metabolizers.

Finally, the Luminex kit used for genotyping in this study did not include some alleles that could change the phenotype for some of the participants.  Chiefly, the *2A allele (an increased activity allele) harbors some *35 alleles (a normal activity allele).  Therefore, some of the *2A alleles may have actually been *35 alleles, and these persons were incorrectly classified as ultra-rapid metabolizers rather than as intermediate to extensive metabolizers.  Thus, the authors excluded all persons with *2A/*2A genotypes in a sensitivity analysis.  Results were very similar to including these persons, suggesting that misclassification was not extensive.  The authors stated that future studies, however, will include typing that allows for the detection of *35 alleles.  Similarly, intermediate to extensive CYP2D6 metabolizers may have highly variable phenotypes due to non-genetic factors.  Therefore, some of these persons were likely incorrectly classified as not having an extreme phenotype.  If these persons experienced adverse outcomes at a lower rate than persons in the “extreme” phenotype groups, then these findings were an over-estimate of the true association.  However, if these persons experienced adverse outcomes at the same or higher rate as the “extreme” phenotype groups, these findings were an under-estimate of the true association.

Many of the pain medications identified in this study were prescribed in conjunction with other medications, making it difficult to definitively attribute some events to a single prescription.  The authors also relied on prescription information and did not know how much or how often the medication was taken.  They did account for the prescription of strong or moderately strong CYP2D6 inhibitor medications, and these findings were consistent regardless of whether persons taking these medications were excluded or re-classified.  Accounting for these medications substantially strengthened associations between extreme metabolizer status and adverse reactions, indicating that it is important to account for use of these medications in future analyses.

Oxycodone and hydrocodone are predominantly metabolized by CYP3A4, and variation in CYP3A4 function may significantly impact an individual’s response to these medications.  The authors conducted a sub-analysis of persons that were only prescribed oxycodone or hydrocodone, and results were attenuated compared to the overall results (as expected, due to smaller sample sizes).  However, all results were still in the same direction, and point estimates were of a similar magnitude, suggesting that variation in CYP2D6 was still an important factor to consider when considering response to oxycodone and hydrocodone.  Finally, while the study sample size was relatively large, the number of outcomes was small, and estimates may not be stable.

Knisely and colleagues (2017) noted that when codeine and tramadol are used for pain management, it is imperative that nurses are able to assess for potential drug-gene and drug-drug-gene interactions that could adversely impact drug metabolism and ultimately pain relief.  Both drugs are metabolized through the CYP2D6 metabolic pathway which can be affected by medications as well the patient's own pharmaco-genotype.  The purpose of this brief report was to identify drug-gene and drug-drug-gene interactions in 30 adult patients prescribed codeine or tramadol for pain.  These researchers used 3 data sources: (i) 6months of EHR data on the number and types of medications prescribed to each patient; (ii) each patient's CYP2D6 pharmaco-genotype, and (iii) published data on known CYP2D6 gene-drug and drug-drug-gene interactions.  A total of 10 patients (33 %) had possible drug-gene or drug-drug-gene interactions; 5 patients had CYP2D6 drug-gene interactions indicating they were not good candidates for codeine or tramadol.  In addition, 5 patients had potential CYP2D6 drug-drug-gene interactions with either codeine or tramadol.  The authors concluded that the findings from this exploratory study underscored the importance of assessing and accounting for drug-gene and drug-drug-gene interactions in patients prescribed codeine or tramadol.

Choi and associates (2017) noted that there is great heterogeneity in the way individuals respond to medications.  Inherited differences, such as single nucleotide polymorphisms (SNP), can influence the efficacy and toxicity of drugs.  This meta-analysis aimed to collate data from studies investigating the effect of SNPs on post-operative and/or intra-operative opioid requirements.  A meta-analysis was conducted following Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines.  Eligibility criteria for studies included were reporting amount of post-operative and/or intra-operative opioid used as the primary outcome and genotyping patients for SNPs in one of the following genes; OPRM1, CYP2D6, CYP3A4, CYP3A5, COMT, UGT2B7, or ABCB1.  A comprehensive systematic search for articles using keywords "opioid-sensitivity", "polymorphisms", "post-operative opioid", "post-surgical opioid", "post-operative pain", and "post-surgical pain" was performed.  A total of 51 studies were included.  Individuals homozygous for AA at the OPRMI (rs1799971) polymorphisms required less post-surgical opioid compared with those homozygous for GG (Hedges g, -0.270; 95 % confidence interval [CI]: -0.433 to -0.108; p = 0.001).  Polymorphisms in CYP2D6, CYP3A4, CYP3A5, COMT, UGT2B7, and ABCB1 did not affect opioid requirements.  The authors concluded that investigation of single changes in 1 gene can only yield limited information regarding genetic effects on opioid requirements.  Rapid development of whole genome sequencing (WES) enables information on all genetic modifications that may affect analgesic response to be collected.  The information collected must include data on the individual's metabolic enzymes, as well as information on drug receptors and enzymes responsible for drug degradation, so that a personal profile can be built up which will predict individual response to drugs, and guide clinicians on the type and dosage of drug to use.

Pharmacogenetic Studies in Patients with Chronic Kidney Disease

Tziastoudi and colleagues (2021) stated that chronic kidney disease (CKD) is an important public health problem due to its high prevalence and morbidity.  Although the treatment of patients with CKD has changed considerably, ineffectiveness as well as side effects of medications represent a major issue.  In an effort to elucidate the contribution of genetic variants located in several genes in the response to treatment of patients with CKD, these investigators carried out a systematic review and meta-analysis of all available pharmacogenetics studies.  The association between genotype distribution and response to medication was examined using the dominant, recessive, and additive inheritance models.  Subgroup analysis based on ethnicity was also carried out.  A total of 29 studies were included in the meta-analysis, which examined the association of 11 genes (16 polymorphisms) with the response to treatment regarding CKD.  Among the 29 studies, 18 studies included patients with renal transplantation, 8 involved patients with nephrotic syndrome, and 3 studies included patients with lupus nephritis.  The authors concluded that the present meta-analysis provided strong evidence for the contribution of variants harbored in the ABCB1, IL-10, ITPA, MIF, and TNF genes that created some genetic pre-disposition that lowered effectiveness or was associated with AEs of medications used in CKD.  Moreover, these researchers stated that future studies are needed to confirm the findings of this meta-analysis, and an appropriate computer program could help guide the selection of the best drugs and doses.

Tests for Patients with Cholangiocarcinoma for the Treatment with Infigratinib (Truseltiq)

On May 28, 2021, the FDA granted accelerated approval to infigratinib (Truseltiq) for the treatment of adults with previously treated, unresectable locally advanced or metastatic cholangiocarcinoma with a fibroblast growth factor receptor 2 (FGFR2) fusion or other re-arrangement as detected by an FDA-approved test.  The FDA also approved FoundationOne CDx for selection of patients with FGFR2 fusion or other re-arrangement as a companion diagnostic device for treatment with infigratinib.

Helsinn Therapeutics, Inc. released an announcement about the discontinuation of distribution of Truseltiq (infigratinib) capsules as of the first quarter of 2023. The discontinuation was based on difficulties in recruiting and enrolling study subjects for the required confirmatory clinical trial (CCA News, 2024).

Mass Spectrometry and Tandem Mass Spectrometry for Busulfan Dose Testing

The product labeling of busulfan supports pharmacokinetic monitoring. 

Lee et al (2019) stated that busulfan, frequently used as a conditioning regimen for hematopoietic stem cell transplantation (HSCT), has a narrow therapeutic range and wide intra-and inter-patient variabilities.  Thus, therapeutic drug monitoring (TDM) of busulfan is needed to ensure that the drug concentrations of patients are within a targeted therapeutic range.  These researchers developed a simple and accurate method for measuring busulfan concentrations using liquid chromatography tandem mass spectrometry (LC-MS/MS).  Separation and detection of busulfan was carried out using T3 column equipped with LC-MS/MS.  Busulfan was isolated from 50 μL human plasma after mixing with busulfan-2H8 (internal standard) solution, calibrator, and quality-control material.  The sample was gradient eluted with a mobile phase composed of ammonium acetate, formic acid, and water or methanol.  The busulfan concentration was quantified using a 6-point standard curve.  Busulfan and busulfan-2H8 were detected in positive-ion multiple-reaction-monitoring mode.  According to the Clinical and Laboratory Standards Institute (CLSI) guideline, these investigators verified the precision, linearity, limit of detection (LOD), limit of quantification (LOQ), and carry-over.  Busulfan and busulfan-2H8 were detected at m/z 264.1>151.1 and 272.2>159.1.  The total run time was 3 mins.  Both intra- and inter-assay coefficients of variation were less than 3 %.  The calibration curve was linear at 25 to 5,000 ng/ml.  The LOD and LOQ were 2.5 ng/ml and 25 ng/ml, respectively.  The recoveries ranged from 92.0 % to 104.8 % and the carry-over was -0.02 %.  The authors concluded that their method for measurement of busulfan concentrations in plasma using LC-MS/MS was very simple and required less time and smaller specimens than previous methods.  Furthermore, these findings demonstrated excellent analytical performance; therefore, this method is expected to be beneficial for therapeutic monitoring of busulfan during the treatment of HSCT patients.

Andersen et la (2021) reported that a fast and reliable method based on 2-channel LC-MS/MS was developed and successfully validated for quantification of busulfan.

Matar et al (2022) noted that busulfan is an alkylating agent commonly used in preparative regimens for hematologic malignant and non-malignant patients undergoing HSCT.  These researchers developed an ultra-performance LC-MS/MS (UPLC-MS/MS) method for quantification of busulfan in human plasma.  A total of 55 patients with hematologic malignancies (n = 34) and non- malignancies (n = 21) received myeloablative busulfan therapy before HSCT.  A MS/MS method was developed and validated to quantify busulfan levels in these patients.  The method was fully validated over the concentration range of 25 to 2,000 ng/ml (r > 0.99).  The assay method demonstrated good precision and accuracy.  Stability studies indicated that the drug was stable in various conditions.  Incurred sample re-analysis findings were within acceptable ranges (less than 15 % of the nominal concentration).  Based on the 1st dose AUC results, 1/3 of hematologic malignant patients and ½ of non-malignant patients needed dose adjustment.  However, in subsequent doses (5th, 9th, and 13th), 77 %, 82 % and 82 %, respectively, of hematologic malignant patients and 71 %, 67 % and 86 %, respectively, of non-malignant patients achieved the target range of busulfan AUC.  The authors concluded that the present MS/MS method is appropriate for routine analysis of busulfan in plasma samples of patients being prepared for HSCT therapy.  The described method is routinely employed in the authors TDM laboratory for quantification of busulfan in human plasma samples of both hematologic malignant and non-malignant patients.  These investigators stated that the use of TDM to busulfan therapy minimized toxicity, maximized effectiveness and improved transplantation outcome.

Measurement of Microsatellite Instability (MSI-H) and Mismatch Repair Deficiency (DMMR) for Patients with Unresectable or Metastatic Solid Tumors being Considered for Dostarlimab

Gomez-Raposo et al (2021) stated that the incidence of endometrial cancer (EC) is increasing worldwide.  The prognosis for patients diagnosed with early-stage EC remains good, whereas for patients with recurrent or metastatic disease, the prognosis is poor and therapeutic options, until recently, were limited.  In 2017, pembrolizumab was approved by the FDA for those patients with DMMR or MSI-H tumors; however, only 20 % to 30 % of EC have MSI, and just over 50 % of these patients benefit from treatment.  In 2019, the FDA granted breakthrough therapy designation to lenvatinib in combination with pembrolizumab for the potential treatment of patients with advanced microsatellite stable EC that has progressed after treatment with at least 1 previous systemic therapy.  It appeared clear that immune check-point inhibitors (ICIs) will have a definite place in the management of EC, both as single-agent or in combination with other targeted agents.  The authors summarized the current evidence of immune check point blockade and the identification of potential biomarkers, beyond MSI-H or DMMR, that could aid in predicting response to this agents in correlation with the genomic EC subtypes.

Redondo et al (2022) noted that between 20 % and 30 % of the EC are associated with a deficiency of a mismatch repair (MMRd) protein or MSI-H, characteristics that render the tumor more sensitive to immune checkpoint inhibitors (ICIs).  Moreover, there is no standard treatment for advanced EC after progression to a platinum-containing regimen.  The GARNET phase-I clinical trial examined the safety, tolerability, and anti-tumor activity of anti-PD1 dostarlimab in patients with advanced solid tumors.  The A1 cohort of this trial enrolled patients with MMRd or MSI-H recurrent or advanced EC who had previously received a platinum-containing regimen.  The results of this cohort showed significant clinical activity, durable responses, and a favorable safety profile, without reducing quality of life (QOL).  Based on these data, dostarlimab achieved accelerated approval.  The authors concluded that although a randomized study has not yet been carried out, dostarlimab monotherapy should be the therapy of choice for patients with advanced MMRd EC in progression after a platinum-containing regimen.  Selecting patients with EC for ICIs using the MMRd predictive biomarker could facilitate more efficient and sustainable health systems and avoid the use of more toxic combinations, leading to personalized medicine.

Oaknin et al (2022) stated that dostarlimab is a humanized monoclonal antibody that binds with high affinity to PD-1, resulting in inhibition of binding to PD-L1 and PD-L2.  These researchers reported interim data from patients with EC participating in a phase-I clinical trial of single-agent dostarlimab.  GARNET, an ongoing, single-arm, open-label, phase-I clinical trial of intravenous dostarlimab in advanced solid tumors, is being undertaken at 123 sites.  Two cohorts of patients with EC were recruited: those with DMMR/MSI-H disease (cohort A1) and those with proficient/stable (MMRp/MSS) disease (cohort A2).  Patients received dostarlimab 500 mg every 3 weeks for 4 cycles, then dostarlimab 1,000 mg every 6 weeks until disease progression.  The primary endpoints were ORR and DoR per RECIST V.1.1, as assessed by blinded independent central review.  Screening began on April 10, 2017, and 129 and 161 patients with advanced EC were enrolled in cohorts A1 and A2, respectively.  The median follow-up duration was 16.3 months (IQR of 9.5 to 22.1) for cohort A1 and 11.5 months (IQR of 11.0 to 25.1) for cohort A2.  In cohort A1, ORR was 43.5 % (95 % CI: 34.0 % to 53.4 %) with 11 complete responses (CRs) and 36 partial responses (PRs).  In cohort A2, ORR was 14.1 % (95 % CI: 9.1 % to 20.6 %) with 3 CRs and 19 PRs.  Median DOR was not reached in either cohort.  In the combined cohorts, the majority of treatment-related adverse events (TRAEs) were grade 1 to 2 (75.5 %), most commonly fatigue (17.6 %), diarrhea (13.8 %), and nausea (13.8 %).  Grade greater than or higher than 3 TRAEs occurred in 16.6 % of patients, and 5.5 % discontinued dostarlimab because of TRAEs.  No deaths were attributable to dostarlimab.  The authors concluded that dostarlimab demonstrated durable antitumor activity in both DMMR/MSI-H (ORR 43.5 %) and MMRp/MSS EC (ORR 14.1 %) with a manageable safety profile.

Maiorano et al (2022) noted that EC represents the 6th most common tumor in women.  In the advanced setting, the prognosis is dismal with limited therapeutic options.  Platinum-based chemotherapy represents the actual standard of care (SOC) in 1st-line chemotherapy; however, no standard 2nd-line chemotherapy is approved, with less than 25 % of patients responding to 2nd-line chemotherapy.  In the past decade, ICIs have changed the treatment landscape of many solid tumors.  These investigators carried out a review according to the PRISMA guidelines.  They searched Embase, Medline, Cochrane Database, and conference abstracts from international societies, up to November 2021.  Clinical trials employing ICIs in advanced EC, written in English, were included.  Reviews, letters, and commentaries were excluded.  The ORR, PFS, OS, and safety (number and grade of TRAEs) were examined.  A total of 15 studies (1,627 patients) were included: 14 non-randomized phase-I/II clinical trials and 1 randomized phase-III clinical trial. A nti-PD1 (pembrolizumab, nivolumab, dostarlimab) and anti-PD-L1 agents (avelumab, atezolizumab, durvalumab) were administered as single agents; pembrolizumab and nivolumab were combined with the tyrosine-kinase inhibitors (TKI) lenvatinib and cabozantinib, respectively; and durvalumab was associated with anti-CTLA4 tremelimumab; 4 studies selected only MSI patients.  Single agents determined an ORR from 26.7 % to 58 % among MSI patients, from 3 % to 26.7 % among MSS patients.  Disease control rate (DCR) ranged from 53.5 % to 88.9 % in MSI, 31.4 % to 35.2 % in MSS patients.  The combination of TKI and ICIs determined 32 % to 63.6 % of ORR in all-comers, 32 % to 36.2 % in MSS patients; 54.2 % to 76 % of patients developed TRAEs.  The combination of ICIs and TKI achieved a higher toxicity rate than single agents (greater than or equal to G3 TRAEs 88.9 %).  The authors concluded that ICIs represented an effective option for pre-treated advanced EC patients with a tolerable profile.  Given the encouraging results in MSI patients, every woman diagnosed with EC should be examined for MS status.  In MSS women, the combination of ICIs and TKI was more effective than monotherapy, notwithstanding safety concerns.  These researchers stated that PD-L1 could not predicted ICI response, whereas other biomarkers such as MSI and tumor mutational burden appeared more accurate.

Henry et al (2022) updated the ASCO biomarkers to guide systemic therapy for metastatic breast cancer (MBC).  The Expert Panel carried out a systematic review to identify randomized clinical trials and prospective-retrospective studies from January 2015 to January 2022.  The search identified 19 studies informing the evidence base.  Candidates for a regimen with a phosphatidylinositol 3-kinase inhibitor and hormonal therapy should undergo testing for PIK3CA mutations using NGS of tumor tissue or circulating tumor DNA (ctDNA) in plasma to determine eligibility for alpelisib plus fulvestrant.  If no mutation is found in ctDNA, testing in tumor tissue, if available, should be used.  Patients who are candidates for PARP inhibitor therapy should undergo testing for germline BRCA1 and BRCA2 pathogenic or likely pathogenic mutations to determine eligibility for a PARP inhibitor.  There is insufficient evidence for or against testing for a germline PALB2 pathogenic variant to determine eligibility for PARP inhibitor therapy in the metastatic setting.  Candidates for ICI therapy should undergo testing for expression of programmed cell death ligand-1 in the tumor and immune cells to determine eligibility for treatment with pembrolizumab plus chemotherapy.  Candidates for an ICI should also undergo testing for DMMR/MSI-H to determine eligibility for dostarlimab-gxly or pembrolizumab, as well as testing for tumor mutational burden.  Clinicians may test for NTRK fusions to determine eligibility for TRK inhibitors.  There are insufficient data to recommend routine testing of tumors for ESR1 mutations, for homologous recombination deficiency (HRD), or for TROP2 expression to guide MBC therapy selection.  There are insufficient data to recommend routine use of ctDNA or circulating tumor cells to monitor response to therapy among patients with MBC.

Estrogen Receptor 1 (ESR1) Gene Mutation Testing for Treatment with Elacestrant

Bidard and colleagues (2022) conducted the EMERALD trial, an international, multicenter, randomized, open-label, phase III clinical study, to compare the efficacy and safety of the novel oral selective estrogen degrader, elacestrant, compared with standard-of-care (SOC) endocrine therapy in patients with estrogen receptor (ER)-positive/human epidermal growth factor receptor 2 (HER2)-negative advanced breast cancer who had progression after first- or second-line treatment with the combination of endocrine therapy and a cyclin-dependent kinase 4/6 (CDK4/6) inhibitor and to compare efficacy between arms in patients with detectable estrogen receptor 1 (ESR1) mutation. Patient randomization was 1:1 for receiving elacestrant 400 mg orally once daily (n = 239) or SOC endocrine monotherapy (investigator’s choice of fulvestrant, anastrozole, letrozole, or exemestane monotherapy and dosed according to the labeling) (n = 238). In addition, random assignment was stratified according to ESR1 mutational status, presence of visceral metastases, and previous treatment with fulvestrant. ESR1 mutation was detected in 47.8% of patients, and 43.4% received two prior endocrine therapies. Evaluation of ESR1 mutational status was performed via cell-free circulating DNA at a central laboratory; blood sample analysis utilized the Guardant360 CDx. Primary end points were progression-free survival (PFS) by blinded independent central review in all patients and patients with detectable ESR1 mutations. PFS was prolonged in all patients (hazard ratio = 0.70; 95% Confidence Interval [CI], 0.55 to 0.88; p = 0.002) and patients with ESR1 mutation (hazard ratio = 0.55; 95% CI, 0.39 to 0.77; p = 0.0005). Treatment-related grade 3/4 adverse events occurred in 7.2% receiving elacestrant and 3.1% receiving SOC. Treatment-related adverse events leading to treatment discontinuations were 3.4% in the elacestrant arm versus 0.9% in SOC. Nausea of any grade occurred in 35.0% receiving elacestrant and 18.8% receiving SOC (grade 3/4, 2.5% and 0.9%, respectively). The investigators concluded that elacestrant demonstrated a significant PFS improvement compared to SOC both in the overall study population and in patients with ESR1 mutations with manageable safety in a phase III trial for patients with ER-positive/HER2-negative advanced breast cancer.

On January 27, 2023, the U.S. Food and Drug Administration (FDA) approved elacestrant (Orserdu) for the treatment of postmenopausal women or adult men with ER-positive, HER2-negative, ESR1-mutated advanced or metastatic breast cancer with disease progression following at least one line of endocrine therapy. The FDA also approved the Guardant360 CDx assay as a companion diagnostic device to identify patients with breast cancer for treatment with elacestrant (Orserdu) (FDA, 2023). The prescribing information for Orserdu with regard to dosage and administration notes that patients should be selected for treatment of ER-positive, HER2-negatve advanced or metastatic breast cancer with Orserdu based on the presence of ESR1 mutation(s) in plasma specimen using an FDA-approved test (Stemline Therapeutics, 2023).

Familial Amyotrophic Lateral Sclerosis (SOD1 Mutation)

Amyotrophic lateral sclerosis (ALS) is a progressive neurodegenerative disease involving both the upper motor neurons (UMN) and lower motor neurons (LMN). UMN signs include hyperreflexia, extensor plantar response, increased muscle tone, and weakness in a topographical representation. LMN signs include weakness, muscle wasting, hyporeflexia, muscle cramps, and fasciculations. In the early stage of the disease, the clinical aspects of ALS can vary. Affected individuals typically present with asymmetric focal weakness of the extremities (stumbling or poor handgrip) or bulbar findings (dysarthria, dysphagia). Other findings include muscle fasciculations, muscle cramps, and lability of affect but not necessarily mood.  Regardless of initial symptoms, atrophy and weakness eventually affect other muscles. Approximately 5,000 people in the U.S. are diagnosed with AML each year.

For most ALS cases, the cause is not known. Cases appear to occur sporadically with no clear associated risk factors and no family history of the disease. However, approximately 10 percent of all cases are found to be inherited and referred to as familial ALS. About 12 to 20 percent of familial cases result from mutations in the superoxide dismutase 1 (SOD1) gene that is involved in production of the enzyme copper-zinc SOD1 (NIH, 2023). Most of these cases are inherited in an autosomal dominant manner. Approximately 50 percent of individuals with pathogenic variants are symptomatic by age 46, and 90 percent are symptomatic by age 70 (McCluskey and Ladha, 2022).

Genetic testing has not routinely been performed to confirm an ALS diagnosis, as testing had offered little or no additional value in making the diagnosis and would not impact medical management or treatment options. Testing has generally been reserved for "rare cases of early-stage disease (i.e., clinically suspected and clinically possible ALS by El Escorial criteria), with or without a family history of ALS" and provide "risk information and subsequent counseling for unaffected at-risk family members". Genetic testing may provide prognostic information for ALS patients with a family history suggesting autosomal dominant inheritance. Individuals with ALS who do not have a positive family history may also consider genetic testing, as some sporadic cases have also been reported to carry this pathogenic genetic variant (McCluskey and Ladha, 2022). Genetic testing for SOD1 is now encouraged to confirm familial ALS diagnosis in adults, as a genotype-specific therapy, tofersen (Qalsody), has become available as a treatment option for that patient population.

Guardant360 Response

The Guardant360 Response (Guardant Health) is a blood-based liquid biopsy that detects changes in circulating tumor DNA (ctDNA) levels to assess early whether a patient is responding to immunotherapy or targeted treatment. The test detects single nucleotide variants (SNV) in a targeted panel of 74 genes, and selected copy number variations (CNV), fusions/rearrangements, and indels for a specific set of genes. All four types of genomic alterations are reported in a single test. Cell free DNA is isolated from whole blood. Following DNA library preparation, next generation sequencing (NGS) of specific gene regions is performed to establish a baseline. Reports provide details of variants detected, a response score and any relevant clinical information.

Rajiv et al (2018) state ctDNA analysis can provide a valuable noninvasive and tumor-specific marker for longitudinal monitoring of tumor burden. Thus, the authors explored the use of ctDNA to predict survival on durvalumab, an anti-PD-L1 therapy, using a broad targeted NGS-based 73-gene panel (Guardant360) in non-small cell lung cancer (NSCLC) and urothelial cancer (UC) cohorts. The study design included  a discovery cohort consisting of 28 patients with NSCLC and two validation NSCLC and UC cohorts of 72 and 29 patients, respectively, to correlate ctDNA changes with clinical outcomes. The authors found that somatic variants were detected in 96% of patients. Changes in variant allele frequencies (VAF) preceded radiographic responses, and patients with reduction in VAF at 6 weeks had significantly greater reduction in tumor volume, with longer progression-free and overall survival. The authors concluded that ctDNA VAF changes were strongly correlated with duration of treatment, antitumor activity, and clinical outcomes in NSCLC and UC, and that early on-treatment reduction in ctDNA VAF may be a useful predictor of long-term benefit from immunotherapy. However, prospective studies are needed to validate these findings and the value of utilizing early changes in ctDNA for therapeutic decision making by identifying non-responders to checkpoint inhibitor monotherapies and guiding combination therapies.

Zhang and colleagues (2020) evaluated the prognostic and predictive impact of ctDNA in patients with advanced cancers treated with immune checkpoint blockade. The authors analyzed pretreatment (n = 978) and on-treatment (n = 171) ctDNA samples across 16 advanced-stage tumor types from three phase I/II trials of durvalumab (± the anti-CTLA4 therapy tremelimumab). A NGS-based targeted panel (Guardant360) was used to characterize somatic genomic alterations (SNV), insertion/deletion mutations (indels), gene amplifications, and gene fusions in 73 genes with known somatic cancer gene variants. All steps, including ctDNA isolation, targeted sequencing, and variant calls, were performed at Guardant Health, a CLIA/CAP-accredited laboratory. The authors found that higher pretreatment VAF were associated with poorer overall survival (OS) and other known prognostic factors, but not objective response, suggesting a prognostic role for patient outcomes. On-treatment reductions in VAF and lower on-treatment VAF were independently associated with longer progression-free survival and OS and increased objective response rate, but not prognostic variables, suggesting that on-treatment ctDNA dynamics are predictive of benefit from immune checkpoint blockade. The authors proposed a concept of “molecular response” using ctDNA, incorporating both pretreatment and on-treatment VAF, that predicted long-term survival similarly to initial radiologic response while also permitting early differentiation of responders among patients with initially radiologically stable disease. The authors acknowledged limitations to their study. Moderate size of the gene panel was used, hindering reliable estimates of TMB as well as other potentially relevant molecular features such as clonality and tumor heterogeneity. They did not have paired tumor tissue or leukocyte cellular DNA to maximize removal of potential germline and clonal hematopoietic variants. Furthermore, although they demonstrated the potential utility of molecular response using a composite ctDNA metric that is similar to the ratio approach used in other studies, the cutoff point applied, the assay resolution around “undetectability,” and adjustment for divergent changes in individual variants will need to be further optimized; thus, these results should therefore be regarded as a proof of concept. The authors concluded that their analysis contributes to the understanding of the role of ctDNA as a prognostic and predictive biomarker and its potential to complement radiologic endpoints and adjudicate radiologically equivocal benefit.

Thompson et al (2021) state that for patients with metastatic NSCLC without a targetable mutation, programmed death 1 (PD-1) or programmed death ligand 1 (PD-L1) antibody therapy alone, or in combination with chemotherapy, has become the standard first-line therapeutic approach. However, only a subset of patients respond to these therapies, and that predicting which patients will experience a durable clinical benefit (DCB) remains a clinical challenge. Therefore, the authors performed plasma NGS on ctDNA from paired blood samples obtained at baseline and at 9 weeks in a prospective cohort of patients with metastatic NSCLC treated with pembrolizumab-based therapy. Samples were analyzed using the Guardant360 74 gene panel and OMNI assay. Among 67 patients, 51 (76.1%) had greater than 1 variant detected at a variant allele fraction greater than 0.3% and thus were eligible for calculation of molecular response from paired baseline and 9-week samples. Molecular response values were significantly lower in patients with an objective radiologic response (log mean 1.25% v 27.7%, P < .001). Patients achieving a DCB had significantly lower molecular response values compared to patients with no durable benefit (log mean 3.5% v 49.4%, P < .001). Molecular responders had significantly longer progression-free survival (hazard ratio, 0.25; 95% CI, 0.13 to 0.50) and overall survival (hazard ratio, 0.27; 95% CI, 0.12 to 0.64) compared with molecular non-responders. The authors concluded that molecular response assessment using ctDNA may serve as a noninvasive, on-therapy predictor of response to pembrolizumab-based therapy in addition to standard of care imaging in metastatic NSCLC. However, the authors note that this strategy requires validation in independent prospective studies.

oncoReveal CDx

The oncoReveal CDx (Pillar Biosciences, Inc.) test is an FDA-approved, qualitative next-generation sequencing (NGS) based in vitro diagnostic (IVD) kit designed to analyze DNA from a tumor sample to identify variants in 22 genes, assisting healthcare professionals to select targeted therapies for cancer patients based on their specific genetic profile. Specifically, the test detects single nucleotide variants (SNVs) and insertions/deletions in 22 genes (AKT1, ALK, BRAF, CTNNB1, DDR2, EGFR, ERBB2, ERBB4, FBXW7, FGFR1, FGFR2, FGFR3, KRAS, MAP2K1, MET, NOTCH1, NRAS, PIK3CA, PTEN, SMAD4, STK11, and TP53) using DNA isolated from a formalin-fixed paraffin-embedded (FFPE) tumor tissue sample. The oncoReveal CDx test is intended to be used as a companion diagnostic to identify patients who may benefit from specific targeted therapies based on their genetic mutation, particularly EGFR in non-small cell lung cancer (targeted therapy EGFR tyrosine kinase inhibitors approved by the FDA) and KRAS in colorectal cancer (targeted therapies cetuximab or panitumumab) (FDA, 2024).

oncoReveal Dx Lung and Colon Cancer Assay

oncoReveal Dx Lung and Colon Cancer Assay (Pillar Biosciences, Inc.) is a next-generation sequencing-based companion diagnostic (CDx) kit that detects somatic mutations in DNA derived from formalin-fixed paraffin-embedded (FFPE) non-small cell lung cancer (NSCLC) and colorectal cancer (CRC) tumor tissue. The assay is intended to be used to select patients with NSCLC or CRC that may benefit from treatment with the targeted therapies of cetuximab or panitumumab for CRC related to the KRAS gene (wild-type variant), and erlotinib, afatinib, gefitinib, or dacomitinib for NSCLC related to the EGFR gene (exon 19 deletions, exon 21 L858R variant), and in accordance with the approved therapeutic product labeling. 

Genomind Pharmacogenetic Report (Genomind PGx)

Genomind, Inc. (King of Prussia, PA) developed Genomind Pharmacogenetic Report (Genomind PGx) which uses pharmacogenetic testing to analyze 26 selected genes reported in several studies to have an implications for response and/or tolerability to treatments used for various psychiatric disorders. The testing separates the 26 genes into two categories: pharmacodynamic genes and pharmacokinetic genes. The pharmacodynamic gene results inform the provider about the function of the genetic variant and how the patient’s genotype may affect tolerability and/or likelihood of responding to specific therapeutic options. The pharmacokinetic gene results inform the provider about the function of the genetic variant and how a patient’s genotype may affect drug metabolism and drug serum levels and/or intestinal absorption and blood-brain barrier penetration of certain drugs. Genomind PGx is intended to inform the provider about drug response and therapeutic management major depressive and select psychiatric disorders. The testing uses a buccal swab specimen and a quantitative polymerase chain reaction methodology (Genomind, 2024; NLM, 2023a).

EffectiveRX Comprehensive Panel

Genetworx, LLC. (Glen Allen, VA) offers the EffectiveRX Comprehensive Panel test which is intended to be an aid to guide medication management decisions for patients who are prescribed to or under consideration for prescription medications associated with but not limited to cardiovascular health, psychiatry, pain management, addiction, or thrombophilia (Genetworx, 2024).

Tempus nP

Tempus Labs, Inc. (Chicago, IL) offers the Tempus nP, a pharmacogenomic test, which analyzes 13 genes and 90+ variants from DNA isolated from buccal swabs or saliva. The test is intended to help inform the provider about medication selection and dosing for patients experiencing depression, anxiety, ADHD and other mental health conditions. The test uses a buccal swab or saliva specimen (NLM, 2023b; Tempus, 2024).

Catechol-O-Methyltransferase (COMT) Genotype, Varies Test

The Mayo Clinic Laboratories (Rochester, MN) offers the Catetchol-O-Methyltransferase (COMT) Genotype, Varies test. Genomic DNA extracted from whole blood or saliva is genotyped for COMT alleles by polymerase chain reaction method. The test is intended to inform the provider about the prediction of a response to nicotine replacement therapy for smoking cessation, investigation of inhibitor dosing for decreasing levodopa metabolism, and research use for assessing estrogen metabolism (Mayo Clinic Laboratories, 2024; NLM, 2024).

Aura Genetics Pharmacogenomics (PGx) Test

Aura Genetics (Youngstown, OH) offers the Aura Genetics Pharmacogenomics (PGx) test, a comprehensive test that analyzes 28 genes and reports on over 120 medications. The test uses a simple mouth swab to identify specific differences in an individual’s DNA that may affect how their body responds to some medications. The test incorporates the Drug-Gene interaction concept which relates an individual’s genetics to response to medications. The Aura Genetics PGx test is intended to help the provider select the best match of medication and dosage specific for the individual based on their genetic makeup (Aura Genetics, 2022).

CRCdx RAS Mutation Detection Kit

The FDA granted premarket approval for the CRCdx RAS Mutation Detection Kit (EntroGen, Inc., Woodland Hills, CA), a laboratory test intended to detect changes in the genetic material in tumor samples from an individual with colorectal cancer. Test results enable the healthcare provider to determine if an individual is a suitable individual for treatment with panitumumab (Vectibix). The test detects for certain RAS gene mutations in tumor samples from an individual with colorectal cancer. If the test detects a mutation in the colorectal cancer tissue, then panitumumab is not recommended. If the test does not detect a mutation, then panitumumab may be an appropriate treatment (FDA, 2024a). The CRCdx RAS Mutation Detection Kit is a qualitative real-time PCR in vitro diagnostic test designed for the detection of 35 variants of KRAS and NRAS exon 2, 3, 4 somatic mutations in genomic DNA extracted from formalin-fixed, paraffin-embedded (FFPE) colorectal cancer (CRC) tissue samples. The test is a companion diagnostic (CDx) to aid in the identification of colorectal cancer (CRC) patients who may benefit from treat with panitumumab based on a no mutation detected test result accordance with the approved therapeutic product labeling (FDA, 2024b).

xT CDx

The FDA granted premarket approval for xT CDx (Tempus Labs, Inc., Chicago, IL), a laboratory test intended to detect the presence of multiple changes in the genetic material (mutations in 648 different genes) in an individual previously diagnoses with solid malignant tumors. The test detects certain mutations in KRAS and NRAS genes to enable healthcare providers identify if an individual with colorectal cancer may benefit from personalized treatment with either of the following: cetuximab (Erbitux) when there is an absence of mutations in codons 12 or 13 of KRAS, or panitumumab (Vectibix) when there is an absence of mutations in exons 2, 3, or 4 of KRAS and an absence of mutations in exons 2, 3, or 4 of NRAS. xT CDx is a companion diagnostic for these personalized treatments (FDA, 2023). xT CDx is a is a qualitative Next Generation Sequencing (NGS)-based in vitro diagnostic device designed for use in the detection of substitutions (single nucleotide variants (SNVs) and multi-nucleotide variants (MNVs)) and insertion and deletion alterations (INDELs) in 648 genes, as well as microsatellite instability (MSI) status, using DNA isolated from formalin-fixed paraffin embedded (FFPE) tumor tissue specimens, and DNA isolated from matched normal blood or saliva specimens, from previously diagnosed cancer patients with solid malignant neoplasms (FDA, 2024c).

PIK3CA Mutation Testing

In the INAVO120 study, a double-blind, placebo-controlled, randomized, phase 3 trial, Turner and colleagues (2024) evaluated the efficacy of inavolisib (Itovebi) in combination with palbociclib and fulvestrant in adult patients with endocrine-resistant PIK3A-mutated, HR-positive, HER2-negative (defined as IHC 0 or 1+, or IHC 2+/ISH-), locally advanced or metastatic breast cancer whose disease progressed during or within 12 months of completing adjuvant endocrine therapy and who have not received prior systemic therapy for locally advanced or metastatic disease.

The investigators prospectively determined PIK3CA mutation status in a central laboratory using the FoundationOne Liquid CDx assay on plasma-derived circulating tumor DNA (ctDNA) or in local laboratories using various validated polymerase chain reaction (PCR) or next-generation sequencing (NGS) assays on tumor tissue or plasma (Genentech, 2024).

TruSight Oncology Comprehensive (TSO Comp) Test

The FDA summary of safety and effectiveness data for the premarket approval of the Trusight Oncology Comprehensive describes it as a qualitative in vitro diagnostic test that uses targeted next-generation sequencing to detect variants in 517 genes using nucleic acids extracted from formalin-fixed, paraffin embedded (FFPE) tumor tissue samples from cancer patients with solid malignant neoplasms using the Illumina® NextSeq™ 550Dx instrument. TruSight Oncology Comprehensive is intended to be used as a companion diagnostic to identify cancer patients who may benefit from treatment with the following targeted therapies in accordance with their approved therapeutic product labeling: larotrectinib (Vitrakvi) for solid tumors with neurotrophic receptor tyrosine kinase (NTRK) 1/2/3 fusions, and selpercatinib (Retevmo) for non-small cell lung cancer (NSCLC) with rearranged during transfection (RET) fusions.

Retevmo was evaluated for efficacy in patients with advanced RET fusion-positive NSCLC enrolled in a multicenter, open-label, multi-cohort clinical trial (LIBRETTO-001, NCT03157128). In this study, local laboratories prospectively identified RET gene alteration by using next generation sequencing (NGS), polymerase chain reaction (PCR), fluorescence in situ hybridization (FISH) or other local testing methods (Lilly USA, 2024).

Vitrakvi was evaluated for efficacy in pediatric and adult patients with unresectable or metastatic solid tumors with a NTRK gene fusion enrolled in one of three multicenter, open-label, single-arm clinical trials: Study LOXO-TRK-14001 (NCT02122913), SCOUT (NCT02637687), and NAVIGATE (NCT02576431). In these studies, local laboratories prospectively identified NTRK gene fusion status by using next generation sequencing (NGS) or fluorescence in situ hybridization (FISH) (Bayer HealthCare Pharmaceuticals, 2023).

Claudin 18 Isoform 2 (CLDN18.2) Testing

Claudin 18 Isoform 2 (CLDN18.2) is a biomarker and therapeutic target in gastric cancer and gastroesophageal junction adenocarcinoma.

Shah and colleagues (2023) state that CLDN18.2 is expressed in normal gastric cells and maintained in malignant gastric and gastroesophageal junction (mG/GEJ) adenocarcinoma cells. The authors conducted a global, double-blind, phase 3 study (GLOW trial), that examined zolbetuximab, a monoclonal antibody that targets CLDN18.2, plus capecitabine and oxaliplatin (CAPOX) as first-line treatment for CLDN18.2-positive, HER2-negative, locally advanced unresectable or mG/GEJ adenocarcinoma. Patients (n = 507) were randomized 1:1 (block sizes of two) to zolbetuximab plus CAPOX or placebo plus CAPOX. GLOW met the primary endpoint of progression-free survival (median, 8.21 months versus 6.80 months with zolbetuximab versus placebo; hazard ratio (HR) = 0.687; 95% confidence interval (CI), 0.544-0.866; P = 0.0007) and key secondary endpoint of overall survival (median, 14.39 months versus 12.16 months; HR = 0.771; 95% CI, 0.615-0.965; P = 0.0118). Grade ≥3 treatment-emergent adverse events were similar with zolbetuximab (72.8%) and placebo (69.9%). The authors concluded that zolbetuximab plus CAPOX represents a potential new first-line therapy for patients with CLDN18.2-positive, HER2-negative, locally advanced unresectable or mG/GEJ adenocarcinoma. 

Shitara et al (2023) report on results from the SPOTLIGHT trial, which investigated the efficacy and safety of first-line zolbetuximab plus mFOLFOX6 (modified folinic acid [or levofolinate], fluorouracil, and oxaliplatin regimen) versus placebo plus mFOLFOX6 in patients with CLDN18.2-positive, HER2-negative, locally advanced unresectable or metastatic gastric or gastro-oesophageal junction adenocarcinoma. The SPOTLIGHT trial was a global, randomized, placebo-controlled, double-blind, phase 3 trial that found that targeting CLDN18.2 with zolbetuximab significantly prolonged progression-free survival and overall survival when combined with mFOLFOX6 versus placebo plus mFOLFOX6 in patients with CLDN18.2-positive, HER2-negative, locally advanced unresectable or metastatic gastric or gastro-oesophageal junction adenocarcinoma. Zolbetuximab plus mFOLFOX6 might represent a new first-line treatment in these patients.

Zolbetuximab-clzb, branded as Vyloy (Astellas Pharma US, Inc) is a claudin 18.2-directed cytolytic antibody that is FDA-approved, in combination with fluoropyrimidine- and platinum-containing chemotherapy, for first-line treatment of adults with locally advanced unresectable or metastatic human epidermal growth factor receptor 2 (HER2)-negative gastric or gastroesophageal junction adenocarcinoma whose tumors are claudin (CLDN) 18.2 positive as determined by an FDA-approved test. 

The VENTANA CLDN18 (43-14A) RxDx Assay is an FDA-approved laboratory test that identifies the amount of CLDN18 protein in tissue samples. It's used to help determine if patients with gastric or gastroesophageal junction (GEJ) cancer are eligible for treatment with zolbetuximab.

The National Comprehensive Cancer Network (NCCN) guidelines for "Gastric cancer" (Version 5.2024) provides a recommendation for assessment of positivity of CLDN18.2 for patients with untreated inoperable locally advanced, recurrent, or metastatic adenocarcinoma of the stomach for whom zolbetuximab therapy is being considered. Moreover, NCCN guidelines for "Esophageal and esophagogastric junction cancers" (Version 5.2024) provides a recommendation for assessment of positivity of CLDN18.2 for patients with untreated inoperable locally advanced, recurrent, or metastatic adenocarcinoma of the esophagus or EGJ for whom zolbetuximab therapy is being considered. NCCN does not specify the use of the VENTANA CLDN18 test.

References

The above policy is based on the following references:

General References
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Cardiovascular pharmacogenomics. TEC Assessment Program. Chicago, IL: BCBSA; 2007;22(7).
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Pharmacogenomics of cancer-candidate genes. TEC Assessment Program. Chicago, IL: BCBSA; 2007;22(5). 
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Evaluating evidence supporting a role for genetic markers in diagnosis, determining disease predisposition, prognosis, or predicting therapeutic response. TEC Assessment in Press. Chicago, IL: BCBSA; September 2009. 
Peterson K, Dieperink E, Ferguson L, et al. Evidence Brief: The Comparative Effectiveness, Harms, and Cost-effectiveness of Pharmacogenomics-guided Antidepressant Treatment versus Usual Care for Major Depressive Disorder. VA ESP Project #09-199; 2016.
Hayes, Inc. Pharmacogenetic testing for selected conditions. Final Evidence Report. Olympia, WA: Washington State Health Care Authority, Health Technology Assessment Program (HTAP); December 9, 2016.
Zeier Z, Carpenter LL, Kalin NH, et al. Clinical implementation of pharmacogenetic decision support tools for antidepressant drug prescribing. Am J Psychiatry. 2018;175(9):873-886.
Li KX, Loshak H. Pharmacogenomic Testing in Depression: A Review of Clinical Effectiveness, Cost-Effectiveness, and Guidelines. CADTH Rapid Response Report: Summary with Critical Appraisal. Ottawa, ON: Canadian Agency for Drugs and Technologies in Health (CADTH); January 31, 2020.
U.S. Food and Drug Administration (FDA). TruSight Oncology Comprehensive. Premarket Approval (PMA). P230011. Silver Spring, MD: FDA; November 4, 2024.
Genotyping for Cytochrome P450 Polymorphisms
Agency for Healthcare Research and Quality (AHRQ). New report finds little evidence to determine the usefulness of genetic tests in the treatment of depression. Press Release. Rockville, MD: AHRQ; January 4, 2006. 
Biovail Corporation. Xenazine (tetrabenazine) tablets. Prescribing Information. Mississauga, ON: Biovail Corporation; September 2009.
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special Report: Genotyping for cytochrome P450 polymorphisms to determine drug-metabolizer status. TEC Assessment Program. Chicago, IL: BCBSA; December 2004;19(9). 
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). CYP2D6 pharmacogenomics of tamoxifen treatment. TEC Assessment Program. Chicago, IL: BCBSA; May 2008:23. 
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Pharmacogenomics-based treatment of Helicobacter pylori infection. TEC Assessment Program. Chicago, IL: BCBSA; May 2008;23. 
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). CYP2D6 pharmacogenomics of tamoxifen treatment. TEC Assessments in Press. Chicago, IL: BCBSA; 2011.
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). CYP2D6 pharmacogenomics of tamoxifen treatment. TEC Assessments in Press. Chicago, IL: BCBSA; December 2013.
Bristol-Myers Squibb/Sanofi Pharmaceuticals Partnership. Plavix (clopidogrel bisulfate) tablets, for oral use. Prescribing Information. Bridgewater, NJ; Bristol-Myers Squibb/Sanofi Pharmaceuticals Partnership; revised May 2019. 
California Technology Assessment Forum (CTAF). Use of genetic testing to guide the initiation of warfarin therapy. Technology Assessment. San Francisco, CA: CTAF; March 2008. 
Desta Z, Zhao X, Shin JG, Flockhart DA. Clinical significance of the cytochrome P450 2C19 genetic polymorphism. Clin Pharmacokinet. 2002;41(12):913-958.
ECRI Institute. Anticoagulation drug warfarin gets new FDA prescribing infromation. Health Technol Trend. 2007;19(9):8.
Eichelbaum M, Ingelman-Sundberg M, Evans WE. Pharmacogenomics and individualized drug therapy. Annu Rev Med. 2006;57:119-137.
Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group. Recommendations from the EGAPP Working Group: Testing for cytochrome P450 polymorphisms in adults with nonpsychotic depression treated with selective serotonin reuptake inhibitors. Genet Med. 2007;9(12):819-825.
Flockhar DA. Drug Interactions: Cytochrome P450 Drug Interaction Table. Indiana University School of Medicine (2007). Indianapolis, IN: Indiana University; updated May 9, 2009. Available at: http://medicine.iupui.edu/clinpharm/ddis. Accessed on August 5, 2009.
Frere C, Cuisset T, Morange PE, et al, Effect of cytochrome P450 polymorphisms on platelet reactivity after treatment with clopidogrel in acute coronary syndrome. Am J Cardiol 2008;101:1088-1093.
Furuta T, Sugimoto M, Shirai N, et al. CYP2C19 pharmacogenomics associated with therapy of Helicobacter pylori infection and gastro-esophageal reflux diseases with a proton pump inhibitor. Pharmacogenomics. 2007;8(9):1199-1210.
Fux R, Morike K, Prohmer AM, et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: A prospective clinical study. Clin Pharmacol Ther. 2005;78(4):378-387.
Gage BF, Bass AR, Lin H, et al. Effect of genotype-guided warfarin dosing on clinical events and anticoagulation control among patients undergoing hip or knee arthroplasty. The GIFT randomized clinical trial. JAMA 2017;318(12):1115-1124.
Goetz MP, Rae JM, Suman VJ, et al. Pharmacogenetics of tamoxifen biotransformation is associated with clinical outcomes of efficacy and hot flashes. J Clin Oncol. 2005;23(36):9312-9318.
Guzey C, Spigset O. Genotyping as a tool to predict adverse drug reactions. Curr Top Med Chem. 2004;4(13):1411-1421.
Holmes DR Jr, Dehmer GJ, Kaul S, et al. ACCF/AHA Clopidogrel Clinical Alert: Approaches to the FDA "Boxed Warning". A Report of the American College of Cardiology Foundation Task Force on Clinical Expert Consensus Documents and the American Heart Association. Circulation. 2010;56(4):321-341.
Humphries SE, Hingorani A. Pharmacogenetics: Progress, pitfalls and clinical potential for coronary heart disease. Vascul Pharmacol. 2006;44(2):119-125.
Ingelman-Sundberg M. Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): Clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 2005;5(1):6-13.
Jannetto PJ, Laleli-Sahin E, Wong SH. Pharmacogenomic genotyping methodologies. Clin Chem Lab Med. 2004;42(11):1256-1264.
Jin Y, Desta Z, Stearns V, et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment.J Natl Cancer Inst. 2005;97(1):30-39.
Juran BD, Egan LJ, Lazaridis KN. The AmpliChip CYP450 test: Principles, challenges, and future clinical utility in digestive disease. Clin Gastroenterol Hepatol. 2006;4(7):822-830.
Kamali F. Genetic influences on the response to warfarin. Curr Opin Hematol. 2006;13(5):357-361.
Kirchheiner J, Bertilsson L, Bruus H, et al. Individualized medicine - implementation of pharmacogenetic diagnostics in antidepressant drug treatment of major depressive disorders. Pharmacopsychiatry. 2003;36 Suppl 3:S235-S243.
Kirchheiner J, Brockmoller J. Clinical consequences of cytochrome P450 2C9 polymorphisms. Clin Pharmacol Ther. 2005;77(1):1-16.
Kushner FG, Hand M, Smith SC Jr, et al.; American College of Cardiology Foundation/American Heart Association Task Force on Practice Guidelines. 2009 Focused Updates: ACC/AHA Guidelines for the Management of Patients With ST-Elevation Myocardial Infarction (updating the 2004 Guideline and 2007 Focused Update) and ACC/AHA/SCAI Guidelines on Percutaneous Coronary Intervention (updating the 2005 Guideline and 2007 Focused Update): A report of the American College of Cardiology Foundation/American Heart Association Task Force on Practice Guidelines. Circulation. 2009;120(22):2271-2306.
Matchar DB, Thakur ME, Grossman I, et al. Testing for cytochrome P450 polymorphisms in adults with non-psychotic depression treated with selective serotonin reuptake inhibitors (SSRIs). Evidence Report/Technology Assessment No. 146. Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); 2007.
Mega JL, Close SL, Wiviott SD, et al. Cytochrome p-450 polymorphisms and response to clopidogrel. NEJM. 2009;360(4):354-362.
Mega JL, Close SL, Wiviott SD, et al. Genetic variants in ABCB1 and CYP2C19 and cardiovascular outcomes after treatment with clopidogrel and prasugrel in the TRITON—TIMI 38 trial: A pharmacogenetic analysis. Lancet. 2010;376(9749):1312-1319.
Ndegwa, S. Pharmacogenomics and warfarin therapy. Issues in Emerging Health Technologies Issue 104. Ottawa, ON: Canadian Agency for Drugs and Technologies in Health (CADTH); 2007. 
Palylyk-Colwell E. CYP450 genotyping for determining drug metabolizer status. Issues in Emerging Health Technologies. Issue 81. Ottawa, ON: Canadian Coordinating Office for Health Technology Assessment; March 2006. 
Paré G, Mehta SR, Yusuf S, et al. Effects of CYP2C19 genotype on outcomes of clopidogrel treatment. N Engl J Med. 2010;363(18):1704-1714.
Pirmohamed M, Park BK. Cytochrome P450 enzyme polymorphisms and adverse drug reactions. Toxicology. 2003;192(1):23-32.
Rae JM, Drury S, Hayes DF, et al; ATAC trialists. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer Inst. 2012;104(6):452-460.
Regan MM, Leyland-Jones B, Bouzyk M, et al; Breast International Group (BIG) 1-98 Collaborative Group. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: The breast international group 1-98 trial. J Natl Cancer Inst. 2012;104(6):441-451.
Ruaño G, Villagra D, Szarek B, et al. Physiogenomic analysis of CYP450 drug metabolism correlates dyslipidemia with pharmacogenetic functional status in psychiatric patients. Biomark Med. 2011;5(4):439-449.
Sanderson S, Emery J, Higgins J. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: A HuGEnet systematic review and meta-analysis. Genet Med. 2005;7(2):97-104.
Simon T, Verstuyft C, Mary-Krause M, et al; French Registry of Acute ST-Elevation and Non-ST-Elevation Myocardial Infarction (FAST-MI) Investigators. Genetic determinants of response to clopidogrel and cardiovascular events. NEJM. 2009;360(4):363-375.
Sugimoto M, Furuta T, Shirai N, et al. Treatment strategy to eradicate Helicobacter pylori infection: Impact of pharmacogenomics-based acid inhibition regimen and alternative antibiotics. Expert Opin Pharmacother. 2007;8(16):2701-2717.
Trenk D, Hochholzer W, Fromm MF, et al. Cytochrome P450 2C19 681G a polymorphism and high on-clopidogrel platelet reactivity associated with adverse 1-year clinical outcome of elective percutaneous coronary intervention with drug eluting or bare-metal stents. J Am Coll Cardiol 2008;51:1925-1934.
Valeant Pharmaceuticals North America, LLC. Xenazine (tetrabenazine) tablets, for oral use. Prescribing Information. Laval QC: Valeant; revised September 2018. 
Villagra D, Goethe J, Schwartz HI, et al . Novel drug metabolism indices for pharmacogenetic functional status based on combinatory genotyping of CYP2C9, CYP2C19 and CYP2D6 genes. Biomark Med. 2011;5(4):427-438.
Visvanathan K, Chlebowski RT, Hurley P, et al; American Society of Clinical Oncology. American society of clinical oncology clinical practice guideline update on the use of pharmacologic interventions including tamoxifen, raloxifene, and aromatase inhibition for breast cancer risk reduction. J Clin Oncol. 2009;27(19):3235-3258.
Wadelius M, Pirmohamed M. Pharmacogenetics of warfarin: Current status and future challenges. Pharmacogenomics J. 2007;7(2):99-111.
Wallentin L, James S, Storey RF, et al.; for the PLATO investigators. Effect of CYP2C19 and ABCB1 single nucleotide polymorphisms on outcomes of treatment with ticagrelor versus clopidogrel for acute coronary syndromes: A genetic substudy of the PLATO trial. Lancet. 2010;376(9749):1320-1328.
Wilkinson GR. Drug metabolism and variability among patients in drug response. N Engl J Med. 2005;352(21):2211-2221.
VKORC1 Assay
Anderson JL, Horne BD, Stevens SM, et al. Randomized trial of genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation. Circulation. 2007;116(22):2563-2570.
Centers for Medicare & Medicaid Services (CMS). Decision Memo for Pharmacogenomic Testing for Warfarin Response (CAG-00400N). Baltimore, MD: CMS; August 3, 2009.
ECRI Institute. Anticoagulation drug warfarin gets new FDA prescribing information. Health Technol Trend. 2007;19(9):8.
Hynicka LM, Cahoon WD, Bukaveckas BL. Genetic testing for warfarin therapy initiation. Ann Pharmacother. 2008;42(9):1298-1303.
Kamali F. Genetic influences on the response to warfarin. Curr Opin Hematol. 2006;13(5):357-361.
Kimmel SE, French B, Kasner SE, et al.; COAG Investigators. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N Engl J Med. 2013;369(24):2283-2293
Ndegwa, S. Pharmacogenomics and warfarin therapy. Issues in Emerging Health Technologies. Issue 104. Ottawa, ON: Canadian Agency for Drugs and Technologies in Health; 2007. 
Rieder MJ, Reiner AP, Gage BF, et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med. 2005;352(22):2285-2293.
U.S. Food and Drug Administration (FDA). FDA clears genetic lab test for warfarin sensitivity. FDA News. Rockville, MD: FDA; September 17, 2007. 
Wadelius M, Pirmohamed M. Pharmacogenetics of warfarin: Current status and future challenges. Pharmacogenomics J. 2007;7(2):99-111.
UGT1A1 Molecular Assay
Ando Y, Saka H, Ando M, et al. Polymorphisms of UDP-glucuronosyltransferase gene and irinotecan toxicity: A pharmacogenetic analysis. Cancer Res. 2000;60(24):6921-6926.
Bosch TM, Meijerman I, Beijnen JH, Schellens JH. Genetic polymorphisms of drug-metabolising enzymes and drug transporters in the chemotherapeutic treatment of cancer. Clin Pharmacokinet. 2006;45(3):253-285.
Carlini LE, Meropol NJ, Bever J, et al. UGT1A7 and UGT1A9 polymorphisms predict response and toxicity in colorectal cancer patients treated with capecitabine/irinotecan. Clin Cancer Res. 2005;11(3):1226-1236.
Dervieux T, Meshkin B, Neri B. Pharmacogenetic testing: Proofs of principle and pharmacoeconomic implications. Mutat Res. 2005;573(1-2):180-194.
Eichelbaum M, Ingelman-Sundberg M, Evans WE. Pharmacogenomics and individualized drug therapy. Annu Rev Med. 2006;57:119-137 .
Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group. Recommendations from the EGAPP Working Group: Can UGT1A1 genotyping reduce morbidity and mortality in patients with metastatic colorectal cancer treated with irinotecan?. Genet Med. 2009;11(1):15-20.
Gardiner SJ, Begg EJ. Pharmacogenetic testing for drug metabolizing enzymes: Is it happening in practice? Pharmacogenet Genomics. 2005;15(5):365-369.
Han JY, Lim HS, Shin ES, et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non-small-cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol. 2006;24(15):2237-2244.
Hasegawa Y, Ando Y, Shimokata K. Screening for adverse reactions to irinotecan treatment using the Invader UGT1A1 Molecular Assay. Expert Rev Mol Diagn. 2006;6(4):527-533.
Huang MJ, Yang SS, Lin MS, Huang CS. Polymorphisms of uridine-diphosphoglucuronosyltransferase 1A7 gene in Taiwan Chinese. World J Gastroenterol. 2005;11(6):797-802.
Innocenti F, Undevia SD, Iyer L, et al. Genetic variants in the UDP-glucuronosyltransferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol. 2004;22(8):1382-1388.
Innocenti F, Vokes EE, Ratain MJ. Irinogenetics: What is the right star? J Clin Oncol. 2006;24(15):2221-2224.
Iyer L, Das S, Janisch L, et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J. 2002;2(1):43-47.
Lentz F, Tran A, Rey E, et al. Pharmacogenomics of fluorouracil, irinotecan, and oxaliplatin in hepatic metastases of colorectal cancer: Clinical implications. Am J Pharmacogenomics. 2005;5(1):21-33.
Maitland ML, Vasisht K, Ratain MJ. TPMT, UGT1A1 and DPYD: Genotyping to ensure safer cancer therapy? Trends Pharmacol Sci. 2006;27(8):432-437.
Marcuello E, Altes A, Menoyo A, et al. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer. 2004;91(4):678-682.
Marsh S, McLeod HL. Pharmacogenetics of irinotecan toxicity. Pharmacogenomics. 2004;5(7):835-843.
McLeod HL, Parodi L, Sargent DJ, et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: Results from Trial N9741. Abstract No. 3520. ASCO Annual Meeting Proceedings Part I. J Clin Oncol. 2006;24(18 Suppl):3520.
McLeod HL, Watters JW. Irinotecan pharmacogenetics: Is it time to intervene? J Clin Oncol. 2004;22(8):1356-1359.
Pfizer Oncology. Camptosar irinotecan hydrochloride injection. Prescribing Information. LAB-0134-9.0. New York, NY: Pfizer Inc.; revised July 2005. 
Pharmacia and Upjohn Co. Camptosar (irinotecan) injection, intravenous infusion. Prescribing Information. LAB-0134-22.0. New York, NY: Pfizer; revised March 2019.
U.S. Food and Drug Administration (FDA). FDA clears genetic test that advances personalized medicine: Test helps determine safety of drug therapy. FDA News.  P05-53. Rockville, MD: FDA; August 22, 2005. 
Waknine Y. FDA safety labeling changes: Camptosar, Keppra, Prempro/Premphase. Medscape Medical News, September 21, 2005. Available at: http://www.medscape.com/viewarticle/513175. Accessed December 9, 2005.
Genotyping for HLA-B*1502
Chung WH, Hung SI, Chen YT. Human leukocyte antigens and drug hypersensitivity. Curr Opin Allergy Clin Immunol. 2007;7(4):317-323.
Hung SI, Chung WH, Jee SH, et al. Genetic susceptibility to carbamazepine-induced cutaneous adverse drug reactions. Pharmacogenet Genomics. 2006;16(4):297-306.
U.S. Food and Drug Administration (FDA). Carbamazepine prescribing information to include recommendation of genetic test for patients with Asian ancestry. FDA News. Rockville, MD: FDA; December 12, 2007. 
HLA-B*5701 Screening
Mallal S, Phillips E, Carosi G. HLA-B*5701 screening for hypersensitivity to abacavir. NEJM. 2008;358(6):568-579.
Rodríguez-Nóvoa S, García-Gascó P, Blanco F, et al.Value of the HLA-B*5701 allele to predict abacavir hypersensitivity in Spaniards. AIDS Res Hum Retroviruses. 2007 Nov;23(11):1374-1376.
Saag M, Balu R, Brachmann P, et al. High sensitivity of HLA-B*5701 in Whites and Blacks in immunologically-confirmed cases of abacavir hypersensitivity (ABC HSR). Oral abstract session: 4th IAS Conference on HIV Pathogenesis, Treatment and Prevention: Abstract no. WEAB305. Sydney, Australia, July 2007. Available at:  http://www.iasociety.org/Abstract.aspx?abstractId=200701433. Accessed February 7, 2008.
Schackman BR, Scott CA, Walensky RP, et al. The cost-effectiveness of HLA-B*5701 genetic screening to guide initial antiretroviral therapy for HIV. AIDS. 2008;22(15):2025-2033.
U.S. Department of Health and Human Services, Panel on Antiretroviral Guidelines for Adult and Adolescents. Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents. Washington, DC: U.S. Department of Health and Human Services; January 29, 2008; 1-128. 
U.S. Food and Drug Administration (FDA).  Fatal hypersensitivity reactions, respiratory symptoms, and Ziagen (abacavir sulfate) [letter]. FDA Medwatch. The FDA Safety Information and Adverse Event Reporting Program. Rockville, MD: FDA; January 2000.  
Genotyping for Apolipoprotein E (Apo E)
Eichner JE, Dunn ST, Perveen G, et al. Apolipoprotein E polymorphism and cardiovascular disease: A HuGE review. Am J Epidemiol. 2002;155:487-495. 
Gerdes LU, Gerdes C, Kervinen K, et al. The apolipoprotein E allele determines prognosis and the effect on prognosis of simvastatin in survivors of myocardial infarction. A substudy of the Scandinavian Simvastatin Survival Study. Circulation 2000;101:1366-1371.
Knijff PD, Stalenhof AFH, Mol MJTM, et al. Influence of apo E polymorphism on the response to simvastatin treatment in patients with heterozygous familial hypercholesterolemia. Atherosclerosis 1990;83:89-97.
Nestel P, Simons L, Barter P, et al. A comparative study of the efficacy of simvastatin and gemfibrozil in combined hyperlipoproteinemia: prediction of response by baseline lipids, apo ? genotype, lipoprotein(a) and insulin. Atherosclerosis 1997;129: 231-239.
Nestruck AC, Bouthillier D, Sing CF, et al. Apolipoprotein E polymorphism and plasma cholesterol response to probucol. Metabolism 1987;36:743-747.
Raman G, Trikalinos TA, Zintzaras E, et al. Reviews of selected pharmacogenetic tests for non-cancer and cancer condtions. Technology Assessment Report. Prepared by the Tufts Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ). Contract No. 290-02-0022. Rockville, MD: AHRQ; November 12, 2008. 
Genotyping for Methylenetetrahydrofolate Reductase (MTHFR)
Derwinger K, Wettergren Y, Odin E, et al. A study of the MTHFR gene polymorphism C677T in colorectal cancer. Clin Colorectal Cancer. 2009;8(1):43-48.
Gemmati D, Ongaro A, Tognazzo S, et al. Methylenetetrahydrofolate reductase C677T and A1298C gene variants in adult non-Hodgkin's lymphoma patients: Association with toxicity and survival. Haematologica. 2007;92(4):478-485.
Raman G, Trikalinos TA, Zintzaras E, et al. Reviews of selected pharmacogenetic tests for non-cancer and cancer condtions. Technology Assessment Report. Prepared by the Tufts Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ). Contract No. 290-02-0022. Rockville, MD: AHRQ; November 12, 2008. 
Zhang W, Press OA, Haiman CA, et al. Association of methylenetetrahydrofolate reductase gene polymorphisms and sex-specific survival in patients with metastatic colon cancer. J Clin Oncol. 2007;25(24):3726-3731.
Measurement of Thromboxane Metabolites in Urine
Altman R, Luciardi HL, Muntaner J, Herrera RN. The antithrombotic profile of aspirin. Aspirin resistance, or simply failure? Thromb J. 2004;2(1):1.
Berger JS, Brown DL, Becker RC. Low-dose aspirin in patients with stable cardiovascular disease: A meta-analysis. Am J Med. 2008;121(1):43-49.
Coma-Canella I, Velasc A. Variability in individual responsiveness to aspirin: Clinical implications and treatment. Cardiovasc Hematol Disord Drug Targets. 2007;7(4):274-287.
Corgenix Medical Corporation. AspirinWorks [website]. Broomfield, CO: Corgenix; 2009. Available at: http://www.aspirinworks.com. Accessed December 30, 2009.
Eikelboom JW, Hirsh J, Weitz JI, et al. Aspirin-resistant thromboxane biosynthesis and the risk of myocardial infarction, stroke, or cardiovascular death in patients at high risk for cardiovascular events. Circulation. 2002;105(14):1650-1655.
Gum PA, Kottke-Marchant K, Poggio ED, et al. Profile and prevalence of aspirin resistance in patients with cardiovascular disease. Am J Cardiol. 2001;88(3):230-235.
Hedegaard SS, Hvas AM, Grove EL, et al. Optical platelet aggregation versus thromboxane metabolites in healthy individuals and patients with stable coronary artery disease after low-dose aspirin administration. Thromb Res. 2009;124(1):96-100.
Krasopoulos G, Brister SJ, Beattie WS, Buchanan MR; Centre for Reviews and Dissemintation (CRD). Aspirin "resistance" and risk of cardiovascular morbidity: Systematic review and meta-analysis. Database of Abstracts of Reviews of Effects (DARE). Accession No. 12008008057. York, UK: University of York; August 9, 2008.
Lordkipanidzé M, Pharand C, Schampaert E, et al. A comparison of six major platelet function tests to determine the prevalence of aspirin resistance in patients with stable coronary artery disease. Eur Heart J. 2007;28(14):1702-1708.
Mansour K, Taher AT, Musallam KM, Alam S. Aspirin resistance. Adv Hematol. 2009;2009:937352.
Martin CP, Talbert RL. Aspirin resistance: An evaluation of current evidence and measurement methods. Pharmacotherapy. 2005;25(7):942-953.
Miyata S, Miyata T, Kada A, Nagatsuka K. [Aspirin resistance] [Article in Japanese] Brain Nerve. 2008;60(11):1357-1364.
Rocca B, Patrono C. Determinants of the interindividual variability in response to antiplatelet drugs. J Thromb Haemost. 2005;3(8):1597-1602.
U.S. Food and Drug Administration (FDA) 510(k). AspirinWorks Test Kit (11 - Dehydro Thromboxane B2). Summary of Safety and Effectiveness. 510(k) No. K062025. Rockville, MD: FDA. May 29, 2007. 
Genetic Testing for rs3798220 Allele (LPA-Aspirin Check)
Chasman DI, Shiffman D, Zee RY, et al. Polymorphism in the apolipoprotein(a) gene, plasma lipoprotein(a), cardiovascular disease, and low-dose aspirin therapy. Atherosclerosis. 2009;203(2):371-376.
Shiffman D, Chasman DI, Ballantyne CM, et al. Coronary heart disease risk, aspirin use, and apolipoprotein(a) 4399Met allele in the Atherosclerosis Risk in Communities (ARIC) study. Thromb Haemost. 2009;102(1):179-180.
U.S. Preventive Services Task Force. Aspirin for the prevention of cardiovascular disease: U.S. Preventive Services Task Force recommendation statement. Ann Intern Med. 2009;150(6):396-404.
AUC-Targeted 5-Fluorouracil Dosing and Pharmacogenetic Testing for 5-FU Toxicity
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Laboratory testing to allow area under the curve (AUC) –targeted 5-fluorouracil dosing for patients administered chemotherapy for cancer, TEC Assessment Program. Chicago, IL: BCBSA; 2010;24(10). 
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Pharmacogenetic testing to predict serious toxicity from 5-fluorouracil (5-FU) for patients administered 5-FU-based chemotherapy for cancer. TEC Assessments in Press. Chicago, IL: BCBSA; January 2010. 
Fety R, Rolland F, Barberi-Heyob M et al. Clinical impact of pharmacokinetically-guided dose adaptation of 5-fluorouracil: Results from a multicentric randomized trial in patients with locally advanced head and neck carcinomas. Clin Cancer Res. 1998;4(9):2039-2045.
Gamelin E, Delva R, Jacob J, et al. Individual fluorouracil dose adjustment based on pharmacokinetic follow-up compared with conventional dosage: Results of a multicenter randomized trial of patients with metastatic colorectal cancer. J Clin Oncol. 2008;26(13):2099-2105.
Walko CM, McLeod HL. Will we ever be ready for blood level-guided therapy? J Clin Oncol. 2008; 26(13):2078-2079.
CYP2D6 Polymorphism and Alzheimer's Disease
Pilotto A, Franceschi M, D'Onofrio G, et al. Effect of a CYP2D6 polymorphism on the efficacy of donepezil in patients with Alzheimer disease. Neurology. 2009;73(10):761-767.
Seripa D, Bizzarro A, Pilotto A, et al. Role of cytochrome P4502D6 functional polymorphisms in the efficacy of donepezil in patients with Alzheimer's disease. Pharmacogenet Genomics. 2011;21(4):225-230.
The Vysis ALK Break Apart FISH Probe Kit
Abbott Laboratories.  Abbott introduces new ALK genetic test for specific form of non-small-cell lung cancer  Press Release. Abbott Park, IL: Abbott; August 30, 2011  
Kwak EL, Bang YJ, Camidge DR, et al. Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer. N Engl J Med. 2010 Oct 28;363(18):1693-1703.
Rodig SJ, Shapiro GI. Crizotinib, a small-molecule dual inhibitor of the c-Met and ALK receptor tyrosine kinases. Curr Opin Investig Drugs. 2010 Dec;11(12):1477-1490.
Stinchcombe TE, Baggstrom MQ, Somaiah N, et al.  Summary of presentations from the 46th Annual Meeting of the American Society of Clinical Oncology: Focus on non-small cell lung cancer.  J Thorac Oncol. 2011;6(1):227-232.
GeneSightRx Testing
Altar CA, Carhart J, Allen JD, et al. Clinical utility of combinatorial pharmacogenomics-guided antidepressant therapy: Evidence from three clinical studies. Mol Neuropsychiatry. 2015;1(3):145-55.
Altar CA, Carhart JM, Allen JD, et al. Clinical validity: Combinatorial pharmacogenomics predicts antidepressant responses and healthcare utilizations better than single gene phenotypes. Pharmacogenomics J. 2015;15(5):443-451. 
Biernacka JM, McElroy SL, Crow S, et al. Pharmacogenomics of antidepressant induced mania: A review and meta-analysis of the serotonin transporter gene (5HTTLPR) association. J Affect Disord. 2012;136(1-2):e21-e29.
Brown LC, Lorenz RA, Li J, Dechairo BM. Economic utility: Combinatorial pharmacogenomics and medication cost savings for mental health care in a primary care setting. Clin Ther. 2017;39(3):592-602.
Fleeman N, McLeod C, Bagust A, et al. The clinical effectiveness and cost-effectiveness of testing for cytochrome P450 polymorphisms in patients with schizophrenia treated with antipsychotics: A systematic review and economic evaluation. Health Technol Assess. 2010;14(3):1-157.
Gelenberg AJ. A review of the current guidelines for depression treatment. J Clin Psychiatry. 2010;71(7):e15.
Greden JF, Parikha SV, Rothschildb AJ, et al. Impact of pharmacogenomics on clinical outcomes in major depressive disorder in the GUIDED trial: A large, patient- and rater-blinded, randomized, controlled study. J Psychiat Res. 2019;111:59-67.
Hall-Flavin DK, Winner JG, Allen JD, et al. Using a pharmacogenomic algorithm to guide the treatment of depression. Transl Psychiatry. 2012;2:e172.
Hall-Flavin DK, Winner JG, Allen JD, et al. Utility of integrated pharmacogenomics testing to support the treatment of major depressive disorder in a psychiatric outpatient setting. Pharmacogenet Genomics. 2013;23(10):535-548.
Hornberger J, Li Q, Quinn B. Cost-effectiveness of combinatorial pharmacogenomic testing for treatment-resistant major depressive disorder patients. Am J Manag Care. 2015;21(6):e357-e365.
Jurgens G, Rasmussen HB, Werge T, et al. Does the medication pattern reflect the CYP2D6 genotype in patients with diagnoses within the schizophrenic spectrum? J Clin Psychopharmacol. 2012;32(1):100-105.
Khani NS, Hudson G, Mills G, et al. A systematic review of pharmacogenetic testing to guide antipsychotic treatment. Nat Ment Health. 2024;2(5):616-626.
Kirchheiner J, Seeringer A, Viviani R. Pharmacogenetics in psychiatry -- a useful clinical tool or wishful thinking for the future? Curr Pharm Des. 2010;16(2):136-144.
Lewis G, Mulligan J, Wiles N, et al. Polymorphism of the 5-HT transporter and response to antidepressants: Randomised controlled trial. Br J Psychiatry. 2011;198(6):464-471.
Lohoff FW, Ferraro TN. Pharmacogenetic considerations in the treatment of psychiatric disorders. Expert Opin Pharmacother. 2010;11(3):423-439.
Plesnicar BK. Personalized antipsychotic treatment: The adverse effects perspectives. Psychiatr Danub. 2010;22(2):329-334.
Vetti HH, Molven A, Eliassen AK, Steen VM. Is pharmacogenetic CYP2D6 testing useful? Tidsskr Nor Laegeforen. 2010;130(22):2224-2228.
Winner J, Allen JD, Altar CA, Spahic-Mihajlovic A. Psychiatric pharmacogenomics predicts health resource utilization of outpatients with anxiety and depression. Transl Psychiatry. 2013;3:e242.
Winner JG, Carhart JM, Altar CA, et al. A prospective, randomized, double-blind study assessing the clinical impact of integrated pharmacogenomic testing for major depressive disorder. Discov Med. 2013;16(89):219-227.
Winner JG, Carhart JM, Altar CA, et al. Combinatorial pharmacogenomic guidance for psychiatric medications reduces overall pharmacy costs in a 1 year prospective evaluation. Curr Med Res Opin. 2015;31(9):1633-1643. 
GeneCept Testing
Bousman CA, Bengesser SA, Aitchison KJ, et al. Review and consensus on pharmacogenomic testing in psychiatry. Pharmacopsychiatry. 2021;54(1):5-17.
Brennan FX, Gardner KR, Lombard J, et al. A naturalistic study of the effectiveness of pharmacogenetic testing to guide treatment in psychiatric patients with mood and anxiety disorders. Prim Care Companion CNS Disord. 2015;17(2).
de Lara DV, de Melo DO, Silva RAM, et al. Pharmacogenetic testing in psychiatry and neurology: An overview of reviews. Pharmacogenomics. 2021;22(8):505-513.
Korchia T, Joober R, Richieri R, et al. Utilizing pharmacogenetics when treating first episode psychosis. J Psychiatry Neurosci. 2023;48(1):E11-E12.
Kumar A, Kearney A. The use of pharmacogenetic testing in psychiatry. J Am Assoc Nurse Pract. 2021;33(11):849-851.
Perlis RH, Mehta R, Edwards 2AM, et al. Pharmacogenetic testing among patients with mood and anxiety disorders is associated with decreased utilization and cost: A propensity-score matched study. Depress Anxiety. 2018;35(10):946-952.
Vasiliu O. The pharmacogenetics of the new-generation antipsychotics -- A scoping review focused on patients with severe psychiatric disorders. Front Psychiatry. 2023;14:1124796.
Virelli CR, Mohiuddin AG, Kennedy JL. Barriers to clinical adoption of pharmacogenomic testing in psychiatry: A critical analysis. Transl Psychiatry. 2021;11(1):509.
IDgenetix Testing / NeuroIDgenetix Test (Pharmacogenetic-Guided Treatment for Anxiety and Depression)
Bousman CA, Bengesser SA, Aitchison KJ, et al. Review and consensus on pharmacogenomic testing in psychiatry. Pharmacopsychiatry. 2021;54(1):5-17.
Bradley P, Shiekh M, Mehra V, et al. Improved efficacy with targeted pharmacogenetic-guided treatment of patients with depression and anxiety: A randomized clinical trial demonstrating clinical utility. J Psychiatr Res. 2018;96:100-107.
Brennan FX, Gardner KR, Lombard J, et al. A naturalistic study of the effectiveness of pharmacogenetic testing to guide treatment in psychiatric patients with mood and anxiety disorders. Prim Care Companion CNS Disord. 2015;17(2).
Brown LC, Stanton JD, Bharthi K, et al. Pharmacogenomic testing and depressive symptom remission: A systematic review and meta-analysis of prospective, controlled clinical trials. Clin Pharmacol Ther. 2022;112(6):1303-1317.
Castle Biosciences, Inc. IDgenetix. How it Works. Friendswood, TX: Castle Biosciences; 2024. Available at: https://castlebiosciences.com/tests/pharmacogenomic-pgx/idgenetix/how-it-works. Accessed May 5, 2024.
Maciel A, Cullors A, Lukowiak AA, Garces J. Estimating cost savings of pharmacogenetic testing for depression in real-world clinical settings. Neuropsychiatr Dis Treat. 2018;14:225-230.
Najafzadeh M, Garces JA, Maciel A. Economic evaluation of implementing a novel pharmacogenomic test (IDgenetix) to guide treatment ofpatients with depression and/or anxiety. Pharmacoeconomics. 2017;35(12):1297-1310.
Olson MC, Maciel A, Gariepy JF, et al. Clinical impact of pharmacogenetic-guided treatment for patients exhibiting neuropsychiatric disorders: A randomized controlled trial. Prim Care Companion CNS Disord. 2017;19(2).
Ontario Health (Quality). Multi-gene pharmacogenomic testing that includes decision-support tools to guide medication selection for major depression: A health technology assessment. Ont Health Technol Assess Ser. 2021;21(13):1-214.
Oslin DW, Lynch KG, Shih M-C, et al. Effect of pharmacogenomic testing for drug-gene interactions on medication selection and remission of symptoms in major depressive disorder: The PRIME Care randomized clinical trial. JAMA. 2022;328(2):151-161.
Roberts TA, Wagner JA, Sandritter T, et al. Retrospective review of pharmacogenetic testing at an academic children’s hospital. Clin Transl Sci. 2021;14(1):412-421.
Sugarman EA, Cullors A, Centeno J, et al. Contribution of pharmacogenetic testing to modeled medication change recommendations in a long-term care population with polypharmacy. Drugs Aging 2016;33:929–936. 
Wang X, Wang C, Zhang Y, An Z. Effect of pharmacogenomics testing guiding on clinical outcomes in major depressive disorder: A systematic review and meta-analysis of RCT. BMC Psychiatry. 2023;23(1):334.
CNSDose Testing
Bousman CA, Müller DJ, Ng CH, et al: Concordance between actual and pharmacogenetic predicted desvenlafaxine dose needed to achieve remission in major depressive disorder: A 10-week open-label study. Pharmacogenet Genomics 2017; 27:1–6.
Singh AB: Improved antidepressant remission in major depression via a pharmacokinetic pathway polygene pharmacogenetic report. Clin Psychopharmacol Neurosci 2015;13:150–156.
ABCB1 Gene Testing
Breitenstein B, Scheuer S, Pfister H, et al. The clinical application of ABCB1 genotyping in antidepressant treatment: a pilot study. CNS Spectr. 2014;19(2):165-175.
EGFR
Alberta Provincial Thoracic Tumour Team. Non-small cell lung cancer stage IV. Clinical Practice Guideline No. LU-004. Edmonton, AB: Alberta Health Services, Cancer Care; June 2011.
Gao G, Ren S, Li A, et al. Epidermal growth factor receptor-tyrosine kinase inhibitor therapy is effective as first-line treatment of advanced non-small-cell lung cancer with mutated EGFR: A meta-analysis from six phase III randomized controlled trials. Int J Cancer. 2012;131(5):E822-E829.
Harris LN, Ismaila N, McShane LM, et al. Use of biomarkers to guide decisions on adjuvant systemic therapy for women with early-stage invasive breast cancer: American Society of Clinical Oncology Clinical Practice Guideline. J Clin Oncol. 2016;34(10):1134-1150.
National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2024. Plymouth Meeting, PA: NCCN; February 9, 2024.
KRAS
Adelstein BA, Dobbins TA, Harris CA, et al. A systematic review and meta-analysis of KRAS status as the determinant of response to anti-EGFR antibodies and the impact of partner chemotherapy in metastatic colorectal cancer. Eur J Cancer. 2011;47(9):1343-1354.
Allegra CJ, Jessup JM, Somerfield MR, et al. American Society of Clinical Oncology provisional clinical opinion: Testing for KRAS gene mutations in patients with metastatic colorectal carcinoma to predict response to anti-epidermal growth factor receptor monoclonal antibody therapy. J Clin Oncol. 2009;27(12):2091-2096.
Amado RG, Wolf M, Peeters M, et al. Wild-type KRAS is required for panitumumab efficacy in patients with metastatic colorectal cancer. J Clin Oncol. 2008;26(10):1626-1634.
Blue Cross Blue Shield Association (BCBSA), Technology Evaluation Center (TEC). KRAS mutations and epidermal growth factor receptor inhibitor therapy in metastatic colorectal cancer. TEC Assessment Program. Chicago, IL: BCBSA; January 2009;25(6).
Bokemeyer C, Bondarenko I, Hartmann J, et al. KRAS status and efficacy of first-line treatment of patients with metastatic colorectal cancer (mCRC) with FOLFOX with or without cetuximab: The OPUS experience. J Clin Oncol. 2008;26:(May 20 Suppl; Abstr. 4000).
De Roock W, Piessevaux H, De Schutter J, et al. KRAS wild-type state predicts survival and is associated to early radiological response in metastatic colorectal cancer treated with cetuximab. Ann Oncol. 2008;19(3):508-515.
Di Fiore F, Blanchard F, Charbonnier F, et al. Clinical relevance of KRAS mutation detection in metastatic colorectal cancer treated by Cetuximab plus chemotherapy. Br J Cancer. 2007;96(8):1166-1169.
Gonçalves A, Esteyries S, Taylor-Smedra B, et al. A polymorphism of EGFR extracellular domain is associated with progression free-survival in metastatic colorectal cancer patients receiving cetuximab-based treatment. BMC Cancer. 2008;8(169).
Hollestelle A, Pelletier C, Hooning M, et al. Prevalence of the variant allele rs61764370 T>G in the 3'UTR of KRAS among Dutch BRCA1, BRCA2 and non-BRCA1/BRCA2 breast cancer families. Breast Cancer Res Treat. 2011;128(1):79-84.
Karapetis CS, Khambata-Ford S, Jonker DJ, et al. K-ras mutations and benefit from cetuximab in advanced colorectal cancer. N Engl J Med. 2008;359(17):1757-1765.
Lièvre A, Bachet JB, Boige V, et al. KRAS mutations as an independent prognostic factor in patients with advanced colorectal cancer treated with cetuximab. J Clin Oncol. 2008;26(3):374-379.
Lièvre A, Bachet JB, Le Corre D, et al. KRAS mutation status is predictive of response to cetuximab therapy in colorectal cancer. Cancer Res. 2006;66(8):3992-3995.
Ratner E, Lu L, Boeke M, et al. A KRAS-variant in ovarian cancer acts as a genetic marker of cancer risk. Cancer Res. 2010;70(16):6509-6515.
Sacher AG, Paweletz C, Dahlberg SE, et al. Prospective validation of rapid plasma genotyping for the detection of EGFR and KRAS mutations in advanced lung cancer. JAMA Oncol. 2016;2(8):1014-1022.
Van Cutsem E, Lang I, D'haens G, et al. KRAS status and efficacy in the first-line treatment of patients with metastatic colorectal cancer (mCRC) treated with FOLFIRI with or without cetuximab: The CRYSTAL experience. J Clin Oncol. 2008;26: Abstr. 2.
Platelet Reactivity Testing (VerifyNow P2Y12 Assay)
Agency for Healthcare Research and Quality (AHRQ). Testing of CYP2C19 variants and platelet reactivity for guiding antiplatelet treatment. Draft Comparative Effectiveness Review. Rockville, MD: AHRQ; 2012.
Brar SS, ten Berg J, Marcucci R, et al. Impact of platelet reactivity on clinical outcomes after percutaneous coronary intervention. A collaborative meta-analysis of individual participant data. J Am Coll Cardiol. 2011;58(19):1945-1954.
Gurbel PA, Tantry US. Clopidogrel response variability and the advent of personalised antiplatelet therapy. A bench to bedside journey. Thromb Haemost. 2011;106(2):265-271.
Holmes MV, Perel P, Shah T, et al. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: A systematic review and meta-analysis. JAMA. 2011;306(24):2704-2714
Kim IS, Jeong YH, Park Y, et al. Interaction analysis between genetic polymorphisms and pharmacodynamic effect in patients treated with adjunctive cilostazol to dual antiplatelet therapy: Results of the ACCEL-TRIPLE (Accelerated Platelet Inhibition by Triple Antiplatelet Therapy According to Gene Polymorphism) study. Br J Clin Pharmacol. 2012;73(4):629-640.
Mallouk N, Labruyère C, Reny JL, et al. Prevalence of poor biological response to clopidogrel: A systematic review. Thromb Haemost. 2012;107(3):494-506.
O'Connor SA, Hulot JS, Silvain J, et al. Pharmacogenetics of clopidogrel. Curr Pharm Des. 2012;18(33):5309-5327.
Price MJ, Berger PB, Teirstein PS, et al; GRAVITAS Investigators. Standard- vs high-dose clopidogrel based on platelet function testing after percutaneous coronary intervention: The GRAVITAS randomized trial. JAMA. 2011;305(11):1097-1105.
Sharma RK, Voelker DJ, Sharma R, et al. Evolving role of platelet function testing in coronary artery interventions. Vasc Health Risk Manag. 2012;8:65-75.
Varenhorst C, Koul S, Erlinge D, et al. Relationship between clopidogrel-induced platelet P2Y12 inhibition and stent thrombosis or myocardial infarction after percutaneous coronary intervention-a case-control study. Am Heart J. 2011;162(2):363-371.
Yu LH, Kim MH, Zhang HZ, et al. Impact of platelet function test on platelet responsiveness and clinical outcome after coronary stent implantation: Platelet responsiveness and clinical outcome. Korean Circ J. 2012;42(6):382-389.
Beta Adrenergic Receptor Genotyping (ADRB2) for Treatment-Resistant Asthma
Basu K, Palmer CN, Tavendale R, et al. Adrenergic beta(2)-receptor genotype predisposes to exacerbations in steroid-treated asthmatic patients taking frequent albuterol or salmeterol. J Allergy Clin Immunol. 2009;124(6):1188-1194.e3
Basu K, Donald HP, Lipworth BJ, et al. Comparison of montelukast versus salmeterol in children with asthma carrying at-risk β2adrenergic receptor polymorphism: A genotype-stratified randomised controlled trial. J Allergy Clin Immunol. 2011;127(2 Suppl):AB212.
Bleecker ER, Nelson HS, Kraft M, et al. Beta2-receptor polymorphisms in patients receiving salmeterol with or without fluticasone propionate. Am J Respir Crit Care Med. 2010;181(7):676-687.
Bleecker ER, Postma DS, Lawrance RM, et al. Effect of ADRB2 polymorphisms on response to long acting beta2-agonist therapy: A pharmacogenetic analysis of two randomised studies. Lancet. 2007;370(9605):2118-2125.
Cleveland Clinic: Current Clinical Medicine. Second Edition. 2010.
Giubergia V, Gravina LP, Castaños C, et al. Influence of beta2-adrenoceptor polymorphisms on the response to chronic use of albuterol inasthmatic children. Pediatr Pulmonol. 2008;43(5):421-425.
Hawkins GA, Weiss ST, Bleecker ER. Asthma pharmacogenomics. Immunol Allergy Clin North Am. 2005;25(4):723-742.
Hawkins GA, Tantisira K, Meyers DA, et al. Sequence, haplotype, and association analysis of ADRbeta2 in a multiethnic asthma case-control study. Am J Respir Crit Care Med. 2006;174(10):1101-1109.
Holloway JW, Arshad SH, Holgate ST. Using genetics to predict the natural history of asthma? J Allergy Clin Immunol. 2010;126(2):200-209; quiz 210-211.
Johnson M. Molecular mechanisms of beta(2)-adrenergic receptor function, response, and regulation. J Allergy Clin Immunol. 2006;117(1):18-24.
Kazani S, Wechsler ME, Israel E. The role of pharmacogenomics in improving the management of asthma. J Allergy Clin Immunol. 2010;125(2):295-302; quiz 303-304.
Liggett SB. The pharmacogenetics of beta2-adrenergic receptors: Relevance to asthma. J Allergy Clin Immunol. 2000;105(2 Pt 2):S487-S492.
Litonjua AA. The significance of beta2-adrenergic receptor polymorphisms in asthma. Curr Opin Pulm Med. 2006;12(1):12-17.
Martin AC, Zhang G, Rueter K, et al. Beta2-adrenoceptor polymorphisms predict response to beta2-agonists in children with acute asthma. J Asthma. 2008;45(5):383-388.
Meurer JR, Lustig JV, Jacob HJ. Genetic aspects of the etiology and treatment of asthma. Pediatr Clin North Am. 2006;53(4):715-725.
Nelson KA et al. Pharmacogenomics of acute asthma: The β 2-adrenergic receptor gene as a model for future therapy. Clin Pediatr Emerg Med. 2009;10(2).
Ortega VE, Hawkins GA, Peters SP, Bleecker ER. Pharmacogenetics of the beta 2-adrenergic receptor gene. Immunol Allergy Clin North Am. 2007;27(4):665-684; vii.
Qiu YY, Zhang XL, Qin Y, et al. Beta(2)-adrenergic receptor haplotype/polymorphisms and asthma susceptibility and clinical phenotype in aChinese Han population. Allergy Asthma Proc. 2010;31(5):91-97.
Taylor DR et al. Bronchodilator response in relation to beta2-adrenoceptor haplotype in patients with asthma. Am J Respir Crit Care Med. 2005;172(6):700-703.
Taylor DR. Pharmacogenetics of beta2-agonist drugs in asthma. Clin Rev Allergy Immunol. 2006;31(2-3):247-258.
Yu IW, Bukaveckas BL. Pharmacogenetic tests in asthma therapy. Clin Lab Med. 2008;28(4):645-665.
MGMT Assay for Glioblastoma
Christians A, Hartmann C, Benner A, et al. Prognostic value of three different methods of MGMT promoter methylation analysis in a prospective trial on newly diagnosed glioblastoma. PLoS One. 2012;7(3):e33449.
Donson AM, Addo-Yobo SO, Handler MH, et al. MGMT promoter methylation correlates with survival benefit and sensitivity to temozolomide in pediatric glioblastoma. Pediatr Blood Cancer. 2007;48(4):403-407.
Hegi ME, Diserens AC, Godard S, et al. Clinical trial substantiates the predictive value of O-6-methylguanine-DNA methyltransferase promoter methylation in glioblastoma patients treated with temozolomide. Clin Cancer Res. 2004;10(6):1871-1874.
Hegi ME, Diserens AC, Gorlia T, et al. MGMT gene silencing and benefit from temozolomide in glioblastoma. N Engl J Med. 2005;352(10):997-1003.
Hegi ME, Liu L, Herman JG, et al. Correlation of O6-methylguanine methyltransferase (MGMT) promoter methylation with clinical outcomes in glioblastoma and clinical strategies to modulate MGMT activity. J Clin Oncol. 2008;26(25):4189-4199.
Idbaih A, Omuro A, Ducray F, et al. Molecular genetic markers as predictors of response to chemotherapy in gliomas. Curr Opin Oncol. 2007;19(6):606-611.
Jansen M, Yip S, Louis DN. Molecular pathology in adult gliomas: Diagnostic, prognostic, and predictive markers. Lancet Neurology. 2010;9(7).
National Comprehensive Cancer Network (NCCN). Central nervous system cancers. NCCN Clinical Practice Guidelines in Oncology v.1.2012. Fort Washington, PA: NCCN; 2012.
National Comprehensive Cancer Network (NCCN). Central nervous system cancers. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Fort Washington, PA: NCCN; 2019.
Perry J, Laperriere N, Zuraw L, et al; Neuro-oncology Disease Site Group. Adjuvant systemic chemotherapy, following surgery and external beam radiotherapy, for adults with newly diagnosed malignant glioma: A clinical practice guideline.  Evidence-based series No.9-2. Toronto, ON: Cancer Care Ontario (CCO); November 2006.
Stupp R, Hegi ME, Mason WP, et al; European Organisation for Research and Treatment of Cancer Brain Tumour and Radiation Oncology Groups; National Cancer Institute of Canada Clinical Trials Group. Effects of radiotherapy with concomitant and adjuvant temozolomide versus radiotherapy alone on survival in glioblastoma in a randomised phase III study: 5-year analysis of the EORTC-NCIC trial. Lancet Oncology. 2009;10(5).
Stupp R,Hegi ME, Gilbert MR, Chakravarti A. Chemoradiotherapy in malignant glioma: standard of care and future directions. J Clin Oncol. 2007;25(26):4127-4136.
Mutation Testing in the BRAF Gene
BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special Report: Companion diagnostics: Example of BRAF gene mutation testing to select patients with melanoma for treatment with BRAF kinase inhibitors. TEC Assessment Program. Chicago, IL: BCBSA; November 2011;26(7).
Bristol-Myers Squibb Co. Daklinza (daclatasvir) tablets, for oral use. Prescribing Information. Princeton, NJ: Bristol-Myers Squibb; revised November 2017.
Daniotti M, Vallacchi V, Rivoltini L, et al. Detection of mutated BRAFV600E variant in circulating DNA of stage III-IV melanoma patients. Int J Cancer. 2007;120(11):2439-2444.
Dienstmann R, Tabernero J. BRAF as a target for cancer therapy. Anticancer Agents Med Chem. 2011;11(3):285-295.
Genentech, Inc. Cotellic (cobimetinib) tablets, for oral use. Prescribing Information. South San Francisco, CA: Genentech; revised January 2018.
Merck & Co, Inc. Keytruda (pembrolizumab) for injection, for intravenous use. Prescribing Information. Whitehouse Station, NJ: Merck; revised September 2019. 
Merck & Co, Inc. Keytruda (pembrolizumab) for injection, for intravenous use. Prescribing Information. Whitehouse Station, NJ: Merck; revised June 2020. 
National Comprehensive Cancer Network (NCCN). Mekinist. NCCN Drugs & Biologics Compendium. Fort Washington, PA: NCCN; 2019.
Nazarian R, Shi H, Wang Q, Melanomas acquire resistance to B-RAF(V600E) inhibition by RTK or N-RAS upregulation. Nature. 2010;468(7326):973-977.
Panka DJ, Sullivan RJ, Mier JW. An inexpensive, specific and highly sensitive protocol to detect the BrafV600E mutation in melanoma tumor biopsies and blood. Melanoma Res. 2010;20(5):401-407.
Pinzani P, Salvianti F, Cascella R, et al. Allele specific Taqman-based real-time PCR assay to quantify circulating BRAFV600E mutated DNA in plasma of melanoma patients. Clin Chim Acta. 2010;411(17-18):1319-1324.
Pinzani P, Santucci C, Mancini I, et al. BRAFV600E detection in melanoma is highly improved by COLD-PCR. Clin Chim Acta. 2011;412(11-12):901-905.
U.S. Food and Drug Administration (FDA). FDA approves encorafenib and binimetinib in combination for unresectable or metastatic melanoma with BRAF mutations. FDA News. Silver Spring, MD: FDA; June 27, 2018. 
U.S. Food and Drug Administration (FDA). FDA approves new uses for two drugs administered together for the treatment of BRAF-positive anaplastic thyroid cancer. FDA News. Silver Spring, MD: FDA; May 4, 2018. 
U.S. Food and Drug Administration (FDA). FDA grants regular approval to dabrafenib and trametinib combination for metastatic NSCLC with BRAF V600E mutation. FDA News. Silver Spring, MD: FDA; June 22, 2017. 
U.S. Food and Drug Administration (FDA). Vemurafenib. Silver Spring, MD: FDA; August 17, 2011. Available at: http://www.fda.gov/AboutFDA/CentersOffices/CDER/ucm268301.htm. Accessed August 18, 2011.
U.S. Food and Drug Adminstration. FDA approves two drugs, companion diagnostic test for advanced skin cancer. FDA News. Silver Spring, MD: FDA; May 29, 2013. 
Millennium PGT
Svetlík S, Hronová K, Bakhouche H, et al. Pharmacogenetics of chronic pain and its treatment. Mediators Inflamm. 2013;2013:864319.
Vuilleumier PH, Stamer UM, Landau R. Pharmacogenomic considerations in opioid analgesia. Pharmgenomics Pers Med. 2012;5:73-87.
PROOVE Opioid Risk Test
Piller C. Proove Biosciences, which sold dubious DNA tests to predict addiction risk, sells off assets as CEO departs amid criminal probe. STAT, August 31, 2017.
PROOVE Biosciences.  PROOVE opiod risk.  Irvine, CA: PROOVE Biosciences; 2016. Available at: https://proove.com/proove-opioid-risk/. Accesed February 3, 2016.
BRACAnalysis CDx
AstraZenica Pharmaceutics, LP. Lynparza (olaparib) capsules, for oral use. Prescribing Information. Reference ID: 4140621. Wilmington, DE: AstraZenica; revised August 2017.
Myriad Genetics. BRACAnalysis CDx. Technical Information Summary. Salt Lake City, UT: Myriad Genetics; 2014.
U.S. Food and Drug Administration (FDA). BRACAnalysis CDx - P140020. Recently Approved Medical Devices. Silver Spring, MD: FDA; updated December 30, 2014.
U.S. Food and Drug Administration (FDA). FDA approves first treatment for breast cancer with a certain inherited genetic mutation. FDA News Release. Silver Spring, MD: FDA; January 12, 2018.
Thiopurine S-Methyltransferase (TPMT)
Coenen MJ, de Jong DJ, van Marrewijk CJ, et al; TOPIC Recruitment Team. Identification of patients with variants in TPMT and dose reduction reduces hematologic events during thiopurine treatment of inflammatory bowel disease. Gastroenterology. 2015;149(4):907-917.
Plumpton CO, Roberts D, Pirmohamed M, Hughes DA. A systematic review of economic evaluations of pharmacogenetic testing for prevention of adverse drug reactions. Pharmacoeconomics. 2016;34(8):771-793.
FLT3 Mutation for Midostaurin (Rydapt)
Novartis Pharmaceuticals Corporation. Rydapt (midostaurin) capsules, for oral use. Prescribing Information. Reference ID: 4090671. East Hanover, NJ: Novartis; revised April 2017.
Aegis Drug-Drug Interaction Test
Chen J, Hou W, Han B, et al. Target-based metabolomics for the quantitative measurement of 37 pathway metabolites in rat brain and serum using hydrophilic interaction ultra-high-performance liquid chromatography-tandem mass spectrometry. Anal Bioanal Chem. 2016;408(10):2527-2542.
OneOme RightMed Pharmacogenomic Test
Berm EJ, Looff Md, Wilffert B, et al. Economic evaluations of pharmacogenetic and pharmacogenomic screening tests: A systematic review. Second update of the literature. PLoS One. 2016;11(1):e0146262.
Del Casale A, Pomes LM, Bonanni L, et al. Pharmacogenomics-guided pharmacotherapy in patients with major depressive disorder or bipolar disorder affected by treatment-resistant depressive episodes: A long-term follow-up study. J Pers Med. 2022;12(2):316.
Hafner S, Haubensak S, Paul T, Zolk O. How to individualize drug therapy based on pharmacogenetic information? A systematic review of published guidelines. Dtsch Med Wochenschr. 2016;141(21):e183-e202.
Health Quality Ontario. Pharmacogenomic testing for psychotropic medication selection: A systematic review of the Assurex GeneSight psychotropic test. Ont Health Technol Assess Ser. 2017;17(4):1-39.
Hinderer M, Boeker M, Wagner SA, et al. Integrating clinical decision support systems for pharmacogenomic testing into clinical routine - a scoping review of designs of user-system interactions in recent system development. BMC Med Inform Decis Mak. 2017;17(1):81.
Peterson K, Dieperink E, Anderson J, et al. Rapid evidence review of the comparative effectiveness, harms, and cost-effectiveness of pharmacogenomics-guided antidepressant treatment versus usual care for major depressive disorder. Psychopharmacology (Berl). 2017;234(11):1649-1661.
Rosenblat JD, Lee Y, McIntyre RS, et al. Does pharmacogenomic testing improve clinical outcomes for major depressive disorder? A systematic review of clinical trials and cost-effectiveness studies. J Clin Psychiatry. 2017;78(6):720-729.
Verbelen M, Weale ME, Lewis CM. Cost-effectiveness of pharmacogenetic-guided treatment: Are we there yet? Pharmacogenomics J. 2017;17(5):395-402.
CYP2C19 Polymorphisms Testing for Individuals on Fluoxetine
Bykov K, Schneeweiss S, Donneyong MM, et al. Impact of an interaction between clopidogrel and selective serotonin reuptake inhibitors. Am J Cardiol. 2017;119(4):651-657.
Dinger J, Meyer MR, Maurer HH. In vitro cytochrome P450 inhibition potential of methylenedioxy-derived designer drugs studied with a two-cocktail approach. Arch Toxicol. 2016;90(2):305-318.
Luk S, Atayee RS, Ma JD, Best BM. Urinary diazepam metabolite distribution in a chronic pain population. J Anal Toxicol. 2014;38(3):135-142.
PGxOnePlus
Bennett S, Walkup JT. Anxiety disorders in children and adolescents: Epidemiology, pathogenesis, clinical manifestations, and course. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed May 2017.
Bonfiglio F, Hysi PG, Ek W, et al. A meta-analysis of reflux genome-wide association studies in 6750 Northern Europeans from the general population. Neurogastroenterol Motil. 2017;29(2).
Ehret GB. Genetic factors in the pathogenesis of hypertension. UpToDate [online serial]  Waltham, MA: UpToDate; reviewed May 2017.
Fass R. Approach to refractory gastroesophageal reflux disease in adults. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed May 2017.
Health Quality Ontario. Pharmacogenomic testing for psychotropic medication selection: A systematic review of the Assurex GeneSight Psychotropic Test. Ont Health Technol Assess Ser. 2017;17(4):1-39.
Genotyping of Interferon-Lambda 3 (IFNL3) for Prediction of Virological Response to Pegylated-Interferon-Alpha and Ribavirin Combination Therapy
Haj-Sheykholeslami A, Keshvari M, Sharafi H, et al. Interferon-λ polymorphisms and response to pegylated interferon in Iranian hepatitis C patients. World J Gastroenterol. 2015;21(29):8935-8942..
Ishiguro H, Abe H, Seki N, et al. Interferon-λ3 polymorphisms in pegylated-interferon-α plus ribavirin therapy for genotype-2 chronic hepatitis C. World J Gastroenterol. 2015;21(13):3904-3911.
Muir AJ, Gong L, Johnson SG, et al. Clinical pharmacogenetics implementation consortium (CPIC) guidelines for IFNL3 (IL28B) genotype and peginterferon alpha based regimens. Clin Pharmacol Ther. 2014;95(2):141-146.
Nakamoto S, Imazeki F, Kanda T, et al. Association of IFNL3 genotype with hepatic steatosis in chronic hepatitis C patients treated with peginterferon and ribavirin combination therapy. Int J Med Sci. 2017;14(11):1088-1093.
UrSure Tenofovir Quantification Test
ClinicalTrials.gov. Tenofovir adherence to rapidly guide and evaluate PrEP and HIV therapy (TARGET). ClinicalTrials.gov. Identifier NCT03012607. Bethesda, MD: National Library of Medicine; updated: October 26, 2017.
Cressey TR, Siriprakaisil O, Klinbuayaem V, et al. A randomized clinical pharmacokinetic trial of tenofovir in blood, plasma and urine in adults with perfect, moderate and low PrEP adherence: The TARGET study. BMC Infect Dis. 2017; 17: 496.
Koenig HC, Mounzer K, Daughtridge GW, et al. Urine assay for tenofovir to monitor adherence in real time to tenofovir disoproxil fumarate/emtricitabine as pre-exposure prophylaxis. HIV Med. 2017;18(6):412-418.
INFINITI Neural Response Panel
Chang S, Hudspeth R, Vairavan, et al. Risk assessment of opioid addiction with a multi-variant genetic panel involved in the dopamine pathway [abstract]. 2017 AACC Anual Scientific Meeting & Clinical Lab Expo. Abstract A-150. Available at: http://www.abstractsonline.com/pp8/#!/4365/presentation/180. Accessed September 28, 2018.
Donaldson K, Demers L, Taylor K, Lopez J, Chang S. Multi-variant Genetic Panel for Genetic Risk of Opioid Addiction. Ann Clin Lab Sci. 2017 Aug;47(4):452-456.
Snapshot Oral Fluid Compliance
Richter LHJ, Jacobs CM, Mahfoud F, et al. Development and application of a LC-HRMS/MS method for analyzing antihypertensive drugs in oral fluid for monitoring drug adherence. Anal Chim Acta. 2019;1070:69-79.
OncolyticAssuranceRX
Cardoso E, Mercier T, Wagner AD, et al. Quantification of the next-generation oral anti-tumor drugs dabrafenib, trametinib, vemurafenib, cobimetinib, pazopanib, regorafenib and two metabolites in human plasma by liquid chromatography-tandem mass spectrometry. J Chromatogr B Analyt Technol Biomed Life Sci. 2018;1083:124-136.
Janssen JM, de Vries N, Venekamp N, et al. Development and validation of a liquid chromatography-tandem mass spectrometry assay for nine oral anticancer drugs in human plasma. J Pharm Biomed Anal. 2019;174:561-566.
van Nuland M, Venekamp N, de Vries N, et al. Development and validation of an UPLC-MS/MS method for the therapeutic drug monitoring of oral anti-hormonal drugs in oncology. J Chromatogr B Analyt Technol Biomed Life Sci. 2019;1106-1107:26-34.
Mutation Testing for microsatellite instability (MSI-H) and mismatch repair deficiency (DMMR) 
Bristol-Myers Squibb Company. Opdivo (nivolumab) injection, for intravenous use . Prescribing Information. Princeton, NJ: Bristol-Myers Squibb Company; June 2020.
Bristol-Myers Squibb Company. Yervoy (ipilimumab) injection, for intravenous use. Prescribing Information. Princeton, NJ: Bristol-Myers Squibb Company; May 2020.
EZH2 Testing for tazemetostat (Tazverik) therapy
Epizyme, Inc. Tazverik (tazemetostat) tablets, for oral use. Prescibing Information. Cambridge, MA: Epizyme, Inc.; June 2020.
FGFR Testing for Pemigatinib therapy
Abou-Alfa GK, Sahai V, Hollebecque A, et al. Pemigatinib for previously treated, locally advanced or metastatic cholangiocarcinoma: a multicentre,open-label, phase 2 study. Lancet Oncol. 2020;21(5):671-684.
CQuentia Pharmacogenetic Comprehensive Panel
Zeier Z, Carpenter LL, Kalin NH, et al. Clinical implementation of pharmacogenetic decision support tools for antidepressant drug prescribing. Am J Psychiatry. 2018;175(9):873-886.
CYP2D6 Genotyping for Uses Associated with Opioid Medications
Choi SW, Lam DMH, Wong SSC, et al. Effects of single nucleotide polymorphisms on surgical and postsurgical opioid requirements: A systematic review and meta-analysis. Clin J Pain. 2017;33(12):1117-1130.
Knisely MR, Carpenter JS, Draucker CB, et al. CYP2D6 drug-gene and drug-drug-gene interactions among patients prescribed pharmacogenetically actionable opioids. Appl Nurs Res. 2017;38:107-110.
Kummer O, Hammann F, Moser C, et al. Effect of the inhibition of CYP3A4 or CYP2D6 on the pharmacokinetics and pharmacodynamics of oxycodone. Eur J Clin Pharmacol. 2011;67(1):63-71.
St Sauver JL, Olson JE, Roger VL, et al. CYP2D6 phenotypes are associated with adverse outcomes related to opioid medications. Pharmgenomics Pers Med. 2017;10:217-227.
Vandenbossche J, Van Peer A, Richards H. Single-dose pharmacokinetic study of tramadol extended-release tablets in children and adolescents. Clin Pharmacol Drug Dev. 2016;5(5):343-353.
VanderVaart S, Berger H, Sistonen J, et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: A pilot study. Ther Drug Monit. 2011;33(4):425-432.
FGFR2 Fusion for Treatment with Infigratinib
CCA News. October 10, 2022: Important information about Truseltiq. Press Release. Langhome, PA: CCA News; 2024. Available at: https://www.ccanewsonline.com/web-exclusives/press-releases/october-10-2022-truseltiq. Accessed May 7, 2024.
U.S. Food and Drug Administration (FDA). FDA grants accelerated approval to infigratinib for metastatic cholangiocarcinoma. FDA News. Silver Spring, MD: FDA; May 2021. FDA: Silver Spring, MD. Available at: https://www.fda.gov/drugs/resources-information-approved-drugs/fda-grants-accelerated-approval-infigratinib-metastatic-cholangiocarcinoma.
Pharmacogenetic Studies in Chronic Kidney Disease
Tziastoudi M, Pissas G, Raptis G, et al. A systematic review and meta-analysis of pharmacogenetic studies in patients with chronic kidney disease. Int J Mol Sci. 2021;22(9):4480.
Genotyping of HLA Class 1 for Hyperuricemia Prior to Initiation of Allopurinol
Hershfield MS, Callaghan JT, Tassaneeyakul, et al. Clinical pharmacogenetics implementation consortium guidelines for human leukocyte antigen-B genotype and allopurinol dosing. Clin Pharmacol Ther. 2013;93(2):153-158.
Khanna D, FitzGerald JD, Khanna PP, et al. 2012 American College of Rheumatology guidelines for management of gout part I: Systematic non-pharmacologic and pharmacologic therapeutic approaches to hyperuricemia. Arthritis Care Res (Hoboken). 2012;64(10):1431-1446.
Pichler WJ. Drug hypersensitivity: Classification and clinical features. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed June 2022.
Mass Spectrometry and Tandem Mass Spectrometry for Busulfan Dose Testing
Andersen AM, Bergan S, Gedde-Dahl T, et al. Fast and reliable quantification of busulfan in blood plasma using two-channel liquid chromatography tandem mass spectrometry: Validation of assay performance in the presence of drug formulation excipients. J Pharm Biomed Anal. 2021;203:114216. 
Lee EJ, Park N, Lee SH, et al. A simple and accurate liquid chromatography-tandem mass spectrometry method for therapeutic drug monitoring of busulfan in plasma. Ann Clin Lab Sci. 2019;49(2):212-217. 
Matar KM, Alshemmari SH, Refaat S, Anwar A. UPLC-tandem mass spectrometry for quantification of busulfan in human plasma: Application to therapeutic drug monitoring. Sci Rep. 2020;10(1):8913.
Measurement of Microsatellite Instability (MSI-H) and Mismatch Repair Deficiency (DMMR) for Patients with Unresectable or Metastatic Solid Tumors being Considered for Dostarlimab
Gomez-Raposo C, Salvador MM, Zamora CA, et al. Immune checkpoint inhibitors in endometrial cancer. Crit Rev Oncol Hematol. 2021;161:103306.
Henry NL, Somerfield MR, Dayao Z, et al. Biomarkers for systemic therapy in metastatic breast cancer: ASCO guideline update. J Clin Oncol. 2022;40(27):3205-3221.
Maiorano BA, Maiorano MFP, Cormio G, et al. How immunotherapy modified the therapeutic scenario of endometrial cancer: A systematic review. Front Oncol. 2022 Apr 14;12:844801.
Oaknin A, Gilbert L, Tinker AV, et al. Safety and antitumor activity of dostarlimab in patients with advanced or recurrent DNA mismatch repair deficient/microsatellite instability-high (dMMR/MSI-H) or proficient/stable (MMRp/MSS) endometrial cancer: Interim results from GARNET-a phase I, single-arm study. J Immunother Cancer. 2022;10(1):e003777.
Redondo A, Gallego A, Mendiola M, et al. Dostarlimab for the treatment of advanced endometrial cancer. Expert Rev Clin Pharmacol. 2022;15(1):1-9.
Estrogen Receptor 1 (ESR1) Gene Mutation Testing for Treatment with Elacestrant
Bidard FC, Kaklamani VG, Neven P, et al. Elacestrant (oral selective estrogen receptor degrader) versus standard endocrine therapy for estrogen receptor-positive, human epidermal growth factor receptor 2-negative advanced breast cancer: results from the randomized phase III EMERALD trial. J Clin Oncol. 2022;40(28):3246-3256.
Stemline Therapeutics, Inc. Orserdu (elacestrant) tablets, for oral use. Prescribing Information. New York, NY: Stemline Therapeutics; revised January 2023.
U.S. Food and Drug Administration (FDA). FDA approves elacestrant for ER-positive, HER2-negative, ESR1-mutated advanced or metastatic breast cancer. Approved Drugs. Silver Spring, MD: FDA; January 27, 2023.
Familial Amyotrophic Lateral Sclerosis (SOD1 Mutation)
McCluskey L. Familial amyotrophic lateral sclerosis. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed April 2022.
National Institute of Health (NIH) / National Institute of Neurological Disorders and Stroke. Amyotrophic lateral sclerosis (ALS). NIH website [online serial]. Last reviewed March 8, 2023.
Guardant360 Response
Raja R, Kuziora M, Brohawn PZ, et al. Early reduction in ctDNA predicts survival in patients with lung and bladder cancer treated with durvalumab. Clin Cancer Res. 2018;24(24):6212-6222.
Thompson JC, Carpenter EL, Silva BA, et al. Serial monitoring of circulating tumor DNA by next-generation gene sequencing as a biomarker of response and survival in patients with advanced NSCLC receiving pembrolizumab-based therapy. JCO Precis Oncol. 2021;5:PO.20.00321.
Zhang Q, Luo J, Wu S, et al. Prognostic and predictive impact of circulating tumor DNA in patients with advanced cancers treated with immune checkpoint blockade. Cancer Discov. 2020;10(12):1842-1853.
Signatera Testing
Bratman SV, Yang SYC, Iafolla MAJ, et al. Personalized circulating tumor DNA analysis as a predictive biomarker in solid tumor patients treated with pembrolizumab. Nat Cancer. 2020;1:873-881.
Coombes RC, Page K, Salari R, et al. Personalized detection of circulating tumor DNA antedates breast cancer metastatic recurrence. Clin Cancer Res. 2019;25(14):4255-4263.
Hufnagl C, Leisch M, Weiss L, et al. Evaluation of circulating cell-free DNA as a molecular monitoring tool in patients with metastatic cancer. Oncol Lett. 2020;19(2):1551-1558.
Kotani D, Oki E, Nakamura Y, et al. Molecular residual disease and efficacy of adjuvant chemotherapy in patients with colorectal cancer. Nat Med. 2023;29(1):127-134.
Loupakis F, Sharma S, Derouazi M, et al. Detection of molecular residual disease using personalized circulating tumor DNA assay in patients with colorectal cancer undergoing resection of metastases. JCO Precis Oncol. 2021;5:PO.21.00101.
Magbanua MJM, Swigart LB, Wu H-T, et al. Circulating tumor DNA in neoadjuvant-treated breast cancer reflects response and survival. Ann Oncol. 2021;32(2):229-239.
Moss J, Zick A, Grinshpun A, et al. Circulating breast-derived DNA allows universal detection and monitoring of localized breast cancer. Ann Oncol. 2020;31(3):395-403.
National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2020. Fort Washington, PA: NCCN; 2020.
National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2020. Fort Washington, PA: NCCN; 2020.
National Institute for Health and Care Excellence (NICE). Signatera for detecting molecular residual disease from solid tumour cancers. Medtech Innovation Briefing (MIB) 307. London, UK: NICE: October 4, 2022.
Oellerich M, Schutz E, Beck J, et al. Using circulating cell-free DNA to monitor personalized cancer therapy. Crit Rev Clin Lab Sci. 2017;54(3):205-218.
Oellerich M, Schutz E, Beck J, Walson PD. Circulating cell-free DNA-diagnostic and prognostic applications in personalized cancer therapy. Ther Drug Monit. 2019;41(2):115-120.
Powles T, Assaf ZJ, Davarpanah N, et al. ctDNA guiding adjuvant immunotherapy in urothelial carcinoma. Nature. 2021;595(7867):432-437.
Reece M, Saluja H, Hollington P, et al. The use of circulating tumor DNA to monitor and predict response to treatment in colorectal cancer. Front Genet. 2019;10:1118.
Reinert T, Henriksen TV, Christensen E, et al. Analysis of plasma cell-free DNA by ultradeep sequencing in patients with stages I to III colorectal cancer. JAMA Oncol. 2019;5(8):1124-1131.
Tie J, Cohen JD, Wang Y, et al. Circulating tumor DNA analyses as markers of recurrence risk and benefit of adjuvant therapy for stage III colon cancer. JAMA Oncol. 2019;5(12):1710-1717.
Volckmar AL, Sultmann H, Riediger A, et al. A field guide for cancer diagnostics using cell-free DNA: From principles to practice and clinical applications. Genes Chromosomes Cancer. 2018;57(3):123-139.
Wang Y, Li L, Cohen JD, et al. Prognostic potential of circulating tumor DNA measurement in postoperative surveillance of nonmetastatic colorectal cancer. JAMA Oncol. 2019 5(8):1118-1123.
Wang R, Zhao A, Cao N, et al. The value of circulation tumor DNA in predicting postoperative recurrence of colorectal cancer: A meta-analysis. Int J Colorectal Dis. 2020;35(8):1463-1475.
Woldu SL, Lotan Y. Re: ctDNA guiding adjuvant immunotherapy in urothelial carcinoma. Euro Urol. 2021;80(4):517-518.
PIK3CA Mutation Testing
Genentech USA, Inc. Itovebi (inavolisib) tablets for oral use. Prescribing Information. South San Francisco, CA: Genentech; revised October 2024.
Novartis Pharmaceuticals Corporation. Piqray (alpelisib) tablets, for oral use. Prescribing Information. East Hanover, NJ: Novartis Pharmaceuticals; Revised May 2019.
Turner NC, Im SA, Saura C, et al. Inavolisib-based therapy in PIK3CA-mutated advanced breast cancer. N Engl J Med. 2024;391(17):1584-1596. 
PIK3CA/AKT1 Activating Mutations or PTEN Loss of Function Alterations Testing
AstraZeneca Pharmaceuticals LP. Truqap (capivasertib) tablets, for oral use. Prescribing Information. Wilmington, DE: AstraZeneca Pharmaceuticals; Revised November 2023.
Genomind Pharmacogenetics Report
Genomind, Inc. Genes included on Genomind’s PGx Test. Genomind PGx Genes. King of Prussia, PA: Genomind; 2024. Available at: https://genomind.com/solutions/pharmacogenetic-testing/genes/. Accessed May 4, 2024.
National Library of Medicine (NLM). Genomind Pharmacogenetic Report. GTR: Genetic Testing Registry. Bethesda, MD: NLM; Updated November 21, 2023a. Available at:  https://www.ncbi.nlm.nih.gov/gtr/tests/classic/523653/. Accessed May 4, 2024.
EffectiveRX Comprehensive Panel
Genetworx, LLC. EffectiveRX Comprehensive Panel. Glen Allen, VA: Genetworx; 2024. Available at: https://genetworx.com/services/effectiverx-comprehensive-tests/. Accessed May 5, 2024.
Tempus nP
National Library of Medicine (NLM). Tempus nP. GTR: Genetic Testing Registry. Bethesda, MD: NLM; Updated August 31, 2023b. Available at: https://www.ncbi.nlm.nih.gov/gtr/tests/classic/595972/. May 7, 2024.
Tempus. Providers Neurology and Psychiatry Overview. Chicago, IL: Tempus; 2024. Available at: https://www.tempus.com/neurology-psychiatry/. Accessed May 7, 2024.
Catechol-O-Methyltransferase (COMT) Genotype, Varies Test
National Library of Medicine (NLM). Catechol-O-Methyltransferase (COMT) Genotype. GTR: Genetic Registry. Bethesda, MD: NLM; Updated April 24, 2024. Available at: https://www.ncbi.nlm.nih.gov/gtr/tests/classic/593359/indication/. Accessed May 7, 2024.
Aura Genetics Pharmacogenomics (PGx) Test
Aura Genetics. PGx. Testing Portfolio. Youngstown, OH: Aura Genetics; 2022. Available at: https://auragenetics.com/testing-portfolio/?goto=pgx. Accessed May 7, 2024.
CRCdx RAS Mutation Detection Kit
U.S. Food & Drug Administration (FDA). CRCdx® RAS Mutation Detection Kit – P220005. Silver Spring, MD: FDA; May 22, 2024a.
U.S. Food & Drug Administration (FDA). Premarket Approval (PMA). CRCDx RAS Mutation Detection Assay Kit. Sliver Spring, MD: FDA; August 5, 2024b.
xT CDx
U.S. Food & Drug Administration (FDA). Premarket Approval (PMA). xT CDx. Silver Spring, MD: FDA; August 5, 2024c.
U.S. Food & Drug Administration (FDA). xT CDx – P210011. Silver Spring, MD: FDA; May 23, 2023.
NTRK
Bayer HealthCare Pharmaceuticals Inc. Vitrakvi (larotrectinib) capsules, for oral use, oral solution. Prescribing Information. Whippany, NJ: Bayer HealthCare Pharmaceuticals; revised November 2023.
RET
Lilly USA, LLC. Retevmo (selpercatinib) capsules, for oral use, tablets for oral use. Prescribing Information. Indianapolis, IN: Lilly USA; revised September 2024.
oncoReveal CDx
Pillar Biosciences, Inc. Pillar Biosciences oncoReveal CDx pan-cancer solid tumor IVD now FDA approved for general tumor profiling on the Illumina MiSeq Dx system. Press Release. Natick, MA: Pillar Biosciences; April 23, 2024.
U.S. Food and Drug Administration (FDA). oncoReveal CDx. Summary of Safety and Effectiveness Data. Premarket Approval Application (PMA) No. P200011/S001. Silver Spring, MD: FDA; April 18, 2024.
Claudin 18 (CLDN18) Testing
Astellas Pharma US, Inc. Vyloy (zolbetuximab-clzb) for injection, for intravenous use. Prescribing Information. Northbrook, IL: Astella Pharma US; revised October 2024.
National Comprehensive Cancer Network (NCCN). Esophageal and esophagogastric junction cancers. NCCN Clinical Practice Guidelines in Oncology, Version 5.2024. Plymouth Meeting, PA: December 20, 2024.
National Comprehensive Cancer Network (NCCN). Gastric cancer. NCCN Clinical Practice Guidelines in Oncology, Version 5.2024. Plymouth Meeting, PA: December 20, 2024.
Shah MA, Shitara K, Ajani JA, et al. Zolbetuximab plus CAPOX in CLDN18.2-positive gastric or gastroesophageal junction adenocarcinoma: The randomized, phase 3 GLOW trial. Nat Med. 2023;29(8):2133-2141.
Shitara K, Lordick F, Bang YJ, et al. Zolbetuximab plus mFOLFOX6 in patients with CLDN18.2-positive, HER2-negative, untreated, locally advanced unresectable or metastatic gastric or gastro-oesophageal junction adenocarcinoma (SPOTLIGHT): A multicentre, randomised, double-blind, phase 3 trial. Lancet. 2023;401(10389):1655-1668.
Dihydropyrimidine Dehydrogenase (DPYD) Gene Variant Testing
National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2025. Plymouth Meeting, PA: NCCN; April 2025.
National Comprehensive Cancer Network (NCCN). Rectal cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2025. Plymouth Meeting, PA: NCCN; April 2025.
U.S. Food and Drug Administration (FDA). Safety announcement: FDA highlights importance of DPD deficiency discussions with patients prior to capecitabine or 5FU treatment. Drugs. Sliver Spring, MD: FDA; January 24, 2025.
van Kuilenburg ABP, Meijer J, Meinsma R, et al. Dihydropyrimidine dehydrogenase deficiency: Homozygosity for an extremely rare variant in DPYD due to uniparental isodisomy of chromosome 1. JIMD Rep. 2019;45:65-69.
Policy History

Last Review  12/03/2025

Effective: 09/16/2005

Next Review: 08/13/2026

Review History

Definitions

Additional Information

Clinical Policy Bulletin Notes

opens a dialog
 
opens a dialog
 
opens a dialog
 
opens a dialog
Copyright Aetna Inc. All rights reserved. Clinical Policy Bulletins are developed by Aetna to assist in administering plan benefits and constitute neither offers of coverage nor medical advice. This Clinical Policy Bulletin contains only a partial, general description of plan or program benefits and does not constitute a contract. Aetna does not provide health care services and, therefore, cannot guarantee any results or outcomes. Participating providers are independent contractors in private practice and are neither employees nor agents of Aetna or its affiliates. Treating providers are solely responsible for medical advice and treatment of members. This Clinical Policy Bulletin may be updated and therefore is subject to change.
 
Glossary
Aetna Mobile App
Careers
Accessibility Services
Terms of Use
Investor Info
FAQs
Program Provisions
Interest-Based Ads Policy
Legal Notices
Plan Disclosures
Nondiscrimination Notice
Site Map
Privacy Center
State Directory
Copyright © 2001-2026 Aetna Inc.
Language services can be provided by calling the number on your member ID card. For additional language assistance: Español 中文 Tiếng Việt 한국어 Tagalog Pусский العربية Kreyòl Français Polski Português Italiano Deutsch 日本語 فارسی Other Languages… 