Liquid Biopsy Log in to use authoring capabilities Open site menu Sites Policy List Toggle Menu Toggle Site Search 0" tag="nav" is-nav id="constituent-nav-collapse" role="navigation"> 0, show: navItem.expanded }" class="nav-item text-nowrap"> {{ navItem.title }} 0 && !(navigationPortalSiteName === 'admin' || isWcmEditMode)"> 0 && navItem.children.filter(function(item) { return item && !item.isHidden && !item.isPrivate }).length > 0" :ref="navItem.expanded ? 'openConstituentNavDropdown' : null" :key="navItem.url" :id="'constituent-nav-' + navItem.urlName + '-dropdown'" :aria-labelledby="'constituent-nav-' + navItem.urlName" :style="{ display: navItem.expanded ? 'block' : 'none' }" :class="{ 'animation-nav-slide-from-left': navItem.animateSubNav }" class="dropdown-menu" @mouseenter="onMouseEnterNavDropdown(navItem, $event)" @mouseleave="onMouseLeaveNavDropdown(navItem, $event)" @keydown="onKeyDownNavDropdown(navItem, $event)" > {{ navItem.title }} Overview Back {{ navItemChild.title }} Quick Links {{ quickLink.title }} Policy List Edit Policy {{ navigationConstituentPage.title || { 'medicalpolicyhb': 'Healthy Blue Medical Policies', 'medicalpolicyih': 'InStil Health Medical Policies' }[navigationSitePage.urlName] || 'Medical Policies' }} Back to list 2 && navigationPath[2].filter(function(item) { return item.selected }).length > 0" class="site-breadcrumb" data-swiftype-index="false" :style="{ 'pointer-events': showDropOverlay ? 'none' : null }"> Home {{ navItem.title }} {{ navItem.title }} Show Related Pages Home {{ navItem.title }} {{ navItem.title }} Hide Related Pages 2 && navigationPath[2].filter(function(item) { return item.selected }).length > 0" class="site-content-header" :style="{ 'pointer-events': showDropOverlay ? 'none' : null }"> Liquid Biopsy Liquid Biopsy - CAM 273 Description The National Cancer Institute (NCI) defines “liquid biopsy” as a test done on a sample of blood, urine, or other bodily fluid to look for cancer cells from a tumor or small pieces of DNA, RNA, or other molecules released by tumor cells into a person’s body fluids. Liquid biopsies are non-invasive blood tests since circulating tumor cells (CTCs) and cell-free tumor DNA (cfDNA) fragments are shed into the bloodstream from existing tumors and can be detected in blood.1,2 The presence of CTCs can be indicative of metastatic disease.4 For guidance concerning Tumor Mutational Burden Testing (TMB) and/or Microsatellite instability (MSI) analysis please refer to AHS-M2178-Microsatellite Instability and Tumor Mutational Burden Testing policy. Policy Application of coverage criteria is dependent upon an individual’s benefit coverage at the time of the request For individuals diagnosed with non-small cell lung cancer (NSCLC), cell-free DNA/circulating tumor DNA (cfDNA/ctDNA) testing is considered MEDICALLY NECESSARY in any of the following situations: When tissue-based testing is infeasible (i.e., quantity not sufficient for tissue-based test or invasive biopsy is medically contraindicated). In the initial diagnostic setting when there is insufficient tissue to allow testing for broad molecular analysis following pathological confirmation of NSCLC (if an oncogenic driver is not identified, follow-up tissue-based analysis should be considered). In the initial diagnostic setting when tissue-based molecular analysis does not completely assess all recommended biomarkers due to tissue quantity or testing methodologies available. Recommended biomarkers include: ALK rearrangements. BRAF mutations. EGFR mutations. ERBB2 ( HER2 ) mutations. KRAS mutations. MET ex14 skipping mutations. NTRK1/2/3 fusions. RET rearrangements. ROS1 rearrangements. PD-L1 expression levels. To aid in biomarker evaluation for treatment selection in the initial diagnostic setting (when the feasibility of timely tissue-based testing is uncertain). For individuals diagnosed with HR-positive/HER2-negative breast cancer and who are being considered for targeted therapy, cfDNA/ctDNA testing for PIK3CA is considered MEDICALLY NECESSARY . For individuals diagnosed with castration-resistant prostate cancer, cfDNA/ctDNA testing of the following biomarkers is considered MEDICALLY NECESSARY : Androgen receptor variant 7 (AR-V2) to guide therapy selection in the post-abiraterone/enzalutamide metastatic CRPC setting. Somatic analysis of BRCA1 and BRCA2 to select patients for rucaparib treatment. For individuals meeting the above criteria, cfDNA/ctDNA testing (annually) is considered MEDICALLY NECESSARY . The following does not meet coverage criteria due to a lack of available published scientific literature confirming that the test(s) is/are required and beneficial for the diagnosis and treatment of an individual’s illness. For all other situations not described above, liquid biopsy testing for screening (e.g., GRAIL Galleri), detecting, and/or monitoring any other malignancy or tumor is considered NOT MEDICALLY NECESSARY. For all situations not addressed above, analysis of PD-L1 by liquid biopsy is considered NOT MEDICALLY NECESSARY . For the screening, detection, and/or diagnosis of cancer, urinary liquid biopsy (i.e., use of cell-free DNA [“UcfDNA”] or circulating tumor DNA obtained in a urine sample) (e.g., SelectMDX) is considered NOT MEDICALLY NECESSARY . Liquid biopsy testing on CSF samples is considered NOT MEDICALLY NECESSARY . Cell capture-enumeration assays of CTCs (e.g., CELLSEARCH ® CTC) is considered NOT MEDICALLY NECESSARY . Table of Terminology Term Definition AACC American Association for Clinical Chemistry ALK Anaplastic lymphoma receptor tyrosine kinase AMP The Association for Molecular Pathology AR Androgen receptor AR-V7 Androgen receptor splice variant 7 ASCO American Society of Clinical Oncology BRAF B-Raf proto-oncogene BRCA1 Breast cancer type 1 susceptibility gene BRCA1/2 Breast cancer type 1/2 susceptibility gene BRCA2 Breast cancer type 2 susceptibility gene CAM Cell adhesion molecule CAP College of American Pathologists CF Cell-free cfDNA Cell-free tumour deoxyribonucleic acid CGP Comprehensive somatic genomic profiling CLIA ’88 Clinical Laboratory Improvement Amendments of 1988 c-MET Cellular mesenchymal epithelial transition CNS Central nervous system CRC Colorectal cancer CRPC Castration-resistant prostate cancer CSCO Chinese Society of Clinical Oncology CSF Cerebrospinal fluid CSF-CTC Circulating Tumor Cells in cerebrospinal fluid CTCs Circulating Tumor Cells ctDNA Circulating Tumor deoxyribonucleic acid CTLA-3 Cytotoxic T-lymphocyte-associated protein 3 DLX1 Distal-less 1 DNA Deoxyribonucleic acid DRE Digital rectal examination EAU European Association of Urology EGFR Epidermal growth factor receptor EpCAM Epithelial cell adhesion molecule ER+ MBCs Estrogen receptor-positive metastatic breast cancer ERCC1 Excision repair cross-complementation group 1 ER-CTCs Estrogen receptor-negative circulating tumor cells ESMO European Society for Medical Oncology ESTRO European Society of Urogenital Radiology EV Extracellular vesicle ExoRNA Exosome ribonucleic acid FDA Food and Drug Administration FFPE Formalin-fixed paraffin-embedded GC Gastric cancer gDNA Genomic deoxyribonucleic acid HCC Hierarchical condition category HDL High-density lipoprotein hENT1 Human equilibrative nucleoside transporter 1 HER2 Human epidermal growth factor receptor 2 HOXC6 Homeobox C6 IASLC International Association for the Study of Lung Cancer InDels Insertions/Deletions KLK3 Kallikreins 3 KRAS Kirsten rat sarcoma viral oncogene homolog LAG-3 Lymphocyte-activation gene 3 LM Leptomeningeal metastasis MDX Molecular diagnostics MET MET Proto-Oncogene MRI Myotubularin 1 mRNA Messenger ribonucleic acid MSI Microsatellite instability MSI-H Microsatellite instability-high NACB National Academy of Clinical Biochemistry NCCN National Comprehensive Cancer Network NCI National Cancer Institute NGS Next-generation sequencing NIH National Institute of Health NK Natural killer NRAS Neuroblastoma rat sarcoma NSCLC Non-small cell lung cancer PCR Polymerase chain reaction PD-L1 Programmed death-ligand 1 PFS Progression-free survival PIK3CA Phosphatidylinositol 3-Kinase PSA Prostate-specific antigen RET Rearranged during transfection RGQ Rapid gas quenching RNA Ribonucleic acid RNases Ribonucleases RRM1 Ribonucleotide reductase, M1 subunit RT-PCR Reverse transcriptase polymerase chain reaction SIOG International Society of Geriatric Oncology SNVs Single nucleotide variants tDNA Tissue deoxyribonucleic acid TEXs Tumor-derived exosomes TIM-3 T cell immunoglobulin and mucin-domain containing-3 TKIs Tyrosine kinase inhibitors TMB Tumor mutational burden TOP1 DNA topoisomerase 1 TOP2A DNA topoisomerase 2 alpha TOP2B DNA topoisomerase 2 beta TP Tumour protein TUBB3 Tubulin beta 3 class III UcfDNA Urinary cell-free deoxyribonucleic acid utDNA Urine-derived tumor deoxyribonucleic acid XRCC1 X-ray repair cross-complementing 1 Rationale The science of non-invasive disease monitoring has advanced greatly since circulating cell-free DNA (cfDNA) was first reported in body fluids by Mandel and Metais. Since then, the evolution of sensitive cfDNA detection technologies has enabled the development of liquid biopsies with many clinical applications. For example, in oncology, the use of liquid biopsy allows for patient stratification, screening, monitoring treatment response and detection of minimal residual disease after surgery or recurrence. Liquid biopsies have grown in importance because the genetic profile of tumors can affect how well patients respond to a certain treatment. However, this characterization is currently achieved through a biopsy despite the inherent problems in procurement of tissue samples and the limitations of tumor analyses. For example, the invasive nature of a biopsy poses a risk to patients and can have a significant cost.5 Tumor sampling from some cancer types also remains difficult resulting in inadequate amount of tissue for genetic testing.5 In the case of advanced or metastatic non-small cell lung cancers (NSCLC), as many as 69% of cases do not have accessible tissue.6 Even when tissue can be collected, preservation methods such as formalin fixation can cause false positive results for genetic tests.7 Finally, due to tumor heterogeneity, biopsies often suffer from sample bias.8 Liquid biopsies are becoming more popular as they provide an opportunity to genotype in a less invasive and expensive manner. However, the low sensitivity (between 60-80%) and higher number of false negative cases compared to traditional tissue biopsy are limitations associated with liquid biopsies.9 Approaches to Liquid Biopsy Analysis Circulating tumor cells (CTCs) According to Brock, et al. (2015), CTCs are cells shed into the vasculature from a primary tumor and may constitute seeds for subsequent growth of additional tumors (metastasis) in distant organs. CTCs generally confer the advantage of containing RNA, DNA, and protein from tumor cells including both nuclear and cytoplasmic biomarkers, which is not attainable from ctDNA or exosomes.10 They have been detected in various metastatic carcinomas11 but are extremely rare in healthy subjects and patients with nonmalignant diseases.5 Clinical evidence indicates that patients with metastatic lesions are more likely to have CTCs amenable to isolation but their frequency is low, often ~1-10 CTCs per mL of whole blood.12 As one mL of blood contains ~7×106 white blood cells and ~5×109 red blood cells, technologies capable of reproducibly isolating a single CTC from the background of all other blood components are fundamental. While such levels of sensitivity are challenging, there are several novel developments in this area, including positive selection, negative selection, physical properties or even enrichment-free assays to efficiently isolate these rare CTCs.4 However, Bettegowda, et al. (2014)) stated that an advantage of ctDNA is that it can be analyzed from bio-banked bio-fluids, such as frozen plasma.13 Typically, CTCs are defined as cells with an intact viable nucleus, cytokeratin positive, epithelial cell adhesion molecule (EpCAM) positive and with the absence of CD45.5 Unfortunately, EpCAM and other markers are not always expressed on CTCs.14 In addition, non-tumor epithelial cells are known to circulate in the blood of patients with prostatitis or patients undergoing surgery.5,15 The heterogeneity of CTCs is a major challenge from a technical standpoint. This has led to alternative strategies of CTC enrichment such as the CTC-iChip which does not rely on tumor antigen expression.5,16 Sequencing the genetic material from CTCs has demonstrated that the majority are not cancer cells, even when the isolated cell(s) fit the phenotypic criteria of being a CTC. One study by Marchetti, et al. (2014) developed a protocol to recover the CTC enriched samples from the cartridge of the Veridex platform and found that from 37 NSCLC patients, the EGFR mutation allele abundance ranged between 0.02% and 24.79% with a mean of 6.34%. Brock, et al. (2015) concluded that the number of CTCs found in the blood is therefore highly dependent on how the platform defines a cell as a CTC.5,17 The CellSearch CTC test, a Food and Drug Administration (FDA) approved actionable CTC test, requires that samples are processed within 96 hours of collection after being drawn into the Cellsave preservative tube. This test does not analyze the molecular genetics of the tumor; rather Cellsave is a platform for CTC numeration. A positive test (more than five detected CTCs for metastatic breast and prostate cancer and more than three CTCs for metastatic colorectal cancer per 7.5 mL of blood) is associated with decreased progression-free survival and decreased overall survival in these patients.18 Overall, although CTCs have produced some promising results in evaluating prognosis of patients with varying cancers, further studies are needed to assess the clinical utility of these biomarkers.19-22 Cell-free DNA (cfDNA) There is currently an intensive research effort to understand the utility of cfDNA in various clinical fields, such as cancer research, non-invasive prenatal testing and transplant rejection diagnostics.5 In a systematic review and meta-analysis of 20 studies and 2012 cases covering assessment of EGFR mutational status in NSCLC, Luo, et al. (2014) found a sensitivity of 0.674, a specificity of 0.935, and area under the curve of 0.93. The authors concluded that detection of EGFR mutation by cfDNA is of adequate diagnostic accuracy and cfDNA analysis could be a promising screening test for NSCLC.23 In a study, Jiang, et al. (2015)observed that most cfDNA in plasma is reportedly fragmented, around 150-180 bp in length with a higher prevalence of tumor associated mutations in the shorter fragments. Per authors, when analyzing the mutation abundance with massively parallel sequencing, a significant correlation was found between mutations and fragments less than 150 bp. Notably, the size of the majority of cfDNA fragments overlaps well with the size of histone DNA.24 A direct comparison of mutation detection on cfDNA vs. CTCs showed a higher abundance of the mutation on the cfDNA from the same patient; moreover, recent large studies comparing the effectiveness of cfDNA analysis to tissue biopsy in NSCLC showed the clinical value of the liquid biopsy approach.25 This positive result led to an approval to use cfDNA analysis for EGFR mutation analysis for IRESSA in Europe (in patients where a tumor sample was not evaluable), making it the first EGFR tyrosine kinase inhibitor for which cfDNA testing is included in the label. Although promising, challenges remain when using cfDNA to characterize the mutation status of a tumor. In addition to the low copy number of mutant alleles, the median half-life of cfDNA in circulation ranges from 15 minutes to a few hours.5 Brock, et al. (2015), in their review, observed that the total concentration of cfDNA in the blood of cancer patients varies considerably with tumor specific mutations ranging from undetectable (less than one copy per five mL of plasma) to patients with over a hundred thousand copies of the mutation per mL of plasma. The authors note that “the challenge of how to maximize the yield of the cfDNA and pair this with a platform sensitive enough to detect rare variants in the background of wild-type DNA remains. Optimally, the ability to detect mutations in plasma should not be limited to a subpopulation of patients with very high mutant copy numbers in circulation.”5 This has been proven to be challenging in early-stage cancers.10 While many analytical platforms report the mutation load with an allelic frequency compared to the wild-type DNA platforms relying solely on the allelic frequency without recording the number of mutations have limitations. This is because the allelic frequency of a gene is affected by the amount of wild-type DNA not related to the tumor. Therefore, it is important to consider the processes that affect the amount of wild-type DNA in circulation.5 For example, exercise increases cfDNA levels almost 10-fold.26 Other pre-analytical variables such as blood collection, the cellular process leading to its release, and extraction protocols affect the amount and size range of cfDNA fragments in a sample.27 Exosomes In the last few years, the exosome field has grown exponentially impacting various areas of research. Studies demonstrating that exosomes are actively released vesicles (carrying RNA, DNA, and protein) and can function as intercellular messengers. Yanez-Mo, et al. (2015) highlights these developments in a review outlining the biological properties of exosomes and other extracellular vesicles (EVs). However, Gould and Raposo (2013) observed that the exosome field still lags behind as the standardization of EV types are not yet firmly established. The majority of exosomes range in size from 30-200 nanometers (nm) in diameter and are isolated from all bio-fluids, including serum, plasma, saliva, urine and cerebrospinal fluid.5 Due to the size of an exosome, on average just over 100 nanometers, the entire transcriptome cannot be packaged inside every vesicle. By way of comparison, retrovirus particles with a similar size can package only around 10 kb, so it is likely that a single vesicle of that size carries only a limited number of transcripts. However, exosomes are extremely abundant (1011 per mL of plasma) and when isolating the vesicle fraction, most of the transcriptome can be detected.5 Per Huang, et al. (2013)and Kahlert, et al. (2014), exosomal RNA can be used for mutation detection as well as global profiling of most types of RNA, and the profile alone (without mutation characterization) can be utilized for diagnostics.5 In the study ‘Immune modulation of T-cell and NK (natural killer) cell activities by TEXs (tumor-derived exosomes)’, Whiteside (2013) observed that exosome investigations have focused on the important physiologic and pathophysiologic functions of these vesicles in micro-metastasis, angiogenesis and immune modulation and as a means for detection of tumor specific mutations in bio-fluids.32 Consequently, in 2012, interest in this new field increased when the National Institute of Health (NIH) dedicated the large strategic Common Fund to study these new entities of extracellular RNA. The goal of this effort is to better understand how exosomes can be utilized for biomarkers and therapeutics as well as understanding this new mechanism of intercellular communication.33 Mutation detection and RNA profiling Analysis of nucleic acids present in bodily fluids can provide a better understanding of the disease, as summarized in Table below. Analysis capability Examples CTCs cfDNA Exosomes Mutations Point mutations, InDels, amplifications, deletions, translocations Yes Yes Yes Epigenetic modifications Methylation patterns Yes Yes Yes RNA transcription profiles Levels/activity of mRNA, microRNA, long non-coding RNA, RNA splice variants Yes No Yes Phenotypic studies of cells from the tumor Cell morphology, protein localization, in vivo studies Yes No No Inflammatory response, stromal and other systemic changes Inflammatory RNA and protein markers No No Yes Analysis of RNA as well as DNA and protein profiles from tumor cells Separate or in combination Yes No Yes Can utilize bio-banked samples Frozen plasma, urine and other bio-fluids No Yes Yes CTCs, circulating tumor cells; cfDNA, cell-free DNA; InDels, insertions/deletions.5 Ribonucleic acid (RNA) profiling from bio-fluids is also difficult. However, since exosomes contain RNA, it was possible to separate the fragile RNA from the large amounts of RNases and PCR inhibitors. As cell-free RNA in blood is immediately degraded, RNAs in serum and plasma were either protected inside vesicles, in protein complexes or associated with HDL particles.5 The levels of these microRNAs are tightly regulated in normal cells, and dysregulation has been implicated in several human diseases, e.g., cardiovascular34 and neurological, and is strongly linked to cancer development and progression. However, microRNAs represent only a minor fraction of the transcriptome. By contrast, the nucleic acids in exosomes can be isolated and the entire transcriptome examined.5 The most significant hurdle for all forms of liquid biopsy remains the relative rarity of nucleic acid derived from a tumor against the background of normal material found in most patient samples. In fact, the majority of cell, cell-free nucleic acids, microRNAs and exosomes in a liquid biopsy will have originated from normal cells with numbers fluctuating as a consequence of biological variations.5 Furthermore, although liquid biopsy was first introduced with serum, other liquid media, such as urine and cerebrospinal fluid (CSF), have been used to evaluate other conditions. Cell-free DNA is not necessarily confined to blood, and other media have been proposed. Urine Urine’s primary advantage over blood is that it is non-invasive, allowing for more convenient testing. Urinary cell-free DNA (UcfDNA) has been proposed as a biomarker for the detection and diagnosis of certain cancers, particularly bladder and prostate cancer.35 An example of this is SelectMDX. SelectMDX evaluates two mRNA cancer-related biomarkers (HOXC6 and DLX1 with KLK3 as a reference gene) to assist a clinician in deciding to continue routine screening or to order a prostate biopsy. This test is considered a “non-invasive urine test” (a liquid biopsy) and reports a binary result of “increased risk” or “very low-risk”.36 Van Neste, et al. (2016) evaluated this test at a 0.90 area under curve in a validation cohort. The authors concluded that the mRNA signature was one of the most significant components of the validation results.37 Shore, et al. (2019) assessed the effect of SelectMDX results on clinical decision-making and found that out of 253 patients SelectMDX evaluated as “negative,” only 12% underwent a biopsy.38 Xu, et al. (2021) assessed the diagnostic value of urinary exosomes for urological tumors. The authors performed a systematic review and meta-analysis of 16 studies with a total of 3224 patients. Diagnostic value was calculated based on the number of true positives, false positives, true negatives, and false negatives. The sensitivity of using urinary exosomes for the diagnosis of urological tumors was 83% and the specificity was 88%. Sensitivity and specificity results were similar regardless of urinary exosome content type and tumor type. The authors conclude that “urinary exosomes may serve as novel non-invasive biomarkers for urological cancer detection.”39 Cerebrospinal Fluid (CSF) Cerebrospinal Fluid is a colorless, clear liquid produced by the choroid plexus. CSF acts to control flow of molecules to the central nervous system (CNS). Due to the tight control of the CSF, it may play a significant role in assessing several conditions. CSF is traditionally used to evaluate conditions such as meningitis, but it has also been used to assess central nervous system cancers, such as leptomeningeal metastases.40,41 In addition to widely-known measures of pathology in CSF (opening pressure, total protein, glucose, cell count with differential), circulating tumor cells in CSF have also been proposed as markers for epithelial tumors.41 Lin, et al. (2017) evaluated the diagnostic accuracy of circulating tumor cells in CSF (CSF-CTC) in patients with leptomeningeal metastasis (LM). There were 30 of 95 total patients diagnosed with LM based on a combination of CSF cytology and MRI. CSF-CTCs were detected in 43 patients (median 19.3 CSF-CTC/mL). Based on receiver operating curve analysis, the optimal cutoff was found to be one CSF-CTC/mL, identifying patients at a rate of 93% sensitivity, 95% specificity, positive predictive value 90%, and negative predictive value 97%.42 Diaz, et al. (2022) studied the clinical utility of CSF-CTC by evaluating how CSF-CTC quantification was able to predict the outcome of LM. The authors performed a single institution retrospective study of 101 LM patients with solid tumors. The CSF-CTC count significantly predicted survival continuously (p=0.0027). The authors conclude that “CSF-CTCs quantification predicts survival in newly diagnosed LM, and outperforms neuroimaging” and suggest CSF-CTC can be used for LM prognosis and to assess disease burden.43 Mathios and Phallen (2022) published a review paper noting “significant strides” towards understanding the molecular mechanisms of brain cancer. Research advances in the field include a focus on the “tumor microenvironment” and identifying molecular biomarkers with liquid-based analyses (such as CSF in liquid biopsy). While it is a rapidly advancing area of research, clinical utility is currently limited, that is, there are currently “no approved non-invasive tests that are clinically useful” for gliomas. The authors point to Cristiano, et al. (2019) as an example of a study that analyzed genome-wide cfDNA fragment features (in a variety of cancers); the authors were able to distinguish patients with cancer from non-cancer patients (as well as isolate the tissue of origin). In another glioma-specific study, Mouliere, et al. (2018) detected five of 13 patients’ brain tumors (38%) using a cfDNA fragmentation-based approach to analyze cfDNA fragments and copy number alterations in CSF. In conclusion, the authors note that, despite recent excitement over promising studies, liquid biopsy approaches to brain cancer are still “in their infancy.”44 Proprietary Testing The FDA approval of use of Roche Cobas EGFR Mutation Test in plasma was based on evaluation of plasma samples from the ENSURE study,47 a multicenter, open-label, randomized, Phase III study of stage IIIB/IV NSCLC patients. A total of 98.6% of the patients enrolled (214/217) had a plasma sample available for testing. The agreement between the Cobas EGFR Mutation Test in plasma and tissue was evaluated for detection of EGFR mutations. In 76.7% of tissue-positive specimens, plasma was also positive for an EGFR mutation. Plasma was negative for EGFR mutation in 98.2% (95.4%, 99.3%) of tissue-negative cases. The patients whose plasma results were positive for exon 19 deletion and/or an L858R mutations treated with erlotinib had improved progression-free survival (PFS) compared to those treated with chemotherapy.48 Another commercially available, FDA-approved test is Guardant360 by Guardant Health Inc. Guardant360 is a gene panel that sequences 74 genes (including 18 amplifications and six fusions) associated with NSCLC and reports the percentage of cfDNA.49 The manufacturer purports that this genetic test will allow providers to make better treatment decisions based on the mutations present in the patient.50 The gene panel was analytically validated, with 99.8% accuracy on 1000 consecutive samples.51 FoundationOne has also created a proprietary FDA-approved test that examines cell-free DNA. FoundationOne’s liquid CDx test evaluates 324 genes using circulating cell-free DNA and is FDA-approved to report short variants in 311 genes.52,53 A prior version of this test (covering 62 genes) was evaluated based on 2666 reference samples. The assay reached >99% sensitivity of short variants of allele frequencies of >0.5%, >95% sensitivity of allele frequencies 0.25%-0.5%, and >70% sensitivity of allele frequencies 0.125%-0.25%. Out of 62 healthy volunteers, no false positives were detected.54 Biodesix is another laboratory that offers a liquid biopsy panel. Biodesix offers two tests; one called GeneStrat, tests EGFR, ALK, ROS1, RET, BRAF, and KRAS.55 Sensitivities of 78%-100% for EGFR, ALK, and KRAS with the GeneStrat test were shown in multiple validation studies.56 GeneStrat also detected over 88% of RET or ROS1-positive patients.57 Biodesix also offers GeneStrat NGS, a broad 52 gene panel also evaluated through blood-based liquid biopsy technology. Other firms that offer liquid biopsy testing include ResolutionBio (now part of Agilent) which offers Agilent Resolution ctDx FIRST (“companion diagnostic to KRAZATI™ (adagrasib) for the detection of KRAS G12C in non-small cell lung cancer [NSCLC]”) and Agilent Resolution ctDx LUNG, which focuses on actionable genes for lung cancer such as EGFR and ALK; Circulogene (tests BRAF, EGFR, KRAS, ALK, ROS1, PD-L1, and MSI), Neogenomics (InvisionFirst, 37-gene panel including 10 actionable genes), and Biocept (CNSide™). As liquid biopsy is a rapidly emerging field, it is possible that many more tests will find their way into the clinical setting.58-61 In addition to panels designed to target cancer or cancer type specific mutations, tests are beginning to emerge that perform the role of a genetic screen in asymptomatic individuals. One such test includes the GRAIL-Galleri multi-cancer early detection (MCED) test, which claims to look for a signal shared by at least 50 types of cancer with a single blood test.62 According to Turnbull C. (2024) the Galleri-MCED test has an overall sensitivity of only 27.5% for early-stage cancer. The test sensitivity improves to 52.8% by restricting analysis to 12 cancers that the Galleri authors specified as being of high unmet need. However, for several of these 12 cancers, including pancreatic, oesophageal, biliary, and liver, the mortality gains from population screening may be low because these cancers are mainly diagnosed in individuals older than 70 years and prognosis is poor regardless of stage.63 Klein, et al. (2021) held a Circulating Cell-free Genome Atlas substudy including 4077 participants in an independent validation set of Galleri (cancer: n=2823; non-cancer: n=1254, non-cancer status confirmed at year-one follow-up). Specificity for cancer signal detection was 99.5%. Overall sensitivity for cancer signal detection was 51.5% with the sensitivity increasing with stage. Stage I-III sensitivity was 67.6% in the 12 pre-specified cancers and was 40.7% in all cancers. Cancer signals were detected across more than 50 cancer types. Overall accuracy of cancer signal origin prediction in true positives was 88.7%. The study concluded that “The MCED test demonstrated high specificity and accuracy of cancer signal origin prediction and detected cancer signals across a wide diversity of cancers. These results support the feasibility of this blood-based MCED test as a complement to existing single-cancer screening tests.”64 Clinical Utility and Validity Seeberg, et al. (2015) conducted a prospective study to assess the prognostic and predictive value of CTCs in 194 patients with colorectal liver metastasis referred to surgery. A total of 153 patients underwent a resection (41 patients had an unresectable tumor), and CTCs were detected in 19.6% of patients. Patients with unresectable tumors had a 46% CTC positivity rate compared to 11.7% for resectable tumors. Patients with two or more CTCs experienced reduced time to relapse/progression. Two or more CTCs was a strong predictor of progression and mortality in all subgroups of patients. The authors concluded that “CTCs predict nonresectability and impaired survival. CTC analysis should be considered as a tool for decision-making before liver resection in these patients.”65 Groot Koerkamp, et al. (2013) performed systematic review and meta-analysis to investigate the prognostic value of CTCs in patients with resectable colorectal liver metastases or widespread metastatic colorectal cancer (CRC). The results of 12 studies representing 1,329 patients were suitable for pooled analysis. The overall survival and progression-free survival were worse in patients with CTCs, with hazard ratios of 2.47 for overall survival rate and 2.07 for progression-free survival. The authors concluded that “the detection of CTCs in peripheral blood of patients with resectable colorectal liver metastases or widespread metastatic CRC is associated with disease progression and poor survival.”66 Zhang, et al. (2012) conducted a meta-analysis of published literature on the prognostic value of CTC in breast cancer. Forty-nine eligible studies enrolling 6,825 patients were identified. The presence of CTC was significantly associated with shorter survival in the total population and the prognostic value of CTC was significant in both early and metastatic breast cancer. The authors concluded that “the detection of CTC is a stable prognosticator in patients with early-stage and metastatic breast cancer. Further studies are required to explore the clinical utility of CTC in breast cancer.”67 Pinzani, et al. (2021) assessed that the clinical validity of CTCs has been demonstrated in cancer screening, prognosis, and monitoring treatment responses. In the original article by Cabel, et al. (2017), using the Cellsearch® technique in early non-metastatic cancer has reported low CTC detection rates (5-30% depending on cancer type), with limited specificity since “some circulating epithelial cells can be found in individuals with inflammatory disease or even in some healthy individuals.” However, in the preliminary report of another study, it was found that a CTC count >25 could “distinguish lung cancer from benign lesions in patients with abnormal lung imaging. CTC count was also shown to be an “independent prognostic factor in non-small cell lung cancer and small cell lung cancer;” despite this, CTCs are rare in the non-metastatic setting and thus cannot be completely utilized as an independent prognostic factor in the localized setting. With respect to the independent cancers, Cabel, et al. (2017) summarizes the clinical validity of CTC detection in Figure 1. On the clinical utility of CTC, Cabel, et al. (2017) initially stated “the clinical utility of CTC detection (i.e. does it improve patient outcome) has yet to be demonstrated before it can be implemented in routine clinical practice.” In recent time, it was seen that specific CTC features may have clinical utility in “[predicting] the sensitivity to specific immunotherapies,” and in the case of ER+ MBCs, ER-CTCs can develop and reflect “acquisition of therapy resistance by the primary tumor.”68 Figure 1. Clinical validity of circulating tumor cells (CTC): level of evidence according to clinical settings.3 Oxnard, et al. (2016) found that: “Sensitivity of plasma genotyping for detection of T790M was 70%. Of 58 patients with T790M-negative tumors, T790M was detected in plasma of 18 (31%). ORR and median PFS were similar in patients with T790M-positive plasma (Objective response rate [ORR], 63%; progression-free survival [PFS], 9.7 months) or T790M-positive tumor (ORR, 62%; PFS, 9.7 months) results. Although patients with T790M-negative plasma had overall favorable outcomes (ORR, 46%; median PFS, 8.2 months), tumor genotyping distinguished a subset of patients positive for T790M who had better outcomes (ORR, 69%; PFS, 16.5 months) as well as a subset of patients negative for T790M with poor outcomes (ORR, 25%; PFS, 2.8 months).”69 The authors concluded that “upon availability of validated plasma T790M assays, some patients could avoid a tumor biopsy for T790M genotyping.”69 A review by Sacher, et al. (2016) genotyped 180 patients with NSCLC using plasma droplet PCR (plasma ddPCR). This was done to validate the plasma droplet PCR technique, and the study identified 115 EGFR mutations and 25 KRAS mutations. The plasma ddPCR was measured to have 82% sensitivity for the EGFR 19 del, 74% for L858R, 77% for T790M, and 64% for KRAS. The positive predictive value was 100% for every mutation apart from T790M at 79%. The authors concluded that the technique “detected EGFR and KRAS mutations rapidly with the high specificity needed to select therapy and avoid repeat biopsies”. The authors also noted that this assay “may also detect EGFR T790M missed by tissue genotyping due to tumor heterogeneity in resistant disease.”70 Kim, et al. (2017) evaluated the clinical utility of Guardant360. This study used the Guardant360 panel to detect mutations in patients with metastatic NSCLC and other cancers. Somatic mutations were detected in 59 patients, 25 of which had actionable mutations. Out of the 73-patient NSCLC cohort, 62 were found to have somatic mutations and 34 had actionable mutations. After these genetic findings were identified, molecularly matched therapy was provided to 10 patients with gastric cancer (GC) and 17 with NSCLC. Response rate was 67% in GC and 87% in patients with NSCLC, while disease control rate was 100% for both types.71 Odegaard, et al. (2018) validated the Guardant360 cell-free DNA sequencing test and aimed to “demonstrate its clinical feasibility”. The authors found that the test could detect variants down to “0.02% to 0.04% allelic fraction/2.12 copies with ≤0.3%/2.24-2.76 copies”. Clinical validation in a cohort of over 750 patients demonstrated high accuracy and specificity, with positive percent agreement (with PCR) of 92%-100% and negative percent agreement of over 99%. In terms of feasibility, the authors performed the test in 10593 patients and found the technical success rate to be over 99.6% and the clinical sensitivity to be 85.9%. The authors also noted that 16.7% of these mutations were targetable with FDA-approved treatments (with 72% with “treatment or trial recommendations”) with as many as 34.5% of non-small cell cancer samples having a targetable mutation.72 Aggarwal, et al. (2019) evaluated the utility of plasma-based sequencing in improving mutation detection in patients with non-small cell lung cancer. The authors first performed next-generation sequencing (NGS) on tissue, then plasma-based sequencing. A total of 229 patients had concurrent sequencing, and NGS alone detected 47 targetable mutations. Addition of plasma sequencing brought that number to 82 targetable mutations. Furthermore, 36 of 42 patients that received “plasma next-generation sequencing–indicated therapy” achieved a “complete or a partial response or stable disease”. The authors concluded that “adding plasma next-generation sequencing testing to the routine management of metastatic non–small cell lung cancer appears to increase targetable mutation detection and improve delivery of targeted therapy.”73 Leighl, et al. (2019) evaluated the utility of “comprehensive cell-free DNA analysis” to identify genomic biomarkers in patients with newly diagnosed metastatic non-small cell lung cancer (NSCLC). A total of 282 patients were included. Tissue genotyping (current standard of care) identified a guideline-recommended biomarker in 60 patients, whereas cell-free DNA identified a relevant biomarker in 77 patients. Concordance between the two methods was 80% (48 biomarkers detected in both methods). For FDA-approved targets (EGFR, ALK, ROS1, BRAF), concordance was >98.2% with 100% positive predictive value for cell-free DNA. Cell-free DNA was also found to have a faster median turnaround time (nine days compared to 15 for tissue genotyping), and “guideline-complete” (assessment of all eight guideline-recommended biomarkers [EGFR, ALK, ROS1, BRAF, RET, MET amplification and exon 14 skipping, and HER2]), was significantly more likely (268 patients vs 51).74 Dudley, et al. (2019) have developed a novel high-throughput sequencing method that uses urine-derived tumor DNA (utDNA) known as utDNA CAPP-Seq (uCAPP-Seq) to detect bladder cancer. This technique was used to analyze samples from 118 patients with early-stage bladder cancer and 67 healthy adults. “We detected utDNA pretreatment in 93% of cases using a tumor mutation-informed approach and in 84% when blinded to tumor mutation status, with 96% to 100% specificity.”75 These results show that utDNA can be used to diagnose early-stage bladder cancer with high sensitivity and specificity. Wang, et al. (2018) performed a meta-analysis to determine the diagnostic performance of cell-free DNA (both blood and urine) assays in bladder cancer. Eleven studies encompassing 802 patients were included. The authors evaluated cell-free DNA assays at the following statistics: “sensitivity 0.71, specificity 0.78 positive likelihood ratio 3.3, negative likelihood ratio 0.37, diagnostic odds ratio nine, and area under curve 0.80. No publication bias was identified. The authors concluded that “cell-free DNA has a high diagnostic value in bladder cancer.”76 Hopefully, cfDNA can be used to indicate prognoses of personalized peptide vaccine therapy in patients with NSCLC. Waki, et al. (2021) identified that cfDNA integrity “decreased after the first cycle of vaccination” and that those with “high prevaccination cfDNA integrity survived longer than those with low prevaccination integrity (median survival time (MST): 17.9 versus 9.0 months, respectively; hazard ratio (HR): 0.58, p=.0049),” showing that monitoring cfDNA levels could contribute to quantifying treatment success and predicting patient lifespans. For exosome-based liquid biopsy, Yu, et al. (2021) have proposed a synergistic alternative of combining cfDNA and exosomal RNA to “increase the sensivity of mutation detection… the exosome component enables a combination of exosomal RNA, cfDNA, and disease specific proteins… the unique composition of the exosome compartment makes these vesicles particularly amenable for multi-analyte testing, since they carry cancer-informative DNA, RNA, proteins, lipids, oligosaccharides, and metabolites. In one study, a high sensitivity (92%) for EGFR mutations was found for utilizing exosomal RNA and ctDNA together and remained high in a subpopulation that’s been difficult for ctDNA assays to detect (88% sensitivity). ExoRNA and ctDNA combined analyses on BRAF, KRAS, and EGFR mutations in exosomes and respective ctDNA have also better correlated the biomarkers with treatment outcomes when compared to ctDNA alone.10 Lee, et al. (2021) analyzed the clinical utility of ctDNA to reliably detect EGFR in ctDNA. The authors compared EGFR analysis results between tissueDNA (tDNA) and ctDNA from 554 NSCLC cases. ctDNA analysis detected EGFR mutation in 57.3% of cases. ctDNA detection correlated with metastatic stage and disease progression (p<0.001). The authors followed up after an average of 41.09 month and found that, “survival analysis revealed ctDNA status and M stage (p < 0.001) to be independent predictors of overall survival in the multivariate analysis.” The authors conclude that ctDNS is clinically useful for EGFR analysis, but note the possibility of false negatives and recommend using tDNA to confirm ctDNA results in some situations.78 Syeda, et al. (2021) evaluated the use of ctDNA as a biomarker for melanoma. The authors measured changes in ctDNA and survival following “BRAF, MEK, or BRAF plus MEK inhibitor therapy” in patients participating in two clinical trials. The BRAFV600-mutant was measured in ctDNA before and during treatment. “Elevated baseline BRAFV600 mutation-positive ctDNA concentration was associated with worse overall survival outcome.” The authors conclude that BRAFV600-mutation ctDNA analysis can be used as a biomarker to predict clinical outcomes.79 Dang and Park (2022) completed a systematic review on the use of ctDNA for liquid biopsy and the potential challenges as a primary cancer screening marker for minimal residual disease. They cite that more studies need to be performed to evaluate the positive and negative predictive values of existing tests utilizing ctDNA for this purpose. Despite this, the figure below details their understanding of positive benefits in the potential utility of ctDNA across the cancer spectrum:80 National Comprehensive Cancer Network (NCCN) The NCCN guidelines for non-small cell lung cancer (NSCLC) strongly advise “broader molecular profiling with the goal of identifying rare driver mutations for which effective drugs may already be available, or to appropriately counsel patients regarding the availability of clinical trials... Broad molecular profiling is considered a key component of improving care for patients with NSCLC.” Furthermore, the NCCN states that “plasma or tissue-based testing via broad molecular profiling should be considered at progression, for the T790M mutation and other genomic resistance mechanisms. If plasma-based testing is negative, tissue-based testing with re-biopsy material is strongly recommended. Practitioners may want to consider scheduling the biopsy concurrently with plasma testing referral.”81 However, the NCCN goes on to state that “cell-free/circulating tumor DNA testing should not be used in lieu of histologic tissue diagnosis.” The NCCN notes that specificity is generally very high for cell-free tumor testing but is lacking in sensitivity (up to 30% false negative rate) and that standards for analytic performance characteristics of cell-free tumor DNA have not been well established. The use of cell-free or circulating tumor DNA may be considered in specific clinical situations, such as if a patient is medically unfit for an invasive tissue sampling or if there is insufficient material for a molecular analysis following pathologic confirmation of an NSCLC diagnosis in the initial diagnostic setting (but “follow-up tissue-based analysis for all patients in which an oncogenic driver is not identified should be planned”). The NCCN notes that “data suggest that plasma ctDNA testing is a useful minimally invasive test that can be used to identify ALK, BRAF, EGFR, HER2, MET exon 14 skipping, RET, ROS1 and other oncogenic biomarkers that would not otherwise be identified in patients with metastatic NSCLC.”81 For NSCLC, the NCCN provides the following specific recommendations for liquid biopsy: “plasma ctDNA testing can be used in specific circumstances if 1) the patient is not medically fit for invasive tissue sampling; or 2) there is insufficient tissue for molecular analysis and follow-up tissue-based analysis will be done if an oncogenic driver is not identified.”81 The NCCN lists “comprehensive germline and somatic profiling to identify candidates for additional targeted therapies” as part of the workup for recurrent stage IV (M1) breast cancer. They go on to specifically note that “tumor tissue or plasma-based circulating tumor DNA (ctDNA) assays may be used and each of these have benefits and limitations for diagnosis and disease progression. Tissue-based assays have greater sensitivity, but ctDNA may reflect tumor heterogeneity more accurately. If one specimen is negative for actionable biomarkers, testing on the alternative specimen can be considered.” The NCCN also states that assessment of the PIK3CA mutation may be performed through liquid biopsy if the tumor is HR-positive, HER2 negative, and if therapy with alpelisib plus fulvestrant is being considered. Finally, for the management of breast cancer with liquid biopsy techniques, the NCCN states that “the clinical use of circulating tumor cells (CTC) or circulating DNA (ctDNA) in metastatic breast cancer is not yet included in the NCCN Guidelines for Breast Cancer for disease assessment and monitoring”, though the sentence that follows would indicate that this statement refers to a count of CTCs, not their use for genotyping: “Patients with persistently increased CTC after three weeks of first-line chemotherapy have a poor PFS and OS.”82 The NCCN states that AR-V7 testing in CTCs “can be considered to help guide selection of therapy in the post-abiraterone/enzalutamide metastatic CRPC [castration-resistant prostate cancer] setting.” The NCCN does not comment on any particular liquid medium over another (e.g., urine, CSF, serum). However, the NCCN does specify the use of circulating DNA for rucaparib treatment, stating that “the preferred method of selecting patients for rucaparib treatment is somatic analysis of BRCA1 and BRCA2 using a circulating tumor DNA sample.”83 SelectMDx is also acknowledged by the NCCN; “the panel believes that SelectMDx score is potentially informative in patients who have never undergone biopsy, and it can therefore be considered in such individuals.”84 With regards to circulating tumor DNA (ctDNA) in colon cancer, the NCCN “panel believes that there are insufficient data to recommend the use of multigene assays, Immunoscore, or post-surgical ctDNA to estimate risk of recurrence or determine adjuvant therapy.”85 For neuroendocrine tumors, NCCN notes that CTCs have been studied as prognostic markers, but state that more research is required. There is no single biomarker available that is satisfactory as a diagnostic, prognostic, or predictive marker.86 For a primary CNS lymphoma, the NCCN remarks that cerebrospinal fluid analysis may “possibly” include gene rearrangement evaluation. For leptomeningeal metastases, the NCCN notes that assessment of ctDNA or CTCs in CSF “increases sensitivity of tumor cell detection and assessment of response to treatment.”87 For pancreatic adenocarcinomas, the NCCN acknowledges that circulating cell-free DNA is being investigated as a biomarker for screening. The NCCN also notes that if tumor tissue is not available, cell-free DNA testing may be considered.88 For esophageal, esophagogastric junction cancers, and gastric cancers, the NCCN states “testing using a validated NGS-based [next-generation sequencing] genomic profiling assay performed in a CLIA-approved laboratory may be considered. A negative result should be interpreted with caution, as this does not exclude the presence of tumor mutations or amplifications.”89 For acute myeloid leukemia, the NCCN notes that “morphologically detectable,” circulating leukemic blasts from peripheral blood may be used to detect molecular abnormalities.90 For bladder cancer, the NCCN mentions RT-PCR testing for FGFR2/3 gene alterations but does not specify whether this can be done through a liquid biopsy or cell-free DNA. The only comment made is that the laboratory should be CLIA-approved.91 The NCCN guidelines for small cell lung cancer and hepatocellular carcinoma do not address use of CTCs, ctDNA, or liquid biopsy for patient management.92,93 In biliary tract cancers, the NCCN states that “a cell-free DNA (cfDNA) test may also be considered for identifying gene mutations. This technique may not reliably identify gene fusions or rearrangements depending on the panel used and the specific partner gene.”94 American Society of Clinical Oncology (ASCO) In 2016, ASCO published updated recommendations for the use of tumor markers in treatment of metastatic breast cancer. ASCO found that although CTCs may be prognostic, they are not predictive for clinical benefit when used to guide or influence decisions on systemic therapy for metastatic breast cancer. ASCO recommends clinicians to not use these markers as adjunctive assessments.95 Similarly, ASCO recommended against use of CTCs to guide decisions about adjuvant systemic therapy for individual’s with early-stage invasive breast cancer.96 In the 2022 update ASCO states that there are insufficient data to recommend routine use of ctDNA and CTCs to monitor response to therapy among patients with metastatic breast cancer.97 However, in a 2023 rapid recommendation update ASCO adds the following: “To aid in treatment selection, the Expert Panel recommends routine testing for emergence of ESR1 mutations at recurrence or progression on ET (with or without CDK4/6 inhibitor) in patients with ER-positive, HER2-negative MBC. Testing should be performed on blood or tissue obtained at the time of progression, as ESR1 mutations develop in response to selection pressure during treatment and are typically undetectable in the primary tumor; blood-based ctDNA is preferred owing to greater sensitivity. If not performed earlier, testing for PIK3CA mutations should also be performed to guide further therapy. Patients whose tumor or ctDNA tests remain ESR1 wild-type may warrant retesting at subsequent progression(s) to determine if an ESR1 mutation has arisen.”98 In 2019, ASCO stated that clinicians “should not use circulating biomarkers as a surveillance strategy for detection of recurrence in patients who have undergone curative-intent treatment of stage I-III NSCLC or SCLC”. ASCO states that further data is required to validate this approach.99 In 2018, ASCO and the College of American Pathologists (CAP) released a joint review on “circulating tumor DNA analysis in patients with cancer.” In it, they note that apart from the assays that have received “regulatory appeal”, most assays have “insufficient evidence” for both clinical validity and clinical utility. They note discordant results between circulating DNA assays and tissue genotyping. Furthermore, they remark on the lack of evidence for use in monitoring therapy effectiveness, diagnosing early-stage cancer, or cancer screening. However, they point to evidence that well-validated assays may support initiation of targeted therapy.100 National Academy of Clinical Biochemistry (NACB), now known as the American Association for Clinical Chemistry (AACC) In 2010, the NACB issued practice guidelines for the use of tumor markers in liver, bladder, cervical, and gastric cancers. It found that CTCs had “questionable” clinical utility in the assessment of liver cancer and did not recommend their use.101 The NACB published an updated guideline in 2020. For liver cancer, they note circulating cell-free serum DNA as “undergoing evaluation” for “predictive marker for distant metastasis of hepatitis C virus–related HCC.” The plasma proteasome is also undergoing evaluation for “assessment of early HCC in patients with chronic viral chronic hepatitis; assessment of metastatic potential of HCC.” Finally, circulating methylated DNA is undergoing evaluation for HCC screening, detection, and prognosis. No other circulating tumor markers for bladder, cervical, and gastric cancers were mentioned.102 College of American Pathologists (CAP), the International Association for the Study of Lung Cancer (IASLC), and the Association for Molecular Pathology (AMP) An Expert Panel was convened to review and update the CAP-IASLC-AMP Molecular Testing Guideline for Selection of Lung Cancer Patients for EGFR and ALK Tyrosine Kinase Inhibitors. This panel consists of practicing pathologists, oncologists, and a methodologist. The panel states there is “insufficient evidence to support the use of circulating cell-free plasma DNA (cfDNA) molecular methods for the diagnosis of primary lung adenocarcinoma”. According to the panel, there is also “insufficient evidence to support the use of circulating tumor cell (CTC) molecular analysis for the diagnosis of primary lung adenocarcinoma, the identification of EGFR or other mutations, or the identification of EGFR T790M mutations at the time of EGFR TKI-resistance.”103 However, the panel acknowledges that “In some clinical settings in which tissue is limited and/or insufficient for molecular testing, physicians may use a cell-free plasma DNA (cfDNA) assay to identify EGFR mutations.”103 In 2021, the IASLC published an updated consensus statement on liquid biopsy testing. They note that liquid biopsy “includes a variety of methodologies for circulating analytes. From a clinical point of view, plasma circulating tumor DNA is the most extensively studied and widely adopted alternative to tissue tumor genotyping in solid tumors, including NSCLC.”104 The following recommendations were presented in a consensus statement: In clinical practice, ctDNA collection, sample handling, and automated processing should be performed using standardized and clinically validated procedures to reduce operator variability and false negative results. Because of the growing number of guideline-recommended oncogene targets to be assessed in advanced NSCLC, testing of plasma ctDNA should be performed by a clinically validated NGS platform rather than single-gene, PCR-based approaches, both in treatment-naive patients and those associated with multiple mechanisms of acquired resistance (MOR) to targeted agents. Where plasma NGS is not available owing to technical and economic constraints, single-gene or low multiplex-based approaches may represent appropriate alternatives. Use of limited PCR analysis for EGFR mutations as the initial step in molecular assessment, for example, remains highly relevant in areas of the world where the EGFR mutation rate is high. Nevertheless, single-gene testing should not be considered complete, and if negative, serial testing for additional actionable biomarkers must be pursued. The benefit of tissue and plasma NGS is now established in several clinical practice settings. It is anticipated, owing to broad-based coverage of requisite oncogenes, decreased turnaround times, and emerging data on cost effectiveness, that in the near future, NGS will become increasingly available worldwide. Implementation of a multidisciplinary MTB to assist clinicians in treatment decision-making is advisable, as described previously. In patients with oncogene-addicted NSCLC, liquid biopsy is emerging as not only complementary to tissue-based analysis but also acceptable as the initial approach (“plasma first”) for biomarker evaluation at the time of diagnosis and for monitoring the efficacy of targeted therapies. Finally, a plasma first approach is appropriate for identification of MOR to targeted therapies in many clinical settings. Indications for liquid biopsy in patients with nononcogene-addicted NSCLC are less well defined at this time, although there are several promising areas of investigation. As noted previously, bTMB is an emerging biomarker, pending completion of ongoing prospective randomized trials and refinement of methodology. American Society for Clinical Pathology, College of American Pathologists, Association for Molecular Pathology, and American Society of Clinical Oncology These joint guidelines from these societies were published regarding molecular biomarkers for colorectal cancer. Despite the potential of liquid biopsy for assessment of tumor recurrence and treatment resistance, the technique “awaits robust validation and further studies to determine their clinical utility”.105 European Society for Medical Oncology (ESMO) and Chinese Society of Clinical Oncology (CSCO) These guidelines state that liquid biopsy can be used as “the initial test for the detection of a T790M mutation [for EGFR in NSCLC], and if tests are negative, a re-biopsy should be attempted if feasible.”106 European Association of Urology (EAU), European Society for Radiotherapy and Oncology (ESTRO), European Society of Urogenital Radiology (ESUR), International Society of Geriatric Oncology (SIOG) The joint guidelines on prostate cancer state that “In asymptomatic [individuals] with a prostate-specific antigen level between three and ps. Go20 ng/mL and a normal [digital rectal examination], use one of the following tools for biopsy indication: risk-calculator - provided it is correctly calibrated to the population prevalence; magnetic resonance imaging of the prostate - STRONG an additional serum, urine biomarker test – WEAK.” These joint guidelines acknowledged SelectMDX as a test to isolate urine biomarkers, but the guidelines noted “the clinically added value of SelectMDX in the era of upfront MRI and targeted biopsies remains unclear.”107 American Society of Colon and Rectal Surgeons (ASCRS) The ASCRS released clinical practice guidelines for the management of colon cancer. The guidelines state that “the use of multigene assays, CDX2 expression analysis, and ctDNA may be used to complement multidisciplinary decision-making for patients with stage II or III colon cancer.”108 References Curigliano G. Liquid biopsies: Tumour diagnosis and treatment monitoring in a blood test | ESMO. ESMO 2014. 2014. http://www.esmo.org/Conferences/Past-Conferences/ESMO-2014-Congress/News-Articles/Liquid-biopsies-Tumour-diagnosis-and-treatment-monitoring-in-a-blood-test Haber DA, Velculescu VE. Blood-based analyses of cancer: circulating tumor cells and circulating tumor DNA. Cancer discovery. Jun 2014;4(6):650-61. doi:10.1158/2159-8290.Cd-13-1014 Cabel L, Proudhon C, Gortais H, et al. Circulating tumor cells: clinical validity and utility. International Journal of Clinical Oncology. 2017/06/01 2017;22(3):421-430. doi:10.1007/s10147-017-1105-2 Coding Section Codes Number Description CPT 81162 BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (e.g., hereditary breast and ovarian cancer) gene analysis; full sequence analysis and full duplication/deletion analysis (i.e., detection of large gene rearrangements) 81163 BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (e.g., hereditary breast and ovarian cancer) gene analysis; full sequence analysis 81164 BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (e.g., hereditary breast and ovarian cancer) gene analysis; full duplication/deletion analysis (i.e., detection of large gene rearrangements) 81194 NTRK (neurotrophic receptor tyrosine kinase 1, 2, and 3) (e.g., solid tumors) translocation analysis 81210 BRAF (B-Raf proto-oncogene, serine/threonine kinase) (e.g., colon cancer, melanoma), gene analysis, V600 variant(s) 81235 EGFR (epidermal growth factor receptor) (e.g., non-small cell lung cancer) gene analysis, common variants (e.g., exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q) 81275 KRAS (Kirsten rat sarcoma viral oncogene homolog) (e.g., carcinoma) gene analysis; variants in exon 2 (e.g., codons 12 and 13) 81276 KRAS (Kirsten rat sarcoma viral oncogene homolog) (e.g., carcinoma) gene analysis; additional variant(s) (e.g., codon 61, codon 146) 81309 PIK3CA (phosphatidylinositol-4, 5-biphosphate 3-kinase, catalytic subunit alpha) (e.g., colorectal and breast cancer) gene analysis, targeted sequence analysis (e.g., exons 7, 9, 20) 81405 Molecular pathology procedure, Level 6 (e.g., analysis of 6 – 10 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 11 – 25 exons, regionally targeted cytogenomic array analysis) 81406 Molecular pathology procedure, Level 7 (e.g., analysis of 11 – 25 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 26 – 50 exons) 81462 (effective 01/01/2024) Solid organ neoplasm, genomic sequence analysis panel, cell-free nucleic acid (e.g., plasma), interrogation for sequence variants; dna analysis or combined dna and rna analysis, copy number variants and rearrangements 81479 Unlisted molecular pathology procedure 86152 Cell enumeration using immunologic selection and identification in fluid specimen (e.g., circulating tumor cells in blood) 86153 Cell enumeration using immunologic selection and identification in fluid specimen (e.g., circulating tumor cells in blood); physician interpretation and report, when required 0011M Oncology, prostate cancer, mRNA expression assay of 12 genes (10 content and 2 housekeeping), RT-PCR test utilizing blood plasma and urine, algorithms to predict high-grade prostate cancer risk Proprietary test: NeoLAB™ Prostate Liquid Biopsy Lab/Manufacturer: NeoGenomics Laboratories 0091U Oncology (colorectal) screening, cell enumeration of circulating tumor cells, utilizing whole blood, algorithm, for the presence of adenoma or cancer, reported as a positive or negative result Proprietary test: FirstSightCRC Lab/Manufacturer: CellMax Life 0155U Oncology (breast cancer), DNA, PIK3CA (phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha) (e.g., breast cancer) gene analysis (i.e., p.C420R, p.E542K, p.E545A, p.E545D [g.1635G>T only], p.E545G, p.E545K, p.Q546E, p.Q546R, p.H1047L, p.H1047R, p.H1047Y), utilizing formalin-fixed paraffin-embedded breast tumor tissue, reported as PIK3CA gene mutation status Proprietary test: therascreen ® PIK3CA RGQ PCR Kit Lab/Manufacturer: QIAGEN 0177U Oncology (breast cancer), DNA, PIK3CA (phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha) gene analysis of 11 gene variants utilizing plasma, reported as PIK3CA gene mutation status Proprietary test: therascreen ® PIK3CA RGQ PCR Kit Lab/Manufacturer: QIAGEN 0179U Oncology (non-small cell lung cancer), cell-free DNA, targeted sequence analysis of 23 genes (single nucleotide variations, insertions and deletions, fusions without prior knowledge of partner/breakpoint, copy number variations), with report of significant mutation(s) Proprietary test: Resolution ctDx Lung™ Lab/Manufacturer: Resolution Bioscience 0229U BCAT1 (Branched chain amino acid transaminase 1) and IKZF1 (IKAROS family zinc finger 1) (e.g., colorectal cancer) promoter methylation analysis Proprietary test: Colvera ® Lab/Manufacturer: Clinical Genomics Pathology Inc 0317U Oncology (lung cancer), four-probe FISH (3q29, 3p22.1, 10q22.3, 10cen) assay, whole blood, predictive algorithm-generated evaluation reported as decreased or increased risk for lung cancer Proprietary test: LungLB ® Lab/Manufacturer: LungLife AI ® 0332U Oncology (pan-tumor), genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative polymerase chain reaction (qPCR), whole blood, reported as a high or low probability of responding to immune checkpoint-inhibitor therapy Protietary test: EpiSwitch ® CiRT (Checkpoint-inhibitor Response Test) Lab/Manufacturer: Next Bio-Research Services LLC 0333U (revised laboratory name ) Oncology (liver), surveillance for hepatocellular carcinoma (HCC) in high-risk patients, analysis of methylation patterns on circulating cellfree DNA (cfDNA) plus measurement of serum of AFP/AFP-L3 and oncoprotein des-gamma-carboxyprothrombin (DCP), algorithm reported as normal or abnormal result 0337U Oncology (plasma cell disorders and myeloma), circulating plasma cell immunologic selection, identification, morphological characterization, and enumeration of plasma cells based on differential CD138, CD38, CD19, and CD45 protein biomarker expression, peripheral blood Protietary test: CELLSEARCH ® Circulating Multiple Myeloma Cell (CMMC) Test Lab/Manufacturer: Menarini Silicon Biosystems Inc. 0338U Oncology (solid tumor), circulating tumor cell selection, identification, morphological characterization, detection and enumeration based on differential EpCAM, cytokeratins 8, 18, and 19, and CD45 protein biomarkers, and quantification of HER2 protein biomarker-expressing cells, peripheral blood Protietary test: CELLSEARCH ® HER2 Circulating Tumor Cell (CTC-HER2) Test Lab/Manufacturer: Menarini Silicon Biosystems Inc. 0343U Oncology (prostate), exosome-based analysis of 442 small noncoding RNAs (sncRNAs) by quantitative reverse transcription polymerase chain reaction (RT-qPCR), urine, reported as molecular evidence of no-, low-, intermediate- or high-risk of prostate cancer Protietary test: miR Sentinel™ Prostate Cancer Test Lab/Manufacturer: miR Scientific LLC 0368U Oncology (colorectal cancer), evaluation for mutations of APC, BRAF, CTNNB1, KRAS, NRAS, PIK3CA, SMAD4, and TP53, and methylation markers (MYO1G, KCNQ5, C9ORF50, FLI1, CLIP4, ZNF132 and TWIST1), multiplex quantitative polymerase chain reaction (qPCR), circulating cell-free DNA (cfDNA), plasma, report of risk score for advanced adenoma or colorectal cancer Proprietary test: ColoScape™ Colorectal Cancer Detection Lab/Manufacturer: DiaCarta Clinical Lab 0388U (effective date 07/01/2023) Oncology (non-small cell lung cancer), next-generation sequencing with identification of single nucleotide variants, copy number variants, insertions and deletions, and structural variants in 37 cancer-related genes, plasma, with report for alteration detection 0395U (effective date 07/01/2023) Oncology (lung), multi-omics (microbial DNA by shotgun next-generation sequencing and carcinoembryonic antigen and osteopontin by immunoassay), plasma, algorithm reported as malignancy risk for lung nodules in early-stage disease 0397U (effective date 07/01/2023) Oncology (non-small cell lung cancer), cell-free DNA from plasma, targeted sequence analysis of at least 109 genes, including sequence variants, substitutions, insertions, deletions, select rearrangements, and copy number variations 0410U (effective date 10/01/2023) Oncology (pancreatic), DNA, whole genome sequencing with 5-hydroxymethylcytosine enrichment, whole blood or plasma, algorithm reported as cancer detected or not detected 0453U Oncology (colorectal cancer), cell-free DNA (cfDNA), methylation based quantitative PCR assay (SEPTIN9, IKZF1, BCAT1, Septin9-2, VAV3, BCAN), plasma, reported as presence or absence of circulating tumor DNA (ctDNA) Proprietary test: ColonAiQ ® Lab/Manufacturer: Breakthrough Genomics, Singlera Genomics Inc. 0490U (effective date 10/01/2024) Oncology (cutaneous or uveal melanoma), circulating tumor cell selection, morphological characterization and enumeration based on differential CD146, high molecular-weight melanomaassociated antigen, CD34 and CD45 protein biomarkers, peripheral blood 0491U (effective date 10/01/2024) Oncology (solid tumor), circulating tumor cell selection, morphological characterization and enumeration based on differential epithelial cell adhesion molecule (EpCAM), cytokeratins 8, 18, and 19, CD45 protein biomarkers, and quantification of estrogen receptor (ER) protein biomarker–expressing cells, peripheral blood 0492U (effective 10/01/2024) Oncology (solid tumor), circulating tumor cell selection, morphological characterization and enumeration based on differential epithelial cell adhesion molecule (EpCAM), cytokeratins 8, 18, and 19, CD45 protein biomarkers, and quantification of PD-L1 protein biomarker-expressing cells, peripheral blood 0496U (effective 10/01/2024) Oncology (colorectal), cell-free DNA, 8 genes for mutations, 7 genes for methylation by real-time RT-PCR, and 4 proteins by enzyme-linked immunosorbent assay, blood, reported positive or negative for colorectal cancer or advanced adenoma risk 0499U (effective 10/01/2024) Oncology (colorectal and lung), DNA from formalin-fixed paraffinembedded (FFPE) tissue, nextgeneration sequencing of 8 genes (NRAS, EGFR, CTNNB1, PIK3CA, APC, BRAF, KRAS, and TP53), mutation detection 0500U (effective 10/01/2024) Autoinflammatory disease (VEXAS syndrome), DNA, UBA1 gene mutations, targeted variant analysis (M41T, M41V, M41L, c.118-2A>C, c.118-1G>C, c.118- 9_118-2del, S56F, S621C) 0501U (effective 10/01/2024) Human papillomavirus (HPV), E6/E7 markers for high-risk types (16, 18, 31, 33, 35, 39, 45, 51, 52, 56, 58, 59, 66, and 68), cervical cells, branched-chain capture hybridization, reported as negative or positive for high risk for HPV 0507U (effective 10/01/2024) Oncology (ovarian), DNA, wholegenome sequencing with 5- hydroxymethylcytosine (5hmC) enrichment, using whole blood or plasma, algorithm reported as cancer detected or not detected 0539U (effective 04/01/2025) Oncology (solid tumor), cellfree circulating tumor DNA (ctDNA), 152 genes, nextgeneration sequencing, interrogation for singlenucleotide variants, insertions/deletions, gene rearrangements, copy number alterations, and microsatellite instability, using whole-blood samples, mutations with clinical actionability reported as actionable variant 0562U (effective 07/01/2025) Oncology (solid tumor), targeted genomic sequence analysis, 33 genes, detection of single-nucleotide variants (SNVs), insertions and deletions, copy-number amplifications, and translocations in human genomic circulating cell-free DNA, plasma, reported as presence of actionable variants 0565U (effective 01/01/2026) Oncology (hepatocellular carcinoma), next-generation sequencing methylation pattern assay to detect 6,626 epigenetic alterations, cellfree DNA, plasma, algorithm reported as cancer signal detected or not detected 0566U (effective 07/01/2025) Oncology (lung), qPCR-based analysis of 13 differentially methylated regions (CCDC181, HOXA7, LRRC8A, MARCHF11, MIR129-2, NCOR2, PANTR1, PRKCB, SLC9A3, TBR1_2, TRAP1, VWC2, ZNF781), pleural fluid, algorithm reported as a qualitative result 0577U (effective 10/01/2025) Oncology (ovarian), serum, analysis of 39 glycoproteins by liquid chromatography with tandem mass spectrometry (LC-MS/MS) in multiple reaction monitoring mode, reported as likelihood of malignancy 0611U (effective 01/01/2026) Oncology (liver), analysis of over 1,000 methylated regions, cell-free DNA from plasma, algorithm reported as a quantitative result (For additional PLA code with identical clinical descriptor, see 0612U. See Appendix O or the most current listing on the AMA CPT website to determine appropriate code assignment) 0612U (effective 01/01/2026) Oncology (liver), analysis of over 1,000 methylated regions, cell-free DNA from plasma, algorithm reported as a quantitative result (For additional PLA code with identical clinical descriptor, see 0611U. See Appendix O or the most current listing on the AMA CPT website to determine appropriate code assignment) Procedure and diagnosis codes on Medical Policy documents are included only as a general reference tool for each policy. They may not be all-inclusive. This medical policy was developed through consideration of peer-reviewed medical literature generally recognized by the relevant medical community, U.S. FDA approval status, nationally accepted standards of medical practice and accepted standards of medical practice in this community, and other nonaffiliated technology evaluation centers, reference to federal regulations, other plan medical policies and accredited national guidelines. "Current Procedural Terminology © American Medical Association. All Rights Reserved" History From 2016 Forward 12/11/2025 Revision to 0565U Long Code Description, added new codes 0611U and 0612U effective 01/01/2026 09/25/2025 Revised laboratory name on code 0333U and adding 0577U code. 06/12/2025 Adding CPT codes 0562U, 0565U, 0566U effective 07/01/2025. 04/10/2025 Annual review, adding "e.g. GRAIL" to criteria #5. Also updating description, rationale, guidelines/recommendations, and references. Removing code 0356U. 02/11/2025 Updated coding section. Added code 0539U. This code will be effective 04/01/2025. No other changes. 09/05/2024 Updated CPT coding. Added codes 0490U, 0491U, 0492U, 0496U, 0499U, 0500U, 0501U and 0507U (effective 10/01/2024). No change in policy intent. 06/25/2024 Interim review to add PLA code 0453U to coding section. 04/24/2024 Annual review, no change to policy intent. Updating rationale and references. 01/11/2024 Added code 81462 effective 01/01/2024 09/11/2023 Updating coding section. Adding code 0410U effective 10/01/2023. No other changes was made. 06/05/2023 Adding codes 0388U, 0395U and 0397U to coding section of policy. No other changes made. 04/12/2023 Annual review, multiple changes to coverage allowances related to expanded NCCN recommendations. Also updating description, rationale, references and coding. 04/24/2022 Annual review, removing "up to 50 genes" from criteria #3, also updating rationale, references and coding. 01/12/2022 Interim review, removing microsatellite instability analysis and tumor mutational burden language as it will migrate to a new policy. Also updating coding, rationale and references. 04/07/2021 Annual review, adding medical necessity statements regarding BRCA1/2. Also updating description, rationale, references and coding. 04/15/2020 Annual review, updating policy verbiage to include additional diagnoses for coverage, microsatellite instability analysis and repeat testing. Also adding more specific criteria regarding testing that is not medically necessary. Also updating coding. 10/15/2019 Interim review to provide verbiage regarding PIK3CA mutation testing, updating medical necessity criteria for Stage IIIb/IV NSCLC testing. Reformatting policy for clarity. 04/04/2019 Annual review, no change to policy intent. Updating coding. 06/27/2018 Interim review of policy to allow for medical necessity criteria related to Guardant360 testing. Also updating description, references and coding. 05/01/2018 Annual review, policy being rewritten entirely to allow for limited indications being medically necessary. 10/23/2017 Annual review, no change to policy intent. 07/19/207 Update review date. No other changes made. No change to policy intent 07/01/2016 NEW POLICY {} Back to list Home Home Complementary Content ${title} ${badge} ${loading}